<!DOCTYPE html>
<html lang="en">

  <head>
    <title>CancerFeed</title>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="robots" content="noindex, nofollow" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="alternate" type="application/rss+xml" title="CancerFeed" href="feed.atom" />
    <link href="index.css?v1.14.4" rel="stylesheet" />
    <style>
  :root {
    /**
     * Solarized Light
     * Ethan Schoonover (modified by aramisgithub)
     */
    --base00: #fdf6e3;
    --base01: #eee8d5;
    --base02: #93a1a1;
    --base03: #839496;
    --base04: #657b83;
    --base05: #586e75;
    --base06: #073642;
    --base07: #002b36;
    --base08: #dc322f;
    --base09: #cb4b16;
    --base0A: #b58900;
    --base0B: #859900;
    --base0C: #2aa198;
    --base0D: #268bd2;
    --base0E: #6c71c4;
    --base0F: #d33682;

    /**
    * Components
    */
    --source-name-color: var(--base0C);
    /**
   * Typograph
   */
    --font-family-default: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell,
      "Helvetica Neue", sans-serif;
    --font-size-L: 3.6rem;
    --font-size-m: 2.2rem;
    --font-size-s: 1.8rem;
    --font-size-scaler: 52.5%; /* 1rem = 10px */
    --line-height-body: 1.4;
    --line-height-heading: 1.2;

  }
</style>

  </head>

  <body>
    <!-- %after-body-begin.html% -->
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-25 datetime="2024-03-25T10:00:00.000Z">2024-03-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-25 datetime="2024-03-25T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38523646"
                        open
                      >
                        <summary class="article-expander__title">Insights into the glioblastoma tumor microenvironment: current and emerging therapeutic approaches</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38523646/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Glioblastoma (GB) is an intrusive and recurrent primary brain tumor with low survivability. The heterogeneity of the tumor microenvironment plays a crucial role in the stemness and proliferation of GB. The tumor microenvironment induces tumor heterogeneity of cancer cells by facilitating clonal evolution and promoting multidrug resistance, leading to cancer cell progression and metastasis. It also plays an important role in angiogenesis to nourish the hypoxic tumor environment. There is a strong...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38523186"
                        open
                      >
                        <summary class="article-expander__title">Multi-omic profiling of simultaneous ductal carcinoma in situ and invasive breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38523186/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: While most co-occurring DCIS and IBC were mutationally similar and suggestive of a common clonal progenitor, transcriptionally the lesions are highly distinct, with IBC expressing key pathways that facilitate invasion and proliferation. These results are suggestive of additional levels of regulation, epigenetic or other, that facilitate the acquisition of invasive properties during tumor evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-24 datetime="2024-03-24T10:00:00.000Z">2024-03-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-24 datetime="2024-03-24T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38522849"
                        open
                      >
                        <summary class="article-expander__title">Natural history of clonal haematopoiesis seen in real-world haematology settings</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38522849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recursive partitioning of healthy consortia led to the development of the Clonal Hematopoiesis Risk Score (CHRS) for clonal haematopoiesis (CH); however, in the practical setting, most cases of CH are diagnosed after patients present with cytopenias or related symptoms. To address this real-world population, we characterize the clinical trajectories of 94 patients with CH and distinguish CH harbouring canonical DNMT3A/TET2/ASXL1 mutations alone (&#x27;sole DTA&#x27;) versus all other groups (&#x27;non-sole...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-22 datetime="2024-03-22T10:00:00.000Z">2024-03-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-22 datetime="2024-03-22T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38514772"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its impact on the aging osteo-hematopoietic niche</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38514772/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) defines a premalignant state predominantly found in older persons that increases the risk of developing hematologic malignancies and age-related inflammatory diseases. However, the risk for malignant transformation or non-malignant disorders is variable and difficult to predict, and defining the clinical relevance of specific candidate driver mutations in individual carriers has proved to be challenging. In addition to the cell-intrinsic mechanisms, mutant cells rely on...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38514772"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its impact on the aging osteo-hematopoietic niche</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38514772/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) defines a premalignant state predominantly found in older persons that increases the risk of developing hematologic malignancies and age-related inflammatory diseases. However, the risk for malignant transformation or non-malignant disorders is variable and difficult to predict, and defining the clinical relevance of specific candidate driver mutations in individual carriers has proved to be challenging. In addition to the cell-intrinsic mechanisms, mutant cells rely on...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38514772"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its impact on the aging osteo-hematopoietic niche</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38514772/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) defines a premalignant state predominantly found in older persons that increases the risk of developing hematologic malignancies and age-related inflammatory diseases. However, the risk for malignant transformation or non-malignant disorders is variable and difficult to predict, and defining the clinical relevance of specific candidate driver mutations in individual carriers has proved to be challenging. In addition to the cell-intrinsic mechanisms, mutant cells rely on...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38514772"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its impact on the aging osteo-hematopoietic niche</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38514772/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) defines a premalignant state predominantly found in older persons that increases the risk of developing hematologic malignancies and age-related inflammatory diseases. However, the risk for malignant transformation or non-malignant disorders is variable and difficult to predict, and defining the clinical relevance of specific candidate driver mutations in individual carriers has proved to be challenging. In addition to the cell-intrinsic mechanisms, mutant cells rely on...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-21 datetime="2024-03-21T10:00:00.000Z">2024-03-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-21 datetime="2024-03-21T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38513819"
                        open
                      >
                        <summary class="article-expander__title">Skin cancer risk is increased by somatic mutations detected noninvasively in healthy-appearing sun-exposed skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38513819/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Skin cancer risk is increased by exposure to ultraviolet radiation (UVR). Since UVR exposure accumulates over time and lighter skin is more susceptible to UVR, age and skin tone are risk factors for skin cancer. However, measurements of somatic mutations in healthy-appearing skin have not been used to calculate skin cancer risk. Herein, we developed a noninvasive test that quantifies somatic mutations in healthy-appearing sun-exposed skin and applied it to a 1,038-subject cohort. Somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38513819"
                        open
                      >
                        <summary class="article-expander__title">Skin cancer risk is increased by somatic mutations detected noninvasively in healthy-appearing sun-exposed skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38513819/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Skin cancer risk is increased by exposure to ultraviolet radiation (UVR). Since UVR exposure accumulates over time and lighter skin is more susceptible to UVR, age and skin tone are risk factors for skin cancer. However, measurements of somatic mutations in healthy-appearing skin have not been used to calculate skin cancer risk. Herein, we developed a noninvasive test that quantifies somatic mutations in healthy-appearing sun-exposed skin and applied it to a 1,038-subject cohort. Somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38513819"
                        open
                      >
                        <summary class="article-expander__title">Skin cancer risk is increased by somatic mutations detected noninvasively in healthy-appearing sun-exposed skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38513819/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Skin cancer risk is increased by exposure to ultraviolet radiation (UVR). Since UVR exposure accumulates over time and lighter skin is more susceptible to UVR, age and skin tone are risk factors for skin cancer. However, measurements of somatic mutations in healthy-appearing skin have not been used to calculate skin cancer risk. Herein, we developed a noninvasive test that quantifies somatic mutations in healthy-appearing sun-exposed skin and applied it to a 1,038-subject cohort. Somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38513819"
                        open
                      >
                        <summary class="article-expander__title">Skin cancer risk is increased by somatic mutations detected noninvasively in healthy-appearing sun-exposed skin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38513819/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Skin cancer risk is increased by exposure to ultraviolet radiation (UVR). Since UVR exposure accumulates over time and lighter skin is more susceptible to UVR, age and skin tone are risk factors for skin cancer. However, measurements of somatic mutations in healthy-appearing skin have not been used to calculate skin cancer risk. Herein, we developed a noninvasive test that quantifies somatic mutations in healthy-appearing sun-exposed skin and applied it to a 1,038-subject cohort. Somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-20 datetime="2024-03-20T10:00:00.000Z">2024-03-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-20 datetime="2024-03-20T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38508850"
                        open
                      >
                        <summary class="article-expander__title">Genetic Factors Altering Immune Responses in Atrial Fibrillation: JACC Review Topic of the Week</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38508850/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Atrial fibrillation (AF) is the most common cardiac arrhythmia worldwide and is associated with a range of adverse clinical outcomes. Accumulating evidence points to inflammatory processes resulting from innate immune responses as a cornerstone in AF pathogenesis. Genetic and epigenetic factors affecting leukocytes have been identified as key modulators of the inflammatory response. Inherited variants in genes encoding proteins involved in the innate immune response have been associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38507736"
                        open
                      >
                        <summary class="article-expander__title">Multiomic Profiling of Human Clonal Hematopoiesis Reveals Genotype and Cell-Specific Inflammatory Pathway Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38507736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-associated phenomenon that increases risk for hematologic malignancy and cardiovascular disease. CH is thought to enhance disease risk through inflammation in the peripheral blood1. Here, we profile peripheral blood gene expression in 66,968 single cells from a cohort of 17 CH patients and 7 controls. Using a novel mitochondrial DNA barcoding approach, we were able to identify and separately compare mutant TET2 and DNMT3A cells to non-mutant counterparts. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38508850"
                        open
                      >
                        <summary class="article-expander__title">Genetic Factors Altering Immune Responses in Atrial Fibrillation: JACC Review Topic of the Week</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38508850/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Atrial fibrillation (AF) is the most common cardiac arrhythmia worldwide and is associated with a range of adverse clinical outcomes. Accumulating evidence points to inflammatory processes resulting from innate immune responses as a cornerstone in AF pathogenesis. Genetic and epigenetic factors affecting leukocytes have been identified as key modulators of the inflammatory response. Inherited variants in genes encoding proteins involved in the innate immune response have been associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38507736"
                        open
                      >
                        <summary class="article-expander__title">Multiomic Profiling of Human Clonal Hematopoiesis Reveals Genotype and Cell-Specific Inflammatory Pathway Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38507736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-associated phenomenon that increases risk for hematologic malignancy and cardiovascular disease. CH is thought to enhance disease risk through inflammation in the peripheral blood1. Here, we profile peripheral blood gene expression in 66,968 single cells from a cohort of 17 CH patients and 7 controls. Using a novel mitochondrial DNA barcoding approach, we were able to identify and separately compare mutant TET2 and DNMT3A cells to non-mutant counterparts. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38508850"
                        open
                      >
                        <summary class="article-expander__title">Genetic Factors Altering Immune Responses in Atrial Fibrillation: JACC Review Topic of the Week</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38508850/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Atrial fibrillation (AF) is the most common cardiac arrhythmia worldwide and is associated with a range of adverse clinical outcomes. Accumulating evidence points to inflammatory processes resulting from innate immune responses as a cornerstone in AF pathogenesis. Genetic and epigenetic factors affecting leukocytes have been identified as key modulators of the inflammatory response. Inherited variants in genes encoding proteins involved in the innate immune response have been associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38507736"
                        open
                      >
                        <summary class="article-expander__title">Multiomic Profiling of Human Clonal Hematopoiesis Reveals Genotype and Cell-Specific Inflammatory Pathway Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38507736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-associated phenomenon that increases risk for hematologic malignancy and cardiovascular disease. CH is thought to enhance disease risk through inflammation in the peripheral blood1. Here, we profile peripheral blood gene expression in 66,968 single cells from a cohort of 17 CH patients and 7 controls. Using a novel mitochondrial DNA barcoding approach, we were able to identify and separately compare mutant TET2 and DNMT3A cells to non-mutant counterparts. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38508850"
                        open
                      >
                        <summary class="article-expander__title">Genetic Factors Altering Immune Responses in Atrial Fibrillation: JACC Review Topic of the Week</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38508850/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Atrial fibrillation (AF) is the most common cardiac arrhythmia worldwide and is associated with a range of adverse clinical outcomes. Accumulating evidence points to inflammatory processes resulting from innate immune responses as a cornerstone in AF pathogenesis. Genetic and epigenetic factors affecting leukocytes have been identified as key modulators of the inflammatory response. Inherited variants in genes encoding proteins involved in the innate immune response have been associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38507736"
                        open
                      >
                        <summary class="article-expander__title">Multiomic Profiling of Human Clonal Hematopoiesis Reveals Genotype and Cell-Specific Inflammatory Pathway Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38507736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-associated phenomenon that increases risk for hematologic malignancy and cardiovascular disease. CH is thought to enhance disease risk through inflammation in the peripheral blood1. Here, we profile peripheral blood gene expression in 66,968 single cells from a cohort of 17 CH patients and 7 controls. Using a novel mitochondrial DNA barcoding approach, we were able to identify and separately compare mutant TET2 and DNMT3A cells to non-mutant counterparts. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38508850"
                        open
                      >
                        <summary class="article-expander__title">Genetic Factors Altering Immune Responses in Atrial Fibrillation: JACC Review Topic of the Week</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38508850/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Atrial fibrillation (AF) is the most common cardiac arrhythmia worldwide and is associated with a range of adverse clinical outcomes. Accumulating evidence points to inflammatory processes resulting from innate immune responses as a cornerstone in AF pathogenesis. Genetic and epigenetic factors affecting leukocytes have been identified as key modulators of the inflammatory response. Inherited variants in genes encoding proteins involved in the innate immune response have been associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38507736"
                        open
                      >
                        <summary class="article-expander__title">Multiomic Profiling of Human Clonal Hematopoiesis Reveals Genotype and Cell-Specific Inflammatory Pathway Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38507736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-associated phenomenon that increases risk for hematologic malignancy and cardiovascular disease. CH is thought to enhance disease risk through inflammation in the peripheral blood1. Here, we profile peripheral blood gene expression in 66,968 single cells from a cohort of 17 CH patients and 7 controls. Using a novel mitochondrial DNA barcoding approach, we were able to identify and separately compare mutant TET2 and DNMT3A cells to non-mutant counterparts. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38507736"
                        open
                      >
                        <summary class="article-expander__title">Multiomic Profiling of Human Clonal Hematopoiesis Reveals Genotype and Cell-Specific Inflammatory Pathway Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38507736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an age-associated phenomenon that increases risk for hematologic malignancy and cardiovascular disease. CH is thought to enhance disease risk through inflammation in the peripheral blood1. Here, we profile peripheral blood gene expression in 66,968 single cells from a cohort of 17 CH patients and 7 controls. Using a novel mitochondrial DNA barcoding approach, we were able to identify and separately compare mutant TET2 and DNMT3A cells to non-mutant counterparts. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38508089"
                        open
                      >
                        <summary class="article-expander__title">B cell clonality in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38508089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Carcinogenesis in the process of long-term co-evolution of tumor cells and immune environment essentially becomes possible due to incorrect decisions made, remembered, and reproduced by the immune system at the level of clonal populations of antigen-specific T- and B-lymphocytes. Tumor-immunity interaction determines the nature of such errors and, consequently, delineates the possible ways of successful immunotherapeutic intervention. It is generally recognized that tumor-infiltrating B cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38503736"
                        open
                      >
                        <summary class="article-expander__title">The bone ecosystem facilitates multiple myeloma relapse and the evolution of heterogeneous drug resistant disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38503736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is an osteolytic malignancy that is incurable due to the emergence of treatment resistant disease. Defining how, when and where myeloma cell intrinsic and extrinsic bone microenvironmental mechanisms cause relapse is challenging with current biological approaches. Here, we report a biology-driven spatiotemporal hybrid agent-based model of the MM-bone microenvironment. Results indicate MM intrinsic mechanisms drive the evolution of treatment resistant disease but that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38508089"
                        open
                      >
                        <summary class="article-expander__title">B cell clonality in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38508089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Carcinogenesis in the process of long-term co-evolution of tumor cells and immune environment essentially becomes possible due to incorrect decisions made, remembered, and reproduced by the immune system at the level of clonal populations of antigen-specific T- and B-lymphocytes. Tumor-immunity interaction determines the nature of such errors and, consequently, delineates the possible ways of successful immunotherapeutic intervention. It is generally recognized that tumor-infiltrating B cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38503736"
                        open
                      >
                        <summary class="article-expander__title">The bone ecosystem facilitates multiple myeloma relapse and the evolution of heterogeneous drug resistant disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38503736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is an osteolytic malignancy that is incurable due to the emergence of treatment resistant disease. Defining how, when and where myeloma cell intrinsic and extrinsic bone microenvironmental mechanisms cause relapse is challenging with current biological approaches. Here, we report a biology-driven spatiotemporal hybrid agent-based model of the MM-bone microenvironment. Results indicate MM intrinsic mechanisms drive the evolution of treatment resistant disease but that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38508089"
                        open
                      >
                        <summary class="article-expander__title">B cell clonality in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38508089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Carcinogenesis in the process of long-term co-evolution of tumor cells and immune environment essentially becomes possible due to incorrect decisions made, remembered, and reproduced by the immune system at the level of clonal populations of antigen-specific T- and B-lymphocytes. Tumor-immunity interaction determines the nature of such errors and, consequently, delineates the possible ways of successful immunotherapeutic intervention. It is generally recognized that tumor-infiltrating B cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38503736"
                        open
                      >
                        <summary class="article-expander__title">The bone ecosystem facilitates multiple myeloma relapse and the evolution of heterogeneous drug resistant disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38503736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is an osteolytic malignancy that is incurable due to the emergence of treatment resistant disease. Defining how, when and where myeloma cell intrinsic and extrinsic bone microenvironmental mechanisms cause relapse is challenging with current biological approaches. Here, we report a biology-driven spatiotemporal hybrid agent-based model of the MM-bone microenvironment. Results indicate MM intrinsic mechanisms drive the evolution of treatment resistant disease but that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38508089"
                        open
                      >
                        <summary class="article-expander__title">B cell clonality in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38508089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Carcinogenesis in the process of long-term co-evolution of tumor cells and immune environment essentially becomes possible due to incorrect decisions made, remembered, and reproduced by the immune system at the level of clonal populations of antigen-specific T- and B-lymphocytes. Tumor-immunity interaction determines the nature of such errors and, consequently, delineates the possible ways of successful immunotherapeutic intervention. It is generally recognized that tumor-infiltrating B cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38503736"
                        open
                      >
                        <summary class="article-expander__title">The bone ecosystem facilitates multiple myeloma relapse and the evolution of heterogeneous drug resistant disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38503736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is an osteolytic malignancy that is incurable due to the emergence of treatment resistant disease. Defining how, when and where myeloma cell intrinsic and extrinsic bone microenvironmental mechanisms cause relapse is challenging with current biological approaches. Here, we report a biology-driven spatiotemporal hybrid agent-based model of the MM-bone microenvironment. Results indicate MM intrinsic mechanisms drive the evolution of treatment resistant disease but that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38508089"
                        open
                      >
                        <summary class="article-expander__title">B cell clonality in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38508089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Carcinogenesis in the process of long-term co-evolution of tumor cells and immune environment essentially becomes possible due to incorrect decisions made, remembered, and reproduced by the immune system at the level of clonal populations of antigen-specific T- and B-lymphocytes. Tumor-immunity interaction determines the nature of such errors and, consequently, delineates the possible ways of successful immunotherapeutic intervention. It is generally recognized that tumor-infiltrating B cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38503736"
                        open
                      >
                        <summary class="article-expander__title">The bone ecosystem facilitates multiple myeloma relapse and the evolution of heterogeneous drug resistant disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38503736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is an osteolytic malignancy that is incurable due to the emergence of treatment resistant disease. Defining how, when and where myeloma cell intrinsic and extrinsic bone microenvironmental mechanisms cause relapse is challenging with current biological approaches. Here, we report a biology-driven spatiotemporal hybrid agent-based model of the MM-bone microenvironment. Results indicate MM intrinsic mechanisms drive the evolution of treatment resistant disease but that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38503736"
                        open
                      >
                        <summary class="article-expander__title">The bone ecosystem facilitates multiple myeloma relapse and the evolution of heterogeneous drug resistant disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38503736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is an osteolytic malignancy that is incurable due to the emergence of treatment resistant disease. Defining how, when and where myeloma cell intrinsic and extrinsic bone microenvironmental mechanisms cause relapse is challenging with current biological approaches. Here, we report a biology-driven spatiotemporal hybrid agent-based model of the MM-bone microenvironment. Results indicate MM intrinsic mechanisms drive the evolution of treatment resistant disease but that the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-19 datetime="2024-03-19T10:00:00.000Z">2024-03-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-19 datetime="2024-03-19T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38502057"
                        open
                      >
                        <summary class="article-expander__title">Distinct molecular profiles drive multifaceted characteristics of colorectal cancer metastatic seeds</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38502057/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis of primary tumors remains a challenge for early diagnosis and prevention. The cellular properties and molecular drivers of metastatically competent clones within primary tumors remain unclear. Here, we generated 10-16 single cell-derived lines from each of three colorectal cancer (CRC) tumors to identify and characterize metastatic seeds. We found that intrinsic factors conferred clones with distinct metastatic potential and cellular communication capabilities, determining...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38502057"
                        open
                      >
                        <summary class="article-expander__title">Distinct molecular profiles drive multifaceted characteristics of colorectal cancer metastatic seeds</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38502057/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis of primary tumors remains a challenge for early diagnosis and prevention. The cellular properties and molecular drivers of metastatically competent clones within primary tumors remain unclear. Here, we generated 10-16 single cell-derived lines from each of three colorectal cancer (CRC) tumors to identify and characterize metastatic seeds. We found that intrinsic factors conferred clones with distinct metastatic potential and cellular communication capabilities, determining...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38502057"
                        open
                      >
                        <summary class="article-expander__title">Distinct molecular profiles drive multifaceted characteristics of colorectal cancer metastatic seeds</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38502057/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis of primary tumors remains a challenge for early diagnosis and prevention. The cellular properties and molecular drivers of metastatically competent clones within primary tumors remain unclear. Here, we generated 10-16 single cell-derived lines from each of three colorectal cancer (CRC) tumors to identify and characterize metastatic seeds. We found that intrinsic factors conferred clones with distinct metastatic potential and cellular communication capabilities, determining...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38502057"
                        open
                      >
                        <summary class="article-expander__title">Distinct molecular profiles drive multifaceted characteristics of colorectal cancer metastatic seeds</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38502057/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis of primary tumors remains a challenge for early diagnosis and prevention. The cellular properties and molecular drivers of metastatically competent clones within primary tumors remain unclear. Here, we generated 10-16 single cell-derived lines from each of three colorectal cancer (CRC) tumors to identify and characterize metastatic seeds. We found that intrinsic factors conferred clones with distinct metastatic potential and cellular communication capabilities, determining...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38502057"
                        open
                      >
                        <summary class="article-expander__title">Distinct molecular profiles drive multifaceted characteristics of colorectal cancer metastatic seeds</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38502057/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis of primary tumors remains a challenge for early diagnosis and prevention. The cellular properties and molecular drivers of metastatically competent clones within primary tumors remain unclear. Here, we generated 10-16 single cell-derived lines from each of three colorectal cancer (CRC) tumors to identify and characterize metastatic seeds. We found that intrinsic factors conferred clones with distinct metastatic potential and cellular communication capabilities, determining...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38502057"
                        open
                      >
                        <summary class="article-expander__title">Distinct molecular profiles drive multifaceted characteristics of colorectal cancer metastatic seeds</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38502057/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis of primary tumors remains a challenge for early diagnosis and prevention. The cellular properties and molecular drivers of metastatically competent clones within primary tumors remain unclear. Here, we generated 10-16 single cell-derived lines from each of three colorectal cancer (CRC) tumors to identify and characterize metastatic seeds. We found that intrinsic factors conferred clones with distinct metastatic potential and cellular communication capabilities, determining...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38502057"
                        open
                      >
                        <summary class="article-expander__title">Distinct molecular profiles drive multifaceted characteristics of colorectal cancer metastatic seeds</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38502057/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis of primary tumors remains a challenge for early diagnosis and prevention. The cellular properties and molecular drivers of metastatically competent clones within primary tumors remain unclear. Here, we generated 10-16 single cell-derived lines from each of three colorectal cancer (CRC) tumors to identify and characterize metastatic seeds. We found that intrinsic factors conferred clones with distinct metastatic potential and cellular communication capabilities, determining...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-18 datetime="2024-03-18T10:00:00.000Z">2024-03-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-18 datetime="2024-03-18T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38498034"
                        open
                      >
                        <summary class="article-expander__title">Malignant progression of pre-leukemic disorders</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38498034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The spectrum of myeloid disorders ranges from aplastic bone marrow failure characterized by an empty bone marrow completely lacking in hematopoiesis to acute myeloid leukemia where the marrow space is replaced by undifferentiated leukemic blasts. Recent advances in the capacity to sequence bulk tumor population as well as at a single cell level has provided significant insight into the stepwise process of transformation to acute myeloid leukemia. Using models of progression in the context of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497167"
                        open
                      >
                        <summary class="article-expander__title">Classical meets malignant hematology: a case of acquired epsilongammadeltabeta-thalassemia in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497167/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hemoglobinopathies including thalassemias are among the most frequent genetic disorders worldwide. Primarily, these entities result from germline variants in the globin gene clusters and their cis-acting regulatory elements, and thus the WHO classifies thalassemias as inherited diseases. Non-inherited disorders of globin chain synthesis mimicking the phenotype of thalassemias have also been described and are referred to as acquired thalassemias. These forms mainly affect the alpha-globin genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496542"
                        open
                      >
                        <summary class="article-expander__title">Developmental hematopoietic stem cell variation explains clonal hematopoiesis later in life</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis becomes increasingly common with age, but its cause is enigmatic because driver mutations are often absent. Serial observations infer weak selection indicating variants are acquired much earlier in life with unexplained initial growth spurts. Here we use fluctuating CpG methylation as a lineage marker to track stem cell clonal dynamics of hematopoiesis. We show, via the shared prenatal circulation of monozygotic twins, that weak selection conferred by stem cell variation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496258"
                        open
                      >
                        <summary class="article-expander__title">Red blood cell distribution width (RDW) is associated with unfavorable functional outcome after transfemoral transcatheter aortic valve implantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Elevated preprocedural RDW identifies patients at risk for impaired functional outcome after TAVI and may represent a useful low-cost parameter to guide intensity of outpatient surveillance strategies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38495711"
                        open
                      >
                        <summary class="article-expander__title">Segmentation of anatomical layers and imaging artifacts in intravascular polarization sensitive optical coherence tomography using attending physician and boundary cardinality losses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38495711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intravascular ultrasound and optical coherence tomography are widely available for assessing coronary stenoses and provide critical information to optimize percutaneous coronary intervention. Intravascular polarization-sensitive optical coherence tomography (PS-OCT) measures the polarization state of the light scattered by the vessel wall in addition to conventional cross-sectional images of subsurface microstructure. This affords reconstruction of tissue polarization properties and reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38498034"
                        open
                      >
                        <summary class="article-expander__title">Malignant progression of pre-leukemic disorders</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38498034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The spectrum of myeloid disorders ranges from aplastic bone marrow failure characterized by an empty bone marrow completely lacking in hematopoiesis to acute myeloid leukemia where the marrow space is replaced by undifferentiated leukemic blasts. Recent advances in the capacity to sequence bulk tumor population as well as at a single cell level has provided significant insight into the stepwise process of transformation to acute myeloid leukemia. Using models of progression in the context of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497167"
                        open
                      >
                        <summary class="article-expander__title">Classical meets malignant hematology: a case of acquired epsilongammadeltabeta-thalassemia in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497167/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hemoglobinopathies including thalassemias are among the most frequent genetic disorders worldwide. Primarily, these entities result from germline variants in the globin gene clusters and their cis-acting regulatory elements, and thus the WHO classifies thalassemias as inherited diseases. Non-inherited disorders of globin chain synthesis mimicking the phenotype of thalassemias have also been described and are referred to as acquired thalassemias. These forms mainly affect the alpha-globin genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496542"
                        open
                      >
                        <summary class="article-expander__title">Developmental hematopoietic stem cell variation explains clonal hematopoiesis later in life</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis becomes increasingly common with age, but its cause is enigmatic because driver mutations are often absent. Serial observations infer weak selection indicating variants are acquired much earlier in life with unexplained initial growth spurts. Here we use fluctuating CpG methylation as a lineage marker to track stem cell clonal dynamics of hematopoiesis. We show, via the shared prenatal circulation of monozygotic twins, that weak selection conferred by stem cell variation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496258"
                        open
                      >
                        <summary class="article-expander__title">Red blood cell distribution width (RDW) is associated with unfavorable functional outcome after transfemoral transcatheter aortic valve implantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Elevated preprocedural RDW identifies patients at risk for impaired functional outcome after TAVI and may represent a useful low-cost parameter to guide intensity of outpatient surveillance strategies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38495711"
                        open
                      >
                        <summary class="article-expander__title">Segmentation of anatomical layers and imaging artifacts in intravascular polarization sensitive optical coherence tomography using attending physician and boundary cardinality losses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38495711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intravascular ultrasound and optical coherence tomography are widely available for assessing coronary stenoses and provide critical information to optimize percutaneous coronary intervention. Intravascular polarization-sensitive optical coherence tomography (PS-OCT) measures the polarization state of the light scattered by the vessel wall in addition to conventional cross-sectional images of subsurface microstructure. This affords reconstruction of tissue polarization properties and reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38498034"
                        open
                      >
                        <summary class="article-expander__title">Malignant progression of pre-leukemic disorders</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38498034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The spectrum of myeloid disorders ranges from aplastic bone marrow failure characterized by an empty bone marrow completely lacking in hematopoiesis to acute myeloid leukemia where the marrow space is replaced by undifferentiated leukemic blasts. Recent advances in the capacity to sequence bulk tumor population as well as at a single cell level has provided significant insight into the stepwise process of transformation to acute myeloid leukemia. Using models of progression in the context of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497167"
                        open
                      >
                        <summary class="article-expander__title">Classical meets malignant hematology: a case of acquired epsilongammadeltabeta-thalassemia in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497167/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hemoglobinopathies including thalassemias are among the most frequent genetic disorders worldwide. Primarily, these entities result from germline variants in the globin gene clusters and their cis-acting regulatory elements, and thus the WHO classifies thalassemias as inherited diseases. Non-inherited disorders of globin chain synthesis mimicking the phenotype of thalassemias have also been described and are referred to as acquired thalassemias. These forms mainly affect the alpha-globin genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496542"
                        open
                      >
                        <summary class="article-expander__title">Developmental hematopoietic stem cell variation explains clonal hematopoiesis later in life</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis becomes increasingly common with age, but its cause is enigmatic because driver mutations are often absent. Serial observations infer weak selection indicating variants are acquired much earlier in life with unexplained initial growth spurts. Here we use fluctuating CpG methylation as a lineage marker to track stem cell clonal dynamics of hematopoiesis. We show, via the shared prenatal circulation of monozygotic twins, that weak selection conferred by stem cell variation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496258"
                        open
                      >
                        <summary class="article-expander__title">Red blood cell distribution width (RDW) is associated with unfavorable functional outcome after transfemoral transcatheter aortic valve implantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Elevated preprocedural RDW identifies patients at risk for impaired functional outcome after TAVI and may represent a useful low-cost parameter to guide intensity of outpatient surveillance strategies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38495711"
                        open
                      >
                        <summary class="article-expander__title">Segmentation of anatomical layers and imaging artifacts in intravascular polarization sensitive optical coherence tomography using attending physician and boundary cardinality losses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38495711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intravascular ultrasound and optical coherence tomography are widely available for assessing coronary stenoses and provide critical information to optimize percutaneous coronary intervention. Intravascular polarization-sensitive optical coherence tomography (PS-OCT) measures the polarization state of the light scattered by the vessel wall in addition to conventional cross-sectional images of subsurface microstructure. This affords reconstruction of tissue polarization properties and reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38498034"
                        open
                      >
                        <summary class="article-expander__title">Malignant progression of pre-leukemic disorders</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38498034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The spectrum of myeloid disorders ranges from aplastic bone marrow failure characterized by an empty bone marrow completely lacking in hematopoiesis to acute myeloid leukemia where the marrow space is replaced by undifferentiated leukemic blasts. Recent advances in the capacity to sequence bulk tumor population as well as at a single cell level has provided significant insight into the stepwise process of transformation to acute myeloid leukemia. Using models of progression in the context of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497167"
                        open
                      >
                        <summary class="article-expander__title">Classical meets malignant hematology: a case of acquired epsilongammadeltabeta-thalassemia in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497167/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hemoglobinopathies including thalassemias are among the most frequent genetic disorders worldwide. Primarily, these entities result from germline variants in the globin gene clusters and their cis-acting regulatory elements, and thus the WHO classifies thalassemias as inherited diseases. Non-inherited disorders of globin chain synthesis mimicking the phenotype of thalassemias have also been described and are referred to as acquired thalassemias. These forms mainly affect the alpha-globin genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496542"
                        open
                      >
                        <summary class="article-expander__title">Developmental hematopoietic stem cell variation explains clonal hematopoiesis later in life</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis becomes increasingly common with age, but its cause is enigmatic because driver mutations are often absent. Serial observations infer weak selection indicating variants are acquired much earlier in life with unexplained initial growth spurts. Here we use fluctuating CpG methylation as a lineage marker to track stem cell clonal dynamics of hematopoiesis. We show, via the shared prenatal circulation of monozygotic twins, that weak selection conferred by stem cell variation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496258"
                        open
                      >
                        <summary class="article-expander__title">Red blood cell distribution width (RDW) is associated with unfavorable functional outcome after transfemoral transcatheter aortic valve implantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Elevated preprocedural RDW identifies patients at risk for impaired functional outcome after TAVI and may represent a useful low-cost parameter to guide intensity of outpatient surveillance strategies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38495711"
                        open
                      >
                        <summary class="article-expander__title">Segmentation of anatomical layers and imaging artifacts in intravascular polarization sensitive optical coherence tomography using attending physician and boundary cardinality losses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38495711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intravascular ultrasound and optical coherence tomography are widely available for assessing coronary stenoses and provide critical information to optimize percutaneous coronary intervention. Intravascular polarization-sensitive optical coherence tomography (PS-OCT) measures the polarization state of the light scattered by the vessel wall in addition to conventional cross-sectional images of subsurface microstructure. This affords reconstruction of tissue polarization properties and reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38498034"
                        open
                      >
                        <summary class="article-expander__title">Malignant progression of pre-leukemic disorders</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38498034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The spectrum of myeloid disorders ranges from aplastic bone marrow failure characterized by an empty bone marrow completely lacking in hematopoiesis to acute myeloid leukemia where the marrow space is replaced by undifferentiated leukemic blasts. Recent advances in the capacity to sequence bulk tumor population as well as at a single cell level has provided significant insight into the stepwise process of transformation to acute myeloid leukemia. Using models of progression in the context of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497167"
                        open
                      >
                        <summary class="article-expander__title">Classical meets malignant hematology: a case of acquired epsilongammadeltabeta-thalassemia in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497167/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hemoglobinopathies including thalassemias are among the most frequent genetic disorders worldwide. Primarily, these entities result from germline variants in the globin gene clusters and their cis-acting regulatory elements, and thus the WHO classifies thalassemias as inherited diseases. Non-inherited disorders of globin chain synthesis mimicking the phenotype of thalassemias have also been described and are referred to as acquired thalassemias. These forms mainly affect the alpha-globin genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496542"
                        open
                      >
                        <summary class="article-expander__title">Developmental hematopoietic stem cell variation explains clonal hematopoiesis later in life</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis becomes increasingly common with age, but its cause is enigmatic because driver mutations are often absent. Serial observations infer weak selection indicating variants are acquired much earlier in life with unexplained initial growth spurts. Here we use fluctuating CpG methylation as a lineage marker to track stem cell clonal dynamics of hematopoiesis. We show, via the shared prenatal circulation of monozygotic twins, that weak selection conferred by stem cell variation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496258"
                        open
                      >
                        <summary class="article-expander__title">Red blood cell distribution width (RDW) is associated with unfavorable functional outcome after transfemoral transcatheter aortic valve implantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Elevated preprocedural RDW identifies patients at risk for impaired functional outcome after TAVI and may represent a useful low-cost parameter to guide intensity of outpatient surveillance strategies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38495711"
                        open
                      >
                        <summary class="article-expander__title">Segmentation of anatomical layers and imaging artifacts in intravascular polarization sensitive optical coherence tomography using attending physician and boundary cardinality losses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38495711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intravascular ultrasound and optical coherence tomography are widely available for assessing coronary stenoses and provide critical information to optimize percutaneous coronary intervention. Intravascular polarization-sensitive optical coherence tomography (PS-OCT) measures the polarization state of the light scattered by the vessel wall in addition to conventional cross-sectional images of subsurface microstructure. This affords reconstruction of tissue polarization properties and reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38498034"
                        open
                      >
                        <summary class="article-expander__title">Malignant progression of pre-leukemic disorders</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38498034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The spectrum of myeloid disorders ranges from aplastic bone marrow failure characterized by an empty bone marrow completely lacking in hematopoiesis to acute myeloid leukemia where the marrow space is replaced by undifferentiated leukemic blasts. Recent advances in the capacity to sequence bulk tumor population as well as at a single cell level has provided significant insight into the stepwise process of transformation to acute myeloid leukemia. Using models of progression in the context of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497167"
                        open
                      >
                        <summary class="article-expander__title">Classical meets malignant hematology: a case of acquired epsilongammadeltabeta-thalassemia in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497167/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hemoglobinopathies including thalassemias are among the most frequent genetic disorders worldwide. Primarily, these entities result from germline variants in the globin gene clusters and their cis-acting regulatory elements, and thus the WHO classifies thalassemias as inherited diseases. Non-inherited disorders of globin chain synthesis mimicking the phenotype of thalassemias have also been described and are referred to as acquired thalassemias. These forms mainly affect the alpha-globin genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496542"
                        open
                      >
                        <summary class="article-expander__title">Developmental hematopoietic stem cell variation explains clonal hematopoiesis later in life</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis becomes increasingly common with age, but its cause is enigmatic because driver mutations are often absent. Serial observations infer weak selection indicating variants are acquired much earlier in life with unexplained initial growth spurts. Here we use fluctuating CpG methylation as a lineage marker to track stem cell clonal dynamics of hematopoiesis. We show, via the shared prenatal circulation of monozygotic twins, that weak selection conferred by stem cell variation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496258"
                        open
                      >
                        <summary class="article-expander__title">Red blood cell distribution width (RDW) is associated with unfavorable functional outcome after transfemoral transcatheter aortic valve implantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Elevated preprocedural RDW identifies patients at risk for impaired functional outcome after TAVI and may represent a useful low-cost parameter to guide intensity of outpatient surveillance strategies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38495711"
                        open
                      >
                        <summary class="article-expander__title">Segmentation of anatomical layers and imaging artifacts in intravascular polarization sensitive optical coherence tomography using attending physician and boundary cardinality losses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38495711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intravascular ultrasound and optical coherence tomography are widely available for assessing coronary stenoses and provide critical information to optimize percutaneous coronary intervention. Intravascular polarization-sensitive optical coherence tomography (PS-OCT) measures the polarization state of the light scattered by the vessel wall in addition to conventional cross-sectional images of subsurface microstructure. This affords reconstruction of tissue polarization properties and reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38498034"
                        open
                      >
                        <summary class="article-expander__title">Malignant progression of pre-leukemic disorders</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38498034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The spectrum of myeloid disorders ranges from aplastic bone marrow failure characterized by an empty bone marrow completely lacking in hematopoiesis to acute myeloid leukemia where the marrow space is replaced by undifferentiated leukemic blasts. Recent advances in the capacity to sequence bulk tumor population as well as at a single cell level has provided significant insight into the stepwise process of transformation to acute myeloid leukemia. Using models of progression in the context of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497167"
                        open
                      >
                        <summary class="article-expander__title">Classical meets malignant hematology: a case of acquired epsilongammadeltabeta-thalassemia in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497167/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hemoglobinopathies including thalassemias are among the most frequent genetic disorders worldwide. Primarily, these entities result from germline variants in the globin gene clusters and their cis-acting regulatory elements, and thus the WHO classifies thalassemias as inherited diseases. Non-inherited disorders of globin chain synthesis mimicking the phenotype of thalassemias have also been described and are referred to as acquired thalassemias. These forms mainly affect the alpha-globin genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496542"
                        open
                      >
                        <summary class="article-expander__title">Developmental hematopoietic stem cell variation explains clonal hematopoiesis later in life</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis becomes increasingly common with age, but its cause is enigmatic because driver mutations are often absent. Serial observations infer weak selection indicating variants are acquired much earlier in life with unexplained initial growth spurts. Here we use fluctuating CpG methylation as a lineage marker to track stem cell clonal dynamics of hematopoiesis. We show, via the shared prenatal circulation of monozygotic twins, that weak selection conferred by stem cell variation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496258"
                        open
                      >
                        <summary class="article-expander__title">Red blood cell distribution width (RDW) is associated with unfavorable functional outcome after transfemoral transcatheter aortic valve implantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Elevated preprocedural RDW identifies patients at risk for impaired functional outcome after TAVI and may represent a useful low-cost parameter to guide intensity of outpatient surveillance strategies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38495711"
                        open
                      >
                        <summary class="article-expander__title">Segmentation of anatomical layers and imaging artifacts in intravascular polarization sensitive optical coherence tomography using attending physician and boundary cardinality losses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38495711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intravascular ultrasound and optical coherence tomography are widely available for assessing coronary stenoses and provide critical information to optimize percutaneous coronary intervention. Intravascular polarization-sensitive optical coherence tomography (PS-OCT) measures the polarization state of the light scattered by the vessel wall in addition to conventional cross-sectional images of subsurface microstructure. This affords reconstruction of tissue polarization properties and reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38498034"
                        open
                      >
                        <summary class="article-expander__title">Malignant progression of pre-leukemic disorders</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38498034/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The spectrum of myeloid disorders ranges from aplastic bone marrow failure characterized by an empty bone marrow completely lacking in hematopoiesis to acute myeloid leukemia where the marrow space is replaced by undifferentiated leukemic blasts. Recent advances in the capacity to sequence bulk tumor population as well as at a single cell level has provided significant insight into the stepwise process of transformation to acute myeloid leukemia. Using models of progression in the context of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497167"
                        open
                      >
                        <summary class="article-expander__title">Classical meets malignant hematology: a case of acquired epsilongammadeltabeta-thalassemia in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497167/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hemoglobinopathies including thalassemias are among the most frequent genetic disorders worldwide. Primarily, these entities result from germline variants in the globin gene clusters and their cis-acting regulatory elements, and thus the WHO classifies thalassemias as inherited diseases. Non-inherited disorders of globin chain synthesis mimicking the phenotype of thalassemias have also been described and are referred to as acquired thalassemias. These forms mainly affect the alpha-globin genes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496542"
                        open
                      >
                        <summary class="article-expander__title">Developmental hematopoietic stem cell variation explains clonal hematopoiesis later in life</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496542/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis becomes increasingly common with age, but its cause is enigmatic because driver mutations are often absent. Serial observations infer weak selection indicating variants are acquired much earlier in life with unexplained initial growth spurts. Here we use fluctuating CpG methylation as a lineage marker to track stem cell clonal dynamics of hematopoiesis. We show, via the shared prenatal circulation of monozygotic twins, that weak selection conferred by stem cell variation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496258"
                        open
                      >
                        <summary class="article-expander__title">Red blood cell distribution width (RDW) is associated with unfavorable functional outcome after transfemoral transcatheter aortic valve implantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Elevated preprocedural RDW identifies patients at risk for impaired functional outcome after TAVI and may represent a useful low-cost parameter to guide intensity of outpatient surveillance strategies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38495711"
                        open
                      >
                        <summary class="article-expander__title">Segmentation of anatomical layers and imaging artifacts in intravascular polarization sensitive optical coherence tomography using attending physician and boundary cardinality losses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38495711/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intravascular ultrasound and optical coherence tomography are widely available for assessing coronary stenoses and provide critical information to optimize percutaneous coronary intervention. Intravascular polarization-sensitive optical coherence tomography (PS-OCT) measures the polarization state of the light scattered by the vessel wall in addition to conventional cross-sectional images of subsurface microstructure. This affords reconstruction of tissue polarization properties and reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496472"
                        open
                      >
                        <summary class="article-expander__title">Visualizing the Spatio-Temporal Dynamics of Clonal Evolution with LinG3D software</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496472/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer clonal evolution, especially following anti-cancer treatments, depends on the locations of the mutated cells within the tumor tissue. Cells near the vessels, exposed to higher concentrations of drugs, will undergo a different evolutionary path than cells residing far from the vasculature in the areas of lower drug levels. However, classical representations of cell lineage trees do not account for this spatial component of emerging cancer clones. Here, we propose the LinG3D (Lineage Graphs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496441"
                        open
                      >
                        <summary class="article-expander__title">De novo detection of somatic variants in long-read single-cell RNA sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496441/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In cancer, genetic and transcriptomic variations generate clonal heterogeneity, possibly leading to treatment resistance. Long-read single-cell RNA sequencing (LR scRNA-seq) has the potential to detect genetic and transcriptomic variations simultaneously. Here, we present LongSom, a computational workflow leveraging LR scRNA-seq data to call de novo somatic single-nucleotide variants (SNVs), copy-number alterations (CNAs), and gene fusions to reconstruct the tumor clonal heterogeneity. For SNV...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496472"
                        open
                      >
                        <summary class="article-expander__title">Visualizing the Spatio-Temporal Dynamics of Clonal Evolution with LinG3D software</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496472/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer clonal evolution, especially following anti-cancer treatments, depends on the locations of the mutated cells within the tumor tissue. Cells near the vessels, exposed to higher concentrations of drugs, will undergo a different evolutionary path than cells residing far from the vasculature in the areas of lower drug levels. However, classical representations of cell lineage trees do not account for this spatial component of emerging cancer clones. Here, we propose the LinG3D (Lineage Graphs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496441"
                        open
                      >
                        <summary class="article-expander__title">De novo detection of somatic variants in long-read single-cell RNA sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496441/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In cancer, genetic and transcriptomic variations generate clonal heterogeneity, possibly leading to treatment resistance. Long-read single-cell RNA sequencing (LR scRNA-seq) has the potential to detect genetic and transcriptomic variations simultaneously. Here, we present LongSom, a computational workflow leveraging LR scRNA-seq data to call de novo somatic single-nucleotide variants (SNVs), copy-number alterations (CNAs), and gene fusions to reconstruct the tumor clonal heterogeneity. For SNV...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496472"
                        open
                      >
                        <summary class="article-expander__title">Visualizing the Spatio-Temporal Dynamics of Clonal Evolution with LinG3D software</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496472/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer clonal evolution, especially following anti-cancer treatments, depends on the locations of the mutated cells within the tumor tissue. Cells near the vessels, exposed to higher concentrations of drugs, will undergo a different evolutionary path than cells residing far from the vasculature in the areas of lower drug levels. However, classical representations of cell lineage trees do not account for this spatial component of emerging cancer clones. Here, we propose the LinG3D (Lineage Graphs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496441"
                        open
                      >
                        <summary class="article-expander__title">De novo detection of somatic variants in long-read single-cell RNA sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496441/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In cancer, genetic and transcriptomic variations generate clonal heterogeneity, possibly leading to treatment resistance. Long-read single-cell RNA sequencing (LR scRNA-seq) has the potential to detect genetic and transcriptomic variations simultaneously. Here, we present LongSom, a computational workflow leveraging LR scRNA-seq data to call de novo somatic single-nucleotide variants (SNVs), copy-number alterations (CNAs), and gene fusions to reconstruct the tumor clonal heterogeneity. For SNV...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496472"
                        open
                      >
                        <summary class="article-expander__title">Visualizing the Spatio-Temporal Dynamics of Clonal Evolution with LinG3D software</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496472/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer clonal evolution, especially following anti-cancer treatments, depends on the locations of the mutated cells within the tumor tissue. Cells near the vessels, exposed to higher concentrations of drugs, will undergo a different evolutionary path than cells residing far from the vasculature in the areas of lower drug levels. However, classical representations of cell lineage trees do not account for this spatial component of emerging cancer clones. Here, we propose the LinG3D (Lineage Graphs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496441"
                        open
                      >
                        <summary class="article-expander__title">De novo detection of somatic variants in long-read single-cell RNA sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496441/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In cancer, genetic and transcriptomic variations generate clonal heterogeneity, possibly leading to treatment resistance. Long-read single-cell RNA sequencing (LR scRNA-seq) has the potential to detect genetic and transcriptomic variations simultaneously. Here, we present LongSom, a computational workflow leveraging LR scRNA-seq data to call de novo somatic single-nucleotide variants (SNVs), copy-number alterations (CNAs), and gene fusions to reconstruct the tumor clonal heterogeneity. For SNV...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496472"
                        open
                      >
                        <summary class="article-expander__title">Visualizing the Spatio-Temporal Dynamics of Clonal Evolution with LinG3D software</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496472/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer clonal evolution, especially following anti-cancer treatments, depends on the locations of the mutated cells within the tumor tissue. Cells near the vessels, exposed to higher concentrations of drugs, will undergo a different evolutionary path than cells residing far from the vasculature in the areas of lower drug levels. However, classical representations of cell lineage trees do not account for this spatial component of emerging cancer clones. Here, we propose the LinG3D (Lineage Graphs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496441"
                        open
                      >
                        <summary class="article-expander__title">De novo detection of somatic variants in long-read single-cell RNA sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496441/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In cancer, genetic and transcriptomic variations generate clonal heterogeneity, possibly leading to treatment resistance. Long-read single-cell RNA sequencing (LR scRNA-seq) has the potential to detect genetic and transcriptomic variations simultaneously. Here, we present LongSom, a computational workflow leveraging LR scRNA-seq data to call de novo somatic single-nucleotide variants (SNVs), copy-number alterations (CNAs), and gene fusions to reconstruct the tumor clonal heterogeneity. For SNV...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496472"
                        open
                      >
                        <summary class="article-expander__title">Visualizing the Spatio-Temporal Dynamics of Clonal Evolution with LinG3D software</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496472/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer clonal evolution, especially following anti-cancer treatments, depends on the locations of the mutated cells within the tumor tissue. Cells near the vessels, exposed to higher concentrations of drugs, will undergo a different evolutionary path than cells residing far from the vasculature in the areas of lower drug levels. However, classical representations of cell lineage trees do not account for this spatial component of emerging cancer clones. Here, we propose the LinG3D (Lineage Graphs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496441"
                        open
                      >
                        <summary class="article-expander__title">De novo detection of somatic variants in long-read single-cell RNA sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496441/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In cancer, genetic and transcriptomic variations generate clonal heterogeneity, possibly leading to treatment resistance. Long-read single-cell RNA sequencing (LR scRNA-seq) has the potential to detect genetic and transcriptomic variations simultaneously. Here, we present LongSom, a computational workflow leveraging LR scRNA-seq data to call de novo somatic single-nucleotide variants (SNVs), copy-number alterations (CNAs), and gene fusions to reconstruct the tumor clonal heterogeneity. For SNV...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38497538"
                        open
                      >
                        <summary class="article-expander__title">Cancer patients with clonal hematopoiesis die from primary malignancy or comorbidities despite higher rates of transformation to myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38497538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In a population including mostly cancer patients, CH was associated with comorbidities and myeloid transformation in patients with higher mutational burdens and anemia. Nevertheless, such patients were less likely to die of their myeloid neoplasm than of primary malignancy or comorbidities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496472"
                        open
                      >
                        <summary class="article-expander__title">Visualizing the Spatio-Temporal Dynamics of Clonal Evolution with LinG3D software</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496472/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer clonal evolution, especially following anti-cancer treatments, depends on the locations of the mutated cells within the tumor tissue. Cells near the vessels, exposed to higher concentrations of drugs, will undergo a different evolutionary path than cells residing far from the vasculature in the areas of lower drug levels. However, classical representations of cell lineage trees do not account for this spatial component of emerging cancer clones. Here, we propose the LinG3D (Lineage Graphs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496441"
                        open
                      >
                        <summary class="article-expander__title">De novo detection of somatic variants in long-read single-cell RNA sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496441/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In cancer, genetic and transcriptomic variations generate clonal heterogeneity, possibly leading to treatment resistance. Long-read single-cell RNA sequencing (LR scRNA-seq) has the potential to detect genetic and transcriptomic variations simultaneously. Here, we present LongSom, a computational workflow leveraging LR scRNA-seq data to call de novo somatic single-nucleotide variants (SNVs), copy-number alterations (CNAs), and gene fusions to reconstruct the tumor clonal heterogeneity. For SNV...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496472"
                        open
                      >
                        <summary class="article-expander__title">Visualizing the Spatio-Temporal Dynamics of Clonal Evolution with LinG3D software</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496472/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer clonal evolution, especially following anti-cancer treatments, depends on the locations of the mutated cells within the tumor tissue. Cells near the vessels, exposed to higher concentrations of drugs, will undergo a different evolutionary path than cells residing far from the vasculature in the areas of lower drug levels. However, classical representations of cell lineage trees do not account for this spatial component of emerging cancer clones. Here, we propose the LinG3D (Lineage Graphs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38496441"
                        open
                      >
                        <summary class="article-expander__title">De novo detection of somatic variants in long-read single-cell RNA sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38496441/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In cancer, genetic and transcriptomic variations generate clonal heterogeneity, possibly leading to treatment resistance. Long-read single-cell RNA sequencing (LR scRNA-seq) has the potential to detect genetic and transcriptomic variations simultaneously. Here, we present LongSom, a computational workflow leveraging LR scRNA-seq data to call de novo somatic single-nucleotide variants (SNVs), copy-number alterations (CNAs), and gene fusions to reconstruct the tumor clonal heterogeneity. For SNV...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-15 datetime="2024-03-15T10:00:00.000Z">2024-03-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-15 datetime="2024-03-15T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38488948"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutations in aging and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38488948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Time always leaves its mark, and our genome is no exception. Mutations in the genome of somatic cells were first hypothesized to be the cause of aging in the 1950s, shortly after the molecular structure of DNA had been described. Somatic mutation theories of aging are based on the fact that mutations in DNA as the ultimate template for all cellular functions are irreversible. However, it took until the 1990s to develop the methods to test if DNA mutations accumulate with age in different organs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38488948"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutations in aging and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38488948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Time always leaves its mark, and our genome is no exception. Mutations in the genome of somatic cells were first hypothesized to be the cause of aging in the 1950s, shortly after the molecular structure of DNA had been described. Somatic mutation theories of aging are based on the fact that mutations in DNA as the ultimate template for all cellular functions are irreversible. However, it took until the 1990s to develop the methods to test if DNA mutations accumulate with age in different organs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38488948"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutations in aging and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38488948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Time always leaves its mark, and our genome is no exception. Mutations in the genome of somatic cells were first hypothesized to be the cause of aging in the 1950s, shortly after the molecular structure of DNA had been described. Somatic mutation theories of aging are based on the fact that mutations in DNA as the ultimate template for all cellular functions are irreversible. However, it took until the 1990s to develop the methods to test if DNA mutations accumulate with age in different organs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38488948"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutations in aging and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38488948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Time always leaves its mark, and our genome is no exception. Mutations in the genome of somatic cells were first hypothesized to be the cause of aging in the 1950s, shortly after the molecular structure of DNA had been described. Somatic mutation theories of aging are based on the fact that mutations in DNA as the ultimate template for all cellular functions are irreversible. However, it took until the 1990s to develop the methods to test if DNA mutations accumulate with age in different organs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38488948"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutations in aging and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38488948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Time always leaves its mark, and our genome is no exception. Mutations in the genome of somatic cells were first hypothesized to be the cause of aging in the 1950s, shortly after the molecular structure of DNA had been described. Somatic mutation theories of aging are based on the fact that mutations in DNA as the ultimate template for all cellular functions are irreversible. However, it took until the 1990s to develop the methods to test if DNA mutations accumulate with age in different organs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38488948"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutations in aging and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38488948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Time always leaves its mark, and our genome is no exception. Mutations in the genome of somatic cells were first hypothesized to be the cause of aging in the 1950s, shortly after the molecular structure of DNA had been described. Somatic mutation theories of aging are based on the fact that mutations in DNA as the ultimate template for all cellular functions are irreversible. However, it took until the 1990s to develop the methods to test if DNA mutations accumulate with age in different organs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38488948"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutations in aging and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38488948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Time always leaves its mark, and our genome is no exception. Mutations in the genome of somatic cells were first hypothesized to be the cause of aging in the 1950s, shortly after the molecular structure of DNA had been described. Somatic mutation theories of aging are based on the fact that mutations in DNA as the ultimate template for all cellular functions are irreversible. However, it took until the 1990s to develop the methods to test if DNA mutations accumulate with age in different organs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38488948"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutations in aging and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38488948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Time always leaves its mark, and our genome is no exception. Mutations in the genome of somatic cells were first hypothesized to be the cause of aging in the 1950s, shortly after the molecular structure of DNA had been described. Somatic mutation theories of aging are based on the fact that mutations in DNA as the ultimate template for all cellular functions are irreversible. However, it took until the 1990s to develop the methods to test if DNA mutations accumulate with age in different organs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38488948"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutations in aging and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38488948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Time always leaves its mark, and our genome is no exception. Mutations in the genome of somatic cells were first hypothesized to be the cause of aging in the 1950s, shortly after the molecular structure of DNA had been described. Somatic mutation theories of aging are based on the fact that mutations in DNA as the ultimate template for all cellular functions are irreversible. However, it took until the 1990s to develop the methods to test if DNA mutations accumulate with age in different organs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38488948"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutations in aging and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38488948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Time always leaves its mark, and our genome is no exception. Mutations in the genome of somatic cells were first hypothesized to be the cause of aging in the 1950s, shortly after the molecular structure of DNA had been described. Somatic mutation theories of aging are based on the fact that mutations in DNA as the ultimate template for all cellular functions are irreversible. However, it took until the 1990s to develop the methods to test if DNA mutations accumulate with age in different organs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38488948"
                        open
                      >
                        <summary class="article-expander__title">Somatic mutations in aging and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38488948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Time always leaves its mark, and our genome is no exception. Mutations in the genome of somatic cells were first hypothesized to be the cause of aging in the 1950s, shortly after the molecular structure of DNA had been described. Somatic mutation theories of aging are based on the fact that mutations in DNA as the ultimate template for all cellular functions are irreversible. However, it took until the 1990s to develop the methods to test if DNA mutations accumulate with age in different organs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38486887"
                        open
                      >
                        <summary class="article-expander__title">Joint inference of clonal structure using single-cell genome and transcriptome sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38486887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Latest advancements in the high-throughput single-cell genome (scDNA) and transcriptome (scRNA) sequencing technologies enabled cell-resolved investigation of tissue clones. However, it remains challenging to cluster and couple single cells for heterogeneous scRNA and scDNA data generated from the same specimen. In this study, we present a computational framework called CCNMF, which employs a novel Coupled-Clone Non-negative Matrix Factorization technique to jointly infer clonal structure for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38486887"
                        open
                      >
                        <summary class="article-expander__title">Joint inference of clonal structure using single-cell genome and transcriptome sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38486887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Latest advancements in the high-throughput single-cell genome (scDNA) and transcriptome (scRNA) sequencing technologies enabled cell-resolved investigation of tissue clones. However, it remains challenging to cluster and couple single cells for heterogeneous scRNA and scDNA data generated from the same specimen. In this study, we present a computational framework called CCNMF, which employs a novel Coupled-Clone Non-negative Matrix Factorization technique to jointly infer clonal structure for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38486887"
                        open
                      >
                        <summary class="article-expander__title">Joint inference of clonal structure using single-cell genome and transcriptome sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38486887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Latest advancements in the high-throughput single-cell genome (scDNA) and transcriptome (scRNA) sequencing technologies enabled cell-resolved investigation of tissue clones. However, it remains challenging to cluster and couple single cells for heterogeneous scRNA and scDNA data generated from the same specimen. In this study, we present a computational framework called CCNMF, which employs a novel Coupled-Clone Non-negative Matrix Factorization technique to jointly infer clonal structure for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38486887"
                        open
                      >
                        <summary class="article-expander__title">Joint inference of clonal structure using single-cell genome and transcriptome sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38486887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Latest advancements in the high-throughput single-cell genome (scDNA) and transcriptome (scRNA) sequencing technologies enabled cell-resolved investigation of tissue clones. However, it remains challenging to cluster and couple single cells for heterogeneous scRNA and scDNA data generated from the same specimen. In this study, we present a computational framework called CCNMF, which employs a novel Coupled-Clone Non-negative Matrix Factorization technique to jointly infer clonal structure for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38486887"
                        open
                      >
                        <summary class="article-expander__title">Joint inference of clonal structure using single-cell genome and transcriptome sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38486887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Latest advancements in the high-throughput single-cell genome (scDNA) and transcriptome (scRNA) sequencing technologies enabled cell-resolved investigation of tissue clones. However, it remains challenging to cluster and couple single cells for heterogeneous scRNA and scDNA data generated from the same specimen. In this study, we present a computational framework called CCNMF, which employs a novel Coupled-Clone Non-negative Matrix Factorization technique to jointly infer clonal structure for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38486887"
                        open
                      >
                        <summary class="article-expander__title">Joint inference of clonal structure using single-cell genome and transcriptome sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38486887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Latest advancements in the high-throughput single-cell genome (scDNA) and transcriptome (scRNA) sequencing technologies enabled cell-resolved investigation of tissue clones. However, it remains challenging to cluster and couple single cells for heterogeneous scRNA and scDNA data generated from the same specimen. In this study, we present a computational framework called CCNMF, which employs a novel Coupled-Clone Non-negative Matrix Factorization technique to jointly infer clonal structure for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38486887"
                        open
                      >
                        <summary class="article-expander__title">Joint inference of clonal structure using single-cell genome and transcriptome sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38486887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Latest advancements in the high-throughput single-cell genome (scDNA) and transcriptome (scRNA) sequencing technologies enabled cell-resolved investigation of tissue clones. However, it remains challenging to cluster and couple single cells for heterogeneous scRNA and scDNA data generated from the same specimen. In this study, we present a computational framework called CCNMF, which employs a novel Coupled-Clone Non-negative Matrix Factorization technique to jointly infer clonal structure for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38486887"
                        open
                      >
                        <summary class="article-expander__title">Joint inference of clonal structure using single-cell genome and transcriptome sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38486887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Latest advancements in the high-throughput single-cell genome (scDNA) and transcriptome (scRNA) sequencing technologies enabled cell-resolved investigation of tissue clones. However, it remains challenging to cluster and couple single cells for heterogeneous scRNA and scDNA data generated from the same specimen. In this study, we present a computational framework called CCNMF, which employs a novel Coupled-Clone Non-negative Matrix Factorization technique to jointly infer clonal structure for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38486887"
                        open
                      >
                        <summary class="article-expander__title">Joint inference of clonal structure using single-cell genome and transcriptome sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38486887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Latest advancements in the high-throughput single-cell genome (scDNA) and transcriptome (scRNA) sequencing technologies enabled cell-resolved investigation of tissue clones. However, it remains challenging to cluster and couple single cells for heterogeneous scRNA and scDNA data generated from the same specimen. In this study, we present a computational framework called CCNMF, which employs a novel Coupled-Clone Non-negative Matrix Factorization technique to jointly infer clonal structure for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38486887"
                        open
                      >
                        <summary class="article-expander__title">Joint inference of clonal structure using single-cell genome and transcriptome sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38486887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Latest advancements in the high-throughput single-cell genome (scDNA) and transcriptome (scRNA) sequencing technologies enabled cell-resolved investigation of tissue clones. However, it remains challenging to cluster and couple single cells for heterogeneous scRNA and scDNA data generated from the same specimen. In this study, we present a computational framework called CCNMF, which employs a novel Coupled-Clone Non-negative Matrix Factorization technique to jointly infer clonal structure for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38486887"
                        open
                      >
                        <summary class="article-expander__title">Joint inference of clonal structure using single-cell genome and transcriptome sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38486887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Latest advancements in the high-throughput single-cell genome (scDNA) and transcriptome (scRNA) sequencing technologies enabled cell-resolved investigation of tissue clones. However, it remains challenging to cluster and couple single cells for heterogeneous scRNA and scDNA data generated from the same specimen. In this study, we present a computational framework called CCNMF, which employs a novel Coupled-Clone Non-negative Matrix Factorization technique to jointly infer clonal structure for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-14 datetime="2024-03-14T10:00:00.000Z">2024-03-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-14 datetime="2024-03-14T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38485690"
                        open
                      >
                        <summary class="article-expander__title">ArCH: Improving the performance of clonal hematopoiesis variant calling and interpretation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38485690/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations in hematopoietic stem and progenitor stem cells (HSPCs) with resultant clonal expansion, termed clonal hematopoiesis (CH), is associated with increased risk of hematologic malignancies and other adverse outcomes. CH is generally present at low allelic fractions, but clonal expansion and acquisition of additional mutations leads to hematologic cancers in a small proportion of individuals. With high depth and high sensitivity sequencing, CH can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38481290"
                        open
                      >
                        <summary class="article-expander__title">p53 biology and reactivation for improved therapy in MDS and AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38481290/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML) originate from preleukemic hematopoietic conditions, such as clonal hematopoiesis of indeterminate potential (CHIP) or clonal cytopenia of undetermined significance (CCUS) and have variable outcomes despite the successful implementation of targeted therapies. The prognosis differs depending on the molecular subgroup. In patients with TP53 mutations, the most inferior outcomes across independent studies were observed. Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38485690"
                        open
                      >
                        <summary class="article-expander__title">ArCH: Improving the performance of clonal hematopoiesis variant calling and interpretation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38485690/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations in hematopoietic stem and progenitor stem cells (HSPCs) with resultant clonal expansion, termed clonal hematopoiesis (CH), is associated with increased risk of hematologic malignancies and other adverse outcomes. CH is generally present at low allelic fractions, but clonal expansion and acquisition of additional mutations leads to hematologic cancers in a small proportion of individuals. With high depth and high sensitivity sequencing, CH can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38481290"
                        open
                      >
                        <summary class="article-expander__title">p53 biology and reactivation for improved therapy in MDS and AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38481290/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML) originate from preleukemic hematopoietic conditions, such as clonal hematopoiesis of indeterminate potential (CHIP) or clonal cytopenia of undetermined significance (CCUS) and have variable outcomes despite the successful implementation of targeted therapies. The prognosis differs depending on the molecular subgroup. In patients with TP53 mutations, the most inferior outcomes across independent studies were observed. Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38485690"
                        open
                      >
                        <summary class="article-expander__title">ArCH: Improving the performance of clonal hematopoiesis variant calling and interpretation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38485690/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations in hematopoietic stem and progenitor stem cells (HSPCs) with resultant clonal expansion, termed clonal hematopoiesis (CH), is associated with increased risk of hematologic malignancies and other adverse outcomes. CH is generally present at low allelic fractions, but clonal expansion and acquisition of additional mutations leads to hematologic cancers in a small proportion of individuals. With high depth and high sensitivity sequencing, CH can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38481290"
                        open
                      >
                        <summary class="article-expander__title">p53 biology and reactivation for improved therapy in MDS and AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38481290/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML) originate from preleukemic hematopoietic conditions, such as clonal hematopoiesis of indeterminate potential (CHIP) or clonal cytopenia of undetermined significance (CCUS) and have variable outcomes despite the successful implementation of targeted therapies. The prognosis differs depending on the molecular subgroup. In patients with TP53 mutations, the most inferior outcomes across independent studies were observed. Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38485690"
                        open
                      >
                        <summary class="article-expander__title">ArCH: Improving the performance of clonal hematopoiesis variant calling and interpretation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38485690/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations in hematopoietic stem and progenitor stem cells (HSPCs) with resultant clonal expansion, termed clonal hematopoiesis (CH), is associated with increased risk of hematologic malignancies and other adverse outcomes. CH is generally present at low allelic fractions, but clonal expansion and acquisition of additional mutations leads to hematologic cancers in a small proportion of individuals. With high depth and high sensitivity sequencing, CH can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38481290"
                        open
                      >
                        <summary class="article-expander__title">p53 biology and reactivation for improved therapy in MDS and AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38481290/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML) originate from preleukemic hematopoietic conditions, such as clonal hematopoiesis of indeterminate potential (CHIP) or clonal cytopenia of undetermined significance (CCUS) and have variable outcomes despite the successful implementation of targeted therapies. The prognosis differs depending on the molecular subgroup. In patients with TP53 mutations, the most inferior outcomes across independent studies were observed. Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38485690"
                        open
                      >
                        <summary class="article-expander__title">ArCH: Improving the performance of clonal hematopoiesis variant calling and interpretation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38485690/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations in hematopoietic stem and progenitor stem cells (HSPCs) with resultant clonal expansion, termed clonal hematopoiesis (CH), is associated with increased risk of hematologic malignancies and other adverse outcomes. CH is generally present at low allelic fractions, but clonal expansion and acquisition of additional mutations leads to hematologic cancers in a small proportion of individuals. With high depth and high sensitivity sequencing, CH can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38481290"
                        open
                      >
                        <summary class="article-expander__title">p53 biology and reactivation for improved therapy in MDS and AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38481290/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML) originate from preleukemic hematopoietic conditions, such as clonal hematopoiesis of indeterminate potential (CHIP) or clonal cytopenia of undetermined significance (CCUS) and have variable outcomes despite the successful implementation of targeted therapies. The prognosis differs depending on the molecular subgroup. In patients with TP53 mutations, the most inferior outcomes across independent studies were observed. Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38485690"
                        open
                      >
                        <summary class="article-expander__title">ArCH: Improving the performance of clonal hematopoiesis variant calling and interpretation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38485690/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations in hematopoietic stem and progenitor stem cells (HSPCs) with resultant clonal expansion, termed clonal hematopoiesis (CH), is associated with increased risk of hematologic malignancies and other adverse outcomes. CH is generally present at low allelic fractions, but clonal expansion and acquisition of additional mutations leads to hematologic cancers in a small proportion of individuals. With high depth and high sensitivity sequencing, CH can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38481290"
                        open
                      >
                        <summary class="article-expander__title">p53 biology and reactivation for improved therapy in MDS and AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38481290/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML) originate from preleukemic hematopoietic conditions, such as clonal hematopoiesis of indeterminate potential (CHIP) or clonal cytopenia of undetermined significance (CCUS) and have variable outcomes despite the successful implementation of targeted therapies. The prognosis differs depending on the molecular subgroup. In patients with TP53 mutations, the most inferior outcomes across independent studies were observed. Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38485690"
                        open
                      >
                        <summary class="article-expander__title">ArCH: Improving the performance of clonal hematopoiesis variant calling and interpretation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38485690/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations in hematopoietic stem and progenitor stem cells (HSPCs) with resultant clonal expansion, termed clonal hematopoiesis (CH), is associated with increased risk of hematologic malignancies and other adverse outcomes. CH is generally present at low allelic fractions, but clonal expansion and acquisition of additional mutations leads to hematologic cancers in a small proportion of individuals. With high depth and high sensitivity sequencing, CH can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38481290"
                        open
                      >
                        <summary class="article-expander__title">p53 biology and reactivation for improved therapy in MDS and AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38481290/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML) originate from preleukemic hematopoietic conditions, such as clonal hematopoiesis of indeterminate potential (CHIP) or clonal cytopenia of undetermined significance (CCUS) and have variable outcomes despite the successful implementation of targeted therapies. The prognosis differs depending on the molecular subgroup. In patients with TP53 mutations, the most inferior outcomes across independent studies were observed. Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38485690"
                        open
                      >
                        <summary class="article-expander__title">ArCH: Improving the performance of clonal hematopoiesis variant calling and interpretation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38485690/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations in hematopoietic stem and progenitor stem cells (HSPCs) with resultant clonal expansion, termed clonal hematopoiesis (CH), is associated with increased risk of hematologic malignancies and other adverse outcomes. CH is generally present at low allelic fractions, but clonal expansion and acquisition of additional mutations leads to hematologic cancers in a small proportion of individuals. With high depth and high sensitivity sequencing, CH can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38481290"
                        open
                      >
                        <summary class="article-expander__title">p53 biology and reactivation for improved therapy in MDS and AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38481290/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML) originate from preleukemic hematopoietic conditions, such as clonal hematopoiesis of indeterminate potential (CHIP) or clonal cytopenia of undetermined significance (CCUS) and have variable outcomes despite the successful implementation of targeted therapies. The prognosis differs depending on the molecular subgroup. In patients with TP53 mutations, the most inferior outcomes across independent studies were observed. Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38485690"
                        open
                      >
                        <summary class="article-expander__title">ArCH: Improving the performance of clonal hematopoiesis variant calling and interpretation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38485690/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations in hematopoietic stem and progenitor stem cells (HSPCs) with resultant clonal expansion, termed clonal hematopoiesis (CH), is associated with increased risk of hematologic malignancies and other adverse outcomes. CH is generally present at low allelic fractions, but clonal expansion and acquisition of additional mutations leads to hematologic cancers in a small proportion of individuals. With high depth and high sensitivity sequencing, CH can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38481290"
                        open
                      >
                        <summary class="article-expander__title">p53 biology and reactivation for improved therapy in MDS and AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38481290/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML) originate from preleukemic hematopoietic conditions, such as clonal hematopoiesis of indeterminate potential (CHIP) or clonal cytopenia of undetermined significance (CCUS) and have variable outcomes despite the successful implementation of targeted therapies. The prognosis differs depending on the molecular subgroup. In patients with TP53 mutations, the most inferior outcomes across independent studies were observed. Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38485690"
                        open
                      >
                        <summary class="article-expander__title">ArCH: Improving the performance of clonal hematopoiesis variant calling and interpretation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38485690/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations in hematopoietic stem and progenitor stem cells (HSPCs) with resultant clonal expansion, termed clonal hematopoiesis (CH), is associated with increased risk of hematologic malignancies and other adverse outcomes. CH is generally present at low allelic fractions, but clonal expansion and acquisition of additional mutations leads to hematologic cancers in a small proportion of individuals. With high depth and high sensitivity sequencing, CH can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38481290"
                        open
                      >
                        <summary class="article-expander__title">p53 biology and reactivation for improved therapy in MDS and AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38481290/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML) originate from preleukemic hematopoietic conditions, such as clonal hematopoiesis of indeterminate potential (CHIP) or clonal cytopenia of undetermined significance (CCUS) and have variable outcomes despite the successful implementation of targeted therapies. The prognosis differs depending on the molecular subgroup. In patients with TP53 mutations, the most inferior outcomes across independent studies were observed. Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38485690"
                        open
                      >
                        <summary class="article-expander__title">ArCH: Improving the performance of clonal hematopoiesis variant calling and interpretation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38485690/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: The acquisition of somatic mutations in hematopoietic stem and progenitor stem cells (HSPCs) with resultant clonal expansion, termed clonal hematopoiesis (CH), is associated with increased risk of hematologic malignancies and other adverse outcomes. CH is generally present at low allelic fractions, but clonal expansion and acquisition of additional mutations leads to hematologic cancers in a small proportion of individuals. With high depth and high sensitivity sequencing, CH can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38481290"
                        open
                      >
                        <summary class="article-expander__title">p53 biology and reactivation for improved therapy in MDS and AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38481290/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML) originate from preleukemic hematopoietic conditions, such as clonal hematopoiesis of indeterminate potential (CHIP) or clonal cytopenia of undetermined significance (CCUS) and have variable outcomes despite the successful implementation of targeted therapies. The prognosis differs depending on the molecular subgroup. In patients with TP53 mutations, the most inferior outcomes across independent studies were observed. Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38481290"
                        open
                      >
                        <summary class="article-expander__title">p53 biology and reactivation for improved therapy in MDS and AML</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38481290/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML) originate from preleukemic hematopoietic conditions, such as clonal hematopoiesis of indeterminate potential (CHIP) or clonal cytopenia of undetermined significance (CCUS) and have variable outcomes despite the successful implementation of targeted therapies. The prognosis differs depending on the molecular subgroup. In patients with TP53 mutations, the most inferior outcomes across independent studies were observed. Myeloid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38480884"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary trajectories of small cell lung cancer under therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38480884/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown^(1-3). Here we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by multiregion sequencing of 160 tumours from 65 patients. Treatment-naive SCLC exhibited clonal homogeneity at distinct tumour sites, whereas first-line platinum-based chemotherapy led to a burst in genomic intratumour heterogeneity and spatial clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38480884"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary trajectories of small cell lung cancer under therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38480884/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown^(1-3). Here we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by multiregion sequencing of 160 tumours from 65 patients. Treatment-naive SCLC exhibited clonal homogeneity at distinct tumour sites, whereas first-line platinum-based chemotherapy led to a burst in genomic intratumour heterogeneity and spatial clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38480884"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary trajectories of small cell lung cancer under therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38480884/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown^(1-3). Here we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by multiregion sequencing of 160 tumours from 65 patients. Treatment-naive SCLC exhibited clonal homogeneity at distinct tumour sites, whereas first-line platinum-based chemotherapy led to a burst in genomic intratumour heterogeneity and spatial clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38480884"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary trajectories of small cell lung cancer under therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38480884/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown^(1-3). Here we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by multiregion sequencing of 160 tumours from 65 patients. Treatment-naive SCLC exhibited clonal homogeneity at distinct tumour sites, whereas first-line platinum-based chemotherapy led to a burst in genomic intratumour heterogeneity and spatial clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38480884"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary trajectories of small cell lung cancer under therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38480884/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown^(1-3). Here we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by multiregion sequencing of 160 tumours from 65 patients. Treatment-naive SCLC exhibited clonal homogeneity at distinct tumour sites, whereas first-line platinum-based chemotherapy led to a burst in genomic intratumour heterogeneity and spatial clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38480884"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary trajectories of small cell lung cancer under therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38480884/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown^(1-3). Here we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by multiregion sequencing of 160 tumours from 65 patients. Treatment-naive SCLC exhibited clonal homogeneity at distinct tumour sites, whereas first-line platinum-based chemotherapy led to a burst in genomic intratumour heterogeneity and spatial clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38480884"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary trajectories of small cell lung cancer under therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38480884/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown^(1-3). Here we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by multiregion sequencing of 160 tumours from 65 patients. Treatment-naive SCLC exhibited clonal homogeneity at distinct tumour sites, whereas first-line platinum-based chemotherapy led to a burst in genomic intratumour heterogeneity and spatial clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38480884"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary trajectories of small cell lung cancer under therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38480884/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown^(1-3). Here we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by multiregion sequencing of 160 tumours from 65 patients. Treatment-naive SCLC exhibited clonal homogeneity at distinct tumour sites, whereas first-line platinum-based chemotherapy led to a burst in genomic intratumour heterogeneity and spatial clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38480884"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary trajectories of small cell lung cancer under therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38480884/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown^(1-3). Here we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by multiregion sequencing of 160 tumours from 65 patients. Treatment-naive SCLC exhibited clonal homogeneity at distinct tumour sites, whereas first-line platinum-based chemotherapy led to a burst in genomic intratumour heterogeneity and spatial clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38480884"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary trajectories of small cell lung cancer under therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38480884/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown^(1-3). Here we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by multiregion sequencing of 160 tumours from 65 patients. Treatment-naive SCLC exhibited clonal homogeneity at distinct tumour sites, whereas first-line platinum-based chemotherapy led to a burst in genomic intratumour heterogeneity and spatial clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38480884"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary trajectories of small cell lung cancer under therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38480884/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown^(1-3). Here we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by multiregion sequencing of 160 tumours from 65 patients. Treatment-naive SCLC exhibited clonal homogeneity at distinct tumour sites, whereas first-line platinum-based chemotherapy led to a burst in genomic intratumour heterogeneity and spatial clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38480884"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary trajectories of small cell lung cancer under therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38480884/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown^(1-3). Here we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by multiregion sequencing of 160 tumours from 65 patients. Treatment-naive SCLC exhibited clonal homogeneity at distinct tumour sites, whereas first-line platinum-based chemotherapy led to a burst in genomic intratumour heterogeneity and spatial clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-13 datetime="2024-03-13T10:00:00.000Z">2024-03-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-13 datetime="2024-03-13T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473705"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Variants Associated with Epigenetic Control and the NOTCH Pathway Are Frequent in Classic Hodgkin Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classic Hodgkin lymphoma (cHL) constitutes a B-cell neoplasm derived from germinal center lymphocytes. Despite high cure rates (80-90%) obtained with the current multiagent protocols, a significant proportion of cHL patients experience recurrences, characterized by a lower sensitivity to second-line treatments. The genomic background of chemorefractory cHL is still poorly understood, limiting personalized treatment strategies based on molecular features. In this study, using a targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473206"
                        open
                      >
                        <summary class="article-expander__title">Modeling the Effect of Spatial Structure on Solid Tumor Evolution and Circulating Tumor DNA Composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473705"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Variants Associated with Epigenetic Control and the NOTCH Pathway Are Frequent in Classic Hodgkin Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classic Hodgkin lymphoma (cHL) constitutes a B-cell neoplasm derived from germinal center lymphocytes. Despite high cure rates (80-90%) obtained with the current multiagent protocols, a significant proportion of cHL patients experience recurrences, characterized by a lower sensitivity to second-line treatments. The genomic background of chemorefractory cHL is still poorly understood, limiting personalized treatment strategies based on molecular features. In this study, using a targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473206"
                        open
                      >
                        <summary class="article-expander__title">Modeling the Effect of Spatial Structure on Solid Tumor Evolution and Circulating Tumor DNA Composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473705"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Variants Associated with Epigenetic Control and the NOTCH Pathway Are Frequent in Classic Hodgkin Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classic Hodgkin lymphoma (cHL) constitutes a B-cell neoplasm derived from germinal center lymphocytes. Despite high cure rates (80-90%) obtained with the current multiagent protocols, a significant proportion of cHL patients experience recurrences, characterized by a lower sensitivity to second-line treatments. The genomic background of chemorefractory cHL is still poorly understood, limiting personalized treatment strategies based on molecular features. In this study, using a targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473206"
                        open
                      >
                        <summary class="article-expander__title">Modeling the Effect of Spatial Structure on Solid Tumor Evolution and Circulating Tumor DNA Composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473705"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Variants Associated with Epigenetic Control and the NOTCH Pathway Are Frequent in Classic Hodgkin Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classic Hodgkin lymphoma (cHL) constitutes a B-cell neoplasm derived from germinal center lymphocytes. Despite high cure rates (80-90%) obtained with the current multiagent protocols, a significant proportion of cHL patients experience recurrences, characterized by a lower sensitivity to second-line treatments. The genomic background of chemorefractory cHL is still poorly understood, limiting personalized treatment strategies based on molecular features. In this study, using a targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473206"
                        open
                      >
                        <summary class="article-expander__title">Modeling the Effect of Spatial Structure on Solid Tumor Evolution and Circulating Tumor DNA Composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473705"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Variants Associated with Epigenetic Control and the NOTCH Pathway Are Frequent in Classic Hodgkin Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classic Hodgkin lymphoma (cHL) constitutes a B-cell neoplasm derived from germinal center lymphocytes. Despite high cure rates (80-90%) obtained with the current multiagent protocols, a significant proportion of cHL patients experience recurrences, characterized by a lower sensitivity to second-line treatments. The genomic background of chemorefractory cHL is still poorly understood, limiting personalized treatment strategies based on molecular features. In this study, using a targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473206"
                        open
                      >
                        <summary class="article-expander__title">Modeling the Effect of Spatial Structure on Solid Tumor Evolution and Circulating Tumor DNA Composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473705"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Variants Associated with Epigenetic Control and the NOTCH Pathway Are Frequent in Classic Hodgkin Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classic Hodgkin lymphoma (cHL) constitutes a B-cell neoplasm derived from germinal center lymphocytes. Despite high cure rates (80-90%) obtained with the current multiagent protocols, a significant proportion of cHL patients experience recurrences, characterized by a lower sensitivity to second-line treatments. The genomic background of chemorefractory cHL is still poorly understood, limiting personalized treatment strategies based on molecular features. In this study, using a targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473206"
                        open
                      >
                        <summary class="article-expander__title">Modeling the Effect of Spatial Structure on Solid Tumor Evolution and Circulating Tumor DNA Composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473705"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Variants Associated with Epigenetic Control and the NOTCH Pathway Are Frequent in Classic Hodgkin Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classic Hodgkin lymphoma (cHL) constitutes a B-cell neoplasm derived from germinal center lymphocytes. Despite high cure rates (80-90%) obtained with the current multiagent protocols, a significant proportion of cHL patients experience recurrences, characterized by a lower sensitivity to second-line treatments. The genomic background of chemorefractory cHL is still poorly understood, limiting personalized treatment strategies based on molecular features. In this study, using a targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473206"
                        open
                      >
                        <summary class="article-expander__title">Modeling the Effect of Spatial Structure on Solid Tumor Evolution and Circulating Tumor DNA Composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473705"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Variants Associated with Epigenetic Control and the NOTCH Pathway Are Frequent in Classic Hodgkin Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classic Hodgkin lymphoma (cHL) constitutes a B-cell neoplasm derived from germinal center lymphocytes. Despite high cure rates (80-90%) obtained with the current multiagent protocols, a significant proportion of cHL patients experience recurrences, characterized by a lower sensitivity to second-line treatments. The genomic background of chemorefractory cHL is still poorly understood, limiting personalized treatment strategies based on molecular features. In this study, using a targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473206"
                        open
                      >
                        <summary class="article-expander__title">Modeling the Effect of Spatial Structure on Solid Tumor Evolution and Circulating Tumor DNA Composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473705"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Variants Associated with Epigenetic Control and the NOTCH Pathway Are Frequent in Classic Hodgkin Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classic Hodgkin lymphoma (cHL) constitutes a B-cell neoplasm derived from germinal center lymphocytes. Despite high cure rates (80-90%) obtained with the current multiagent protocols, a significant proportion of cHL patients experience recurrences, characterized by a lower sensitivity to second-line treatments. The genomic background of chemorefractory cHL is still poorly understood, limiting personalized treatment strategies based on molecular features. In this study, using a targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473206"
                        open
                      >
                        <summary class="article-expander__title">Modeling the Effect of Spatial Structure on Solid Tumor Evolution and Circulating Tumor DNA Composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473705"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Variants Associated with Epigenetic Control and the NOTCH Pathway Are Frequent in Classic Hodgkin Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classic Hodgkin lymphoma (cHL) constitutes a B-cell neoplasm derived from germinal center lymphocytes. Despite high cure rates (80-90%) obtained with the current multiagent protocols, a significant proportion of cHL patients experience recurrences, characterized by a lower sensitivity to second-line treatments. The genomic background of chemorefractory cHL is still poorly understood, limiting personalized treatment strategies based on molecular features. In this study, using a targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473206"
                        open
                      >
                        <summary class="article-expander__title">Modeling the Effect of Spatial Structure on Solid Tumor Evolution and Circulating Tumor DNA Composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473705"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Variants Associated with Epigenetic Control and the NOTCH Pathway Are Frequent in Classic Hodgkin Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classic Hodgkin lymphoma (cHL) constitutes a B-cell neoplasm derived from germinal center lymphocytes. Despite high cure rates (80-90%) obtained with the current multiagent protocols, a significant proportion of cHL patients experience recurrences, characterized by a lower sensitivity to second-line treatments. The genomic background of chemorefractory cHL is still poorly understood, limiting personalized treatment strategies based on molecular features. In this study, using a targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473206"
                        open
                      >
                        <summary class="article-expander__title">Modeling the Effect of Spatial Structure on Solid Tumor Evolution and Circulating Tumor DNA Composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473705"
                        open
                      >
                        <summary class="article-expander__title">Pathogenic Variants Associated with Epigenetic Control and the NOTCH Pathway Are Frequent in Classic Hodgkin Lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classic Hodgkin lymphoma (cHL) constitutes a B-cell neoplasm derived from germinal center lymphocytes. Despite high cure rates (80-90%) obtained with the current multiagent protocols, a significant proportion of cHL patients experience recurrences, characterized by a lower sensitivity to second-line treatments. The genomic background of chemorefractory cHL is still poorly understood, limiting personalized treatment strategies based on molecular features. In this study, using a targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473206"
                        open
                      >
                        <summary class="article-expander__title">Modeling the Effect of Spatial Structure on Solid Tumor Evolution and Circulating Tumor DNA Composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38473206"
                        open
                      >
                        <summary class="article-expander__title">Modeling the Effect of Spatial Structure on Solid Tumor Evolution and Circulating Tumor DNA Composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38473206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-12 datetime="2024-03-12T10:00:00.000Z">2024-03-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-12 datetime="2024-03-12T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38470216"
                        open
                      >
                        <summary class="article-expander__title">Impact of clonal hematopoiesis of indeterminate potential on hepatocellular carcinoma in individuals with steatotic liver disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38470216/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We observed an independent association between CHIP, particularly related to non-DNTM3A and TET2 genetic lesions, and MASLD-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38470216"
                        open
                      >
                        <summary class="article-expander__title">Impact of clonal hematopoiesis of indeterminate potential on hepatocellular carcinoma in individuals with steatotic liver disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38470216/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We observed an independent association between CHIP, particularly related to non-DNTM3A and TET2 genetic lesions, and MASLD-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38470216"
                        open
                      >
                        <summary class="article-expander__title">Impact of clonal hematopoiesis of indeterminate potential on hepatocellular carcinoma in individuals with steatotic liver disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38470216/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We observed an independent association between CHIP, particularly related to non-DNTM3A and TET2 genetic lesions, and MASLD-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38470216"
                        open
                      >
                        <summary class="article-expander__title">Impact of clonal hematopoiesis of indeterminate potential on hepatocellular carcinoma in individuals with steatotic liver disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38470216/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We observed an independent association between CHIP, particularly related to non-DNTM3A and TET2 genetic lesions, and MASLD-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38470216"
                        open
                      >
                        <summary class="article-expander__title">Impact of clonal hematopoiesis of indeterminate potential on hepatocellular carcinoma in individuals with steatotic liver disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38470216/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We observed an independent association between CHIP, particularly related to non-DNTM3A and TET2 genetic lesions, and MASLD-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38470216"
                        open
                      >
                        <summary class="article-expander__title">Impact of clonal hematopoiesis of indeterminate potential on hepatocellular carcinoma in individuals with steatotic liver disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38470216/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We observed an independent association between CHIP, particularly related to non-DNTM3A and TET2 genetic lesions, and MASLD-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38470216"
                        open
                      >
                        <summary class="article-expander__title">Impact of clonal hematopoiesis of indeterminate potential on hepatocellular carcinoma in individuals with steatotic liver disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38470216/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We observed an independent association between CHIP, particularly related to non-DNTM3A and TET2 genetic lesions, and MASLD-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38470216"
                        open
                      >
                        <summary class="article-expander__title">Impact of clonal hematopoiesis of indeterminate potential on hepatocellular carcinoma in individuals with steatotic liver disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38470216/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We observed an independent association between CHIP, particularly related to non-DNTM3A and TET2 genetic lesions, and MASLD-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38470216"
                        open
                      >
                        <summary class="article-expander__title">Impact of clonal hematopoiesis of indeterminate potential on hepatocellular carcinoma in individuals with steatotic liver disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38470216/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We observed an independent association between CHIP, particularly related to non-DNTM3A and TET2 genetic lesions, and MASLD-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38470216"
                        open
                      >
                        <summary class="article-expander__title">Impact of clonal hematopoiesis of indeterminate potential on hepatocellular carcinoma in individuals with steatotic liver disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38470216/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We observed an independent association between CHIP, particularly related to non-DNTM3A and TET2 genetic lesions, and MASLD-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38470216"
                        open
                      >
                        <summary class="article-expander__title">Impact of clonal hematopoiesis of indeterminate potential on hepatocellular carcinoma in individuals with steatotic liver disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38470216/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We observed an independent association between CHIP, particularly related to non-DNTM3A and TET2 genetic lesions, and MASLD-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38470216"
                        open
                      >
                        <summary class="article-expander__title">Impact of clonal hematopoiesis of indeterminate potential on hepatocellular carcinoma in individuals with steatotic liver disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38470216/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We observed an independent association between CHIP, particularly related to non-DNTM3A and TET2 genetic lesions, and MASLD-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38470216"
                        open
                      >
                        <summary class="article-expander__title">Impact of clonal hematopoiesis of indeterminate potential on hepatocellular carcinoma in individuals with steatotic liver disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38470216/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We observed an independent association between CHIP, particularly related to non-DNTM3A and TET2 genetic lesions, and MASLD-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38470216"
                        open
                      >
                        <summary class="article-expander__title">Impact of clonal hematopoiesis of indeterminate potential on hepatocellular carcinoma in individuals with steatotic liver disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38470216/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We observed an independent association between CHIP, particularly related to non-DNTM3A and TET2 genetic lesions, and MASLD-HCC.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-09 datetime="2024-03-09T11:00:00.000Z">2024-03-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-09 datetime="2024-03-09T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459936"
                        open
                      >
                        <summary class="article-expander__title">Clonal origin and genomic diversity in Lynch syndrome-associated endometrial cancer with multiple synchronous tumors: Identification of the pathogenicity of MLH1 p.L582H</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459936/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lynch syndrome-associated endometrial cancer patients often present multiple synchronous tumors and this assessment can affect treatment strategies. We present a case of a 27-year-old woman with tumors in the uterine corpus, cervix, and ovaries who was diagnosed with endometrial cancer and exhibited cervical invasion and ovarian metastasis. Her family history suggested Lynch syndrome, and genetic testing identified a variant of uncertain significance, MLH1 p.L582H. We conducted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-08 datetime="2024-03-08T11:00:00.000Z">2024-03-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-08 datetime="2024-03-08T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38459554"
                        open
                      >
                        <summary class="article-expander__title">Joint analysis of mutational and transcriptional landscapes in human cancer reveals key perturbations during cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38459554/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results show that lung cancer and chronic myeloid leukemia maintain a high clonal and transcriptional diversity, and we find little evidence in favor of clonal sweeps. This suggests that for these cancers selection based solely on growth rate is unlikely to be the dominating driving force during cancer evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-07 datetime="2024-03-07T11:00:00.000Z">2024-03-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-07 datetime="2024-03-07T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240325205025&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38454125"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38454125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age is a predominant risk factor for acute kidney injury (AKI), yet the biological mechanisms underlying this risk are largely unknown. Clonal hematopoiesis of indeterminate potential (CHIP) confers increased risk for several chronic diseases associated with aging. Here we sought to test whether CHIP increases the risk of AKI. In three population-based epidemiology cohorts, we found that CHIP was associated with a greater risk of incident AKI, which was more pronounced in patients with AKI...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-06 datetime="2024-03-06T11:00:00.000Z">2024-03-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-06 datetime="2024-03-06T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38446993"
                        open
                      >
                        <summary class="article-expander__title">Plasma Proteomic Signature Predicts Myeloid Neoplasm Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38446993/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These data highlight the role of immune cell regulation in the progression of CH to MN and the promise of leveraging multi-omic characterization of CH to improve MN risk stratification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38448455"
                        open
                      >
                        <summary class="article-expander__title">Evolving copy number gains promote tumor expansion and bolster mutational diversification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38448455/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The timing and fitness effect of somatic copy number alterations (SCNA) in cancer evolution remains poorly understood. Here we present a framework to determine the timing of a clonal SCNA that encompasses multiple gains. This involves calculating the proportion of time from its last gain to the onset of population expansion (lead time) as well as the proportion of time prior to its first gain (initiation time). Our method capitalizes on the observation that a genomic segment, while in a specific...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-05 datetime="2024-03-05T11:00:00.000Z">2024-03-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-05 datetime="2024-03-05T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38440231"
                        open
                      >
                        <summary class="article-expander__title">The dark side of stemness - the role of hematopoietic stem cells in development of blood malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38440231/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem cells (HSCs) produce all blood cells throughout the life of the organism. However, the high self-renewal and longevity of HSCs predispose them to accumulate mutations. The acquired mutations drive preleukemic clonal hematopoiesis, which is frequent among elderly people. The preleukemic state, although often asymptomatic, increases the risk of blood cancers. Nevertheless, the direct role of preleukemic HSCs is well-evidenced in adult myeloid leukemia (AML), while their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240324205157&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240323205541&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240322204906&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38441194"
                        open
                      >
                        <summary class="article-expander__title">Molecular and nutritional markers in head and neck cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38441194/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: There is still a need of biomarkers in the induction and neoadjuvant settings for squamous cell carcinoma of the head and neck (SCCHN). The objective of this concise review article is to give an overview on both predictive and prognostic biomarkers potentially useful for the management of SCCHN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-04 datetime="2024-03-04T11:00:00.000Z">2024-03-04</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-04 datetime="2024-03-04T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38438628"
                        open
                      >
                        <summary class="article-expander__title">Tissue mosaicism following stem cell aging: blood as an exemplar</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38438628/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of stem cell regenerative potential underlies aging of all tissues. Somatic mosaicism, the emergence of cellular patchworks within tissues, increases with age and has been observed in every organ yet examined. In the hematopoietic system, as in most tissues, stem cell aging through a variety of mechanisms occurs in lockstep with the emergence of somatic mosaicism. Here, we draw on insights from aging hematopoiesis to illustrate fundamental principles of stem cell aging and somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240321205008&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38435425"
                        open
                      >
                        <summary class="article-expander__title">Analysis of immunoglobulin/T-cell receptor repertoires by high-throughput RNA sequencing reveals a continuous dynamic of positive clonal selection in follicular lymphoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38435425/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Follicular lymphoma (FL) course is highly variable, making its clinical management challenging. In this incurable and recurring pathology, the interval between relapses tends to decrease while aggressiveness increases, sometimes resulting in the transformation to higher-grade lymphoma. These evolutions are particularly difficult to anticipate, resulting from complex clonal evolutions where multiple subclones compete and thrive due to their capacity to proliferate and resist therapies. Here, to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-02 datetime="2024-03-02T11:00:00.000Z">2024-03-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-02 datetime="2024-03-02T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38431463"
                        open
                      >
                        <summary class="article-expander__title">TET2 mutation as prototypic clonal hematopoiesis lesion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38431463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Loss of function TET2 mutation (TET2^(MT)) is one of the most frequently observed lesions in clonal hematopoiesis (CH). TET2 a member TET-dioxygenase family of enzymes that along with TET1 and TET3, progressively oxidize 5-methyl cytosine (mC) resulting in regulated demethylation of promoter, enhancer and silencer elements of the genome. This process is critical for efficient transcription that determine cell lineage fate, proliferation and survival and the maintenance of the genomic fidelity...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-03-01 datetime="2024-03-01T11:00:00.000Z">2024-03-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-03-01 datetime="2024-03-01T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429201"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in the setting of hematopoietic cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429201/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) in autologous transplant recipients and allogeneic transplant donors has genetic features and clinical associations that are distinct from each other and from non-cancer populations. CH in the setting of autologous transplant is enriched for mutations in DNA damage response pathway genes and is associated with adverse outcomes, including an increased risk of therapy-related myeloid neoplasm and inferior overall survival. Studies of CH in allogeneic transplant donors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38429084"
                        open
                      >
                        <summary class="article-expander__title">EBV+ nodal T/NK-cell lymphoma associated with clonal hematopoiesis and structural variations of the viral genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38429084/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epstein-Barr virus (EBV)+ nodal T- and NK-cell lymphoma (EBV+ nPTCL) is a peripheral T-cell lymphoma (PTCL) that presents as a primary nodal disease with T-cell phenotype and EBV harboring on tumor cells. To date, the genetic aspect of EBV+ nPTCL has not been fully investigated. In this study, whole-exome and/or genome sequencing was performed on 22 cases of EBV+ nPTCL. TET2 (68%) and DNMT3A (32%) were observed to be the most frequently mutated genes whose presence was associated with poor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38427753"
                        open
                      >
                        <summary class="article-expander__title">ctDNA improves prognostic prediction in relapsed/refractory MM receiving ixazomib, lenalidomide, and dexamethasone</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38427753/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It remains elusive how driver mutations, including those detected in circulating tumor DNA (ctDNA), affect prognosis in relapsed/refractory multiple myeloma (RRMM). Here we performed targeted-capture sequencing using bone marrow plasma cells (BMPC) and ctDNA of 261 RRMM cases uniformly treated with ixazomib, lenalidomide, and dexamethasone in a multicenter, prospective, observational study. We detected 24 and 47 recurrently mutated genes in BMPC and ctDNA, respectively. In addition to clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38426096"
                        open
                      >
                        <summary class="article-expander__title">When inflammatory stressors dramatically change, disease phenotypes may transform between autoimmune hematopoietic failure and myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38426096/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aplastic anemia (AA) and hypoplastic myelodysplastic syndrome are paradigms of autoimmune hematopoietic failure (AHF). Myelodysplastic syndrome and acute myeloid leukemia are unequivocal myeloid neoplasms (MNs). Currently, AA is also known to be a clonal hematological disease. Genetic aberrations typically observed in MNs are detected in approximately one-third of AA patients. In AA patients harboring MN-related genetic aberrations, a poor response to immunosuppressive therapy (IST) and an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-29 datetime="2024-02-29T11:00:00.000Z">2024-02-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-29 datetime="2024-02-29T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38421682"
                        open
                      >
                        <summary class="article-expander__title">Spatial Mapping of Hematopoietic Clones in Human Bone Marrow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38421682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the expansion of somatically mutated cells in the hematopoietic compartment of individuals without hematopoietic dysfunction. Large CH clones (i.e. &gt;2% variant allele fraction) predispose to hematologic malignancy, but CH is detected at lower levels in nearly all middle-aged individuals. Prior work has extensively characterized CH in peripheral blood, but the spatial distribution of hematopoietic clones in human bone marrow is largely undescribed. To understand CH at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38421682"
                        open
                      >
                        <summary class="article-expander__title">Spatial Mapping of Hematopoietic Clones in Human Bone Marrow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38421682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the expansion of somatically mutated cells in the hematopoietic compartment of individuals without hematopoietic dysfunction. Large CH clones (i.e. &gt;2% variant allele fraction) predispose to hematologic malignancy, but CH is detected at lower levels in nearly all middle-aged individuals. Prior work has extensively characterized CH in peripheral blood, but the spatial distribution of hematopoietic clones in human bone marrow is largely undescribed. To understand CH at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38421682"
                        open
                      >
                        <summary class="article-expander__title">Spatial Mapping of Hematopoietic Clones in Human Bone Marrow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38421682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the expansion of somatically mutated cells in the hematopoietic compartment of individuals without hematopoietic dysfunction. Large CH clones (i.e. &gt;2% variant allele fraction) predispose to hematologic malignancy, but CH is detected at lower levels in nearly all middle-aged individuals. Prior work has extensively characterized CH in peripheral blood, but the spatial distribution of hematopoietic clones in human bone marrow is largely undescribed. To understand CH at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38421682"
                        open
                      >
                        <summary class="article-expander__title">Spatial Mapping of Hematopoietic Clones in Human Bone Marrow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38421682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the expansion of somatically mutated cells in the hematopoietic compartment of individuals without hematopoietic dysfunction. Large CH clones (i.e. &gt;2% variant allele fraction) predispose to hematologic malignancy, but CH is detected at lower levels in nearly all middle-aged individuals. Prior work has extensively characterized CH in peripheral blood, but the spatial distribution of hematopoietic clones in human bone marrow is largely undescribed. To understand CH at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38421682"
                        open
                      >
                        <summary class="article-expander__title">Spatial Mapping of Hematopoietic Clones in Human Bone Marrow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38421682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the expansion of somatically mutated cells in the hematopoietic compartment of individuals without hematopoietic dysfunction. Large CH clones (i.e. &gt;2% variant allele fraction) predispose to hematologic malignancy, but CH is detected at lower levels in nearly all middle-aged individuals. Prior work has extensively characterized CH in peripheral blood, but the spatial distribution of hematopoietic clones in human bone marrow is largely undescribed. To understand CH at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38421682"
                        open
                      >
                        <summary class="article-expander__title">Spatial Mapping of Hematopoietic Clones in Human Bone Marrow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38421682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the expansion of somatically mutated cells in the hematopoietic compartment of individuals without hematopoietic dysfunction. Large CH clones (i.e. &gt;2% variant allele fraction) predispose to hematologic malignancy, but CH is detected at lower levels in nearly all middle-aged individuals. Prior work has extensively characterized CH in peripheral blood, but the spatial distribution of hematopoietic clones in human bone marrow is largely undescribed. To understand CH at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38421682"
                        open
                      >
                        <summary class="article-expander__title">Spatial Mapping of Hematopoietic Clones in Human Bone Marrow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38421682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the expansion of somatically mutated cells in the hematopoietic compartment of individuals without hematopoietic dysfunction. Large CH clones (i.e. &gt;2% variant allele fraction) predispose to hematologic malignancy, but CH is detected at lower levels in nearly all middle-aged individuals. Prior work has extensively characterized CH in peripheral blood, but the spatial distribution of hematopoietic clones in human bone marrow is largely undescribed. To understand CH at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38421682"
                        open
                      >
                        <summary class="article-expander__title">Spatial Mapping of Hematopoietic Clones in Human Bone Marrow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38421682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the expansion of somatically mutated cells in the hematopoietic compartment of individuals without hematopoietic dysfunction. Large CH clones (i.e. &gt;2% variant allele fraction) predispose to hematologic malignancy, but CH is detected at lower levels in nearly all middle-aged individuals. Prior work has extensively characterized CH in peripheral blood, but the spatial distribution of hematopoietic clones in human bone marrow is largely undescribed. To understand CH at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38421682"
                        open
                      >
                        <summary class="article-expander__title">Spatial Mapping of Hematopoietic Clones in Human Bone Marrow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38421682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the expansion of somatically mutated cells in the hematopoietic compartment of individuals without hematopoietic dysfunction. Large CH clones (i.e. &gt;2% variant allele fraction) predispose to hematologic malignancy, but CH is detected at lower levels in nearly all middle-aged individuals. Prior work has extensively characterized CH in peripheral blood, but the spatial distribution of hematopoietic clones in human bone marrow is largely undescribed. To understand CH at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38421682"
                        open
                      >
                        <summary class="article-expander__title">Spatial Mapping of Hematopoietic Clones in Human Bone Marrow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38421682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the expansion of somatically mutated cells in the hematopoietic compartment of individuals without hematopoietic dysfunction. Large CH clones (i.e. &gt;2% variant allele fraction) predispose to hematologic malignancy, but CH is detected at lower levels in nearly all middle-aged individuals. Prior work has extensively characterized CH in peripheral blood, but the spatial distribution of hematopoietic clones in human bone marrow is largely undescribed. To understand CH at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38421682"
                        open
                      >
                        <summary class="article-expander__title">Spatial Mapping of Hematopoietic Clones in Human Bone Marrow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38421682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the expansion of somatically mutated cells in the hematopoietic compartment of individuals without hematopoietic dysfunction. Large CH clones (i.e. &gt;2% variant allele fraction) predispose to hematologic malignancy, but CH is detected at lower levels in nearly all middle-aged individuals. Prior work has extensively characterized CH in peripheral blood, but the spatial distribution of hematopoietic clones in human bone marrow is largely undescribed. To understand CH at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38421682"
                        open
                      >
                        <summary class="article-expander__title">Spatial Mapping of Hematopoietic Clones in Human Bone Marrow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38421682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the expansion of somatically mutated cells in the hematopoietic compartment of individuals without hematopoietic dysfunction. Large CH clones (i.e. &gt;2% variant allele fraction) predispose to hematologic malignancy, but CH is detected at lower levels in nearly all middle-aged individuals. Prior work has extensively characterized CH in peripheral blood, but the spatial distribution of hematopoietic clones in human bone marrow is largely undescribed. To understand CH at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38424288"
                        open
                      >
                        <summary class="article-expander__title">Heart and blood: clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38424288/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cardiovascular diseases are among the leading causes of death worldwide, with well-known modifiable risk factors, such as smoking, overweight, lipid metabolism disorders, lack of physical activity and high blood pressure playing a significant role. Recent studies have now identified &quot;clonal hematopoiesis&quot; as a novel blood-based risk factor. Clonal hematopoiesis arises from mutations in hematopoietic stem cells, which lead to the expansion of mutated blood cells. Mutated cell clones can be...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38423847"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and autoimmunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38423847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) has been associated with aging, occurring in about 10% of individuals aged &gt;70 years, and immune dysfunction. Aged hematopoietic stem and progenitor cells exhibit pathological changes in immune function and activation of inflammatory pathways. CH clones commonly harbor a loss of function mutation in DNMT3A or TET2, which causes increased expression of inflammatory signaling genes, a proposed mechanism connected to CH and the development of age-related diseases....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38421682"
                        open
                      >
                        <summary class="article-expander__title">Spatial Mapping of Hematopoietic Clones in Human Bone Marrow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38421682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the expansion of somatically mutated cells in the hematopoietic compartment of individuals without hematopoietic dysfunction. Large CH clones (i.e. &gt;2% variant allele fraction) predispose to hematologic malignancy, but CH is detected at lower levels in nearly all middle-aged individuals. Prior work has extensively characterized CH in peripheral blood, but the spatial distribution of hematopoietic clones in human bone marrow is largely undescribed. To understand CH at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422464"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndromes and Myelodysplastic Syndromes/Myeloproliferative Neoplasms: A Real-World Experience From a Developing Country</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422464/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: To our knowledge, our study is the first comprehensive study examining the epidemiology and prognostic factors in patients with MDS and patients with MDS/MPN in Jordan. Our results show that MDS and MDS/MPN epidemiology in Jordan is different compared with Western countries. Our results also show that baseline serum ferritin levels can be used as a prognostic marker for patients with MDS.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38422020"
                        open
                      >
                        <summary class="article-expander__title">Aging and comprehensive molecular profiling in acute myeloid leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38422020/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is an aging-related and heterogeneous hematopoietic malignancy. In this study, a total of 1,474 newly diagnosed AML patients with RNA sequencing data were enrolled, and targeted or whole exome sequencing data were obtained in 94% cases. The correlation of aging-related factors including age and clonal hematopoiesis (CH), gender, and genomic/transcriptomic profiles (gene fusions, genetic mutations, and gene expression networks or pathways) was systematically analyzed....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38421682"
                        open
                      >
                        <summary class="article-expander__title">Spatial Mapping of Hematopoietic Clones in Human Bone Marrow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38421682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is the expansion of somatically mutated cells in the hematopoietic compartment of individuals without hematopoietic dysfunction. Large CH clones (i.e. &gt;2% variant allele fraction) predispose to hematologic malignancy, but CH is detected at lower levels in nearly all middle-aged individuals. Prior work has extensively characterized CH in peripheral blood, but the spatial distribution of hematopoietic clones in human bone marrow is largely undescribed. To understand CH at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-28 datetime="2024-02-28T11:00:00.000Z">2024-02-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-28 datetime="2024-02-28T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38417801"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Risk: Atherosclerosis, Thrombosis, and beyond</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38417801/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired mutations that lead to clonal hematopoiesis have emerged as a new and potent risk factor for atherosclerotic cardiovascular disease and other cardiovascular conditions. Human sequencing studies and experiments in mouse models provide compelling evidence supporting that this condition, particularly when driven by specific mutated genes, contributes to the development of atherosclerosis by exacerbating inflammatory responses. The insights gained from these studies are paving the way for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38417801"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Risk: Atherosclerosis, Thrombosis, and beyond</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38417801/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired mutations that lead to clonal hematopoiesis have emerged as a new and potent risk factor for atherosclerotic cardiovascular disease and other cardiovascular conditions. Human sequencing studies and experiments in mouse models provide compelling evidence supporting that this condition, particularly when driven by specific mutated genes, contributes to the development of atherosclerosis by exacerbating inflammatory responses. The insights gained from these studies are paving the way for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38417801"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Risk: Atherosclerosis, Thrombosis, and beyond</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38417801/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired mutations that lead to clonal hematopoiesis have emerged as a new and potent risk factor for atherosclerotic cardiovascular disease and other cardiovascular conditions. Human sequencing studies and experiments in mouse models provide compelling evidence supporting that this condition, particularly when driven by specific mutated genes, contributes to the development of atherosclerosis by exacerbating inflammatory responses. The insights gained from these studies are paving the way for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38417801"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Risk: Atherosclerosis, Thrombosis, and beyond</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38417801/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired mutations that lead to clonal hematopoiesis have emerged as a new and potent risk factor for atherosclerotic cardiovascular disease and other cardiovascular conditions. Human sequencing studies and experiments in mouse models provide compelling evidence supporting that this condition, particularly when driven by specific mutated genes, contributes to the development of atherosclerosis by exacerbating inflammatory responses. The insights gained from these studies are paving the way for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38417801"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Risk: Atherosclerosis, Thrombosis, and beyond</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38417801/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired mutations that lead to clonal hematopoiesis have emerged as a new and potent risk factor for atherosclerotic cardiovascular disease and other cardiovascular conditions. Human sequencing studies and experiments in mouse models provide compelling evidence supporting that this condition, particularly when driven by specific mutated genes, contributes to the development of atherosclerosis by exacerbating inflammatory responses. The insights gained from these studies are paving the way for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38414863"
                        open
                      >
                        <summary class="article-expander__title">Hematologic DNMT3A reduction and high-fat diet synergize to promote weight gain and tissue inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38414863/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240307195005&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, blood cell production becomes dominated by a limited number of variant hematopoietic stem cell (HSC) clones. Differentiated progeny of variant HSCs are thought to mediate the detrimental effects of such clonal hematopoiesis on organismal health, but the mechanisms are poorly understood. While somatic mutations in DNA methyltransferase 3A (DNMT3A) frequently drive clonal dominance, the aging milieu also likely contributes. Here, we examined in mice the interaction between high-fat...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38417801"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Risk: Atherosclerosis, Thrombosis, and beyond</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38417801/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired mutations that lead to clonal hematopoiesis have emerged as a new and potent risk factor for atherosclerotic cardiovascular disease and other cardiovascular conditions. Human sequencing studies and experiments in mouse models provide compelling evidence supporting that this condition, particularly when driven by specific mutated genes, contributes to the development of atherosclerosis by exacerbating inflammatory responses. The insights gained from these studies are paving the way for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38414863"
                        open
                      >
                        <summary class="article-expander__title">Hematologic DNMT3A reduction and high-fat diet synergize to promote weight gain and tissue inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38414863/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, blood cell production becomes dominated by a limited number of variant hematopoietic stem cell (HSC) clones. Differentiated progeny of variant HSCs are thought to mediate the detrimental effects of such clonal hematopoiesis on organismal health, but the mechanisms are poorly understood. While somatic mutations in DNA methyltransferase 3A (DNMT3A) frequently drive clonal dominance, the aging milieu also likely contributes. Here, we examined in mice the interaction between high-fat...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38417801"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Risk: Atherosclerosis, Thrombosis, and beyond</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38417801/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired mutations that lead to clonal hematopoiesis have emerged as a new and potent risk factor for atherosclerotic cardiovascular disease and other cardiovascular conditions. Human sequencing studies and experiments in mouse models provide compelling evidence supporting that this condition, particularly when driven by specific mutated genes, contributes to the development of atherosclerosis by exacerbating inflammatory responses. The insights gained from these studies are paving the way for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38414863"
                        open
                      >
                        <summary class="article-expander__title">Hematologic DNMT3A reduction and high-fat diet synergize to promote weight gain and tissue inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38414863/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, blood cell production becomes dominated by a limited number of variant hematopoietic stem cell (HSC) clones. Differentiated progeny of variant HSCs are thought to mediate the detrimental effects of such clonal hematopoiesis on organismal health, but the mechanisms are poorly understood. While somatic mutations in DNA methyltransferase 3A (DNMT3A) frequently drive clonal dominance, the aging milieu also likely contributes. Here, we examined in mice the interaction between high-fat...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38417801"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Risk: Atherosclerosis, Thrombosis, and beyond</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38417801/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired mutations that lead to clonal hematopoiesis have emerged as a new and potent risk factor for atherosclerotic cardiovascular disease and other cardiovascular conditions. Human sequencing studies and experiments in mouse models provide compelling evidence supporting that this condition, particularly when driven by specific mutated genes, contributes to the development of atherosclerosis by exacerbating inflammatory responses. The insights gained from these studies are paving the way for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38414863"
                        open
                      >
                        <summary class="article-expander__title">Hematologic DNMT3A reduction and high-fat diet synergize to promote weight gain and tissue inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38414863/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, blood cell production becomes dominated by a limited number of variant hematopoietic stem cell (HSC) clones. Differentiated progeny of variant HSCs are thought to mediate the detrimental effects of such clonal hematopoiesis on organismal health, but the mechanisms are poorly understood. While somatic mutations in DNA methyltransferase 3A (DNMT3A) frequently drive clonal dominance, the aging milieu also likely contributes. Here, we examined in mice the interaction between high-fat...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38417801"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Risk: Atherosclerosis, Thrombosis, and beyond</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38417801/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired mutations that lead to clonal hematopoiesis have emerged as a new and potent risk factor for atherosclerotic cardiovascular disease and other cardiovascular conditions. Human sequencing studies and experiments in mouse models provide compelling evidence supporting that this condition, particularly when driven by specific mutated genes, contributes to the development of atherosclerosis by exacerbating inflammatory responses. The insights gained from these studies are paving the way for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38414863"
                        open
                      >
                        <summary class="article-expander__title">Hematologic DNMT3A reduction and high-fat diet synergize to promote weight gain and tissue inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38414863/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, blood cell production becomes dominated by a limited number of variant hematopoietic stem cell (HSC) clones. Differentiated progeny of variant HSCs are thought to mediate the detrimental effects of such clonal hematopoiesis on organismal health, but the mechanisms are poorly understood. While somatic mutations in DNA methyltransferase 3A (DNMT3A) frequently drive clonal dominance, the aging milieu also likely contributes. Here, we examined in mice the interaction between high-fat...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38417801"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Risk: Atherosclerosis, Thrombosis, and beyond</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38417801/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired mutations that lead to clonal hematopoiesis have emerged as a new and potent risk factor for atherosclerotic cardiovascular disease and other cardiovascular conditions. Human sequencing studies and experiments in mouse models provide compelling evidence supporting that this condition, particularly when driven by specific mutated genes, contributes to the development of atherosclerosis by exacerbating inflammatory responses. The insights gained from these studies are paving the way for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38414863"
                        open
                      >
                        <summary class="article-expander__title">Hematologic DNMT3A reduction and high-fat diet synergize to promote weight gain and tissue inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38414863/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, blood cell production becomes dominated by a limited number of variant hematopoietic stem cell (HSC) clones. Differentiated progeny of variant HSCs are thought to mediate the detrimental effects of such clonal hematopoiesis on organismal health, but the mechanisms are poorly understood. While somatic mutations in DNA methyltransferase 3A (DNMT3A) frequently drive clonal dominance, the aging milieu also likely contributes. Here, we examined in mice the interaction between high-fat...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38417801"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Risk: Atherosclerosis, Thrombosis, and beyond</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38417801/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired mutations that lead to clonal hematopoiesis have emerged as a new and potent risk factor for atherosclerotic cardiovascular disease and other cardiovascular conditions. Human sequencing studies and experiments in mouse models provide compelling evidence supporting that this condition, particularly when driven by specific mutated genes, contributes to the development of atherosclerosis by exacerbating inflammatory responses. The insights gained from these studies are paving the way for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38414863"
                        open
                      >
                        <summary class="article-expander__title">Hematologic DNMT3A reduction and high-fat diet synergize to promote weight gain and tissue inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38414863/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, blood cell production becomes dominated by a limited number of variant hematopoietic stem cell (HSC) clones. Differentiated progeny of variant HSCs are thought to mediate the detrimental effects of such clonal hematopoiesis on organismal health, but the mechanisms are poorly understood. While somatic mutations in DNA methyltransferase 3A (DNMT3A) frequently drive clonal dominance, the aging milieu also likely contributes. Here, we examined in mice the interaction between high-fat...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38417801"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Risk: Atherosclerosis, Thrombosis, and beyond</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38417801/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired mutations that lead to clonal hematopoiesis have emerged as a new and potent risk factor for atherosclerotic cardiovascular disease and other cardiovascular conditions. Human sequencing studies and experiments in mouse models provide compelling evidence supporting that this condition, particularly when driven by specific mutated genes, contributes to the development of atherosclerosis by exacerbating inflammatory responses. The insights gained from these studies are paving the way for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38414863"
                        open
                      >
                        <summary class="article-expander__title">Hematologic DNMT3A reduction and high-fat diet synergize to promote weight gain and tissue inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38414863/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, blood cell production becomes dominated by a limited number of variant hematopoietic stem cell (HSC) clones. Differentiated progeny of variant HSCs are thought to mediate the detrimental effects of such clonal hematopoiesis on organismal health, but the mechanisms are poorly understood. While somatic mutations in DNA methyltransferase 3A (DNMT3A) frequently drive clonal dominance, the aging milieu also likely contributes. Here, we examined in mice the interaction between high-fat...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38414863"
                        open
                      >
                        <summary class="article-expander__title">Hematologic DNMT3A reduction and high-fat diet synergize to promote weight gain and tissue inflammation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38414863/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>During aging, blood cell production becomes dominated by a limited number of variant hematopoietic stem cell (HSC) clones. Differentiated progeny of variant HSCs are thought to mediate the detrimental effects of such clonal hematopoiesis on organismal health, but the mechanisms are poorly understood. While somatic mutations in DNA methyltransferase 3A (DNMT3A) frequently drive clonal dominance, the aging milieu also likely contributes. Here, we examined in mice the interaction between high-fat...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240320205145&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38418463"
                        open
                      >
                        <summary class="article-expander__title">A Phase II trial of alternating osimertinib and gefitinib therapy in advanced EGFR-T790M positive non-small cell lung cancer: OSCILLATE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38418463/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this phase II, single arm trial (ACTRN12617000720314), we investigate if alternating osimertinib and gefitinib would delay the development of resistance to osimertinib in advanced, non-small cell lung cancer (NSCLC) with the epidermal growth factor receptor (EGFR) T790M mutation (n &#x3D; 47) by modulating selective pressure on resistant clones. The primary endpoint is progression free-survival (PFS) rate at 12 months, and secondary endpoints include: feasibility of alternating therapy, overall...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-27 datetime="2024-02-27T11:00:00.000Z">2024-02-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-27 datetime="2024-02-27T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240319205002&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38410652"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of simulation methods for tumor subclonal reconstruction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38410652/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most neoplastic tumors originate from a single cell, and their evolution can be genetically traced through lineages characterized by common alterations such as small somatic mutations (SSMs), copy number alterations (CNAs), structural variants (SVs), and aneuploidies. Due to the complexity of these alterations in most tumors and the errors introduced by sequencing protocols and calling algorithms, tumor subclonal reconstruction algorithms are necessary to recapitulate the DNA sequence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240318205006&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38410652"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of simulation methods for tumor subclonal reconstruction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38410652/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most neoplastic tumors originate from a single cell, and their evolution can be genetically traced through lineages characterized by common alterations such as small somatic mutations (SSMs), copy number alterations (CNAs), structural variants (SVs), and aneuploidies. Due to the complexity of these alterations in most tumors and the errors introduced by sequencing protocols and calling algorithms, tumor subclonal reconstruction algorithms are necessary to recapitulate the DNA sequence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38410652"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of simulation methods for tumor subclonal reconstruction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38410652/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most neoplastic tumors originate from a single cell, and their evolution can be genetically traced through lineages characterized by common alterations such as small somatic mutations (SSMs), copy number alterations (CNAs), structural variants (SVs), and aneuploidies. Due to the complexity of these alterations in most tumors and the errors introduced by sequencing protocols and calling algorithms, tumor subclonal reconstruction algorithms are necessary to recapitulate the DNA sequence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38410652"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of simulation methods for tumor subclonal reconstruction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38410652/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most neoplastic tumors originate from a single cell, and their evolution can be genetically traced through lineages characterized by common alterations such as small somatic mutations (SSMs), copy number alterations (CNAs), structural variants (SVs), and aneuploidies. Due to the complexity of these alterations in most tumors and the errors introduced by sequencing protocols and calling algorithms, tumor subclonal reconstruction algorithms are necessary to recapitulate the DNA sequence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38410652"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of simulation methods for tumor subclonal reconstruction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38410652/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most neoplastic tumors originate from a single cell, and their evolution can be genetically traced through lineages characterized by common alterations such as small somatic mutations (SSMs), copy number alterations (CNAs), structural variants (SVs), and aneuploidies. Due to the complexity of these alterations in most tumors and the errors introduced by sequencing protocols and calling algorithms, tumor subclonal reconstruction algorithms are necessary to recapitulate the DNA sequence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38410652"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of simulation methods for tumor subclonal reconstruction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38410652/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most neoplastic tumors originate from a single cell, and their evolution can be genetically traced through lineages characterized by common alterations such as small somatic mutations (SSMs), copy number alterations (CNAs), structural variants (SVs), and aneuploidies. Due to the complexity of these alterations in most tumors and the errors introduced by sequencing protocols and calling algorithms, tumor subclonal reconstruction algorithms are necessary to recapitulate the DNA sequence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38410652"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of simulation methods for tumor subclonal reconstruction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38410652/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most neoplastic tumors originate from a single cell, and their evolution can be genetically traced through lineages characterized by common alterations such as small somatic mutations (SSMs), copy number alterations (CNAs), structural variants (SVs), and aneuploidies. Due to the complexity of these alterations in most tumors and the errors introduced by sequencing protocols and calling algorithms, tumor subclonal reconstruction algorithms are necessary to recapitulate the DNA sequence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38410652"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of simulation methods for tumor subclonal reconstruction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38410652/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most neoplastic tumors originate from a single cell, and their evolution can be genetically traced through lineages characterized by common alterations such as small somatic mutations (SSMs), copy number alterations (CNAs), structural variants (SVs), and aneuploidies. Due to the complexity of these alterations in most tumors and the errors introduced by sequencing protocols and calling algorithms, tumor subclonal reconstruction algorithms are necessary to recapitulate the DNA sequence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38410652"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of simulation methods for tumor subclonal reconstruction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38410652/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most neoplastic tumors originate from a single cell, and their evolution can be genetically traced through lineages characterized by common alterations such as small somatic mutations (SSMs), copy number alterations (CNAs), structural variants (SVs), and aneuploidies. Due to the complexity of these alterations in most tumors and the errors introduced by sequencing protocols and calling algorithms, tumor subclonal reconstruction algorithms are necessary to recapitulate the DNA sequence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38410652"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of simulation methods for tumor subclonal reconstruction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38410652/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most neoplastic tumors originate from a single cell, and their evolution can be genetically traced through lineages characterized by common alterations such as small somatic mutations (SSMs), copy number alterations (CNAs), structural variants (SVs), and aneuploidies. Due to the complexity of these alterations in most tumors and the errors introduced by sequencing protocols and calling algorithms, tumor subclonal reconstruction algorithms are necessary to recapitulate the DNA sequence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38410652"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of simulation methods for tumor subclonal reconstruction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38410652/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most neoplastic tumors originate from a single cell, and their evolution can be genetically traced through lineages characterized by common alterations such as small somatic mutations (SSMs), copy number alterations (CNAs), structural variants (SVs), and aneuploidies. Due to the complexity of these alterations in most tumors and the errors introduced by sequencing protocols and calling algorithms, tumor subclonal reconstruction algorithms are necessary to recapitulate the DNA sequence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38410652"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of simulation methods for tumor subclonal reconstruction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38410652/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most neoplastic tumors originate from a single cell, and their evolution can be genetically traced through lineages characterized by common alterations such as small somatic mutations (SSMs), copy number alterations (CNAs), structural variants (SVs), and aneuploidies. Due to the complexity of these alterations in most tumors and the errors introduced by sequencing protocols and calling algorithms, tumor subclonal reconstruction algorithms are necessary to recapitulate the DNA sequence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38410652"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of simulation methods for tumor subclonal reconstruction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38410652/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most neoplastic tumors originate from a single cell, and their evolution can be genetically traced through lineages characterized by common alterations such as small somatic mutations (SSMs), copy number alterations (CNAs), structural variants (SVs), and aneuploidies. Due to the complexity of these alterations in most tumors and the errors introduced by sequencing protocols and calling algorithms, tumor subclonal reconstruction algorithms are necessary to recapitulate the DNA sequence...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38412093"
                        open
                      >
                        <summary class="article-expander__title">The genomic and evolutionary landscapes of anaplastic thyroid carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38412093/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Anaplastic thyroid carcinoma is arguably the most lethal human malignancy. It often co-occurs with differentiated thyroid cancers, yet the molecular origins of its aggressivity are unknown. We sequenced tumor DNA from 329 regions of thyroid cancer, including 213 from patients with primary anaplastic thyroid carcinomas. We also whole genome sequenced 9 patients using multi-region sequencing of both differentiated and anaplastic thyroid cancer components. Using these data, we demonstrate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38409885"
                        open
                      >
                        <summary class="article-expander__title">Whole-exome sequencing has revealed novel genetic characteristics in intracranial germ cell tumours in the Chinese</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38409885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study indicated that Chinese IGCT patients may have distinct genetic characteristics and identified several possible genetic alterations that have the potential to become prognostic biomarkers of NGGCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-26 datetime="2024-02-26T11:00:00.000Z">2024-02-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-26 datetime="2024-02-26T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405837"
                        open
                      >
                        <summary class="article-expander__title">Metformin reduces the clonal fitness of Dnmt3a(R878H) hematopoietic stem and progenitor cells by reversing their aberrant metabolic and epigenetic state</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a hematopoietic stem cell (HSC) acquires a mutation that confers a competitive advantage over wild-type (WT) HSCs, resulting in its clonal expansion. Individuals with CH are at an increased risk of developing hematologic neoplasms and a range of age-related inflammatory illnesses^(1-3). Therapeutic interventions that suppress the expansion of mutant HSCs have the potential to prevent these CH-related illnesses; however, such interventions have not yet been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405837"
                        open
                      >
                        <summary class="article-expander__title">Metformin reduces the clonal fitness of Dnmt3a(R878H) hematopoietic stem and progenitor cells by reversing their aberrant metabolic and epigenetic state</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a hematopoietic stem cell (HSC) acquires a mutation that confers a competitive advantage over wild-type (WT) HSCs, resulting in its clonal expansion. Individuals with CH are at an increased risk of developing hematologic neoplasms and a range of age-related inflammatory illnesses^(1-3). Therapeutic interventions that suppress the expansion of mutant HSCs have the potential to prevent these CH-related illnesses; however, such interventions have not yet been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405837"
                        open
                      >
                        <summary class="article-expander__title">Metformin reduces the clonal fitness of Dnmt3aR878H hematopoietic stem and progenitor cells by reversing their aberrant metabolic and epigenetic state</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a hematopoietic stem cell (HSC) acquires a mutation that confers a competitive advantage over wild-type (WT) HSCs, resulting in its clonal expansion. Individuals with CH are at an increased risk of developing hematologic neoplasms and a range of age-related inflammatory illnesses1-3. Therapeutic interventions that suppress the expansion of mutant HSCs have the potential to prevent these CH-related illnesses; however, such interventions have not yet been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405837"
                        open
                      >
                        <summary class="article-expander__title">Metformin reduces the clonal fitness of Dnmt3aR878H hematopoietic stem and progenitor cells by reversing their aberrant metabolic and epigenetic state</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a hematopoietic stem cell (HSC) acquires a mutation that confers a competitive advantage over wild-type (WT) HSCs, resulting in its clonal expansion. Individuals with CH are at an increased risk of developing hematologic neoplasms and a range of age-related inflammatory illnesses1-3. Therapeutic interventions that suppress the expansion of mutant HSCs have the potential to prevent these CH-related illnesses; however, such interventions have not yet been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405837"
                        open
                      >
                        <summary class="article-expander__title">Metformin reduces the clonal fitness of Dnmt3aR878H hematopoietic stem and progenitor cells by reversing their aberrant metabolic and epigenetic state</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a hematopoietic stem cell (HSC) acquires a mutation that confers a competitive advantage over wild-type (WT) HSCs, resulting in its clonal expansion. Individuals with CH are at an increased risk of developing hematologic neoplasms and a range of age-related inflammatory illnesses1-3. Therapeutic interventions that suppress the expansion of mutant HSCs have the potential to prevent these CH-related illnesses; however, such interventions have not yet been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405837"
                        open
                      >
                        <summary class="article-expander__title">Metformin reduces the clonal fitness of Dnmt3aR878H hematopoietic stem and progenitor cells by reversing their aberrant metabolic and epigenetic state</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a hematopoietic stem cell (HSC) acquires a mutation that confers a competitive advantage over wild-type (WT) HSCs, resulting in its clonal expansion. Individuals with CH are at an increased risk of developing hematologic neoplasms and a range of age-related inflammatory illnesses1-3. Therapeutic interventions that suppress the expansion of mutant HSCs have the potential to prevent these CH-related illnesses; however, such interventions have not yet been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405837"
                        open
                      >
                        <summary class="article-expander__title">Metformin reduces the clonal fitness of Dnmt3aR878H hematopoietic stem and progenitor cells by reversing their aberrant metabolic and epigenetic state</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a hematopoietic stem cell (HSC) acquires a mutation that confers a competitive advantage over wild-type (WT) HSCs, resulting in its clonal expansion. Individuals with CH are at an increased risk of developing hematologic neoplasms and a range of age-related inflammatory illnesses1-3. Therapeutic interventions that suppress the expansion of mutant HSCs have the potential to prevent these CH-related illnesses; however, such interventions have not yet been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405837"
                        open
                      >
                        <summary class="article-expander__title">Metformin reduces the clonal fitness of Dnmt3aR878H hematopoietic stem and progenitor cells by reversing their aberrant metabolic and epigenetic state</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a hematopoietic stem cell (HSC) acquires a mutation that confers a competitive advantage over wild-type (WT) HSCs, resulting in its clonal expansion. Individuals with CH are at an increased risk of developing hematologic neoplasms and a range of age-related inflammatory illnesses1-3. Therapeutic interventions that suppress the expansion of mutant HSCs have the potential to prevent these CH-related illnesses; however, such interventions have not yet been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405837"
                        open
                      >
                        <summary class="article-expander__title">Metformin reduces the clonal fitness of Dnmt3aR878H hematopoietic stem and progenitor cells by reversing their aberrant metabolic and epigenetic state</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a hematopoietic stem cell (HSC) acquires a mutation that confers a competitive advantage over wild-type (WT) HSCs, resulting in its clonal expansion. Individuals with CH are at an increased risk of developing hematologic neoplasms and a range of age-related inflammatory illnesses1-3. Therapeutic interventions that suppress the expansion of mutant HSCs have the potential to prevent these CH-related illnesses; however, such interventions have not yet been...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38405882"
                        open
                      >
                        <summary class="article-expander__title">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38405882/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-25 datetime="2024-02-25T11:00:00.000Z">2024-02-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-25 datetime="2024-02-25T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240317205113&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240316205329&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240315204911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38403501"
                        open
                      >
                        <summary class="article-expander__title">The crossroads of cancer therapies and clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38403501/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The intricate interplay between Clonal Hematopoiesis (CH) and the repercussions of cancer therapies has garnered significant research focus in recent years. Previously perceived as an age-related phenomenon, CH is now closely linked to inflammation (&quot;Inflammaging&quot;) and cancer, impacting leukemogenesis, cancer progression, and treatment responses. This review explores the complex interplay between CH and diverse cancer therapies, including chemotherapy, targeted treatments, radiation, stem cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-24 datetime="2024-02-24T11:00:00.000Z">2024-02-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-24 datetime="2024-02-24T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240314205048&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38396286"
                        open
                      >
                        <summary class="article-expander__title">The evolution of preclinical models for myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38396286/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic Neoplasms (MDS) are a group of clonal disorders characterized by ineffective hematopoiesis and morphologic dysplasia. Clinical manifestations of MDS vary widely and are dictated in large part by a range of genetic aberrations. The lack of robust in vitro models for MDS has limited the ability to conduct high throughput drug screens, which in turn has hampered the development of novel therapies for MDS. There are very few well-characterized MDS cell lines, and the available cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-23 datetime="2024-02-23T11:00:00.000Z">2024-02-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-23 datetime="2024-02-23T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392038"
                        open
                      >
                        <summary class="article-expander__title">Homing and Engraftment of Hematopoietic Stem Cells Following Transplantation: A Pre-Clinical Perspective</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392038/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem-cell (HSC) transplantation (HSCT) is used to treat various hematologic disorders. Use of genetically modified mouse models of hematopoietic cell transplantation has been critical in our fundamental understanding of HSC biology and in developing approaches for human patients. Pre-clinical studies in animal models provide insight into the journey of transplanted HSCs from infusion to engraftment in bone-marrow (BM) niches. Various signaling molecules and growth factors secreted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392038"
                        open
                      >
                        <summary class="article-expander__title">Homing and Engraftment of Hematopoietic Stem Cells Following Transplantation: A Pre-Clinical Perspective</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392038/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem-cell (HSC) transplantation (HSCT) is used to treat various hematologic disorders. Use of genetically modified mouse models of hematopoietic cell transplantation has been critical in our fundamental understanding of HSC biology and in developing approaches for human patients. Pre-clinical studies in animal models provide insight into the journey of transplanted HSCs from infusion to engraftment in bone-marrow (BM) niches. Various signaling molecules and growth factors secreted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392038"
                        open
                      >
                        <summary class="article-expander__title">Homing and Engraftment of Hematopoietic Stem Cells Following Transplantation: A Pre-Clinical Perspective</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392038/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem-cell (HSC) transplantation (HSCT) is used to treat various hematologic disorders. Use of genetically modified mouse models of hematopoietic cell transplantation has been critical in our fundamental understanding of HSC biology and in developing approaches for human patients. Pre-clinical studies in animal models provide insight into the journey of transplanted HSCs from infusion to engraftment in bone-marrow (BM) niches. Various signaling molecules and growth factors secreted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392038"
                        open
                      >
                        <summary class="article-expander__title">Homing and Engraftment of Hematopoietic Stem Cells Following Transplantation: A Pre-Clinical Perspective</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392038/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem-cell (HSC) transplantation (HSCT) is used to treat various hematologic disorders. Use of genetically modified mouse models of hematopoietic cell transplantation has been critical in our fundamental understanding of HSC biology and in developing approaches for human patients. Pre-clinical studies in animal models provide insight into the journey of transplanted HSCs from infusion to engraftment in bone-marrow (BM) niches. Various signaling molecules and growth factors secreted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392038"
                        open
                      >
                        <summary class="article-expander__title">Homing and Engraftment of Hematopoietic Stem Cells Following Transplantation: A Pre-Clinical Perspective</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392038/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem-cell (HSC) transplantation (HSCT) is used to treat various hematologic disorders. Use of genetically modified mouse models of hematopoietic cell transplantation has been critical in our fundamental understanding of HSC biology and in developing approaches for human patients. Pre-clinical studies in animal models provide insight into the journey of transplanted HSCs from infusion to engraftment in bone-marrow (BM) niches. Various signaling molecules and growth factors secreted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392038"
                        open
                      >
                        <summary class="article-expander__title">Homing and Engraftment of Hematopoietic Stem Cells Following Transplantation: A Pre-Clinical Perspective</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392038/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem-cell (HSC) transplantation (HSCT) is used to treat various hematologic disorders. Use of genetically modified mouse models of hematopoietic cell transplantation has been critical in our fundamental understanding of HSC biology and in developing approaches for human patients. Pre-clinical studies in animal models provide insight into the journey of transplanted HSCs from infusion to engraftment in bone-marrow (BM) niches. Various signaling molecules and growth factors secreted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392038"
                        open
                      >
                        <summary class="article-expander__title">Homing and Engraftment of Hematopoietic Stem Cells Following Transplantation: A Pre-Clinical Perspective</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392038/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem-cell (HSC) transplantation (HSCT) is used to treat various hematologic disorders. Use of genetically modified mouse models of hematopoietic cell transplantation has been critical in our fundamental understanding of HSC biology and in developing approaches for human patients. Pre-clinical studies in animal models provide insight into the journey of transplanted HSCs from infusion to engraftment in bone-marrow (BM) niches. Various signaling molecules and growth factors secreted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392038"
                        open
                      >
                        <summary class="article-expander__title">Homing and Engraftment of Hematopoietic Stem Cells Following Transplantation: A Pre-Clinical Perspective</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392038/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem-cell (HSC) transplantation (HSCT) is used to treat various hematologic disorders. Use of genetically modified mouse models of hematopoietic cell transplantation has been critical in our fundamental understanding of HSC biology and in developing approaches for human patients. Pre-clinical studies in animal models provide insight into the journey of transplanted HSCs from infusion to engraftment in bone-marrow (BM) niches. Various signaling molecules and growth factors secreted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392038"
                        open
                      >
                        <summary class="article-expander__title">Homing and Engraftment of Hematopoietic Stem Cells Following Transplantation: A Pre-Clinical Perspective</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392038/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoietic stem-cell (HSC) transplantation (HSCT) is used to treat various hematologic disorders. Use of genetically modified mouse models of hematopoietic cell transplantation has been critical in our fundamental understanding of HSC biology and in developing approaches for human patients. Pre-clinical studies in animal models provide insight into the journey of transplanted HSCs from infusion to engraftment in bone-marrow (BM) niches. Various signaling molecules and growth factors secreted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38392306"
                        open
                      >
                        <summary class="article-expander__title">Current Applications and Challenges of Next-Generation Sequencing in Plasma Circulating Tumour DNA of Ovarian Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38392306/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumour DNA (ctDNA) facilitates longitudinal study of the tumour genome, which, unlike tumour tissue biopsies, globally reflects intratumor and intermetastatis heterogeneity. Despite its costs, next-generation sequencing (NGS) has revolutionised the study of ctDNA, ensuring a more comprehensive and multimodal approach, increasing data collection, and introducing new variables that can be correlated with clinical outcomes. Current NGS strategies can comprise a tumour-informed set of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-22 datetime="2024-02-22T11:00:00.000Z">2024-02-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-22 datetime="2024-02-22T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240313205031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240312205055&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240311204947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240310205110&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240309195348&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387932"
                        open
                      >
                        <summary class="article-expander__title">Clinical Value of Detecting ABL Kinase Domain Mutations in Patients with Chronic Myeloid Leukemia Based on High-Throughput Sequencing Technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387932/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: High-throughput sequencing is more sensitive and accurate than Sanger sequencing in mutation detection, which is helpful to accurately and visually evaluate TKI treatment response and optimize treatment strategy for CML.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38387920"
                        open
                      >
                        <summary class="article-expander__title">Analysis of CHIP-Related Mutation and Risk of Cardio-Cerebrovasculars Events in Patients with Myeloproliferative Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38387920/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: There is a higher mutation rate of CHIP-related genes in MPN patients, especially those over 60 years old. Older age, thrombosis history, CHIP-related mutations and IL-1 elevated levels are independent risk factors for CCEs in MPN. DNMT3A and ASXL1 mutations are independent risk factors for CCEs in PV patients. CHIP-related gene mutations and inflammatory cytokine IL-1  elevated levels may be the novel risk factors for CCEs in MPN.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38388848"
                        open
                      >
                        <summary class="article-expander__title">Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38388848/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-21 datetime="2024-02-21T11:00:00.000Z">2024-02-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-21 datetime="2024-02-21T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240308194754&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240307195004&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38383412"
                        open
                      >
                        <summary class="article-expander__title">Genomic and transcriptomic profiling of combined small-cell lung cancer through microdissection: unveiling the transformational pathway of mixed subtype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38383412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Together, these findings delved into the clonal origin and TIME heterogeneity of different components in cSCLC, shedding new light on the evolutionary processes underlying this enigmatic subtype.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-19 datetime="2024-02-19T11:00:00.000Z">2024-02-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-19 datetime="2024-02-19T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370823"
                        open
                      >
                        <summary class="article-expander__title">Increased Frequency of Clonal Hematopoiesis of Indeterminate Potential in Bloom Syndrome Probands and Carriers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings of increased CHIP in BSyn probands and carriers suggest that one or two germline pathogenic variants in BLM could be sufficient to increase the risk of clonal hematopoiesis. These findings warrant further studies in larger cohorts to determine the significance of CHIP as a potential biomarker of aging, cancer, cardiovascular disease, morbidity and mortality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370823"
                        open
                      >
                        <summary class="article-expander__title">Increased Frequency of Clonal Hematopoiesis of Indeterminate Potential in Bloom Syndrome Probands and Carriers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings of increased CHIP in BSyn probands and carriers suggest that one or two germline pathogenic variants in BLM could be sufficient to increase the risk of clonal hematopoiesis. These findings warrant further studies in larger cohorts to determine the significance of CHIP as a potential biomarker of aging, cancer, cardiovascular disease, morbidity and mortality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370823"
                        open
                      >
                        <summary class="article-expander__title">Increased Frequency of Clonal Hematopoiesis of Indeterminate Potential in Bloom Syndrome Probands and Carriers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings of increased CHIP in BSyn probands and carriers suggest that one or two germline pathogenic variants in BLM could be sufficient to increase the risk of clonal hematopoiesis. These findings warrant further studies in larger cohorts to determine the significance of CHIP as a potential biomarker of aging, cancer, cardiovascular disease, morbidity and mortality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370823"
                        open
                      >
                        <summary class="article-expander__title">Increased Frequency of Clonal Hematopoiesis of Indeterminate Potential in Bloom Syndrome Probands and Carriers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings of increased CHIP in BSyn probands and carriers suggest that one or two germline pathogenic variants in BLM could be sufficient to increase the risk of clonal hematopoiesis. These findings warrant further studies in larger cohorts to determine the significance of CHIP as a potential biomarker of aging, cancer, cardiovascular disease, morbidity and mortality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370823"
                        open
                      >
                        <summary class="article-expander__title">Increased Frequency of Clonal Hematopoiesis of Indeterminate Potential in Bloom Syndrome Probands and Carriers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings of increased CHIP in BSyn probands and carriers suggest that one or two germline pathogenic variants in BLM could be sufficient to increase the risk of clonal hematopoiesis. These findings warrant further studies in larger cohorts to determine the significance of CHIP as a potential biomarker of aging, cancer, cardiovascular disease, morbidity and mortality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370823"
                        open
                      >
                        <summary class="article-expander__title">Increased Frequency of Clonal Hematopoiesis of Indeterminate Potential in Bloom Syndrome Probands and Carriers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings of increased CHIP in BSyn probands and carriers suggest that one or two germline pathogenic variants in BLM could be sufficient to increase the risk of clonal hematopoiesis. These findings warrant further studies in larger cohorts to determine the significance of CHIP as a potential biomarker of aging, cancer, cardiovascular disease, morbidity and mortality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370823"
                        open
                      >
                        <summary class="article-expander__title">Increased Frequency of Clonal Hematopoiesis of Indeterminate Potential in Bloom Syndrome Probands and Carriers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings of increased CHIP in BSyn probands and carriers suggest that one or two germline pathogenic variants in BLM could be sufficient to increase the risk of clonal hematopoiesis. These findings warrant further studies in larger cohorts to determine the significance of CHIP as a potential biomarker of aging, cancer, cardiovascular disease, morbidity and mortality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370823"
                        open
                      >
                        <summary class="article-expander__title">Increased Frequency of Clonal Hematopoiesis of Indeterminate Potential in Bloom Syndrome Probands and Carriers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings of increased CHIP in BSyn probands and carriers suggest that one or two germline pathogenic variants in BLM could be sufficient to increase the risk of clonal hematopoiesis. These findings warrant further studies in larger cohorts to determine the significance of CHIP as a potential biomarker of aging, cancer, cardiovascular disease, morbidity and mortality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370823"
                        open
                      >
                        <summary class="article-expander__title">Increased Frequency of Clonal Hematopoiesis of Indeterminate Potential in Bloom Syndrome Probands and Carriers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings of increased CHIP in BSyn probands and carriers suggest that one or two germline pathogenic variants in BLM could be sufficient to increase the risk of clonal hematopoiesis. These findings warrant further studies in larger cohorts to determine the significance of CHIP as a potential biomarker of aging, cancer, cardiovascular disease, morbidity and mortality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370823"
                        open
                      >
                        <summary class="article-expander__title">Increased Frequency of Clonal Hematopoiesis of Indeterminate Potential in Bloom Syndrome Probands and Carriers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings of increased CHIP in BSyn probands and carriers suggest that one or two germline pathogenic variants in BLM could be sufficient to increase the risk of clonal hematopoiesis. These findings warrant further studies in larger cohorts to determine the significance of CHIP as a potential biomarker of aging, cancer, cardiovascular disease, morbidity and mortality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370823"
                        open
                      >
                        <summary class="article-expander__title">Increased Frequency of Clonal Hematopoiesis of Indeterminate Potential in Bloom Syndrome Probands and Carriers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings of increased CHIP in BSyn probands and carriers suggest that one or two germline pathogenic variants in BLM could be sufficient to increase the risk of clonal hematopoiesis. These findings warrant further studies in larger cohorts to determine the significance of CHIP as a potential biomarker of aging, cancer, cardiovascular disease, morbidity and mortality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370823"
                        open
                      >
                        <summary class="article-expander__title">Increased Frequency of Clonal Hematopoiesis of Indeterminate Potential in Bloom Syndrome Probands and Carriers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370823/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our findings of increased CHIP in BSyn probands and carriers suggest that one or two germline pathogenic variants in BLM could be sufficient to increase the risk of clonal hematopoiesis. These findings warrant further studies in larger cohorts to determine the significance of CHIP as a potential biomarker of aging, cancer, cardiovascular disease, morbidity and mortality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240306193958&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240305195013&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38374116"
                        open
                      >
                        <summary class="article-expander__title">Intra-prostatic tumour evolution, steps in metastatic spread and histogenomic associations revealed by integration of multi-region whole-genome sequencing with histopathological features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38374116/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results link histological transformation with specific genomic alterations and phylogenetic branching. These findings have implications for diagnosis and risk stratification, in addition to providing a rationale for further studies to characterise the genetic changes causally linked to morphological transformation. Our study demonstrates the value of integrating multi-region sequencing with histopathological data to understand tumour evolution and identify mechanisms of prostate...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38370717"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal profiling defines persistence and resistance dynamics during targeted treatment of melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38370717/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. Using spatial transcriptomics in patient derived xenograft models, we capture clonal lineage evolution during treatment, finding the persister state to show increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-16 datetime="2024-02-16T11:00:00.000Z">2024-02-16</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-16 datetime="2024-02-16T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362345"
                        open
                      >
                        <summary class="article-expander__title">Clonal Expansion in Cardiovascular Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansion refers to the proliferation and selection of advantageous &quot;clones&quot; that are better suited for survival in a Darwinian manner. In recent years, we have greatly enhanced our understanding of cell clonality in the cardiovascular context. However, our knowledge of the underlying mechanisms behind this clonal selection is still severely limited. There is a transpiring pattern of clonal expansion of smooth muscle cells and endothelial cells-and, in some cases, macrophages-in numerous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362011"
                        open
                      >
                        <summary class="article-expander__title">Suppression of IL-1beta promotes beneficial accumulation of fibroblast-like cells in atherosclerotic plaques in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an independent risk factor for atherosclerotic cardiovascular disease. Murine models of CH suggest a central role of inflammasomes and IL-1 in accelerated atherosclerosis and plaque destabilization. Here we show using single-cell RNA sequencing in human carotid plaques that inflammasome components are enriched in macrophages, while the receptor for IL-1 is enriched in fibroblasts and smooth muscle cells (SMCs). To address the role of inflammatory crosstalk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362345"
                        open
                      >
                        <summary class="article-expander__title">Clonal Expansion in Cardiovascular Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansion refers to the proliferation and selection of advantageous &quot;clones&quot; that are better suited for survival in a Darwinian manner. In recent years, we have greatly enhanced our understanding of cell clonality in the cardiovascular context. However, our knowledge of the underlying mechanisms behind this clonal selection is still severely limited. There is a transpiring pattern of clonal expansion of smooth muscle cells and endothelial cells-and, in some cases, macrophages-in numerous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362011"
                        open
                      >
                        <summary class="article-expander__title">Suppression of IL-1beta promotes beneficial accumulation of fibroblast-like cells in atherosclerotic plaques in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an independent risk factor for atherosclerotic cardiovascular disease. Murine models of CH suggest a central role of inflammasomes and IL-1 in accelerated atherosclerosis and plaque destabilization. Here we show using single-cell RNA sequencing in human carotid plaques that inflammasome components are enriched in macrophages, while the receptor for IL-1 is enriched in fibroblasts and smooth muscle cells (SMCs). To address the role of inflammatory crosstalk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362345"
                        open
                      >
                        <summary class="article-expander__title">Clonal Expansion in Cardiovascular Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansion refers to the proliferation and selection of advantageous &quot;clones&quot; that are better suited for survival in a Darwinian manner. In recent years, we have greatly enhanced our understanding of cell clonality in the cardiovascular context. However, our knowledge of the underlying mechanisms behind this clonal selection is still severely limited. There is a transpiring pattern of clonal expansion of smooth muscle cells and endothelial cells-and, in some cases, macrophages-in numerous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362011"
                        open
                      >
                        <summary class="article-expander__title">Suppression of IL-1beta promotes beneficial accumulation of fibroblast-like cells in atherosclerotic plaques in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an independent risk factor for atherosclerotic cardiovascular disease. Murine models of CH suggest a central role of inflammasomes and IL-1 in accelerated atherosclerosis and plaque destabilization. Here we show using single-cell RNA sequencing in human carotid plaques that inflammasome components are enriched in macrophages, while the receptor for IL-1 is enriched in fibroblasts and smooth muscle cells (SMCs). To address the role of inflammatory crosstalk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362345"
                        open
                      >
                        <summary class="article-expander__title">Clonal Expansion in Cardiovascular Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansion refers to the proliferation and selection of advantageous &quot;clones&quot; that are better suited for survival in a Darwinian manner. In recent years, we have greatly enhanced our understanding of cell clonality in the cardiovascular context. However, our knowledge of the underlying mechanisms behind this clonal selection is still severely limited. There is a transpiring pattern of clonal expansion of smooth muscle cells and endothelial cells-and, in some cases, macrophages-in numerous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362011"
                        open
                      >
                        <summary class="article-expander__title">Suppression of IL-1beta promotes beneficial accumulation of fibroblast-like cells in atherosclerotic plaques in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an independent risk factor for atherosclerotic cardiovascular disease. Murine models of CH suggest a central role of inflammasomes and IL-1 in accelerated atherosclerosis and plaque destabilization. Here we show using single-cell RNA sequencing in human carotid plaques that inflammasome components are enriched in macrophages, while the receptor for IL-1 is enriched in fibroblasts and smooth muscle cells (SMCs). To address the role of inflammatory crosstalk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362345"
                        open
                      >
                        <summary class="article-expander__title">Clonal Expansion in Cardiovascular Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansion refers to the proliferation and selection of advantageous &quot;clones&quot; that are better suited for survival in a Darwinian manner. In recent years, we have greatly enhanced our understanding of cell clonality in the cardiovascular context. However, our knowledge of the underlying mechanisms behind this clonal selection is still severely limited. There is a transpiring pattern of clonal expansion of smooth muscle cells and endothelial cells-and, in some cases, macrophages-in numerous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362011"
                        open
                      >
                        <summary class="article-expander__title">Suppression of IL-1beta promotes beneficial accumulation of fibroblast-like cells in atherosclerotic plaques in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an independent risk factor for atherosclerotic cardiovascular disease. Murine models of CH suggest a central role of inflammasomes and IL-1 in accelerated atherosclerosis and plaque destabilization. Here we show using single-cell RNA sequencing in human carotid plaques that inflammasome components are enriched in macrophages, while the receptor for IL-1 is enriched in fibroblasts and smooth muscle cells (SMCs). To address the role of inflammatory crosstalk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362345"
                        open
                      >
                        <summary class="article-expander__title">Clonal Expansion in Cardiovascular Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansion refers to the proliferation and selection of advantageous &quot;clones&quot; that are better suited for survival in a Darwinian manner. In recent years, we have greatly enhanced our understanding of cell clonality in the cardiovascular context. However, our knowledge of the underlying mechanisms behind this clonal selection is still severely limited. There is a transpiring pattern of clonal expansion of smooth muscle cells and endothelial cells-and, in some cases, macrophages-in numerous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362011"
                        open
                      >
                        <summary class="article-expander__title">Suppression of IL-1beta promotes beneficial accumulation of fibroblast-like cells in atherosclerotic plaques in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an independent risk factor for atherosclerotic cardiovascular disease. Murine models of CH suggest a central role of inflammasomes and IL-1 in accelerated atherosclerosis and plaque destabilization. Here we show using single-cell RNA sequencing in human carotid plaques that inflammasome components are enriched in macrophages, while the receptor for IL-1 is enriched in fibroblasts and smooth muscle cells (SMCs). To address the role of inflammatory crosstalk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362345"
                        open
                      >
                        <summary class="article-expander__title">Clonal Expansion in Cardiovascular Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansion refers to the proliferation and selection of advantageous &quot;clones&quot; that are better suited for survival in a Darwinian manner. In recent years, we have greatly enhanced our understanding of cell clonality in the cardiovascular context. However, our knowledge of the underlying mechanisms behind this clonal selection is still severely limited. There is a transpiring pattern of clonal expansion of smooth muscle cells and endothelial cells-and, in some cases, macrophages-in numerous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362011"
                        open
                      >
                        <summary class="article-expander__title">Suppression of IL-1beta promotes beneficial accumulation of fibroblast-like cells in atherosclerotic plaques in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an independent risk factor for atherosclerotic cardiovascular disease. Murine models of CH suggest a central role of inflammasomes and IL-1 in accelerated atherosclerosis and plaque destabilization. Here we show using single-cell RNA sequencing in human carotid plaques that inflammasome components are enriched in macrophages, while the receptor for IL-1 is enriched in fibroblasts and smooth muscle cells (SMCs). To address the role of inflammatory crosstalk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362345"
                        open
                      >
                        <summary class="article-expander__title">Clonal Expansion in Cardiovascular Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansion refers to the proliferation and selection of advantageous &quot;clones&quot; that are better suited for survival in a Darwinian manner. In recent years, we have greatly enhanced our understanding of cell clonality in the cardiovascular context. However, our knowledge of the underlying mechanisms behind this clonal selection is still severely limited. There is a transpiring pattern of clonal expansion of smooth muscle cells and endothelial cells-and, in some cases, macrophages-in numerous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362011"
                        open
                      >
                        <summary class="article-expander__title">Suppression of IL-1beta promotes beneficial accumulation of fibroblast-like cells in atherosclerotic plaques in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an independent risk factor for atherosclerotic cardiovascular disease. Murine models of CH suggest a central role of inflammasomes and IL-1 in accelerated atherosclerosis and plaque destabilization. Here we show using single-cell RNA sequencing in human carotid plaques that inflammasome components are enriched in macrophages, while the receptor for IL-1 is enriched in fibroblasts and smooth muscle cells (SMCs). To address the role of inflammatory crosstalk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362345"
                        open
                      >
                        <summary class="article-expander__title">Clonal Expansion in Cardiovascular Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansion refers to the proliferation and selection of advantageous &quot;clones&quot; that are better suited for survival in a Darwinian manner. In recent years, we have greatly enhanced our understanding of cell clonality in the cardiovascular context. However, our knowledge of the underlying mechanisms behind this clonal selection is still severely limited. There is a transpiring pattern of clonal expansion of smooth muscle cells and endothelial cells-and, in some cases, macrophages-in numerous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362011"
                        open
                      >
                        <summary class="article-expander__title">Suppression of IL-1beta promotes beneficial accumulation of fibroblast-like cells in atherosclerotic plaques in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an independent risk factor for atherosclerotic cardiovascular disease. Murine models of CH suggest a central role of inflammasomes and IL-1 in accelerated atherosclerosis and plaque destabilization. Here we show using single-cell RNA sequencing in human carotid plaques that inflammasome components are enriched in macrophages, while the receptor for IL-1 is enriched in fibroblasts and smooth muscle cells (SMCs). To address the role of inflammatory crosstalk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362345"
                        open
                      >
                        <summary class="article-expander__title">Clonal Expansion in Cardiovascular Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansion refers to the proliferation and selection of advantageous &quot;clones&quot; that are better suited for survival in a Darwinian manner. In recent years, we have greatly enhanced our understanding of cell clonality in the cardiovascular context. However, our knowledge of the underlying mechanisms behind this clonal selection is still severely limited. There is a transpiring pattern of clonal expansion of smooth muscle cells and endothelial cells-and, in some cases, macrophages-in numerous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362011"
                        open
                      >
                        <summary class="article-expander__title">Suppression of IL-1 promotes beneficial accumulation of fibroblast-like cells in atherosclerotic plaques in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an independent risk factor for atherosclerotic cardiovascular disease. Murine models of CH suggest a central role of inflammasomes and IL-1 in accelerated atherosclerosis and plaque destabilization. Here we show using single-cell RNA sequencing in human carotid plaques that inflammasome components are enriched in macrophages, while the receptor for IL-1 is enriched in fibroblasts and smooth muscle cells (SMCs). To address the role of inflammatory crosstalk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362345"
                        open
                      >
                        <summary class="article-expander__title">Clonal Expansion in Cardiovascular Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansion refers to the proliferation and selection of advantageous &quot;clones&quot; that are better suited for survival in a Darwinian manner. In recent years, we have greatly enhanced our understanding of cell clonality in the cardiovascular context. However, our knowledge of the underlying mechanisms behind this clonal selection is still severely limited. There is a transpiring pattern of clonal expansion of smooth muscle cells and endothelial cells-and, in some cases, macrophages-in numerous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362011"
                        open
                      >
                        <summary class="article-expander__title">Suppression of IL-1 promotes beneficial accumulation of fibroblast-like cells in atherosclerotic plaques in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an independent risk factor for atherosclerotic cardiovascular disease. Murine models of CH suggest a central role of inflammasomes and IL-1 in accelerated atherosclerosis and plaque destabilization. Here we show using single-cell RNA sequencing in human carotid plaques that inflammasome components are enriched in macrophages, while the receptor for IL-1 is enriched in fibroblasts and smooth muscle cells (SMCs). To address the role of inflammatory crosstalk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362345"
                        open
                      >
                        <summary class="article-expander__title">Clonal Expansion in Cardiovascular Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansion refers to the proliferation and selection of advantageous &quot;clones&quot; that are better suited for survival in a Darwinian manner. In recent years, we have greatly enhanced our understanding of cell clonality in the cardiovascular context. However, our knowledge of the underlying mechanisms behind this clonal selection is still severely limited. There is a transpiring pattern of clonal expansion of smooth muscle cells and endothelial cells-and, in some cases, macrophages-in numerous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362011"
                        open
                      >
                        <summary class="article-expander__title">Suppression of IL-1 promotes beneficial accumulation of fibroblast-like cells in atherosclerotic plaques in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an independent risk factor for atherosclerotic cardiovascular disease. Murine models of CH suggest a central role of inflammasomes and IL-1 in accelerated atherosclerosis and plaque destabilization. Here we show using single-cell RNA sequencing in human carotid plaques that inflammasome components are enriched in macrophages, while the receptor for IL-1 is enriched in fibroblasts and smooth muscle cells (SMCs). To address the role of inflammatory crosstalk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362345"
                        open
                      >
                        <summary class="article-expander__title">Clonal Expansion in Cardiovascular Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansion refers to the proliferation and selection of advantageous &quot;clones&quot; that are better suited for survival in a Darwinian manner. In recent years, we have greatly enhanced our understanding of cell clonality in the cardiovascular context. However, our knowledge of the underlying mechanisms behind this clonal selection is still severely limited. There is a transpiring pattern of clonal expansion of smooth muscle cells and endothelial cells-and, in some cases, macrophages-in numerous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362011"
                        open
                      >
                        <summary class="article-expander__title">Suppression of IL-1 promotes beneficial accumulation of fibroblast-like cells in atherosclerotic plaques in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an independent risk factor for atherosclerotic cardiovascular disease. Murine models of CH suggest a central role of inflammasomes and IL-1 in accelerated atherosclerosis and plaque destabilization. Here we show using single-cell RNA sequencing in human carotid plaques that inflammasome components are enriched in macrophages, while the receptor for IL-1 is enriched in fibroblasts and smooth muscle cells (SMCs). To address the role of inflammatory crosstalk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362345"
                        open
                      >
                        <summary class="article-expander__title">Clonal Expansion in Cardiovascular Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansion refers to the proliferation and selection of advantageous &quot;clones&quot; that are better suited for survival in a Darwinian manner. In recent years, we have greatly enhanced our understanding of cell clonality in the cardiovascular context. However, our knowledge of the underlying mechanisms behind this clonal selection is still severely limited. There is a transpiring pattern of clonal expansion of smooth muscle cells and endothelial cells-and, in some cases, macrophages-in numerous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38362011"
                        open
                      >
                        <summary class="article-expander__title">Suppression of IL-1 promotes beneficial accumulation of fibroblast-like cells in atherosclerotic plaques in clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38362011/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an independent risk factor for atherosclerotic cardiovascular disease. Murine models of CH suggest a central role of inflammasomes and IL-1 in accelerated atherosclerosis and plaque destabilization. Here we show using single-cell RNA sequencing in human carotid plaques that inflammasome components are enriched in macrophages, while the receptor for IL-1 is enriched in fibroblasts and smooth muscle cells (SMCs). To address the role of inflammatory crosstalk in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-15 datetime="2024-02-15T11:00:00.000Z">2024-02-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-15 datetime="2024-02-15T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38360546"
                        open
                      >
                        <summary class="article-expander__title">TET Enzymes in the Immune System: From DNA Demethylation to Immunotherapy, Inflammation, and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38360546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ten-eleven translocation (TET) proteins are iron-dependent and -ketoglutarate-dependent dioxygenases that sequentially oxidize the methyl group of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC). All three epigenetic modifications are intermediates in DNA demethylation. TET proteins are recruited by transcription factors and by RNA polymerase II to modify 5mC at enhancers and gene bodies, thereby regulating gene expression during...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38360546"
                        open
                      >
                        <summary class="article-expander__title">TET Enzymes in the Immune System: From DNA Demethylation to Immunotherapy, Inflammation, and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38360546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ten-eleven translocation (TET) proteins are iron-dependent and -ketoglutarate-dependent dioxygenases that sequentially oxidize the methyl group of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC). All three epigenetic modifications are intermediates in DNA demethylation. TET proteins are recruited by transcription factors and by RNA polymerase II to modify 5mC at enhancers and gene bodies, thereby regulating gene expression during...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359087"
                        open
                      >
                        <summary class="article-expander__title">Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in additional sex combs like 1 (ASXL1) confer poor prognosis both in myeloid malignancies and in pre-malignant clonal hematopoiesis (CH). However, the mechanisms by which these mutations contribute to disease initiation remain unresolved and mutation-specific targeting has remained elusive. To address this, we developed a human disease model that recapitulates the disease trajectory from ASXL1-mutant CH to lethal myeloid malignancy. We demonstrate that mutations in ASXL1 lead to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38357791"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential With Loss of Tet2 Enhances Risk for Atrial Fibrillation Through Nlrp3 Inflammasome Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38357791/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We identified a modest association between CHIP, particularly TET2 CHIP, and incident AF in the UK Biobank population. In a mouse model of AF resulting from hematopoietic-specific inactivation of Tet2, we propose altered calcium handling as an arrhythmogenic mechanism, dependent on Nlrp3 inflammasome activation. Our data are in keeping with previous studies of CHIP in cardiovascular disease, and further studies into the therapeutic potential of NLRP3 inhibition for individuals with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38360546"
                        open
                      >
                        <summary class="article-expander__title">TET Enzymes in the Immune System: From DNA Demethylation to Immunotherapy, Inflammation, and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38360546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ten-eleven translocation (TET) proteins are iron-dependent and -ketoglutarate-dependent dioxygenases that sequentially oxidize the methyl group of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC). All three epigenetic modifications are intermediates in DNA demethylation. TET proteins are recruited by transcription factors and by RNA polymerase II to modify 5mC at enhancers and gene bodies, thereby regulating gene expression during...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359087"
                        open
                      >
                        <summary class="article-expander__title">Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in additional sex combs like 1 (ASXL1) confer poor prognosis both in myeloid malignancies and in pre-malignant clonal hematopoiesis (CH). However, the mechanisms by which these mutations contribute to disease initiation remain unresolved and mutation-specific targeting has remained elusive. To address this, we developed a human disease model that recapitulates the disease trajectory from ASXL1-mutant CH to lethal myeloid malignancy. We demonstrate that mutations in ASXL1 lead to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38357791"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential With Loss of Tet2 Enhances Risk for Atrial Fibrillation Through Nlrp3 Inflammasome Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38357791/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We identified a modest association between CHIP, particularly TET2 CHIP, and incident AF in the UK Biobank population. In a mouse model of AF resulting from hematopoietic-specific inactivation of Tet2, we propose altered calcium handling as an arrhythmogenic mechanism, dependent on Nlrp3 inflammasome activation. Our data are in keeping with previous studies of CHIP in cardiovascular disease, and further studies into the therapeutic potential of NLRP3 inhibition for individuals with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38360546"
                        open
                      >
                        <summary class="article-expander__title">TET Enzymes in the Immune System: From DNA Demethylation to Immunotherapy, Inflammation, and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38360546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ten-eleven translocation (TET) proteins are iron-dependent and -ketoglutarate-dependent dioxygenases that sequentially oxidize the methyl group of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC). All three epigenetic modifications are intermediates in DNA demethylation. TET proteins are recruited by transcription factors and by RNA polymerase II to modify 5mC at enhancers and gene bodies, thereby regulating gene expression during...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359087"
                        open
                      >
                        <summary class="article-expander__title">Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in additional sex combs like 1 (ASXL1) confer poor prognosis both in myeloid malignancies and in pre-malignant clonal hematopoiesis (CH). However, the mechanisms by which these mutations contribute to disease initiation remain unresolved and mutation-specific targeting has remained elusive. To address this, we developed a human disease model that recapitulates the disease trajectory from ASXL1-mutant CH to lethal myeloid malignancy. We demonstrate that mutations in ASXL1 lead to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38357791"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential With Loss of Tet2 Enhances Risk for Atrial Fibrillation Through Nlrp3 Inflammasome Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38357791/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We identified a modest association between CHIP, particularly TET2 CHIP, and incident AF in the UK Biobank population. In a mouse model of AF resulting from hematopoietic-specific inactivation of Tet2, we propose altered calcium handling as an arrhythmogenic mechanism, dependent on Nlrp3 inflammasome activation. Our data are in keeping with previous studies of CHIP in cardiovascular disease, and further studies into the therapeutic potential of NLRP3 inhibition for individuals with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38360546"
                        open
                      >
                        <summary class="article-expander__title">TET Enzymes in the Immune System: From DNA Demethylation to Immunotherapy, Inflammation, and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38360546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ten-eleven translocation (TET) proteins are iron-dependent and -ketoglutarate-dependent dioxygenases that sequentially oxidize the methyl group of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC). All three epigenetic modifications are intermediates in DNA demethylation. TET proteins are recruited by transcription factors and by RNA polymerase II to modify 5mC at enhancers and gene bodies, thereby regulating gene expression during...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359087"
                        open
                      >
                        <summary class="article-expander__title">Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in additional sex combs like 1 (ASXL1) confer poor prognosis both in myeloid malignancies and in pre-malignant clonal hematopoiesis (CH). However, the mechanisms by which these mutations contribute to disease initiation remain unresolved and mutation-specific targeting has remained elusive. To address this, we developed a human disease model that recapitulates the disease trajectory from ASXL1-mutant CH to lethal myeloid malignancy. We demonstrate that mutations in ASXL1 lead to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38357791"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential With Loss of Tet2 Enhances Risk for Atrial Fibrillation Through Nlrp3 Inflammasome Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38357791/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We identified a modest association between CHIP, particularly TET2 CHIP, and incident AF in the UK Biobank population. In a mouse model of AF resulting from hematopoietic-specific inactivation of Tet2, we propose altered calcium handling as an arrhythmogenic mechanism, dependent on Nlrp3 inflammasome activation. Our data are in keeping with previous studies of CHIP in cardiovascular disease, and further studies into the therapeutic potential of NLRP3 inhibition for individuals with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38360546"
                        open
                      >
                        <summary class="article-expander__title">TET Enzymes in the Immune System: From DNA Demethylation to Immunotherapy, Inflammation, and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38360546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ten-eleven translocation (TET) proteins are iron-dependent and -ketoglutarate-dependent dioxygenases that sequentially oxidize the methyl group of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC). All three epigenetic modifications are intermediates in DNA demethylation. TET proteins are recruited by transcription factors and by RNA polymerase II to modify 5mC at enhancers and gene bodies, thereby regulating gene expression during...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359087"
                        open
                      >
                        <summary class="article-expander__title">Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in additional sex combs like 1 (ASXL1) confer poor prognosis both in myeloid malignancies and in pre-malignant clonal hematopoiesis (CH). However, the mechanisms by which these mutations contribute to disease initiation remain unresolved and mutation-specific targeting has remained elusive. To address this, we developed a human disease model that recapitulates the disease trajectory from ASXL1-mutant CH to lethal myeloid malignancy. We demonstrate that mutations in ASXL1 lead to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38357791"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential With Loss of Tet2 Enhances Risk for Atrial Fibrillation Through Nlrp3 Inflammasome Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38357791/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We identified a modest association between CHIP, particularly TET2 CHIP, and incident AF in the UK Biobank population. In a mouse model of AF resulting from hematopoietic-specific inactivation of Tet2, we propose altered calcium handling as an arrhythmogenic mechanism, dependent on Nlrp3 inflammasome activation. Our data are in keeping with previous studies of CHIP in cardiovascular disease, and further studies into the therapeutic potential of NLRP3 inhibition for individuals with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38360546"
                        open
                      >
                        <summary class="article-expander__title">TET Enzymes in the Immune System: From DNA Demethylation to Immunotherapy, Inflammation, and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38360546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ten-eleven translocation (TET) proteins are iron-dependent and -ketoglutarate-dependent dioxygenases that sequentially oxidize the methyl group of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC). All three epigenetic modifications are intermediates in DNA demethylation. TET proteins are recruited by transcription factors and by RNA polymerase II to modify 5mC at enhancers and gene bodies, thereby regulating gene expression during...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359087"
                        open
                      >
                        <summary class="article-expander__title">Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in additional sex combs like 1 (ASXL1) confer poor prognosis both in myeloid malignancies and in pre-malignant clonal hematopoiesis (CH). However, the mechanisms by which these mutations contribute to disease initiation remain unresolved and mutation-specific targeting has remained elusive. To address this, we developed a human disease model that recapitulates the disease trajectory from ASXL1-mutant CH to lethal myeloid malignancy. We demonstrate that mutations in ASXL1 lead to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38357791"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential With Loss of Tet2 Enhances Risk for Atrial Fibrillation Through Nlrp3 Inflammasome Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38357791/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We identified a modest association between CHIP, particularly TET2 CHIP, and incident AF in the UK Biobank population. In a mouse model of AF resulting from hematopoietic-specific inactivation of Tet2, we propose altered calcium handling as an arrhythmogenic mechanism, dependent on Nlrp3 inflammasome activation. Our data are in keeping with previous studies of CHIP in cardiovascular disease, and further studies into the therapeutic potential of NLRP3 inhibition for individuals with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38360546"
                        open
                      >
                        <summary class="article-expander__title">TET Enzymes in the Immune System: From DNA Demethylation to Immunotherapy, Inflammation, and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38360546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ten-eleven translocation (TET) proteins are iron-dependent and -ketoglutarate-dependent dioxygenases that sequentially oxidize the methyl group of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC). All three epigenetic modifications are intermediates in DNA demethylation. TET proteins are recruited by transcription factors and by RNA polymerase II to modify 5mC at enhancers and gene bodies, thereby regulating gene expression during...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359087"
                        open
                      >
                        <summary class="article-expander__title">Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in additional sex combs like 1 (ASXL1) confer poor prognosis both in myeloid malignancies and in pre-malignant clonal hematopoiesis (CH). However, the mechanisms by which these mutations contribute to disease initiation remain unresolved and mutation-specific targeting has remained elusive. To address this, we developed a human disease model that recapitulates the disease trajectory from ASXL1-mutant CH to lethal myeloid malignancy. We demonstrate that mutations in ASXL1 lead to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38357791"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential With Loss of Tet2 Enhances Risk for Atrial Fibrillation Through Nlrp3 Inflammasome Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38357791/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We identified a modest association between CHIP, particularly TET2 CHIP, and incident AF in the UK Biobank population. In a mouse model of AF resulting from hematopoietic-specific inactivation of Tet2, we propose altered calcium handling as an arrhythmogenic mechanism, dependent on Nlrp3 inflammasome activation. Our data are in keeping with previous studies of CHIP in cardiovascular disease, and further studies into the therapeutic potential of NLRP3 inhibition for individuals with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38360546"
                        open
                      >
                        <summary class="article-expander__title">TET Enzymes in the Immune System: From DNA Demethylation to Immunotherapy, Inflammation, and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38360546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ten-eleven translocation (TET) proteins are iron-dependent and -ketoglutarate-dependent dioxygenases that sequentially oxidize the methyl group of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC). All three epigenetic modifications are intermediates in DNA demethylation. TET proteins are recruited by transcription factors and by RNA polymerase II to modify 5mC at enhancers and gene bodies, thereby regulating gene expression during...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359087"
                        open
                      >
                        <summary class="article-expander__title">Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in additional sex combs like 1 (ASXL1) confer poor prognosis both in myeloid malignancies and in pre-malignant clonal hematopoiesis (CH). However, the mechanisms by which these mutations contribute to disease initiation remain unresolved and mutation-specific targeting has remained elusive. To address this, we developed a human disease model that recapitulates the disease trajectory from ASXL1-mutant CH to lethal myeloid malignancy. We demonstrate that mutations in ASXL1 lead to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38357791"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential With Loss of Tet2 Enhances Risk for Atrial Fibrillation Through Nlrp3 Inflammasome Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38357791/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We identified a modest association between CHIP, particularly TET2 CHIP, and incident AF in the UK Biobank population. In a mouse model of AF resulting from hematopoietic-specific inactivation of Tet2, we propose altered calcium handling as an arrhythmogenic mechanism, dependent on Nlrp3 inflammasome activation. Our data are in keeping with previous studies of CHIP in cardiovascular disease, and further studies into the therapeutic potential of NLRP3 inhibition for individuals with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38360546"
                        open
                      >
                        <summary class="article-expander__title">TET Enzymes in the Immune System: From DNA Demethylation to Immunotherapy, Inflammation, and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38360546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ten-eleven translocation (TET) proteins are iron-dependent and -ketoglutarate-dependent dioxygenases that sequentially oxidize the methyl group of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC). All three epigenetic modifications are intermediates in DNA demethylation. TET proteins are recruited by transcription factors and by RNA polymerase II to modify 5mC at enhancers and gene bodies, thereby regulating gene expression during...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359087"
                        open
                      >
                        <summary class="article-expander__title">Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in additional sex combs like 1 (ASXL1) confer poor prognosis both in myeloid malignancies and in pre-malignant clonal hematopoiesis (CH). However, the mechanisms by which these mutations contribute to disease initiation remain unresolved and mutation-specific targeting has remained elusive. To address this, we developed a human disease model that recapitulates the disease trajectory from ASXL1-mutant CH to lethal myeloid malignancy. We demonstrate that mutations in ASXL1 lead to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38357791"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential With Loss of Tet2 Enhances Risk for Atrial Fibrillation Through Nlrp3 Inflammasome Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38357791/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We identified a modest association between CHIP, particularly TET2 CHIP, and incident AF in the UK Biobank population. In a mouse model of AF resulting from hematopoietic-specific inactivation of Tet2, we propose altered calcium handling as an arrhythmogenic mechanism, dependent on Nlrp3 inflammasome activation. Our data are in keeping with previous studies of CHIP in cardiovascular disease, and further studies into the therapeutic potential of NLRP3 inhibition for individuals with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38360546"
                        open
                      >
                        <summary class="article-expander__title">TET Enzymes in the Immune System: From DNA Demethylation to Immunotherapy, Inflammation, and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38360546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ten-eleven translocation (TET) proteins are iron-dependent and -ketoglutarate-dependent dioxygenases that sequentially oxidize the methyl group of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC). All three epigenetic modifications are intermediates in DNA demethylation. TET proteins are recruited by transcription factors and by RNA polymerase II to modify 5mC at enhancers and gene bodies, thereby regulating gene expression during...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359087"
                        open
                      >
                        <summary class="article-expander__title">Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in additional sex combs like 1 (ASXL1) confer poor prognosis both in myeloid malignancies and in pre-malignant clonal hematopoiesis (CH). However, the mechanisms by which these mutations contribute to disease initiation remain unresolved and mutation-specific targeting has remained elusive. To address this, we developed a human disease model that recapitulates the disease trajectory from ASXL1-mutant CH to lethal myeloid malignancy. We demonstrate that mutations in ASXL1 lead to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38357791"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential With Loss of Tet2 Enhances Risk for Atrial Fibrillation Through Nlrp3 Inflammasome Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38357791/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We identified a modest association between CHIP, particularly TET2 CHIP, and incident AF in the UK Biobank population. In a mouse model of AF resulting from hematopoietic-specific inactivation of Tet2, we propose altered calcium handling as an arrhythmogenic mechanism, dependent on Nlrp3 inflammasome activation. Our data are in keeping with previous studies of CHIP in cardiovascular disease, and further studies into the therapeutic potential of NLRP3 inhibition for individuals with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38360546"
                        open
                      >
                        <summary class="article-expander__title">TET Enzymes in the Immune System: From DNA Demethylation to Immunotherapy, Inflammation, and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38360546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ten-eleven translocation (TET) proteins are iron-dependent and -ketoglutarate-dependent dioxygenases that sequentially oxidize the methyl group of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC). All three epigenetic modifications are intermediates in DNA demethylation. TET proteins are recruited by transcription factors and by RNA polymerase II to modify 5mC at enhancers and gene bodies, thereby regulating gene expression during...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359087"
                        open
                      >
                        <summary class="article-expander__title">Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in additional sex combs like 1 (ASXL1) confer poor prognosis both in myeloid malignancies and in pre-malignant clonal hematopoiesis (CH). However, the mechanisms by which these mutations contribute to disease initiation remain unresolved and mutation-specific targeting has remained elusive. To address this, we developed a human disease model that recapitulates the disease trajectory from ASXL1-mutant CH to lethal myeloid malignancy. We demonstrate that mutations in ASXL1 lead to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38357791"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential With Loss of Tet2 Enhances Risk for Atrial Fibrillation Through Nlrp3 Inflammasome Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38357791/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We identified a modest association between CHIP, particularly TET2 CHIP, and incident AF in the UK Biobank population. In a mouse model of AF resulting from hematopoietic-specific inactivation of Tet2, we propose altered calcium handling as an arrhythmogenic mechanism, dependent on Nlrp3 inflammasome activation. Our data are in keeping with previous studies of CHIP in cardiovascular disease, and further studies into the therapeutic potential of NLRP3 inhibition for individuals with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38360546"
                        open
                      >
                        <summary class="article-expander__title">TET Enzymes in the Immune System: From DNA Demethylation to Immunotherapy, Inflammation, and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38360546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ten-eleven translocation (TET) proteins are iron-dependent and -ketoglutarate-dependent dioxygenases that sequentially oxidize the methyl group of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC). All three epigenetic modifications are intermediates in DNA demethylation. TET proteins are recruited by transcription factors and by RNA polymerase II to modify 5mC at enhancers and gene bodies, thereby regulating gene expression during...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359087"
                        open
                      >
                        <summary class="article-expander__title">Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in additional sex combs like 1 (ASXL1) confer poor prognosis both in myeloid malignancies and in pre-malignant clonal hematopoiesis (CH). However, the mechanisms by which these mutations contribute to disease initiation remain unresolved and mutation-specific targeting has remained elusive. To address this, we developed a human disease model that recapitulates the disease trajectory from ASXL1-mutant CH to lethal myeloid malignancy. We demonstrate that mutations in ASXL1 lead to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38357791"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential With Loss of Tet2 Enhances Risk for Atrial Fibrillation Through Nlrp3 Inflammasome Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38357791/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We identified a modest association between CHIP, particularly TET2 CHIP, and incident AF in the UK Biobank population. In a mouse model of AF resulting from hematopoietic-specific inactivation of Tet2, we propose altered calcium handling as an arrhythmogenic mechanism, dependent on Nlrp3 inflammasome activation. Our data are in keeping with previous studies of CHIP in cardiovascular disease, and further studies into the therapeutic potential of NLRP3 inhibition for individuals with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38360546"
                        open
                      >
                        <summary class="article-expander__title">TET Enzymes in the Immune System: From DNA Demethylation to Immunotherapy, Inflammation, and Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38360546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ten-eleven translocation (TET) proteins are iron-dependent and -ketoglutarate-dependent dioxygenases that sequentially oxidize the methyl group of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC). All three epigenetic modifications are intermediates in DNA demethylation. TET proteins are recruited by transcription factors and by RNA polymerase II to modify 5mC at enhancers and gene bodies, thereby regulating gene expression during...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359087"
                        open
                      >
                        <summary class="article-expander__title">Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in additional sex combs like 1 (ASXL1) confer poor prognosis both in myeloid malignancies and in pre-malignant clonal hematopoiesis (CH). However, the mechanisms by which these mutations contribute to disease initiation remain unresolved and mutation-specific targeting has remained elusive. To address this, we developed a human disease model that recapitulates the disease trajectory from ASXL1-mutant CH to lethal myeloid malignancy. We demonstrate that mutations in ASXL1 lead to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38357791"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential With Loss of Tet2 Enhances Risk for Atrial Fibrillation Through Nlrp3 Inflammasome Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38357791/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We identified a modest association between CHIP, particularly TET2 CHIP, and incident AF in the UK Biobank population. In a mouse model of AF resulting from hematopoietic-specific inactivation of Tet2, we propose altered calcium handling as an arrhythmogenic mechanism, dependent on Nlrp3 inflammasome activation. Our data are in keeping with previous studies of CHIP in cardiovascular disease, and further studies into the therapeutic potential of NLRP3 inhibition for individuals with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359296"
                        open
                      >
                        <summary class="article-expander__title">The impact of age and number of mutations on the size of clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359296/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) represents the clonal expansion of hematopoietic stem cells and their progeny driven by somatic mutations. Accurate risk assessment of CH is critical for disease prevention and clinical decision-making. The size of CH has been showed to associate with higher disease risk, yet, factors influencing the size of CH are unknown. In addition, the characteristics of CH in long-lived individuals are not well documented. Here, we report an in-depth analysis of CH in longevous...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38359087"
                        open
                      >
                        <summary class="article-expander__title">Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38359087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in additional sex combs like 1 (ASXL1) confer poor prognosis both in myeloid malignancies and in pre-malignant clonal hematopoiesis (CH). However, the mechanisms by which these mutations contribute to disease initiation remain unresolved and mutation-specific targeting has remained elusive. To address this, we developed a human disease model that recapitulates the disease trajectory from ASXL1-mutant CH to lethal myeloid malignancy. We demonstrate that mutations in ASXL1 lead to the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38357791"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis of Indeterminate Potential With Loss of Tet2 Enhances Risk for Atrial Fibrillation Through Nlrp3 Inflammasome Activation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38357791/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We identified a modest association between CHIP, particularly TET2 CHIP, and incident AF in the UK Biobank population. In a mouse model of AF resulting from hematopoietic-specific inactivation of Tet2, we propose altered calcium handling as an arrhythmogenic mechanism, dependent on Nlrp3 inflammasome activation. Our data are in keeping with previous studies of CHIP in cardiovascular disease, and further studies into the therapeutic potential of NLRP3 inhibition for individuals with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-14 datetime="2024-02-14T11:00:00.000Z">2024-02-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-14 datetime="2024-02-14T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240304194947&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38355834"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution of BRCA2 reversion mutations under therapeutic pressure by PARP inhibition and platinum chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38355834/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reversion mutations that restore wild-type function of the BRCA gene have been described as a key mechanism of resistance to Poly(ADP-ribose) polymerase (PARP) inhibitor therapy in BRCA-associated cancers. Here, we report a case of a patient with metastatic castration-resistant prostate cancer (mCRPC) with a germline BRCA2 mutation who developed acquired resistance to PARP inhibition. Extensive genomic interrogation of cell-free DNA (cfDNA) and tissue at baseline, post-progression, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-13 datetime="2024-02-13T11:00:00.000Z">2024-02-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-13 datetime="2024-02-13T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240303195045&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240302195240&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240301194836&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240229195439&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38350765"
                        open
                      >
                        <summary class="article-expander__title">A tower of babel of acronyms? The shadowlands of MGUS/MBL/CHIP/TCUS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38350765/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With the advent of outperforming and massive laboratory tools, such as multiparameter flow cytometry and next-generation sequencing, hematopoietic cell clones with putative abnormalities for a variety of blood malignancies have been appreciated in otherwise healthy individuals. These conditions do not fulfill the criteria of their presumed cancer counterparts, and thus have been recognized as their precursor states. This is the case of monoclonal gammopathy of unknown significance (MGUS), the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38349293"
                        open
                      >
                        <summary class="article-expander__title">Section E6.1-6.6 of the American College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38349293/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytogenomic analyses of acquired clonal chromosomal abnormalities in neoplastic blood, bone marrow, and/or lymph nodes are instrumental in the clinical management of patients with hematologic neoplasms. Cytogenetic analyses assist in the diagnosis of such disorders and can provide important prognostic information. Furthermore, cytogenetic studies can provide crucial information regarding specific genetically defined subtypes of these neoplasms that may have targeted therapies. At time of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-11 datetime="2024-02-11T11:00:00.000Z">2024-02-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-11 datetime="2024-02-11T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240228194944&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38342613"
                        open
                      >
                        <summary class="article-expander__title">Minimal residual disease monitoring in acute myeloid leukemia: Focus on MFC-MRD and treatment guidance for elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38342613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) is distinguished by clonal growth of myeloid precursor cells, which impairs normal hematopoiesis. Minimal residual disease (MRD) refers to the residual leukemia cells that persist after chemotherapy. Patients who test positive for MRD have a higher likelihood of experiencing a recurrence, regardless of the specific chemotherapy approach used. Multi-parameter flow cytometry (MFC), polymerase chain reaction (PCR), and next-generation sequencing (NGS) are commonly...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-10 datetime="2024-02-10T11:00:00.000Z">2024-02-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-10 datetime="2024-02-10T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240227194957&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38341340"
                        open
                      >
                        <summary class="article-expander__title">Immune-driven clonal cell selection at the intersection among cancer, infections, autoimmunity and senescence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38341340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immune surveillance mechanisms play a crucial role in maintaining lifelong immune homeostasis in response to pathologic stimuli and aberrant cell states. However, their persistence, especially in the context of chronic antigenic exposure, can create a fertile ground for immune evasion. These escaping cell phenotypes, harboring a variety of genomic and transcriptomic aberrances, chiefly in human leukocyte antigen (HLA) and antigen presentation machinery genes, may survive and proliferate,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38338802"
                        open
                      >
                        <summary class="article-expander__title">Exploring the Molecular Aspects of Myeloproliferative Neoplasms Associated with Unusual Site Vein Thrombosis: Review of the Literature and Latest Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38338802/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms (MPNs) are the leading causes of unusual site thrombosis, affecting nearly 40% of individuals with conditions like Budd-Chiari syndrome or portal vein thrombosis. Diagnosing MPNs in these cases is challenging because common indicators, such as spleen enlargement and elevated blood cell counts, can be obscured by portal hypertension or bleeding issues. Recent advancements in diagnostic tools have enhanced the accuracy of MPN diagnosis and classification. While bone...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240226194911&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339305"
                        open
                      >
                        <summary class="article-expander__title">Macromutations Yielding Karyotype Alterations (and the Process(es) behind Them) Are the Favored Route of Carcinogenesis and Speciation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339305/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>It is argued that carcinogenesis and speciation are evolutionary events which are based on changes in the &#x27;karyotypic code&#x27; through a phase of &#x27;genome instability&#x27;, followed by a bottleneck of selection for the viability and adaptability of the initial cells. Genomic (i.e., chromosomal) instability is caused by (massive) DNA breakage and the subsequent mis-repair of DNA double-strand breaks (DSBs) resulting in various chromosome rearrangements. Potential tumor cells are selected for rapid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339265"
                        open
                      >
                        <summary class="article-expander__title">Molecular Genetic Profile of Myelofibrosis: Implications in the Diagnosis, Prognosis, and Treatment Advancements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339265/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is an essential element of primary myelofibrosis, whereas secondary MF may develop in the advanced stages of other myeloid neoplasms, especially polycythemia vera and essential thrombocythemia. Over the last two decades, advances in molecular diagnostic techniques, particularly the integration of next-generation sequencing in clinical laboratories, have revolutionized the diagnosis, classification, and clinical decision making of myelofibrosis. Driver mutations involving JAK2,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38339250"
                        open
                      >
                        <summary class="article-expander__title">Novel Insights into the Initiation, Evolution, and Progression of Multiple Myeloma by Multi-Omics Investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38339250/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The evolutionary history of multiple myeloma (MM) includes malignant transformation, followed by progression to pre-malignant stages and overt malignancy, ultimately leading to more aggressive and resistant forms. Over the past decade, large effort has been made to identify the potential therapeutic targets in MM. However, MM remains largely incurable. Most patients experience multiple relapses and inevitably become refractory to treatment. Tumor-initiating cell populations are the postulated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-09 datetime="2024-02-09T11:00:00.000Z">2024-02-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-09 datetime="2024-02-09T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38333211"
                        open
                      >
                        <summary class="article-expander__title">Case report: Cytopenias in VEXAS syndrome - a WHO 2022 based approach in a single-center cohort</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38333211/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>VEXAS syndrome is an acquired autoinflammatory disease characterized in most cases by cytopenias and macrocytic anemia. Dyshematopoiesis is a frequent finding in chronic inflammatory conditions and therefore, cytopenias are not easily classified in VEXAS patients. Here we report a series of 7 patients affected by VEXAS associated cytopenias, treated at our center. The use of NGS, together with morphological assays, integrated with the WHO 2022 criteria, allowed to identify three subsets of VEXAS...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-08 datetime="2024-02-08T11:00:00.000Z">2024-02-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-08 datetime="2024-02-08T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38327653"
                        open
                      >
                        <summary class="article-expander__title">Oxford Nanopore Technology and its Application in Liquid Biopsies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38327653/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advanced medical technologies are transforming the future of healthcare, in particular, the screening and detection of molecular-genetic changes in patients suspected of having a neoplasm. They are based on the assumption that neoplasms release small amounts of various neoplasm-specific molecules, such as tumor DNA, called circulating DNA (cirDNA), into the extracellular space and subsequently into the blood. The detection of tumor-specific molecules and specific molecular changes in body fluids...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-07 datetime="2024-02-07T11:00:00.000Z">2024-02-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-07 datetime="2024-02-07T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38323355"
                        open
                      >
                        <summary class="article-expander__title">Genomic evolutional analysis of surgical resected specimen to assess osimertinib as a first-line therapy in two patients with lung cancer harboring EGFR mutation: Case series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38323355/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) is crucial for patients with lung cancer harboring EGFR mutations. However, almost all patients experience disease progression, regardless of their response to the targeted therapy, necessitating the development of additional treatment options. Two patients with lung cancer harboring EGFR-L858R mutations in exon 21 were treated by surgical resection during successful osimertinib treatment. Because the pathological diagnosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-06 datetime="2024-02-06T11:00:00.000Z">2024-02-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-06 datetime="2024-02-06T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320547"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential is associated with current smoking status and history of exacerbation in patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Given that CHIP in specific genes was significantly associated with current smoking status and acute exacerbation, CHIP can be considered as a candidate biomarker for COPD patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240206194820&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240225195107&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240224195344&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240223194725&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38320009"
                        open
                      >
                        <summary class="article-expander__title">How Great Is the Threat of Clonal Hematopoiesis? Let This New Risk Score Be Your Guide</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38320009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recent expanding compendium of sequencing analysis has offered insight into the pathobiology of myeloid neoplasms. This molecular evidence of clonal hematopoiesis provides information to allow earlier identification of predisposition states to myeloid neoplasms, such as clonal hematopoiesis of indeterminate potential (CHIP) and clonal cytopenia of unknown significance (CCUS). The ability to risk-stratify cases of clonal hematopoiesis that may evolve to frank myeloid neoplasms is essential to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38321573"
                        open
                      >
                        <summary class="article-expander__title">Rare subclonal sequencing of breast cancers indicates putative metastatic driver mutations are predominately acquired after dissemination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38321573/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results strongly suggest that metastatic driver mutations are sequentially acquired and selected within the same clonal lineage both before, but more commonly after, dissemination from the primary tumor, and that these mutations are biologically consequential. Despite inherent limitations in sampling archival primary tumors, our findings indicate that tumor cells in most patients continue to undergo clinically relevant genomic evolution after their dissemination from the primary...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38318640"
                        open
                      >
                        <summary class="article-expander__title">Resolving the heterogeneous tumour microenvironment in cardiac myxoma through single-cell and spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38318640/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our study presents the first comprehensive characterisation of the TME in CM at both single-cell and spatial resolution. Our study provides novel insight into the differentiation of myxoma tumour cells and advance our understanding of the TME in CM. Given the rarity of cardiac tumours, our study provides invaluable datasets and promotes the development of medical therapies for CM.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-05 datetime="2024-02-05T11:00:00.000Z">2024-02-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-05 datetime="2024-02-05T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural History of Germline BRCA1 Mutated and BRCA Wild-type Triple-negative Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240222194922&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing analysis of 13 sequentially obtained tumor samples, eight sequentially obtained circulating tumor DNA (ctDNA) samples and three germline DNA samples over the life history of 3 patients with triple-negative breast cancer (TNBC), 2 of whom had germline pathogenic BRCA1 mutation, to unravel tumor evolution. Tumor tissue from all timepoints and germline DNA was subjected to whole-exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural History of Germline BRCA1 Mutated and BRCA Wild-type Triple-negative Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing analysis of 13 sequentially obtained tumor samples, eight sequentially obtained circulating tumor DNA (ctDNA) samples and three germline DNA samples over the life history of 3 patients with triple-negative breast cancer (TNBC), 2 of whom had germline pathogenic BRCA1 mutation, to unravel tumor evolution. Tumor tissue from all timepoints and germline DNA was subjected to whole-exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural History of Germline BRCA1 Mutated and BRCA Wild-type Triple-negative Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing analysis of 13 sequentially obtained tumor samples, eight sequentially obtained circulating tumor DNA (ctDNA) samples and three germline DNA samples over the life history of 3 patients with triple-negative breast cancer (TNBC), 2 of whom had germline pathogenic BRCA1 mutation, to unravel tumor evolution. Tumor tissue from all timepoints and germline DNA was subjected to whole-exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural History of Germline BRCA1 Mutated and BRCA Wild-type Triple-negative Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing analysis of 13 sequentially obtained tumor samples, eight sequentially obtained circulating tumor DNA (ctDNA) samples and three germline DNA samples over the life history of 3 patients with triple-negative breast cancer (TNBC), 2 of whom had germline pathogenic BRCA1 mutation, to unravel tumor evolution. Tumor tissue from all timepoints and germline DNA was subjected to whole-exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural History of Germline BRCA1 Mutated and BRCA Wild-type Triple-negative Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing analysis of 13 sequentially obtained tumor samples, eight sequentially obtained circulating tumor DNA (ctDNA) samples and three germline DNA samples over the life history of 3 patients with triple-negative breast cancer (TNBC), 2 of whom had germline pathogenic BRCA1 mutation, to unravel tumor evolution. Tumor tissue from all timepoints and germline DNA was subjected to whole-exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural History of Germline BRCA1 Mutated and BRCA Wild-type Triple-negative Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing analysis of 13 sequentially obtained tumor samples, eight sequentially obtained circulating tumor DNA (ctDNA) samples and three germline DNA samples over the life history of 3 patients with triple-negative breast cancer (TNBC), 2 of whom had germline pathogenic BRCA1 mutation, to unravel tumor evolution. Tumor tissue from all timepoints and germline DNA was subjected to whole-exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural History of Germline BRCA1 Mutated and BRCA Wild-type Triple-negative Breast Cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing analysis of 13 sequentially obtained tumor samples, eight sequentially obtained circulating tumor DNA (ctDNA) samples and three germline DNA samples over the life history of 3 patients with triple-negative breast cancer (TNBC), 2 of whom had germline pathogenic BRCA1 mutation, to unravel tumor evolution. Tumor tissue from all timepoints and germline DNA was subjected to whole-exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural history of germline BRCA1 mutated and BRCA wild-type triple-negative breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing (NGS) analysis of 13 sequentially sampled tumours, eight sequentially sampled circulating tumour DNA (ctDNA) and three germline DNA samples over the life history of three triple-negative breast cancer (TNBC) patients, two of whom had germline pathogenic BRCA1 mutation, to unravel tumour evolution. Tumour tissue from all timepoints and germline DNA was subjected to whole exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a WES-derived...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural history of germline BRCA1 mutated and BRCA wild-type triple-negative breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing (NGS) analysis of 13 sequentially sampled tumours, eight sequentially sampled circulating tumour DNA (ctDNA) and three germline DNA samples over the life history of three triple-negative breast cancer (TNBC) patients, two of whom had germline pathogenic BRCA1 mutation, to unravel tumour evolution. Tumour tissue from all timepoints and germline DNA was subjected to whole exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a WES-derived...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural history of germline BRCA1 mutated and BRCA wild-type triple-negative breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing (NGS) analysis of 13 sequentially sampled tumours, eight sequentially sampled circulating tumour DNA (ctDNA) and three germline DNA samples over the life history of three triple-negative breast cancer (TNBC) patients, two of whom had germline pathogenic BRCA1 mutation, to unravel tumour evolution. Tumour tissue from all timepoints and germline DNA was subjected to whole exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a WES-derived...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural history of germline BRCA1 mutated and BRCA wild-type triple-negative breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing (NGS) analysis of 13 sequentially sampled tumours, eight sequentially sampled circulating tumour DNA (ctDNA) and three germline DNA samples over the life history of three triple-negative breast cancer (TNBC) patients, two of whom had germline pathogenic BRCA1 mutation, to unravel tumour evolution. Tumour tissue from all timepoints and germline DNA was subjected to whole exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a WES-derived...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural history of germline BRCA1 mutated and BRCA wild-type triple-negative breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing (NGS) analysis of 13 sequentially sampled tumours, eight sequentially sampled circulating tumour DNA (ctDNA) and three germline DNA samples over the life history of three triple-negative breast cancer (TNBC) patients, two of whom had germline pathogenic BRCA1 mutation, to unravel tumour evolution. Tumour tissue from all timepoints and germline DNA was subjected to whole exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a WES-derived...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural history of germline BRCA1 mutated and BRCA wild-type triple-negative breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing (NGS) analysis of 13 sequentially sampled tumours, eight sequentially sampled circulating tumour DNA (ctDNA) and three germline DNA samples over the life history of three triple-negative breast cancer (TNBC) patients, two of whom had germline pathogenic BRCA1 mutation, to unravel tumour evolution. Tumour tissue from all timepoints and germline DNA was subjected to whole exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a WES-derived...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural history of germline BRCA1 mutated and BRCA wild-type triple-negative breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing (NGS) analysis of 13 sequentially sampled tumours, eight sequentially sampled circulating tumour DNA (ctDNA) and three germline DNA samples over the life history of three triple-negative breast cancer (TNBC) patients, two of whom had germline pathogenic BRCA1 mutation, to unravel tumour evolution. Tumour tissue from all timepoints and germline DNA was subjected to whole exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a WES-derived...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural history of germline BRCA1 mutated and BRCA wild-type triple-negative breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing (NGS) analysis of 13 sequentially sampled tumours, eight sequentially sampled circulating tumour DNA (ctDNA) and three germline DNA samples over the life history of three triple-negative breast cancer (TNBC) patients, two of whom had germline pathogenic BRCA1 mutation, to unravel tumour evolution. Tumour tissue from all timepoints and germline DNA was subjected to whole exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a WES-derived...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural history of germline BRCA1 mutated and BRCA wild-type triple-negative breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing (NGS) analysis of 13 sequentially sampled tumours, eight sequentially sampled circulating tumour DNA (ctDNA) and three germline DNA samples over the life history of three triple-negative breast cancer (TNBC) patients, two of whom had germline pathogenic BRCA1 mutation, to unravel tumour evolution. Tumour tissue from all timepoints and germline DNA was subjected to whole exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a WES-derived...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural history of germline BRCA1 mutated and BRCA wild-type triple-negative breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240206194820&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing (NGS) analysis of 13 sequentially sampled tumours, eight sequentially sampled circulating tumour DNA (ctDNA) and three germline DNA samples over the life history of three triple-negative breast cancer (TNBC) patients, two of whom had germline pathogenic BRCA1 mutation, to unravel tumour evolution. Tumour tissue from all timepoints and germline DNA was subjected to whole exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a WES-derived...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38315150"
                        open
                      >
                        <summary class="article-expander__title">Natural history of germline BRCA1 mutated and BRCA wild-type triple-negative breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38315150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240205194943&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We report a deep next-generation sequencing (NGS) analysis of 13 sequentially sampled tumours, eight sequentially sampled circulating tumour DNA (ctDNA) and three germline DNA samples over the life history of three triple-negative breast cancer (TNBC) patients, two of whom had germline pathogenic BRCA1 mutation, to unravel tumour evolution. Tumour tissue from all timepoints and germline DNA was subjected to whole exome sequencing (WES), custom amplicon deep sequencing (30,000X) of a WES-derived...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-04 datetime="2024-02-04T11:00:00.000Z">2024-02-04</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-04 datetime="2024-02-04T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311515"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in children with predisposing conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311515/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis in children and young adults differs from that occuring in the older adult population. A variety of stressors drive this phenomenon, sometimes independent of age-related processes. For the purposes of this review, we adopt the term clonal hematopoiesis in predisposed individuals (CHIPI) to differentiate it from classical, age-related clonal hematopoiesis of indeterminate potential (CHIP). Stress-induced CHIPI selection can be extrinsic, such as following immunologic,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311515"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in children with predisposing conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311515/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis in children and young adults differs from that occuring in the older adult population. A variety of stressors drive this phenomenon, sometimes independent of age-related processes. For the purposes of this review, we adopt the term clonal hematopoiesis in predisposed individuals (CHIPI) to differentiate it from classical, age-related clonal hematopoiesis of indeterminate potential (CHIP). Stress-induced CHIPI selection can be extrinsic, such as following immunologic,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311515"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in children with predisposing conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311515/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis in children and young adults differs from that occuring in the older adult population. A variety of stressors drive this phenomenon, sometimes independent of age-related processes. For the purposes of this review, we adopt the term clonal hematopoiesis in predisposed individuals (CHIPI) to differentiate it from classical, age-related clonal hematopoiesis of indeterminate potential (CHIP). Stress-induced CHIPI selection can be extrinsic, such as following immunologic,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311515"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in children with predisposing conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311515/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis in children and young adults differs from that occuring in the older adult population. A variety of stressors drive this phenomenon, sometimes independent of age-related processes. For the purposes of this review, we adopt the term clonal hematopoiesis in predisposed individuals (CHIPI) to differentiate it from classical, age-related clonal hematopoiesis of indeterminate potential (CHIP). Stress-induced CHIPI selection can be extrinsic, such as following immunologic,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311514"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition for clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311514/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an entity hallmarked by skewed hematopoiesis with persistent overrepresentation of cells from a common stem/progenitor lineage harboring single-nucleotide variants and/or insertions/deletions. CH is a common and age-related phenomenon that is associated with an increased risk of hematological malignancies, cardiovascular disease, and all-cause mortality. While CH is a term of the hematological aspect, there exists a complex interaction with other organ systems,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38310350"
                        open
                      >
                        <summary class="article-expander__title">Detecting Multiple Driver Mutations in a Patient with Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38310350/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Three driver mutations have been identified in patients with essential thrombocythemia - JAK2 V617F, CALR, and MPL. Out of these, JAK2 V617F is mostly common. These mutations are thought to be mutually exclusive; therefore, the initial workup may not include the identification of all mutations separately. CASE REPORT We present a case of a 55-year-old woman who was referred to the hematology clinic for an elevated platelet count noted when she was hospitalized for a renal stone. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311515"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in children with predisposing conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311515/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis in children and young adults differs from that occuring in the older adult population. A variety of stressors drive this phenomenon, sometimes independent of age-related processes. For the purposes of this review, we adopt the term clonal hematopoiesis in predisposed individuals (CHIPI) to differentiate it from classical, age-related clonal hematopoiesis of indeterminate potential (CHIP). Stress-induced CHIPI selection can be extrinsic, such as following immunologic,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311514"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition for clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311514/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an entity hallmarked by skewed hematopoiesis with persistent overrepresentation of cells from a common stem/progenitor lineage harboring single-nucleotide variants and/or insertions/deletions. CH is a common and age-related phenomenon that is associated with an increased risk of hematological malignancies, cardiovascular disease, and all-cause mortality. While CH is a term of the hematological aspect, there exists a complex interaction with other organ systems,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38310350"
                        open
                      >
                        <summary class="article-expander__title">Detecting Multiple Driver Mutations in a Patient with Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38310350/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Three driver mutations have been identified in patients with essential thrombocythemia - JAK2 V617F, CALR, and MPL. Out of these, JAK2 V617F is mostly common. These mutations are thought to be mutually exclusive; therefore, the initial workup may not include the identification of all mutations separately. CASE REPORT We present a case of a 55-year-old woman who was referred to the hematology clinic for an elevated platelet count noted when she was hospitalized for a renal stone. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311515"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in children with predisposing conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311515/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis in children and young adults differs from that occuring in the older adult population. A variety of stressors drive this phenomenon, sometimes independent of age-related processes. For the purposes of this review, we adopt the term clonal hematopoiesis in predisposed individuals (CHIPI) to differentiate it from classical, age-related clonal hematopoiesis of indeterminate potential (CHIP). Stress-induced CHIPI selection can be extrinsic, such as following immunologic,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311514"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition for clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311514/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an entity hallmarked by skewed hematopoiesis with persistent overrepresentation of cells from a common stem/progenitor lineage harboring single-nucleotide variants and/or insertions/deletions. CH is a common and age-related phenomenon that is associated with an increased risk of hematological malignancies, cardiovascular disease, and all-cause mortality. While CH is a term of the hematological aspect, there exists a complex interaction with other organ systems,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38310350"
                        open
                      >
                        <summary class="article-expander__title">Detecting Multiple Driver Mutations in a Patient with Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38310350/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Three driver mutations have been identified in patients with essential thrombocythemia - JAK2 V617F, CALR, and MPL. Out of these, JAK2 V617F is mostly common. These mutations are thought to be mutually exclusive; therefore, the initial workup may not include the identification of all mutations separately. CASE REPORT We present a case of a 55-year-old woman who was referred to the hematology clinic for an elevated platelet count noted when she was hospitalized for a renal stone. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311515"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in children with predisposing conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311515/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis in children and young adults differs from that occuring in the older adult population. A variety of stressors drive this phenomenon, sometimes independent of age-related processes. For the purposes of this review, we adopt the term clonal hematopoiesis in predisposed individuals (CHIPI) to differentiate it from classical, age-related clonal hematopoiesis of indeterminate potential (CHIP). Stress-induced CHIPI selection can be extrinsic, such as following immunologic,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311514"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition for clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311514/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an entity hallmarked by skewed hematopoiesis with persistent overrepresentation of cells from a common stem/progenitor lineage harboring single-nucleotide variants and/or insertions/deletions. CH is a common and age-related phenomenon that is associated with an increased risk of hematological malignancies, cardiovascular disease, and all-cause mortality. While CH is a term of the hematological aspect, there exists a complex interaction with other organ systems,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38310350"
                        open
                      >
                        <summary class="article-expander__title">Detecting Multiple Driver Mutations in a Patient with Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38310350/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Three driver mutations have been identified in patients with essential thrombocythemia - JAK2 V617F, CALR, and MPL. Out of these, JAK2 V617F is mostly common. These mutations are thought to be mutually exclusive; therefore, the initial workup may not include the identification of all mutations separately. CASE REPORT We present a case of a 55-year-old woman who was referred to the hematology clinic for an elevated platelet count noted when she was hospitalized for a renal stone. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311515"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in children with predisposing conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311515/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis in children and young adults differs from that occuring in the older adult population. A variety of stressors drive this phenomenon, sometimes independent of age-related processes. For the purposes of this review, we adopt the term clonal hematopoiesis in predisposed individuals (CHIPI) to differentiate it from classical, age-related clonal hematopoiesis of indeterminate potential (CHIP). Stress-induced CHIPI selection can be extrinsic, such as following immunologic,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311514"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition for clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311514/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an entity hallmarked by skewed hematopoiesis with persistent overrepresentation of cells from a common stem/progenitor lineage harboring single-nucleotide variants and/or insertions/deletions. CH is a common and age-related phenomenon that is associated with an increased risk of hematological malignancies, cardiovascular disease, and all-cause mortality. While CH is a term of the hematological aspect, there exists a complex interaction with other organ systems,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38310350"
                        open
                      >
                        <summary class="article-expander__title">Detecting Multiple Driver Mutations in a Patient with Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38310350/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Three driver mutations have been identified in patients with essential thrombocythemia - JAK2 V617F, CALR, and MPL. Out of these, JAK2 V617F is mostly common. These mutations are thought to be mutually exclusive; therefore, the initial workup may not include the identification of all mutations separately. CASE REPORT We present a case of a 55-year-old woman who was referred to the hematology clinic for an elevated platelet count noted when she was hospitalized for a renal stone. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311515"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in children with predisposing conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311515/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis in children and young adults differs from that occuring in the older adult population. A variety of stressors drive this phenomenon, sometimes independent of age-related processes. For the purposes of this review, we adopt the term clonal hematopoiesis in predisposed individuals (CHIPI) to differentiate it from classical, age-related clonal hematopoiesis of indeterminate potential (CHIP). Stress-induced CHIPI selection can be extrinsic, such as following immunologic,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311514"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition for clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311514/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an entity hallmarked by skewed hematopoiesis with persistent overrepresentation of cells from a common stem/progenitor lineage harboring single-nucleotide variants and/or insertions/deletions. CH is a common and age-related phenomenon that is associated with an increased risk of hematological malignancies, cardiovascular disease, and all-cause mortality. While CH is a term of the hematological aspect, there exists a complex interaction with other organ systems,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38310350"
                        open
                      >
                        <summary class="article-expander__title">Detecting Multiple Driver Mutations in a Patient with Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38310350/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Three driver mutations have been identified in patients with essential thrombocythemia - JAK2 V617F, CALR, and MPL. Out of these, JAK2 V617F is mostly common. These mutations are thought to be mutually exclusive; therefore, the initial workup may not include the identification of all mutations separately. CASE REPORT We present a case of a 55-year-old woman who was referred to the hematology clinic for an elevated platelet count noted when she was hospitalized for a renal stone. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311515"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in children with predisposing conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311515/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis in children and young adults differs from that occuring in the older adult population. A variety of stressors drive this phenomenon, sometimes independent of age-related processes. For the purposes of this review, we adopt the term clonal hematopoiesis in predisposed individuals (CHIPI) to differentiate it from classical, age-related clonal hematopoiesis of indeterminate potential (CHIP). Stress-induced CHIPI selection can be extrinsic, such as following immunologic,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311514"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition for clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311514/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an entity hallmarked by skewed hematopoiesis with persistent overrepresentation of cells from a common stem/progenitor lineage harboring single-nucleotide variants and/or insertions/deletions. CH is a common and age-related phenomenon that is associated with an increased risk of hematological malignancies, cardiovascular disease, and all-cause mortality. While CH is a term of the hematological aspect, there exists a complex interaction with other organ systems,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38310350"
                        open
                      >
                        <summary class="article-expander__title">Detecting Multiple Driver Mutations in a Patient with Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38310350/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Three driver mutations have been identified in patients with essential thrombocythemia - JAK2 V617F, CALR, and MPL. Out of these, JAK2 V617F is mostly common. These mutations are thought to be mutually exclusive; therefore, the initial workup may not include the identification of all mutations separately. CASE REPORT We present a case of a 55-year-old woman who was referred to the hematology clinic for an elevated platelet count noted when she was hospitalized for a renal stone. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311515"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in children with predisposing conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311515/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis in children and young adults differs from that occuring in the older adult population. A variety of stressors drive this phenomenon, sometimes independent of age-related processes. For the purposes of this review, we adopt the term clonal hematopoiesis in predisposed individuals (CHIPI) to differentiate it from classical, age-related clonal hematopoiesis of indeterminate potential (CHIP). Stress-induced CHIPI selection can be extrinsic, such as following immunologic,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311514"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition for clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311514/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an entity hallmarked by skewed hematopoiesis with persistent overrepresentation of cells from a common stem/progenitor lineage harboring single-nucleotide variants and/or insertions/deletions. CH is a common and age-related phenomenon that is associated with an increased risk of hematological malignancies, cardiovascular disease, and all-cause mortality. While CH is a term of the hematological aspect, there exists a complex interaction with other organ systems,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38310350"
                        open
                      >
                        <summary class="article-expander__title">Detecting Multiple Driver Mutations in a Patient with Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38310350/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Three driver mutations have been identified in patients with essential thrombocythemia - JAK2 V617F, CALR, and MPL. Out of these, JAK2 V617F is mostly common. These mutations are thought to be mutually exclusive; therefore, the initial workup may not include the identification of all mutations separately. CASE REPORT We present a case of a 55-year-old woman who was referred to the hematology clinic for an elevated platelet count noted when she was hospitalized for a renal stone. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311515"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in children with predisposing conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311515/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis in children and young adults differs from that occuring in the older adult population. A variety of stressors drive this phenomenon, sometimes independent of age-related processes. For the purposes of this review, we adopt the term clonal hematopoiesis in predisposed individuals (CHIPI) to differentiate it from classical, age-related clonal hematopoiesis of indeterminate potential (CHIP). Stress-induced CHIPI selection can be extrinsic, such as following immunologic,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311514"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition for clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311514/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an entity hallmarked by skewed hematopoiesis with persistent overrepresentation of cells from a common stem/progenitor lineage harboring single-nucleotide variants and/or insertions/deletions. CH is a common and age-related phenomenon that is associated with an increased risk of hematological malignancies, cardiovascular disease, and all-cause mortality. While CH is a term of the hematological aspect, there exists a complex interaction with other organ systems,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38310350"
                        open
                      >
                        <summary class="article-expander__title">Detecting Multiple Driver Mutations in a Patient with Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38310350/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Three driver mutations have been identified in patients with essential thrombocythemia - JAK2 V617F, CALR, and MPL. Out of these, JAK2 V617F is mostly common. These mutations are thought to be mutually exclusive; therefore, the initial workup may not include the identification of all mutations separately. CASE REPORT We present a case of a 55-year-old woman who was referred to the hematology clinic for an elevated platelet count noted when she was hospitalized for a renal stone. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311515"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in children with predisposing conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311515/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240206194820&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis in children and young adults differs from that occuring in the older adult population. A variety of stressors drive this phenomenon, sometimes independent of age-related processes. For the purposes of this review, we adopt the term clonal hematopoiesis in predisposed individuals (CHIPI) to differentiate it from classical, age-related clonal hematopoiesis of indeterminate potential (CHIP). Stress-induced CHIPI selection can be extrinsic, such as following immunologic,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311514"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition for clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311514/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240206194820&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an entity hallmarked by skewed hematopoiesis with persistent overrepresentation of cells from a common stem/progenitor lineage harboring single-nucleotide variants and/or insertions/deletions. CH is a common and age-related phenomenon that is associated with an increased risk of hematological malignancies, cardiovascular disease, and all-cause mortality. While CH is a term of the hematological aspect, there exists a complex interaction with other organ systems,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38310350"
                        open
                      >
                        <summary class="article-expander__title">Detecting Multiple Driver Mutations in a Patient with Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38310350/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240206194820&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Three driver mutations have been identified in patients with essential thrombocythemia - JAK2 V617F, CALR, and MPL. Out of these, JAK2 V617F is mostly common. These mutations are thought to be mutually exclusive; therefore, the initial workup may not include the identification of all mutations separately. CASE REPORT We present a case of a 55-year-old woman who was referred to the hematology clinic for an elevated platelet count noted when she was hospitalized for a renal stone. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311515"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in children with predisposing conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311515/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240205194943&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis in children and young adults differs from that occuring in the older adult population. A variety of stressors drive this phenomenon, sometimes independent of age-related processes. For the purposes of this review, we adopt the term clonal hematopoiesis in predisposed individuals (CHIPI) to differentiate it from classical, age-related clonal hematopoiesis of indeterminate potential (CHIP). Stress-induced CHIPI selection can be extrinsic, such as following immunologic,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38311514"
                        open
                      >
                        <summary class="article-expander__title">Germline predisposition for clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38311514/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240205194943&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is an entity hallmarked by skewed hematopoiesis with persistent overrepresentation of cells from a common stem/progenitor lineage harboring single-nucleotide variants and/or insertions/deletions. CH is a common and age-related phenomenon that is associated with an increased risk of hematological malignancies, cardiovascular disease, and all-cause mortality. While CH is a term of the hematological aspect, there exists a complex interaction with other organ systems,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38310350"
                        open
                      >
                        <summary class="article-expander__title">Detecting Multiple Driver Mutations in a Patient with Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38310350/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240205194943&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Three driver mutations have been identified in patients with essential thrombocythemia - JAK2 V617F, CALR, and MPL. Out of these, JAK2 V617F is mostly common. These mutations are thought to be mutually exclusive; therefore, the initial workup may not include the identification of all mutations separately. CASE REPORT We present a case of a 55-year-old woman who was referred to the hematology clinic for an elevated platelet count noted when she was hospitalized for a renal stone. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38310350"
                        open
                      >
                        <summary class="article-expander__title">Detecting Multiple Driver Mutations in a Patient with Essential Thrombocythemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38310350/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240204195148&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Three driver mutations have been identified in patients with essential thrombocythemia - JAK2 V617F, CALR, and MPL. Out of these, JAK2 V617F is mostly common. These mutations are thought to be mutually exclusive; therefore, the initial workup may not include the identification of all mutations separately. CASE REPORT We present a case of a 55-year-old woman who was referred to the hematology clinic for an elevated platelet count noted when she was hospitalized for a renal stone. The...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-02-01 datetime="2024-02-01T11:00:00.000Z">2024-02-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-02-01 datetime="2024-02-01T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240221194954&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240220195031&amp;v&#x3D;2.18.0.post9+e462414">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38300528"
                        open
                      >
                        <summary class="article-expander__title">Single-cell Multiomics Analysis of Myelodysplastic Syndromes and Clinical Response to Hypomethylating Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38300528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240219194927&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Alterations in epigenetic marks, such as DNA methylation, represent a hallmark of cancer that has been successfully exploited for therapy in myeloid malignancies. Hypomethylating agents (HMA), such as azacitidine, have become standard-of-care therapy to treat myelodysplastic syndromes (MDS), myeloid neoplasms that can evolve into acute myeloid leukemia. However, our capacity to identify who will respond to HMAs, and the duration of response, remains limited. To shed light on this question, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38300528"
                        open
                      >
                        <summary class="article-expander__title">Single-cell Multiomics Analysis of Myelodysplastic Syndromes and Clinical Response to Hypomethylating Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38300528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Alterations in epigenetic marks, such as DNA methylation, represent a hallmark of cancer that has been successfully exploited for therapy in myeloid malignancies. Hypomethylating agents (HMA), such as azacitidine, have become standard-of-care therapy to treat myelodysplastic syndromes (MDS), myeloid neoplasms that can evolve into acute myeloid leukemia. However, our capacity to identify who will respond to HMAs, and the duration of response, remains limited. To shed light on this question, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38300528"
                        open
                      >
                        <summary class="article-expander__title">Single-cell Multiomics Analysis of Myelodysplastic Syndromes and Clinical Response to Hypomethylating Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38300528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Alterations in epigenetic marks, such as DNA methylation, represent a hallmark of cancer that has been successfully exploited for therapy in myeloid malignancies. Hypomethylating agents (HMA), such as azacitidine, have become standard-of-care therapy to treat myelodysplastic syndromes (MDS), myeloid neoplasms that can evolve into acute myeloid leukemia. However, our capacity to identify who will respond to HMAs, and the duration of response, remains limited. To shed light on this question, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38300528"
                        open
                      >
                        <summary class="article-expander__title">Single-cell Multiomics Analysis of Myelodysplastic Syndromes and Clinical Response to Hypomethylating Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38300528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Alterations in epigenetic marks, such as DNA methylation, represent a hallmark of cancer that has been successfully exploited for therapy in myeloid malignancies. Hypomethylating agents (HMA), such as azacitidine, have become standard-of-care therapy to treat myelodysplastic syndromes (MDS), myeloid neoplasms that can evolve into acute myeloid leukemia. However, our capacity to identify who will respond to HMAs, and the duration of response, remains limited. To shed light on this question, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38300528"
                        open
                      >
                        <summary class="article-expander__title">Single-cell Multiomics Analysis of Myelodysplastic Syndromes and Clinical Response to Hypomethylating Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38300528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Alterations in epigenetic marks, such as DNA methylation, represent a hallmark of cancer that has been successfully exploited for therapy in myeloid malignancies. Hypomethylating agents (HMA), such as azacitidine, have become standard-of-care therapy to treat myelodysplastic syndromes (MDS), myeloid neoplasms that can evolve into acute myeloid leukemia. However, our capacity to identify who will respond to HMAs, and the duration of response, remains limited. To shed light on this question, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38300528"
                        open
                      >
                        <summary class="article-expander__title">Single-cell Multiomics Analysis of Myelodysplastic Syndromes and Clinical Response to Hypomethylating Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38300528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Alterations in epigenetic marks, such as DNA methylation, represent a hallmark of cancer that has been successfully exploited for therapy in myeloid malignancies. Hypomethylating agents (HMA), such as azacitidine, have become standard-of-care therapy to treat myelodysplastic syndromes (MDS), myeloid neoplasms that can evolve into acute myeloid leukemia. However, our capacity to identify who will respond to HMAs, and the duration of response, remains limited. To shed light on this question, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38300528"
                        open
                      >
                        <summary class="article-expander__title">Single-cell Multiomics Analysis of Myelodysplastic Syndromes and Clinical Response to Hypomethylating Therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38300528/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Alterations in epigenetic marks, such as DNA methylation, represent a hallmark of cancer that has been successfully exploited for therapy in myeloid malignancies. Hypomethylating agents (HMA), such as azacitidine, have become standard-of-care therapy to treat myelodysplastic syndromes (MDS), myeloid neoplasms that can evolve into acute myeloid leukemia. However, our capacity to identify who will respond to HMAs, and the duration of response, remains limited. To shed light on this question, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240206194820&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240205194943&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240204195148&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240203195418&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38301422"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic neoplasms evolving from inherited bone marrow failure syndromes / germline predisposition syndromes: Back under the microscope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38301422/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240202194939&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes and germline predisposition syndromes (IBMFS/GPS) are associated with increased risk for hematologic malignancies, particularly myeloid neoplasms, such as myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). The diagnosis of MDS in these syndromes poses difficulty due to frequent bone marrow hypocellularity and the presence of some degree of dysplastic features related to the underlying germline defect causing abnormal maturation of one or...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-01-31 datetime="2024-01-31T11:00:00.000Z">2024-01-31</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-01-31 datetime="2024-01-31T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240206194820&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240205194943&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240204195148&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240203195418&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240202194939&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240201195024&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293139"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential-associated non-small cell lung cancer risk is potentiated by small particulate matter air pollution among non-smokers: a novel somatic variant-environment interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293139/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240131195240&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Small particulate matter air pollution (PM (2.5) ) is a recognized driver of non-small cell lung cancer (NSCLC) among non-smoking individuals. Inhaled PM (2.5) recruits pro-inflammatory macrophages to the air-lung interface, which promotes malignant lung epithelial cell growth and progression to overt cancer. We sought to determine whether clonal hematopoiesis of indeterminate potential (CHIP), a common age-related condition characterized by hyperinflammatory macrophages, exacerbates PM (2.5)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_source&#x3D;axios&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240218195129&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240217195333&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240216194916&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240215194931&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38294947"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Control of Tumor Growth</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38294947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer treatment optimizations select the most optimum combinations of drugs, sequencing schedules, and appropriate doses that would limit toxicity and yield an improved patient quality of life. However, these optimizations often lack an adequate consideration of cancer&#x27;s near-infinite potential for evolutionary adaptation to therapeutic interventions. Adapting cancer therapy based on monitored tumor burden and clonal composition is an intuitively sound approach to the treatment of cancer as an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38294947"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Control of Tumor Growth</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38294947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer treatment optimizations select the most optimum combinations of drugs, sequencing schedules, and appropriate doses that would limit toxicity and yield an improved patient quality of life. However, these optimizations often lack an adequate consideration of cancer&#x27;s near-infinite potential for evolutionary adaptation to therapeutic interventions. Adapting cancer therapy based on monitored tumor burden and clonal composition is an intuitively sound approach to the treatment of cancer as an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293103"
                        open
                      >
                        <summary class="article-expander__title">Single cell RNA-sequencing of Ewing sarcoma tumors demonstrates transcriptional heterogeneity and clonal evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293103/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ewing sarcoma is the second most common bone cancer in children, accounting for 2% of pediatric cancer diagnoses. Patients who present with metastatic disease at the time of diagnosis have a dismal prognosis, compared to the &gt;70% 5-year survival of those with localized disease. Here, we utilized single cell RNA-sequencing to characterize the transcriptional landscape of primary Ewing sarcoma tumors and surrounding tumor microenvironment (TME). Copy-number analysis identified subclonal evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38294947"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Control of Tumor Growth</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38294947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer treatment optimizations select the most optimum combinations of drugs, sequencing schedules, and appropriate doses that would limit toxicity and yield an improved patient quality of life. However, these optimizations often lack an adequate consideration of cancer&#x27;s near-infinite potential for evolutionary adaptation to therapeutic interventions. Adapting cancer therapy based on monitored tumor burden and clonal composition is an intuitively sound approach to the treatment of cancer as an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293103"
                        open
                      >
                        <summary class="article-expander__title">Single cell RNA-sequencing of Ewing sarcoma tumors demonstrates transcriptional heterogeneity and clonal evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293103/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ewing sarcoma is the second most common bone cancer in children, accounting for 2% of pediatric cancer diagnoses. Patients who present with metastatic disease at the time of diagnosis have a dismal prognosis, compared to the &gt;70% 5-year survival of those with localized disease. Here, we utilized single cell RNA-sequencing to characterize the transcriptional landscape of primary Ewing sarcoma tumors and surrounding tumor microenvironment (TME). Copy-number analysis identified subclonal evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38292240"
                        open
                      >
                        <summary class="article-expander__title">TGF- and BMP signaling are associated with the transformation of glioblastoma to gliosarcoma and then osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38292240/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This unusual clinical case provided an opportunity to explore the modulators of longitudinal sarcomatous transformation, potentially uncovering markers indicating predisposition to this change and identification of novel therapeutic targets.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38294947"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Control of Tumor Growth</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38294947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer treatment optimizations select the most optimum combinations of drugs, sequencing schedules, and appropriate doses that would limit toxicity and yield an improved patient quality of life. However, these optimizations often lack an adequate consideration of cancer&#x27;s near-infinite potential for evolutionary adaptation to therapeutic interventions. Adapting cancer therapy based on monitored tumor burden and clonal composition is an intuitively sound approach to the treatment of cancer as an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293103"
                        open
                      >
                        <summary class="article-expander__title">Single cell RNA-sequencing of Ewing sarcoma tumors demonstrates transcriptional heterogeneity and clonal evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293103/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ewing sarcoma is the second most common bone cancer in children, accounting for 2% of pediatric cancer diagnoses. Patients who present with metastatic disease at the time of diagnosis have a dismal prognosis, compared to the &gt;70% 5-year survival of those with localized disease. Here, we utilized single cell RNA-sequencing to characterize the transcriptional landscape of primary Ewing sarcoma tumors and surrounding tumor microenvironment (TME). Copy-number analysis identified subclonal evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38292240"
                        open
                      >
                        <summary class="article-expander__title">TGF- and BMP signaling are associated with the transformation of glioblastoma to gliosarcoma and then osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38292240/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This unusual clinical case provided an opportunity to explore the modulators of longitudinal sarcomatous transformation, potentially uncovering markers indicating predisposition to this change and identification of novel therapeutic targets.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38294947"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Control of Tumor Growth</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38294947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer treatment optimizations select the most optimum combinations of drugs, sequencing schedules, and appropriate doses that would limit toxicity and yield an improved patient quality of life. However, these optimizations often lack an adequate consideration of cancer&#x27;s near-infinite potential for evolutionary adaptation to therapeutic interventions. Adapting cancer therapy based on monitored tumor burden and clonal composition is an intuitively sound approach to the treatment of cancer as an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293103"
                        open
                      >
                        <summary class="article-expander__title">Single cell RNA-sequencing of Ewing sarcoma tumors demonstrates transcriptional heterogeneity and clonal evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293103/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ewing sarcoma is the second most common bone cancer in children, accounting for 2% of pediatric cancer diagnoses. Patients who present with metastatic disease at the time of diagnosis have a dismal prognosis, compared to the &gt;70% 5-year survival of those with localized disease. Here, we utilized single cell RNA-sequencing to characterize the transcriptional landscape of primary Ewing sarcoma tumors and surrounding tumor microenvironment (TME). Copy-number analysis identified subclonal evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38292240"
                        open
                      >
                        <summary class="article-expander__title">TGF- and BMP signaling are associated with the transformation of glioblastoma to gliosarcoma and then osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38292240/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This unusual clinical case provided an opportunity to explore the modulators of longitudinal sarcomatous transformation, potentially uncovering markers indicating predisposition to this change and identification of novel therapeutic targets.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38294947"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Control of Tumor Growth</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38294947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer treatment optimizations select the most optimum combinations of drugs, sequencing schedules, and appropriate doses that would limit toxicity and yield an improved patient quality of life. However, these optimizations often lack an adequate consideration of cancer&#x27;s near-infinite potential for evolutionary adaptation to therapeutic interventions. Adapting cancer therapy based on monitored tumor burden and clonal composition is an intuitively sound approach to the treatment of cancer as an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293103"
                        open
                      >
                        <summary class="article-expander__title">Single cell RNA-sequencing of Ewing sarcoma tumors demonstrates transcriptional heterogeneity and clonal evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293103/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ewing sarcoma is the second most common bone cancer in children, accounting for 2% of pediatric cancer diagnoses. Patients who present with metastatic disease at the time of diagnosis have a dismal prognosis, compared to the &gt;70% 5-year survival of those with localized disease. Here, we utilized single cell RNA-sequencing to characterize the transcriptional landscape of primary Ewing sarcoma tumors and surrounding tumor microenvironment (TME). Copy-number analysis identified subclonal evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38292240"
                        open
                      >
                        <summary class="article-expander__title">TGF- and BMP signaling are associated with the transformation of glioblastoma to gliosarcoma and then osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38292240/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This unusual clinical case provided an opportunity to explore the modulators of longitudinal sarcomatous transformation, potentially uncovering markers indicating predisposition to this change and identification of novel therapeutic targets.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38294947"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Control of Tumor Growth</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38294947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer treatment optimizations select the most optimum combinations of drugs, sequencing schedules, and appropriate doses that would limit toxicity and yield an improved patient quality of life. However, these optimizations often lack an adequate consideration of cancer&#x27;s near-infinite potential for evolutionary adaptation to therapeutic interventions. Adapting cancer therapy based on monitored tumor burden and clonal composition is an intuitively sound approach to the treatment of cancer as an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293103"
                        open
                      >
                        <summary class="article-expander__title">Single cell RNA-sequencing of Ewing sarcoma tumors demonstrates transcriptional heterogeneity and clonal evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293103/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ewing sarcoma is the second most common bone cancer in children, accounting for 2% of pediatric cancer diagnoses. Patients who present with metastatic disease at the time of diagnosis have a dismal prognosis, compared to the &gt;70% 5-year survival of those with localized disease. Here, we utilized single cell RNA-sequencing to characterize the transcriptional landscape of primary Ewing sarcoma tumors and surrounding tumor microenvironment (TME). Copy-number analysis identified subclonal evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38292240"
                        open
                      >
                        <summary class="article-expander__title">TGF- and BMP signaling are associated with the transformation of glioblastoma to gliosarcoma and then osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38292240/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This unusual clinical case provided an opportunity to explore the modulators of longitudinal sarcomatous transformation, potentially uncovering markers indicating predisposition to this change and identification of novel therapeutic targets.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38294947"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Control of Tumor Growth</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38294947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer treatment optimizations select the most optimum combinations of drugs, sequencing schedules, and appropriate doses that would limit toxicity and yield an improved patient quality of life. However, these optimizations often lack an adequate consideration of cancer&#x27;s near-infinite potential for evolutionary adaptation to therapeutic interventions. Adapting cancer therapy based on monitored tumor burden and clonal composition is an intuitively sound approach to the treatment of cancer as an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293103"
                        open
                      >
                        <summary class="article-expander__title">Single cell RNA-sequencing of Ewing sarcoma tumors demonstrates transcriptional heterogeneity and clonal evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293103/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ewing sarcoma is the second most common bone cancer in children, accounting for 2% of pediatric cancer diagnoses. Patients who present with metastatic disease at the time of diagnosis have a dismal prognosis, compared to the &gt;70% 5-year survival of those with localized disease. Here, we utilized single cell RNA-sequencing to characterize the transcriptional landscape of primary Ewing sarcoma tumors and surrounding tumor microenvironment (TME). Copy-number analysis identified subclonal evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38292240"
                        open
                      >
                        <summary class="article-expander__title">TGF- and BMP signaling are associated with the transformation of glioblastoma to gliosarcoma and then osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38292240/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This unusual clinical case provided an opportunity to explore the modulators of longitudinal sarcomatous transformation, potentially uncovering markers indicating predisposition to this change and identification of novel therapeutic targets.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240206194820&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38294947"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Control of Tumor Growth</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38294947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240206194820&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer treatment optimizations select the most optimum combinations of drugs, sequencing schedules, and appropriate doses that would limit toxicity and yield an improved patient quality of life. However, these optimizations often lack an adequate consideration of cancer&#x27;s near-infinite potential for evolutionary adaptation to therapeutic interventions. Adapting cancer therapy based on monitored tumor burden and clonal composition is an intuitively sound approach to the treatment of cancer as an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293103"
                        open
                      >
                        <summary class="article-expander__title">Single cell RNA-sequencing of Ewing sarcoma tumors demonstrates transcriptional heterogeneity and clonal evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293103/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240206194820&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ewing sarcoma is the second most common bone cancer in children, accounting for 2% of pediatric cancer diagnoses. Patients who present with metastatic disease at the time of diagnosis have a dismal prognosis, compared to the &gt;70% 5-year survival of those with localized disease. Here, we utilized single cell RNA-sequencing to characterize the transcriptional landscape of primary Ewing sarcoma tumors and surrounding tumor microenvironment (TME). Copy-number analysis identified subclonal evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38292240"
                        open
                      >
                        <summary class="article-expander__title">TGF- and BMP signaling are associated with the transformation of glioblastoma to gliosarcoma and then osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38292240/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240206194820&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This unusual clinical case provided an opportunity to explore the modulators of longitudinal sarcomatous transformation, potentially uncovering markers indicating predisposition to this change and identification of novel therapeutic targets.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240205194943&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38294947"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Control of Tumor Growth</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38294947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240205194943&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer treatment optimizations select the most optimum combinations of drugs, sequencing schedules, and appropriate doses that would limit toxicity and yield an improved patient quality of life. However, these optimizations often lack an adequate consideration of cancer&#x27;s near-infinite potential for evolutionary adaptation to therapeutic interventions. Adapting cancer therapy based on monitored tumor burden and clonal composition is an intuitively sound approach to the treatment of cancer as an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293103"
                        open
                      >
                        <summary class="article-expander__title">Single cell RNA-sequencing of Ewing sarcoma tumors demonstrates transcriptional heterogeneity and clonal evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293103/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240205194943&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ewing sarcoma is the second most common bone cancer in children, accounting for 2% of pediatric cancer diagnoses. Patients who present with metastatic disease at the time of diagnosis have a dismal prognosis, compared to the &gt;70% 5-year survival of those with localized disease. Here, we utilized single cell RNA-sequencing to characterize the transcriptional landscape of primary Ewing sarcoma tumors and surrounding tumor microenvironment (TME). Copy-number analysis identified subclonal evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38292240"
                        open
                      >
                        <summary class="article-expander__title">TGF- and BMP signaling are associated with the transformation of glioblastoma to gliosarcoma and then osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38292240/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240205194943&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This unusual clinical case provided an opportunity to explore the modulators of longitudinal sarcomatous transformation, potentially uncovering markers indicating predisposition to this change and identification of novel therapeutic targets.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240204195148&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38294947"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Control of Tumor Growth</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38294947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240204195148&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer treatment optimizations select the most optimum combinations of drugs, sequencing schedules, and appropriate doses that would limit toxicity and yield an improved patient quality of life. However, these optimizations often lack an adequate consideration of cancer&#x27;s near-infinite potential for evolutionary adaptation to therapeutic interventions. Adapting cancer therapy based on monitored tumor burden and clonal composition is an intuitively sound approach to the treatment of cancer as an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293103"
                        open
                      >
                        <summary class="article-expander__title">Single cell RNA-sequencing of Ewing sarcoma tumors demonstrates transcriptional heterogeneity and clonal evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293103/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240204195148&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ewing sarcoma is the second most common bone cancer in children, accounting for 2% of pediatric cancer diagnoses. Patients who present with metastatic disease at the time of diagnosis have a dismal prognosis, compared to the &gt;70% 5-year survival of those with localized disease. Here, we utilized single cell RNA-sequencing to characterize the transcriptional landscape of primary Ewing sarcoma tumors and surrounding tumor microenvironment (TME). Copy-number analysis identified subclonal evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38292240"
                        open
                      >
                        <summary class="article-expander__title">TGF- and BMP signaling are associated with the transformation of glioblastoma to gliosarcoma and then osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38292240/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240204195148&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This unusual clinical case provided an opportunity to explore the modulators of longitudinal sarcomatous transformation, potentially uncovering markers indicating predisposition to this change and identification of novel therapeutic targets.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240203195418&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38294947"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Control of Tumor Growth</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38294947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240203195418&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer treatment optimizations select the most optimum combinations of drugs, sequencing schedules, and appropriate doses that would limit toxicity and yield an improved patient quality of life. However, these optimizations often lack an adequate consideration of cancer&#x27;s near-infinite potential for evolutionary adaptation to therapeutic interventions. Adapting cancer therapy based on monitored tumor burden and clonal composition is an intuitively sound approach to the treatment of cancer as an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293103"
                        open
                      >
                        <summary class="article-expander__title">Single cell RNA-sequencing of Ewing sarcoma tumors demonstrates transcriptional heterogeneity and clonal evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293103/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240203195418&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ewing sarcoma is the second most common bone cancer in children, accounting for 2% of pediatric cancer diagnoses. Patients who present with metastatic disease at the time of diagnosis have a dismal prognosis, compared to the &gt;70% 5-year survival of those with localized disease. Here, we utilized single cell RNA-sequencing to characterize the transcriptional landscape of primary Ewing sarcoma tumors and surrounding tumor microenvironment (TME). Copy-number analysis identified subclonal evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38292240"
                        open
                      >
                        <summary class="article-expander__title">TGF- and BMP signaling are associated with the transformation of glioblastoma to gliosarcoma and then osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38292240/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240203195418&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This unusual clinical case provided an opportunity to explore the modulators of longitudinal sarcomatous transformation, potentially uncovering markers indicating predisposition to this change and identification of novel therapeutic targets.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240202194939&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38294947"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Control of Tumor Growth</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38294947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240202194939&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer treatment optimizations select the most optimum combinations of drugs, sequencing schedules, and appropriate doses that would limit toxicity and yield an improved patient quality of life. However, these optimizations often lack an adequate consideration of cancer&#x27;s near-infinite potential for evolutionary adaptation to therapeutic interventions. Adapting cancer therapy based on monitored tumor burden and clonal composition is an intuitively sound approach to the treatment of cancer as an...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293103"
                        open
                      >
                        <summary class="article-expander__title">Single cell RNA-sequencing of Ewing sarcoma tumors demonstrates transcriptional heterogeneity and clonal evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293103/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240202194939&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ewing sarcoma is the second most common bone cancer in children, accounting for 2% of pediatric cancer diagnoses. Patients who present with metastatic disease at the time of diagnosis have a dismal prognosis, compared to the &gt;70% 5-year survival of those with localized disease. Here, we utilized single cell RNA-sequencing to characterize the transcriptional landscape of primary Ewing sarcoma tumors and surrounding tumor microenvironment (TME). Copy-number analysis identified subclonal evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38292240"
                        open
                      >
                        <summary class="article-expander__title">TGF- and BMP signaling are associated with the transformation of glioblastoma to gliosarcoma and then osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38292240/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240202194939&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This unusual clinical case provided an opportunity to explore the modulators of longitudinal sarcomatous transformation, potentially uncovering markers indicating predisposition to this change and identification of novel therapeutic targets.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38297376"
                        open
                      >
                        <summary class="article-expander__title">Computational validation of clonal and subclonal copy number alterations from bulk tumor sequencing using CNAqc</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38297376/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240201195024&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Copy number alterations (CNAs) are among the most important genetic events in cancer, but their detection from sequencing data is challenging because of unknown sample purity, tumor ploidy, and general intra-tumor heterogeneity. Here, we present CNAqc, an evolution-inspired method to perform the computational validation of clonal and subclonal CNAs detected from bulk DNA sequencing. CNAqc is validated using single-cell data and simulations, is applied to over 4000 TCGA and PCAWG samples, and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293103"
                        open
                      >
                        <summary class="article-expander__title">Single cell RNA-sequencing of Ewing sarcoma tumors demonstrates transcriptional heterogeneity and clonal evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293103/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240201195024&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ewing sarcoma is the second most common bone cancer in children, accounting for 2% of pediatric cancer diagnoses. Patients who present with metastatic disease at the time of diagnosis have a dismal prognosis, compared to the &gt;70% 5-year survival of those with localized disease. Here, we utilized single cell RNA-sequencing to characterize the transcriptional landscape of primary Ewing sarcoma tumors and surrounding tumor microenvironment (TME). Copy-number analysis identified subclonal evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38292240"
                        open
                      >
                        <summary class="article-expander__title">TGF- and BMP signaling are associated with the transformation of glioblastoma to gliosarcoma and then osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38292240/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240201195024&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This unusual clinical case provided an opportunity to explore the modulators of longitudinal sarcomatous transformation, potentially uncovering markers indicating predisposition to this change and identification of novel therapeutic targets.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38293103"
                        open
                      >
                        <summary class="article-expander__title">Single cell RNA-sequencing of Ewing sarcoma tumors demonstrates transcriptional heterogeneity and clonal evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38293103/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240131195240&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ewing sarcoma is the second most common bone cancer in children, accounting for 2% of pediatric cancer diagnoses. Patients who present with metastatic disease at the time of diagnosis have a dismal prognosis, compared to the &gt;70% 5-year survival of those with localized disease. Here, we utilized single cell RNA-sequencing to characterize the transcriptional landscape of primary Ewing sarcoma tumors and surrounding tumor microenvironment (TME). Copy-number analysis identified subclonal evolution...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38292240"
                        open
                      >
                        <summary class="article-expander__title">TGF- and BMP signaling are associated with the transformation of glioblastoma to gliosarcoma and then osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38292240/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20240131195240&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: This unusual clinical case provided an opportunity to explore the modulators of longitudinal sarcomatous transformation, potentially uncovering markers indicating predisposition to this change and identification of novel therapeutic targets.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2024-01-29 datetime="2024-01-29T11:00:00.000Z">2024-01-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2024-01-29 datetime="2024-01-29T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.18.0.post9+e462414&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;ff&#x3D;20240325205025&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240214195122&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240213195048&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240212195038&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240211195044&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240210195415&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240209194815&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240208194923&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240207194932&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240206194820&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240205194943&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240204195148&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240203195418&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240202194939&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240201195024&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240131195240&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240130195039&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38283344"
                        open
                      >
                        <summary class="article-expander__title">Engineered T cells from induced pluripotent stem cells: from research towards clinical implementation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38283344/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20240129195013&amp;v&#x3D;2.18.0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Induced pluripotent stem cell (iPSC)-derived T (iT) cells represent a groundbreaking frontier in adoptive cell therapies with engineered T cells, poised to overcome pivotal limitations associated with conventional manufacturing methods. iPSCs offer an off-the-shelf source of therapeutic T cells with the potential for infinite expansion and straightforward genetic manipulation to ensure hypo-immunogenicity and introduce specific therapeutic functions, such as antigen specificity through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z">2022-06-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="72a3a11d39c217b1a80cbb551444e6f6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nodal interferon exposure drives tumour-specific tolerance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1533878403463405569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nodal interferon exposure drives tumour-specific tolerance https://t.co/FT4NeBcFTg pic.twitter.com/NRHkVDUsOr
 NatureReviewsCancer (@NatureRevCancer) June 6, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z">2022-05-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="bb6bc242bcbc356c4a38db850e74d812"
                        open
                      >
                        <summary class="article-expander__title">@VIBConferences: #Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.Registration Deadline: Thursday 2 June. Read more</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VIBConferences/status/1531132720952266752">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>#Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.
Registration Deadline: Thursday 2 June. Read more https://t.co/cc0HC5cj6W pic.twitter.com/u8LFuzBbsV
 VIB Conferences (@VIBConferences) May 30, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z">2022-05-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="ed6c996dc1a718afb523de6229a53131"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530198403686072320">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy https://t.co/wjuHplkpof pic.twitter.com/1SoKYhRsPO
 NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="d15a3a4b24853812d7fa21fd56de5815"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content </summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530174481313972224">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content 
 https://t.co/Fo6XRWV3OZ@natrescancer pic.twitter.com/rbxRPqLV6g
 NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-26 datetime="2022-05-26T08:53:50.000Z">2022-05-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-26 datetime="2022-05-26T08:53:50.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fe13755a2693ea65459e5e21df097d2a"
                        open
                      >
                        <summary class="article-expander__title">@UlrikeHarjes: This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer:  do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/UlrikeHarjes/status/1529747629936193537">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer: https://t.co/p3Y2cdNhco
I do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest https://t.co/qWH2dTv8j9
 Ulrike Harjes (@UlrikeHarjes) May 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z">2022-05-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9ba5279424b31b0122a4ae20c18e6d03"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Developing dietary interventions as therapy for cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1529516411059437568">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Developing dietary interventions as therapy for cancer https://t.co/ijuqctk1wJ pic.twitter.com/KHPpMKnCFa
 NatureReviewsCancer (@NatureRevCancer) May 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z">2022-05-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="611d40f2e6ea2796b028bebda41a86dd"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Digging for treasures in the tumour interactome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526928478490095616">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Digging for treasures in the tumour interactome https://t.co/NhHYAwUgLE
 NatureReviewsCancer (@NatureRevCancer) May 18, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z">2022-05-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9590a971b00448cd312078183965b21d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: As part of our overall commitment to diversity, equity and inclusion at Springer Nature, wed like to better understand what DEI means within the global research community. Take the survey here</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526560254091313156">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>As part of our overall commitment to diversity, equity and inclusion at Springer Nature, wed like to better understand what DEI means within the global research community. Take the survey here https://t.co/03yLiUxFTq
 NatureReviewsCancer (@NatureRevCancer) May 17, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z">2022-05-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="aa38ff4245b93e964773f191844ddc7a"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT   where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial </summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524772831904509953">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT   where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial  https://t.co/VbyoUJmTRJ
 NatureReviewsCancer (@NatureRevCancer) May 12, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z">2022-05-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="65aca76b9f794aaa668d84ad32dca4a0"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tools for the next generation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524068259712036864">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tools for the next generation https://t.co/lEzTPFYDf5
 NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="afa9b6784b4ccc7c05ed4efd26ddfb2d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524022709386190856">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon https://t.co/XXT0c9SAYX
 NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z">2022-04-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="43e2d5ccebc9030f737d656f47ac9d98"
                        open
                      >
                        <summary class="article-expander__title">@VarunVenkatara2: Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VarunVenkatara2/status/1520050291986604033">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter https://t.co/S55cKTGNFd
 Varun Venkataramani (@VarunVenkatara2) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9f6b8644378881f4c2e6b6833c93fc2f"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! An inhospitable site</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020349068316673">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! An inhospitable site https://t.co/0dvqIKCWxh pic.twitter.com/PtY8J9Fz7r
 NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="f6d91efcd60dae769e2722585bdfa8df"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Disconnecting multicellular networks in brain tumours</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020340201598978">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Disconnecting multicellular networks in brain tumours https://t.co/LJrzy8bRma pic.twitter.com/CO7BnooP0r
 NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z">2022-04-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="76e3b436bc8e43ad21366a80c120e395"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519678964364623872">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.
 https://t.co/vQwHFJhpwv@natrescancer pic.twitter.com/6kYgjRYhdT
 NatureReviewsCancer (@NatureRevCancer) April 28, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fdf224ea293cdf039086d16e5699a59a"
                        open
                      >
                        <summary class="article-expander__title">@tmurraystewart: Our latest publication on #polyamines is online!@HopkinsMedicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/tmurraystewart/status/1519413460978704384">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our latest publication on #polyamines is online!@HopkinsMedicine https://t.co/ehRGgN1w9R
 Tracy Murray Stewart (@tmurraystewart) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z">2022-04-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="4f43226ec4dc8e531ff6e64a9c5ab8e6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519381375874265088">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity https://t.co/YCJX1hRQLO pic.twitter.com/zRUCRNXyxH
 NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="a44ed79b9f76aad377b9272fb392f553"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519317944995569664">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.
 https://t.co/JTUhX0DHQ1@natrescancer pic.twitter.com/2PdCmsDu2p
 NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z">2022-04-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="06ff7641d8f5fcc0dff91f835ac3bbc1"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino  @natrescancer @Columbia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518967636561956865">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino
 https://t.co/GeTVCMJQsp @natrescancer
 @Columbia pic.twitter.com/ZXW6A0Fe0c
 NatureReviewsCancer (@NatureRevCancer) April 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z">2022-04-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="96b2ddff4eb269cf343213c6a1b1ce95"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tracking dormant disseminated tumour cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518563607440621569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tracking dormant disseminated tumour cells https://t.co/kSXyWAvCxF
 NatureReviewsCancer (@NatureRevCancer) April 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>

    <footer>
      <a class="footer-link" href="https://github.com/MacwanEric/cancer.rss.feed/actions/runs/8429091581">
        <time id="build-timestamp" datetime="2024-03-26T00:50:28.307Z">2024-03-26T00:50:28.307Z</time>
      </a>
      <a class="footer-link" href="https://github.com/osmoscraft/osmosfeed">osmosfeed 1.15.1</a>
    </footer>
    <script src="index.js?v1.14.4"></script>
    <!-- %before-body-end.html% -->
  </body>

</html>