<!DOCTYPE html>
<html lang="en">

  <head>
    <title>CancerFeed</title>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="robots" content="noindex, nofollow" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="alternate" type="application/rss+xml" title="CancerFeed" href="feed.atom" />
    <link href="index.css?v1.14.4" rel="stylesheet" />
    <style>
  :root {
    /**
     * Solarized Light
     * Ethan Schoonover (modified by aramisgithub)
     */
    --base00: #fdf6e3;
    --base01: #eee8d5;
    --base02: #93a1a1;
    --base03: #839496;
    --base04: #657b83;
    --base05: #586e75;
    --base06: #073642;
    --base07: #002b36;
    --base08: #dc322f;
    --base09: #cb4b16;
    --base0A: #b58900;
    --base0B: #859900;
    --base0C: #2aa198;
    --base0D: #268bd2;
    --base0E: #6c71c4;
    --base0F: #d33682;

    /**
    * Components
    */
    --source-name-color: var(--base0C);
    /**
   * Typograph
   */
    --font-family-default: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell,
      "Helvetica Neue", sans-serif;
    --font-size-L: 3.6rem;
    --font-size-m: 2.2rem;
    --font-size-s: 1.8rem;
    --font-size-scaler: 52.5%; /* 1rem = 10px */
    --line-height-body: 1.4;
    --line-height-heading: 1.2;

  }
</style>

  </head>

  <body>
    <!-- %after-body-begin.html% -->
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-12-06 datetime="2023-12-06T11:00:00.000Z">2023-12-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-12-06 datetime="2023-12-06T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38055913"
                        open
                      >
                        <summary class="article-expander__title">Genetic Characterization of Primary Mediastinal B-Cell Lymphoma: Pathogenesis and Patient Outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38055913/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This large-scale genomic characterization of PMBCL identified novel treatment targets and genetic lesions for refined risk stratification. DUSP2 and CD58 mutation analyses may guide treatment decisions between rituximab plus cyclophosphamide, doxorubicin, vincristine, and prednisone and dose-adjusted etoposide, prednisone, vincristine, cyclophosphamide, doxorubicin, and rituximab.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-12-05 datetime="2023-12-05T11:00:00.000Z">2023-12-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-12-05 datetime="2023-12-05T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38051659"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential: implications for the cardiologists</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38051659/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms, including polycythemia vera, essential thrombocythemia, and myelofibrosis, are characterized by somatic gene mutations in bone marrow stem cells, which trigger an inflammatory response influencing the development of associated cardiovascular complications. In recent years, the same mutations were found in individuals with cardiovascular diseases even in the absence of hematological alterations. These genetic events allow the identification of a new entity called...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38052482"
                        open
                      >
                        <summary class="article-expander__title">Mutations, Bottlenecks, and Clonal Sweeps: How Environmental Carcinogens and Genomic Changes Shape Clonal Evolution during Tumor Progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38052482/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The transition from a single, initiated cell to a full-blown malignant tumor involves significant genomic evolution. Exposure to carcinogens-whether directly mutagenic or not-can drive progression toward malignancy, as can stochastic acquisition of cancer-promoting genetic events. Mouse models using both carcinogens and germline genetic manipulations have enabled precise inquiry into the evolutionary dynamics that take place as a tumor progresses from benign to malignant to metastatic stages....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-12-04 datetime="2023-12-04T11:00:00.000Z">2023-12-04</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-12-04 datetime="2023-12-04T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38049337"
                        open
                      >
                        <summary class="article-expander__title">Effect of clonal hematopoiesis in remission on hematopoiesis recovery in patients with NPM1 mutated acute myeloid leukemia after chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38049337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Objective: To investigate the effect of clonal hematopoiesis (CH) in remission on hematopoiesis recovery in patients with NPM1 mutated acute myeloid leukemia (AML) after chemotherapy. Methods: Retrospective analysis was performed on 86 patients with NPM1(mut) AML newly diagnosed and treated in the First Affiliated Hospital of Soochow University between July 2016 and June 2019. Their clinical data and NGS test results at diagnosis were analyzed. Moreover, bone marrow samples in remission were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38049337"
                        open
                      >
                        <summary class="article-expander__title">Effect of clonal hematopoiesis in remission on hematopoiesis recovery in patients with NPM1 mutated acute myeloid leukemia after chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38049337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Objective: To investigate the effect of clonal hematopoiesis (CH) in remission on hematopoiesis recovery in patients with NPM1 mutated acute myeloid leukemia (AML) after chemotherapy. Methods: Retrospective analysis was performed on 86 patients with NPM1(mut) AML newly diagnosed and treated in the First Affiliated Hospital of Soochow University between July 2016 and June 2019. Their clinical data and NGS test results at diagnosis were analyzed. Moreover, bone marrow samples in remission were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38049337"
                        open
                      >
                        <summary class="article-expander__title">Effect of clonal hematopoiesis in remission on hematopoiesis recovery in patients with NPM1 mutated acute myeloid leukemia after chemotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38049337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Objective: To investigate the effect of clonal hematopoiesis (CH) in remission on hematopoiesis recovery in patients with NPM1 mutated acute myeloid leukemia (AML) after chemotherapy. Methods: Retrospective analysis was performed on 86 patients with NPM1(mut) AML newly diagnosed and treated in the First Affiliated Hospital of Soochow University between July 2016 and June 2019. Their clinical data and NGS test results at diagnosis were analyzed. Moreover, bone marrow samples in remission were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38046281"
                        open
                      >
                        <summary class="article-expander__title">Recombinant Human Adenovirus Type 5 (H101) Intra-Tumor Therapy in Patients with Persistent, Recurrent, or Metastatic Cervical Cancer: Genomic Profiling Relating to Clinical Efficacy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38046281/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conducted an unprecedented work via a WES-based approach and provided preliminary insights into H101 treatment-induced genetic aberrations in which some genes (TTN, KMT2D, ALDOA, DNAH7, ADAP1, PTPN23, and THEMIS2) could be considered potential therapeutic targets of H101-containing treatment in cervical carcinoma. Moreover, the therapy-associated characteristics such as clonal evolution and a mutational signature may warrant further evaluation of H101 in clinical settings for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38046281"
                        open
                      >
                        <summary class="article-expander__title">Recombinant Human Adenovirus Type 5 (H101) Intra-Tumor Therapy in Patients with Persistent, Recurrent, or Metastatic Cervical Cancer: Genomic Profiling Relating to Clinical Efficacy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38046281/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conducted an unprecedented work via a WES-based approach and provided preliminary insights into H101 treatment-induced genetic aberrations in which some genes (TTN, KMT2D, ALDOA, DNAH7, ADAP1, PTPN23, and THEMIS2) could be considered potential therapeutic targets of H101-containing treatment in cervical carcinoma. Moreover, the therapy-associated characteristics such as clonal evolution and a mutational signature may warrant further evaluation of H101 in clinical settings for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38046281"
                        open
                      >
                        <summary class="article-expander__title">Recombinant Human Adenovirus Type 5 (H101) Intra-Tumor Therapy in Patients with Persistent, Recurrent, or Metastatic Cervical Cancer: Genomic Profiling Relating to Clinical Efficacy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38046281/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conducted an unprecedented work via a WES-based approach and provided preliminary insights into H101 treatment-induced genetic aberrations in which some genes (TTN, KMT2D, ALDOA, DNAH7, ADAP1, PTPN23, and THEMIS2) could be considered potential therapeutic targets of H101-containing treatment in cervical carcinoma. Moreover, the therapy-associated characteristics such as clonal evolution and a mutational signature may warrant further evaluation of H101 in clinical settings for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-12-02 datetime="2023-12-02T11:00:00.000Z">2023-12-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-12-02 datetime="2023-12-02T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38042915"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity in ER+ breast cancer reveals the proteasome and PKC as potential therapeutic targets</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38042915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity impacts the success or failure of anti-cancer therapies. Here, we investigated the evolution and mechanistic heterogeneity in clonal populations of cell models for estrogen receptor positive breast cancer. To this end, we established barcoded models of luminal breast cancer and rendered them resistant to commonly applied first line endocrine therapies. By isolating single clones from the resistant cell pools and characterizing replicates of individual clones we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38042915"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity in ER+ breast cancer reveals the proteasome and PKC as potential therapeutic targets</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38042915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity impacts the success or failure of anti-cancer therapies. Here, we investigated the evolution and mechanistic heterogeneity in clonal populations of cell models for estrogen receptor positive breast cancer. To this end, we established barcoded models of luminal breast cancer and rendered them resistant to commonly applied first line endocrine therapies. By isolating single clones from the resistant cell pools and characterizing replicates of individual clones we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38042915"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity in ER+ breast cancer reveals the proteasome and PKC as potential therapeutic targets</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38042915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity impacts the success or failure of anti-cancer therapies. Here, we investigated the evolution and mechanistic heterogeneity in clonal populations of cell models for estrogen receptor positive breast cancer. To this end, we established barcoded models of luminal breast cancer and rendered them resistant to commonly applied first line endocrine therapies. By isolating single clones from the resistant cell pools and characterizing replicates of individual clones we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38042915"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity in ER+ breast cancer reveals the proteasome and PKC as potential therapeutic targets</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38042915/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity impacts the success or failure of anti-cancer therapies. Here, we investigated the evolution and mechanistic heterogeneity in clonal populations of cell models for estrogen receptor positive breast cancer. To this end, we established barcoded models of luminal breast cancer and rendered them resistant to commonly applied first line endocrine therapies. By isolating single clones from the resistant cell pools and characterizing replicates of individual clones we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-12-01 datetime="2023-12-01T11:00:00.000Z">2023-12-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-12-01 datetime="2023-12-01T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38039845"
                        open
                      >
                        <summary class="article-expander__title">Mechanistic target of rapamycin in regulating macrophage function in inflammatory cardiovascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38039845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mechanistic target of rapamycin (mTOR) is evolutionarily conserved from yeast to humans and is one of the most fundamental pathways of living organisms. Since its discovery three decades ago, mTOR has been recognized as the center of nutrient sensing and growth, homeostasis, metabolism, life span, and aging. The role of dysregulated mTOR in common diseases, especially cancer, has been extensively studied and reported. Emerging evidence supports that mTOR critically regulates innate immune...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38037868"
                        open
                      >
                        <summary class="article-expander__title">Circulating tumor DNA profiling approach based on in silico background elimination guides chemotherapy in nasopharyngeal carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38037868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) analysis increasingly provides a promising minimally invasive alternative to tissue biopsies in precision oncology. However, there are no ctDNA analysis approaches available in nasopharyngeal carcinoma (NPC) and current methods of ctDNA mutation profiling have limited resolution because of the high background noise and false-positive rate caused by benign variants in plasma cell-free DNA (cfDNA), majorly generated during clonal hematopoiesis. Although personalized...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38037800"
                        open
                      >
                        <summary class="article-expander__title">Pathobiology of nodal peripheral T-cell lymphomas: current understanding and future directions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38037800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Predominantly nodal is the most common clinical presentation of peripheral T- (and NK-) cell lymphomas (PTCL), which comprise three main groups of diseases: (i) systemic anaplastic large cell lymphomas (ALCL), whether positive or negative for anaplastic lymphoma kinase (ALK); (ii) follicular helper T-cell lymphomas (TFHL); and (iii) PTCL, not otherwise specified (NOS). Recent advances in the genomic and molecular characterization of PTCL, with enhanced understanding of pathobiology, have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38039845"
                        open
                      >
                        <summary class="article-expander__title">Mechanistic target of rapamycin in regulating macrophage function in inflammatory cardiovascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38039845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mechanistic target of rapamycin (mTOR) is evolutionarily conserved from yeast to humans and is one of the most fundamental pathways of living organisms. Since its discovery three decades ago, mTOR has been recognized as the center of nutrient sensing and growth, homeostasis, metabolism, life span, and aging. The role of dysregulated mTOR in common diseases, especially cancer, has been extensively studied and reported. Emerging evidence supports that mTOR critically regulates innate immune...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38037868"
                        open
                      >
                        <summary class="article-expander__title">Circulating tumor DNA profiling approach based on in silico background elimination guides chemotherapy in nasopharyngeal carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38037868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) analysis increasingly provides a promising minimally invasive alternative to tissue biopsies in precision oncology. However, there are no ctDNA analysis approaches available in nasopharyngeal carcinoma (NPC) and current methods of ctDNA mutation profiling have limited resolution because of the high background noise and false-positive rate caused by benign variants in plasma cell-free DNA (cfDNA), majorly generated during clonal hematopoiesis. Although personalized...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38037800"
                        open
                      >
                        <summary class="article-expander__title">Pathobiology of nodal peripheral T-cell lymphomas: current understanding and future directions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38037800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Predominantly nodal is the most common clinical presentation of peripheral T- (and NK-) cell lymphomas (PTCL), which comprise three main groups of diseases: (i) systemic anaplastic large cell lymphomas (ALCL), whether positive or negative for anaplastic lymphoma kinase (ALK); (ii) follicular helper T-cell lymphomas (TFHL); and (iii) PTCL, not otherwise specified (NOS). Recent advances in the genomic and molecular characterization of PTCL, with enhanced understanding of pathobiology, have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38039845"
                        open
                      >
                        <summary class="article-expander__title">Mechanistic target of rapamycin in regulating macrophage function in inflammatory cardiovascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38039845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mechanistic target of rapamycin (mTOR) is evolutionarily conserved from yeast to humans and is one of the most fundamental pathways of living organisms. Since its discovery three decades ago, mTOR has been recognized as the center of nutrient sensing and growth, homeostasis, metabolism, life span, and aging. The role of dysregulated mTOR in common diseases, especially cancer, has been extensively studied and reported. Emerging evidence supports that mTOR critically regulates innate immune...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38037868"
                        open
                      >
                        <summary class="article-expander__title">Circulating tumor DNA profiling approach based on in silico background elimination guides chemotherapy in nasopharyngeal carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38037868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) analysis increasingly provides a promising minimally invasive alternative to tissue biopsies in precision oncology. However, there are no ctDNA analysis approaches available in nasopharyngeal carcinoma (NPC) and current methods of ctDNA mutation profiling have limited resolution because of the high background noise and false-positive rate caused by benign variants in plasma cell-free DNA (cfDNA), majorly generated during clonal hematopoiesis. Although personalized...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38037800"
                        open
                      >
                        <summary class="article-expander__title">Pathobiology of nodal peripheral T-cell lymphomas: current understanding and future directions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38037800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Predominantly nodal is the most common clinical presentation of peripheral T- (and NK-) cell lymphomas (PTCL), which comprise three main groups of diseases: (i) systemic anaplastic large cell lymphomas (ALCL), whether positive or negative for anaplastic lymphoma kinase (ALK); (ii) follicular helper T-cell lymphomas (TFHL); and (iii) PTCL, not otherwise specified (NOS). Recent advances in the genomic and molecular characterization of PTCL, with enhanced understanding of pathobiology, have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38039845"
                        open
                      >
                        <summary class="article-expander__title">Mechanistic target of rapamycin in regulating macrophage function in inflammatory cardiovascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38039845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mechanistic target of rapamycin (mTOR) is evolutionarily conserved from yeast to humans and is one of the most fundamental pathways of living organisms. Since its discovery three decades ago, mTOR has been recognized as the center of nutrient sensing and growth, homeostasis, metabolism, life span, and aging. The role of dysregulated mTOR in common diseases, especially cancer, has been extensively studied and reported. Emerging evidence supports that mTOR critically regulates innate immune...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38037868"
                        open
                      >
                        <summary class="article-expander__title">Circulating tumor DNA profiling approach based on in silico background elimination guides chemotherapy in nasopharyngeal carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38037868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) analysis increasingly provides a promising minimally invasive alternative to tissue biopsies in precision oncology. However, there are no ctDNA analysis approaches available in nasopharyngeal carcinoma (NPC) and current methods of ctDNA mutation profiling have limited resolution because of the high background noise and false-positive rate caused by benign variants in plasma cell-free DNA (cfDNA), majorly generated during clonal hematopoiesis. Although personalized...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38037800"
                        open
                      >
                        <summary class="article-expander__title">Pathobiology of nodal peripheral T-cell lymphomas: current understanding and future directions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38037800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Predominantly nodal is the most common clinical presentation of peripheral T- (and NK-) cell lymphomas (PTCL), which comprise three main groups of diseases: (i) systemic anaplastic large cell lymphomas (ALCL), whether positive or negative for anaplastic lymphoma kinase (ALK); (ii) follicular helper T-cell lymphomas (TFHL); and (iii) PTCL, not otherwise specified (NOS). Recent advances in the genomic and molecular characterization of PTCL, with enhanced understanding of pathobiology, have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38039845"
                        open
                      >
                        <summary class="article-expander__title">Mechanistic target of rapamycin in regulating macrophage function in inflammatory cardiovascular diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38039845/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mechanistic target of rapamycin (mTOR) is evolutionarily conserved from yeast to humans and is one of the most fundamental pathways of living organisms. Since its discovery three decades ago, mTOR has been recognized as the center of nutrient sensing and growth, homeostasis, metabolism, life span, and aging. The role of dysregulated mTOR in common diseases, especially cancer, has been extensively studied and reported. Emerging evidence supports that mTOR critically regulates innate immune...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38037868"
                        open
                      >
                        <summary class="article-expander__title">Circulating tumor DNA profiling approach based on in silico background elimination guides chemotherapy in nasopharyngeal carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38037868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) analysis increasingly provides a promising minimally invasive alternative to tissue biopsies in precision oncology. However, there are no ctDNA analysis approaches available in nasopharyngeal carcinoma (NPC) and current methods of ctDNA mutation profiling have limited resolution because of the high background noise and false-positive rate caused by benign variants in plasma cell-free DNA (cfDNA), majorly generated during clonal hematopoiesis. Although personalized...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38037800"
                        open
                      >
                        <summary class="article-expander__title">Pathobiology of nodal peripheral T-cell lymphomas: current understanding and future directions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38037800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Predominantly nodal is the most common clinical presentation of peripheral T- (and NK-) cell lymphomas (PTCL), which comprise three main groups of diseases: (i) systemic anaplastic large cell lymphomas (ALCL), whether positive or negative for anaplastic lymphoma kinase (ALK); (ii) follicular helper T-cell lymphomas (TFHL); and (iii) PTCL, not otherwise specified (NOS). Recent advances in the genomic and molecular characterization of PTCL, with enhanced understanding of pathobiology, have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38037868"
                        open
                      >
                        <summary class="article-expander__title">Circulating tumor DNA profiling approach based on in silico background elimination guides chemotherapy in nasopharyngeal carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38037868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) analysis increasingly provides a promising minimally invasive alternative to tissue biopsies in precision oncology. However, there are no ctDNA analysis approaches available in nasopharyngeal carcinoma (NPC) and current methods of ctDNA mutation profiling have limited resolution because of the high background noise and false-positive rate caused by benign variants in plasma cell-free DNA (cfDNA), majorly generated during clonal hematopoiesis. Although personalized...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38037800"
                        open
                      >
                        <summary class="article-expander__title">Pathobiology of nodal peripheral T-cell lymphomas: current understanding and future directions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38037800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Predominantly nodal is the most common clinical presentation of peripheral T- (and NK-) cell lymphomas (PTCL), which comprise three main groups of diseases: (i) systemic anaplastic large cell lymphomas (ALCL), whether positive or negative for anaplastic lymphoma kinase (ALK); (ii) follicular helper T-cell lymphomas (TFHL); and (iii) PTCL, not otherwise specified (NOS). Recent advances in the genomic and molecular characterization of PTCL, with enhanced understanding of pathobiology, have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-29 datetime="2023-11-29T11:00:00.000Z">2023-11-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-29 datetime="2023-11-29T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028538"
                        open
                      >
                        <summary class="article-expander__title">Noncoding rules of survival: epigenetic regulation of normal and malignant hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoiesis is an essential process for organismal development and homeostasis. Epigenetic regulation of gene expression is critical for stem cell self-renewal and differentiation in normal hematopoiesis. Increasing evidence shows that disrupting the balance between self-renewal and cell fate decisions can give rise to hematological diseases such as bone marrow failure and leukemia. Consequently, next-generation sequencing studies have identified various aberrations in histone modifications,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019913"
                        open
                      >
                        <summary class="article-expander__title">ROBO1 deficiency impairs HSPC homeostasis and erythropoiesis via CDC42 and predicts poor prognosis in MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019913/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of clonal hematopoietic neoplasms originating from hematopoietic stem progenitor cells (HSPCs). We previously identified frequent roundabout guidance receptor 1 (ROBO1) mutations in patients with MDS, while the exact role of ROBO1 in hematopoiesis remains poorly delineated. Here, we report that ROBO1 deficiency confers MDS-like disease with anemia and multilineage dysplasia in mice and predicts poor prognosis in patients with MDS. More specifically,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019014"
                        open
                      >
                        <summary class="article-expander__title">Genomic Landscape of Patients with Germline RUNX1 Variants and Familial Platelet Disorder with Myeloid Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019014/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Familial platelet disorder with associated myeloid malignancies (FPDMM) is caused by germline RUNX1 mutations and characterized by thrombocytopenia and increased risk of hematological malignancies. We recently launched a longitudinal natural history study for patients with FPDMM. Among 27 families with research genomic data by the end of 2021, 26 different germline RUNX1 variants were detected. Besides missense mutations enriched in Runt homology domain and loss-of-function mutations distributed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028538"
                        open
                      >
                        <summary class="article-expander__title">Noncoding rules of survival: epigenetic regulation of normal and malignant hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoiesis is an essential process for organismal development and homeostasis. Epigenetic regulation of gene expression is critical for stem cell self-renewal and differentiation in normal hematopoiesis. Increasing evidence shows that disrupting the balance between self-renewal and cell fate decisions can give rise to hematological diseases such as bone marrow failure and leukemia. Consequently, next-generation sequencing studies have identified various aberrations in histone modifications,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019913"
                        open
                      >
                        <summary class="article-expander__title">ROBO1 deficiency impairs HSPC homeostasis and erythropoiesis via CDC42 and predicts poor prognosis in MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019913/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of clonal hematopoietic neoplasms originating from hematopoietic stem progenitor cells (HSPCs). We previously identified frequent roundabout guidance receptor 1 (ROBO1) mutations in patients with MDS, while the exact role of ROBO1 in hematopoiesis remains poorly delineated. Here, we report that ROBO1 deficiency confers MDS-like disease with anemia and multilineage dysplasia in mice and predicts poor prognosis in patients with MDS. More specifically,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019014"
                        open
                      >
                        <summary class="article-expander__title">Genomic Landscape of Patients with Germline RUNX1 Variants and Familial Platelet Disorder with Myeloid Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019014/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Familial platelet disorder with associated myeloid malignancies (FPDMM) is caused by germline RUNX1 mutations and characterized by thrombocytopenia and increased risk of hematological malignancies. We recently launched a longitudinal natural history study for patients with FPDMM. Among 27 families with research genomic data by the end of 2021, 26 different germline RUNX1 variants were detected. Besides missense mutations enriched in Runt homology domain and loss-of-function mutations distributed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028538"
                        open
                      >
                        <summary class="article-expander__title">Noncoding rules of survival: epigenetic regulation of normal and malignant hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoiesis is an essential process for organismal development and homeostasis. Epigenetic regulation of gene expression is critical for stem cell self-renewal and differentiation in normal hematopoiesis. Increasing evidence shows that disrupting the balance between self-renewal and cell fate decisions can give rise to hematological diseases such as bone marrow failure and leukemia. Consequently, next-generation sequencing studies have identified various aberrations in histone modifications,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019913"
                        open
                      >
                        <summary class="article-expander__title">ROBO1 deficiency impairs HSPC homeostasis and erythropoiesis via CDC42 and predicts poor prognosis in MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019913/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of clonal hematopoietic neoplasms originating from hematopoietic stem progenitor cells (HSPCs). We previously identified frequent roundabout guidance receptor 1 (ROBO1) mutations in patients with MDS, while the exact role of ROBO1 in hematopoiesis remains poorly delineated. Here, we report that ROBO1 deficiency confers MDS-like disease with anemia and multilineage dysplasia in mice and predicts poor prognosis in patients with MDS. More specifically,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019014"
                        open
                      >
                        <summary class="article-expander__title">Genomic Landscape of Patients with Germline RUNX1 Variants and Familial Platelet Disorder with Myeloid Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019014/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Familial platelet disorder with associated myeloid malignancies (FPDMM) is caused by germline RUNX1 mutations and characterized by thrombocytopenia and increased risk of hematological malignancies. We recently launched a longitudinal natural history study for patients with FPDMM. Among 27 families with research genomic data by the end of 2021, 26 different germline RUNX1 variants were detected. Besides missense mutations enriched in Runt homology domain and loss-of-function mutations distributed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028538"
                        open
                      >
                        <summary class="article-expander__title">Noncoding rules of survival: epigenetic regulation of normal and malignant hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoiesis is an essential process for organismal development and homeostasis. Epigenetic regulation of gene expression is critical for stem cell self-renewal and differentiation in normal hematopoiesis. Increasing evidence shows that disrupting the balance between self-renewal and cell fate decisions can give rise to hematological diseases such as bone marrow failure and leukemia. Consequently, next-generation sequencing studies have identified various aberrations in histone modifications,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019913"
                        open
                      >
                        <summary class="article-expander__title">ROBO1 deficiency impairs HSPC homeostasis and erythropoiesis via CDC42 and predicts poor prognosis in MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019913/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of clonal hematopoietic neoplasms originating from hematopoietic stem progenitor cells (HSPCs). We previously identified frequent roundabout guidance receptor 1 (ROBO1) mutations in patients with MDS, while the exact role of ROBO1 in hematopoiesis remains poorly delineated. Here, we report that ROBO1 deficiency confers MDS-like disease with anemia and multilineage dysplasia in mice and predicts poor prognosis in patients with MDS. More specifically,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019014"
                        open
                      >
                        <summary class="article-expander__title">Genomic Landscape of Patients with Germline RUNX1 Variants and Familial Platelet Disorder with Myeloid Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019014/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Familial platelet disorder with associated myeloid malignancies (FPDMM) is caused by germline RUNX1 mutations and characterized by thrombocytopenia and increased risk of hematological malignancies. We recently launched a longitudinal natural history study for patients with FPDMM. Among 27 families with research genomic data by the end of 2021, 26 different germline RUNX1 variants were detected. Besides missense mutations enriched in Runt homology domain and loss-of-function mutations distributed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028538"
                        open
                      >
                        <summary class="article-expander__title">Noncoding rules of survival: epigenetic regulation of normal and malignant hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoiesis is an essential process for organismal development and homeostasis. Epigenetic regulation of gene expression is critical for stem cell self-renewal and differentiation in normal hematopoiesis. Increasing evidence shows that disrupting the balance between self-renewal and cell fate decisions can give rise to hematological diseases such as bone marrow failure and leukemia. Consequently, next-generation sequencing studies have identified various aberrations in histone modifications,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019913"
                        open
                      >
                        <summary class="article-expander__title">ROBO1 deficiency impairs HSPC homeostasis and erythropoiesis via CDC42 and predicts poor prognosis in MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019913/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of clonal hematopoietic neoplasms originating from hematopoietic stem progenitor cells (HSPCs). We previously identified frequent roundabout guidance receptor 1 (ROBO1) mutations in patients with MDS, while the exact role of ROBO1 in hematopoiesis remains poorly delineated. Here, we report that ROBO1 deficiency confers MDS-like disease with anemia and multilineage dysplasia in mice and predicts poor prognosis in patients with MDS. More specifically,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019014"
                        open
                      >
                        <summary class="article-expander__title">Genomic Landscape of Patients with Germline RUNX1 Variants and Familial Platelet Disorder with Myeloid Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019014/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Familial platelet disorder with associated myeloid malignancies (FPDMM) is caused by germline RUNX1 mutations and characterized by thrombocytopenia and increased risk of hematological malignancies. We recently launched a longitudinal natural history study for patients with FPDMM. Among 27 families with research genomic data by the end of 2021, 26 different germline RUNX1 variants were detected. Besides missense mutations enriched in Runt homology domain and loss-of-function mutations distributed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028538"
                        open
                      >
                        <summary class="article-expander__title">Noncoding rules of survival: epigenetic regulation of normal and malignant hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoiesis is an essential process for organismal development and homeostasis. Epigenetic regulation of gene expression is critical for stem cell self-renewal and differentiation in normal hematopoiesis. Increasing evidence shows that disrupting the balance between self-renewal and cell fate decisions can give rise to hematological diseases such as bone marrow failure and leukemia. Consequently, next-generation sequencing studies have identified various aberrations in histone modifications,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019913"
                        open
                      >
                        <summary class="article-expander__title">ROBO1 deficiency impairs HSPC homeostasis and erythropoiesis via CDC42 and predicts poor prognosis in MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019913/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of clonal hematopoietic neoplasms originating from hematopoietic stem progenitor cells (HSPCs). We previously identified frequent roundabout guidance receptor 1 (ROBO1) mutations in patients with MDS, while the exact role of ROBO1 in hematopoiesis remains poorly delineated. Here, we report that ROBO1 deficiency confers MDS-like disease with anemia and multilineage dysplasia in mice and predicts poor prognosis in patients with MDS. More specifically,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019014"
                        open
                      >
                        <summary class="article-expander__title">Genomic Landscape of Patients with Germline RUNX1 Variants and Familial Platelet Disorder with Myeloid Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019014/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Familial platelet disorder with associated myeloid malignancies (FPDMM) is caused by germline RUNX1 mutations and characterized by thrombocytopenia and increased risk of hematological malignancies. We recently launched a longitudinal natural history study for patients with FPDMM. Among 27 families with research genomic data by the end of 2021, 26 different germline RUNX1 variants were detected. Besides missense mutations enriched in Runt homology domain and loss-of-function mutations distributed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028538"
                        open
                      >
                        <summary class="article-expander__title">Noncoding rules of survival: epigenetic regulation of normal and malignant hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoiesis is an essential process for organismal development and homeostasis. Epigenetic regulation of gene expression is critical for stem cell self-renewal and differentiation in normal hematopoiesis. Increasing evidence shows that disrupting the balance between self-renewal and cell fate decisions can give rise to hematological diseases such as bone marrow failure and leukemia. Consequently, next-generation sequencing studies have identified various aberrations in histone modifications,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019913"
                        open
                      >
                        <summary class="article-expander__title">ROBO1 deficiency impairs HSPC homeostasis and erythropoiesis via CDC42 and predicts poor prognosis in MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019913/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of clonal hematopoietic neoplasms originating from hematopoietic stem progenitor cells (HSPCs). We previously identified frequent roundabout guidance receptor 1 (ROBO1) mutations in patients with MDS, while the exact role of ROBO1 in hematopoiesis remains poorly delineated. Here, we report that ROBO1 deficiency confers MDS-like disease with anemia and multilineage dysplasia in mice and predicts poor prognosis in patients with MDS. More specifically,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019014"
                        open
                      >
                        <summary class="article-expander__title">Genomic Landscape of Patients with Germline RUNX1 Variants and Familial Platelet Disorder with Myeloid Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019014/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Familial platelet disorder with associated myeloid malignancies (FPDMM) is caused by germline RUNX1 mutations and characterized by thrombocytopenia and increased risk of hematological malignancies. We recently launched a longitudinal natural history study for patients with FPDMM. Among 27 families with research genomic data by the end of 2021, 26 different germline RUNX1 variants were detected. Besides missense mutations enriched in Runt homology domain and loss-of-function mutations distributed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028538"
                        open
                      >
                        <summary class="article-expander__title">Noncoding rules of survival: epigenetic regulation of normal and malignant hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028538/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematopoiesis is an essential process for organismal development and homeostasis. Epigenetic regulation of gene expression is critical for stem cell self-renewal and differentiation in normal hematopoiesis. Increasing evidence shows that disrupting the balance between self-renewal and cell fate decisions can give rise to hematological diseases such as bone marrow failure and leukemia. Consequently, next-generation sequencing studies have identified various aberrations in histone modifications,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019913"
                        open
                      >
                        <summary class="article-expander__title">ROBO1 deficiency impairs HSPC homeostasis and erythropoiesis via CDC42 and predicts poor prognosis in MDS</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019913/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of clonal hematopoietic neoplasms originating from hematopoietic stem progenitor cells (HSPCs). We previously identified frequent roundabout guidance receptor 1 (ROBO1) mutations in patients with MDS, while the exact role of ROBO1 in hematopoiesis remains poorly delineated. Here, we report that ROBO1 deficiency confers MDS-like disease with anemia and multilineage dysplasia in mice and predicts poor prognosis in patients with MDS. More specifically,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019014"
                        open
                      >
                        <summary class="article-expander__title">Genomic Landscape of Patients with Germline RUNX1 Variants and Familial Platelet Disorder with Myeloid Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019014/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Familial platelet disorder with associated myeloid malignancies (FPDMM) is caused by germline RUNX1 mutations and characterized by thrombocytopenia and increased risk of hematological malignancies. We recently launched a longitudinal natural history study for patients with FPDMM. Among 27 families with research genomic data by the end of 2021, 26 different germline RUNX1 variants were detected. Besides missense mutations enriched in Runt homology domain and loss-of-function mutations distributed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38030613"
                        open
                      >
                        <summary class="article-expander__title">Contribution of pks+ E. coli mutations to colorectal carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38030613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The dominant mutational signature in colorectal cancer genomes is C &gt; T deamination (COSMIC Signature 1) and, in a small subgroup, mismatch repair signature (COSMIC signatures 6 and 44). Mutations in common colorectal cancer driver genes are often not consistent with those signatures. Here we perform whole-genome sequencing of normal colon crypts from cancer patients, matched to a previous multi-omic tumour dataset. We analyse normal crypts that were distant vs adjacent to the cancer. In...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38022603"
                        open
                      >
                        <summary class="article-expander__title">Modeling the crosstalk between malignant B cells and their microenvironment in B-cell lymphomas: challenges and opportunities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38022603/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell lymphomas are a group of heterogeneous neoplasms resulting from the clonal expansion of mature B cells arrested at various stages of differentiation. Specifically, two lymphoma subtypes arise from germinal centers (GCs), namely follicular lymphoma (FL) and GC B-cell diffuse large B-cell lymphoma (GCB-DLBCL). In addition to recent advances in describing the genetic landscape of FL and GCB-DLBCL, tumor microenvironment (TME) has progressively emerged as a central determinant of early...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018058"
                        open
                      >
                        <summary class="article-expander__title">Research status and challenges of advanced myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018058/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPN), also known as Ph^(-)MPN, includes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF). Secondary myelofibrosis (sMF) and secondary acute myeloid leukemia (sAML) are important disease progressions of MPN. After MPN disease progression, hematopoietic stem cells undergo new clonal evolution, leading to drug resistance, poor treatment effect and poor survival of patients. In recent years, the exploration of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017473"
                        open
                      >
                        <summary class="article-expander__title">Elucidating the clonal relationship of esophageal second primary tumors in patients with laryngeal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017473/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Laryngeal cancer ranks as the second most prevalent upper airway malignancy, following Lung cancer. Although some progress has been made in managing laryngeal cancer, the 5-year survival rate is disappointing. The gradual increase in the incidence of second primary tumors (SPTs) plays a crucial role in determining survival outcomes during long-term follow-up, and the esophagus was the most common site with a worse prognosis. In clinical practice, the treatment of esophageal second primary tumors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38030613"
                        open
                      >
                        <summary class="article-expander__title">Contribution of pks+ E. coli mutations to colorectal carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38030613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The dominant mutational signature in colorectal cancer genomes is C &gt; T deamination (COSMIC Signature 1) and, in a small subgroup, mismatch repair signature (COSMIC signatures 6 and 44). Mutations in common colorectal cancer driver genes are often not consistent with those signatures. Here we perform whole-genome sequencing of normal colon crypts from cancer patients, matched to a previous multi-omic tumour dataset. We analyse normal crypts that were distant vs adjacent to the cancer. In...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38022603"
                        open
                      >
                        <summary class="article-expander__title">Modeling the crosstalk between malignant B cells and their microenvironment in B-cell lymphomas: challenges and opportunities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38022603/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell lymphomas are a group of heterogeneous neoplasms resulting from the clonal expansion of mature B cells arrested at various stages of differentiation. Specifically, two lymphoma subtypes arise from germinal centers (GCs), namely follicular lymphoma (FL) and GC B-cell diffuse large B-cell lymphoma (GCB-DLBCL). In addition to recent advances in describing the genetic landscape of FL and GCB-DLBCL, tumor microenvironment (TME) has progressively emerged as a central determinant of early...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018058"
                        open
                      >
                        <summary class="article-expander__title">Research status and challenges of advanced myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018058/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPN), also known as Ph^(-)MPN, includes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF). Secondary myelofibrosis (sMF) and secondary acute myeloid leukemia (sAML) are important disease progressions of MPN. After MPN disease progression, hematopoietic stem cells undergo new clonal evolution, leading to drug resistance, poor treatment effect and poor survival of patients. In recent years, the exploration of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017473"
                        open
                      >
                        <summary class="article-expander__title">Elucidating the clonal relationship of esophageal second primary tumors in patients with laryngeal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017473/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Laryngeal cancer ranks as the second most prevalent upper airway malignancy, following Lung cancer. Although some progress has been made in managing laryngeal cancer, the 5-year survival rate is disappointing. The gradual increase in the incidence of second primary tumors (SPTs) plays a crucial role in determining survival outcomes during long-term follow-up, and the esophagus was the most common site with a worse prognosis. In clinical practice, the treatment of esophageal second primary tumors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38030613"
                        open
                      >
                        <summary class="article-expander__title">Contribution of pks+ E. coli mutations to colorectal carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38030613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The dominant mutational signature in colorectal cancer genomes is C &gt; T deamination (COSMIC Signature 1) and, in a small subgroup, mismatch repair signature (COSMIC signatures 6 and 44). Mutations in common colorectal cancer driver genes are often not consistent with those signatures. Here we perform whole-genome sequencing of normal colon crypts from cancer patients, matched to a previous multi-omic tumour dataset. We analyse normal crypts that were distant vs adjacent to the cancer. In...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38022603"
                        open
                      >
                        <summary class="article-expander__title">Modeling the crosstalk between malignant B cells and their microenvironment in B-cell lymphomas: challenges and opportunities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38022603/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell lymphomas are a group of heterogeneous neoplasms resulting from the clonal expansion of mature B cells arrested at various stages of differentiation. Specifically, two lymphoma subtypes arise from germinal centers (GCs), namely follicular lymphoma (FL) and GC B-cell diffuse large B-cell lymphoma (GCB-DLBCL). In addition to recent advances in describing the genetic landscape of FL and GCB-DLBCL, tumor microenvironment (TME) has progressively emerged as a central determinant of early...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018058"
                        open
                      >
                        <summary class="article-expander__title">Research status and challenges of advanced myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018058/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPN), also known as Ph^(-)MPN, includes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF). Secondary myelofibrosis (sMF) and secondary acute myeloid leukemia (sAML) are important disease progressions of MPN. After MPN disease progression, hematopoietic stem cells undergo new clonal evolution, leading to drug resistance, poor treatment effect and poor survival of patients. In recent years, the exploration of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017473"
                        open
                      >
                        <summary class="article-expander__title">Elucidating the clonal relationship of esophageal second primary tumors in patients with laryngeal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017473/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Laryngeal cancer ranks as the second most prevalent upper airway malignancy, following Lung cancer. Although some progress has been made in managing laryngeal cancer, the 5-year survival rate is disappointing. The gradual increase in the incidence of second primary tumors (SPTs) plays a crucial role in determining survival outcomes during long-term follow-up, and the esophagus was the most common site with a worse prognosis. In clinical practice, the treatment of esophageal second primary tumors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38030613"
                        open
                      >
                        <summary class="article-expander__title">Contribution of pks+ E. coli mutations to colorectal carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38030613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The dominant mutational signature in colorectal cancer genomes is C &gt; T deamination (COSMIC Signature 1) and, in a small subgroup, mismatch repair signature (COSMIC signatures 6 and 44). Mutations in common colorectal cancer driver genes are often not consistent with those signatures. Here we perform whole-genome sequencing of normal colon crypts from cancer patients, matched to a previous multi-omic tumour dataset. We analyse normal crypts that were distant vs adjacent to the cancer. In...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38022603"
                        open
                      >
                        <summary class="article-expander__title">Modeling the crosstalk between malignant B cells and their microenvironment in B-cell lymphomas: challenges and opportunities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38022603/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell lymphomas are a group of heterogeneous neoplasms resulting from the clonal expansion of mature B cells arrested at various stages of differentiation. Specifically, two lymphoma subtypes arise from germinal centers (GCs), namely follicular lymphoma (FL) and GC B-cell diffuse large B-cell lymphoma (GCB-DLBCL). In addition to recent advances in describing the genetic landscape of FL and GCB-DLBCL, tumor microenvironment (TME) has progressively emerged as a central determinant of early...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018058"
                        open
                      >
                        <summary class="article-expander__title">Research status and challenges of advanced myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018058/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPN), also known as Ph^(-)MPN, includes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF). Secondary myelofibrosis (sMF) and secondary acute myeloid leukemia (sAML) are important disease progressions of MPN. After MPN disease progression, hematopoietic stem cells undergo new clonal evolution, leading to drug resistance, poor treatment effect and poor survival of patients. In recent years, the exploration of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017473"
                        open
                      >
                        <summary class="article-expander__title">Elucidating the clonal relationship of esophageal second primary tumors in patients with laryngeal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017473/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Laryngeal cancer ranks as the second most prevalent upper airway malignancy, following Lung cancer. Although some progress has been made in managing laryngeal cancer, the 5-year survival rate is disappointing. The gradual increase in the incidence of second primary tumors (SPTs) plays a crucial role in determining survival outcomes during long-term follow-up, and the esophagus was the most common site with a worse prognosis. In clinical practice, the treatment of esophageal second primary tumors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38030613"
                        open
                      >
                        <summary class="article-expander__title">Contribution of pks+ E. coli mutations to colorectal carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38030613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The dominant mutational signature in colorectal cancer genomes is C &gt; T deamination (COSMIC Signature 1) and, in a small subgroup, mismatch repair signature (COSMIC signatures 6 and 44). Mutations in common colorectal cancer driver genes are often not consistent with those signatures. Here we perform whole-genome sequencing of normal colon crypts from cancer patients, matched to a previous multi-omic tumour dataset. We analyse normal crypts that were distant vs adjacent to the cancer. In...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38022603"
                        open
                      >
                        <summary class="article-expander__title">Modeling the crosstalk between malignant B cells and their microenvironment in B-cell lymphomas: challenges and opportunities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38022603/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell lymphomas are a group of heterogeneous neoplasms resulting from the clonal expansion of mature B cells arrested at various stages of differentiation. Specifically, two lymphoma subtypes arise from germinal centers (GCs), namely follicular lymphoma (FL) and GC B-cell diffuse large B-cell lymphoma (GCB-DLBCL). In addition to recent advances in describing the genetic landscape of FL and GCB-DLBCL, tumor microenvironment (TME) has progressively emerged as a central determinant of early...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018058"
                        open
                      >
                        <summary class="article-expander__title">Research status and challenges of advanced myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018058/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPN), also known as Ph^(-)MPN, includes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF). Secondary myelofibrosis (sMF) and secondary acute myeloid leukemia (sAML) are important disease progressions of MPN. After MPN disease progression, hematopoietic stem cells undergo new clonal evolution, leading to drug resistance, poor treatment effect and poor survival of patients. In recent years, the exploration of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017473"
                        open
                      >
                        <summary class="article-expander__title">Elucidating the clonal relationship of esophageal second primary tumors in patients with laryngeal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017473/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Laryngeal cancer ranks as the second most prevalent upper airway malignancy, following Lung cancer. Although some progress has been made in managing laryngeal cancer, the 5-year survival rate is disappointing. The gradual increase in the incidence of second primary tumors (SPTs) plays a crucial role in determining survival outcomes during long-term follow-up, and the esophagus was the most common site with a worse prognosis. In clinical practice, the treatment of esophageal second primary tumors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38030613"
                        open
                      >
                        <summary class="article-expander__title">Contribution of pks+ E. coli mutations to colorectal carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38030613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The dominant mutational signature in colorectal cancer genomes is C &gt; T deamination (COSMIC Signature 1) and, in a small subgroup, mismatch repair signature (COSMIC signatures 6 and 44). Mutations in common colorectal cancer driver genes are often not consistent with those signatures. Here we perform whole-genome sequencing of normal colon crypts from cancer patients, matched to a previous multi-omic tumour dataset. We analyse normal crypts that were distant vs adjacent to the cancer. In...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38022603"
                        open
                      >
                        <summary class="article-expander__title">Modeling the crosstalk between malignant B cells and their microenvironment in B-cell lymphomas: challenges and opportunities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38022603/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell lymphomas are a group of heterogeneous neoplasms resulting from the clonal expansion of mature B cells arrested at various stages of differentiation. Specifically, two lymphoma subtypes arise from germinal centers (GCs), namely follicular lymphoma (FL) and GC B-cell diffuse large B-cell lymphoma (GCB-DLBCL). In addition to recent advances in describing the genetic landscape of FL and GCB-DLBCL, tumor microenvironment (TME) has progressively emerged as a central determinant of early...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018058"
                        open
                      >
                        <summary class="article-expander__title">Research status and challenges of advanced myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018058/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPN), also known as Ph^(-)MPN, includes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF). Secondary myelofibrosis (sMF) and secondary acute myeloid leukemia (sAML) are important disease progressions of MPN. After MPN disease progression, hematopoietic stem cells undergo new clonal evolution, leading to drug resistance, poor treatment effect and poor survival of patients. In recent years, the exploration of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017473"
                        open
                      >
                        <summary class="article-expander__title">Elucidating the clonal relationship of esophageal second primary tumors in patients with laryngeal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017473/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Laryngeal cancer ranks as the second most prevalent upper airway malignancy, following Lung cancer. Although some progress has been made in managing laryngeal cancer, the 5-year survival rate is disappointing. The gradual increase in the incidence of second primary tumors (SPTs) plays a crucial role in determining survival outcomes during long-term follow-up, and the esophagus was the most common site with a worse prognosis. In clinical practice, the treatment of esophageal second primary tumors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38030613"
                        open
                      >
                        <summary class="article-expander__title">Contribution of pks+ E. coli mutations to colorectal carcinogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38030613/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The dominant mutational signature in colorectal cancer genomes is C &gt; T deamination (COSMIC Signature 1) and, in a small subgroup, mismatch repair signature (COSMIC signatures 6 and 44). Mutations in common colorectal cancer driver genes are often not consistent with those signatures. Here we perform whole-genome sequencing of normal colon crypts from cancer patients, matched to a previous multi-omic tumour dataset. We analyse normal crypts that were distant vs adjacent to the cancer. In...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38022603"
                        open
                      >
                        <summary class="article-expander__title">Modeling the crosstalk between malignant B cells and their microenvironment in B-cell lymphomas: challenges and opportunities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38022603/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell lymphomas are a group of heterogeneous neoplasms resulting from the clonal expansion of mature B cells arrested at various stages of differentiation. Specifically, two lymphoma subtypes arise from germinal centers (GCs), namely follicular lymphoma (FL) and GC B-cell diffuse large B-cell lymphoma (GCB-DLBCL). In addition to recent advances in describing the genetic landscape of FL and GCB-DLBCL, tumor microenvironment (TME) has progressively emerged as a central determinant of early...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018058"
                        open
                      >
                        <summary class="article-expander__title">Research status and challenges of advanced myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018058/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPN), also known as Ph^(-)MPN, includes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF). Secondary myelofibrosis (sMF) and secondary acute myeloid leukemia (sAML) are important disease progressions of MPN. After MPN disease progression, hematopoietic stem cells undergo new clonal evolution, leading to drug resistance, poor treatment effect and poor survival of patients. In recent years, the exploration of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017473"
                        open
                      >
                        <summary class="article-expander__title">Elucidating the clonal relationship of esophageal second primary tumors in patients with laryngeal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017473/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Laryngeal cancer ranks as the second most prevalent upper airway malignancy, following Lung cancer. Although some progress has been made in managing laryngeal cancer, the 5-year survival rate is disappointing. The gradual increase in the incidence of second primary tumors (SPTs) plays a crucial role in determining survival outcomes during long-term follow-up, and the esophagus was the most common site with a worse prognosis. In clinical practice, the treatment of esophageal second primary tumors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38028397"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Myeloid Neoplasms: Predisposition and Clonal Evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38028397/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related Myeloid Neoplasm (t-MN) represents one of the worst long-term consequences of cytotoxic therapy for primary tumors and autoimmune disease. Poor survival and refractoriness to current treatment strategies characterize affected patients from a clinical point of view. In our aging societies, where newer therapies and ameliorated cancer management protocols are improving the life expectancy of cancer patients, therapy-related Myeloid Neoplasms are an emerging problem. Although...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38022603"
                        open
                      >
                        <summary class="article-expander__title">Modeling the crosstalk between malignant B cells and their microenvironment in B-cell lymphomas: challenges and opportunities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38022603/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell lymphomas are a group of heterogeneous neoplasms resulting from the clonal expansion of mature B cells arrested at various stages of differentiation. Specifically, two lymphoma subtypes arise from germinal centers (GCs), namely follicular lymphoma (FL) and GC B-cell diffuse large B-cell lymphoma (GCB-DLBCL). In addition to recent advances in describing the genetic landscape of FL and GCB-DLBCL, tumor microenvironment (TME) has progressively emerged as a central determinant of early...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38019104"
                        open
                      >
                        <summary class="article-expander__title">Clinical outcomes and evolution of clonal hematopoiesis in patients with newly diagnosed multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38019104/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) at time of autologous stem cell transplant (ASCT), has been shown to be associated with decreased overall survival (OS) and progression-free survival (PFS) in multiple myeloma (MM) patients not receiving immunomodulatory drugs (IMiDs). However, the significance of CH in newly diagnosed patients, including transplant-ineligible patients, and its effect on clonal evolution during MM therapy, has not been well studied. Using our new algorithm to differentiate tumor and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018059"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and its evolution of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018059/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutations of myeloproliferative neoplasma (MPN) mainly include driver mutations and non-driver mutations. The driver mutations mainly include JAK2 mutations, CALR mutations and MPL mutations and non-driver mutations mainly include ASXL1, DNMT3A, TET2, SF3B1, EZH2, TP53, SRSF2, USAF1, etc. Driver and non-driver mutations and their clonal evolution affect the thrombosis and disease transformation of MPN. Clonal hematopoiesis of MPN can occur decades before diagnosis, even in the fetal stage....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38018058"
                        open
                      >
                        <summary class="article-expander__title">Research status and challenges of advanced myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38018058/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Classical myeloproliferative neoplasms (MPN), also known as Ph^(-)MPN, includes polycythemia vera (PV), essential thrombocythemia (ET) and primary myelofibrosis (PMF). Secondary myelofibrosis (sMF) and secondary acute myeloid leukemia (sAML) are important disease progressions of MPN. After MPN disease progression, hematopoietic stem cells undergo new clonal evolution, leading to drug resistance, poor treatment effect and poor survival of patients. In recent years, the exploration of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017473"
                        open
                      >
                        <summary class="article-expander__title">Elucidating the clonal relationship of esophageal second primary tumors in patients with laryngeal squamous cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017473/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Laryngeal cancer ranks as the second most prevalent upper airway malignancy, following Lung cancer. Although some progress has been made in managing laryngeal cancer, the 5-year survival rate is disappointing. The gradual increase in the incidence of second primary tumors (SPTs) plays a crucial role in determining survival outcomes during long-term follow-up, and the esophagus was the most common site with a worse prognosis. In clinical practice, the treatment of esophageal second primary tumors...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-28 datetime="2023-11-28T11:00:00.000Z">2023-11-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-28 datetime="2023-11-28T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38016423"
                        open
                      >
                        <summary class="article-expander__title">Cytogenetics in the management of clonal chromosomal abnormalities of undetermined significance and persistent polyclonal B-cell lymphocytosis: Guidelines from the Groupe Francophone de Cytogénétique Hématologique (GFCH)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38016423/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired clonal chromosomal abnormalities (CAs) are usually considered to be disease-related. However, when a CA of this type is the only abnormality present (and especially in small clones), the clinical significance is unclear. Here, we review the literature on recurrent CAs whose significance is regularly subject to debate. Our objective was to help with their interpretation and develop guidelines for sex chromosome loss, trisomy 15, trisomy 8, deletion 20q and other isolated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38016423"
                        open
                      >
                        <summary class="article-expander__title">Cytogenetics in the management of clonal chromosomal abnormalities of undetermined significance and persistent polyclonal B-cell lymphocytosis: Guidelines from the Groupe Francophone de Cytogénétique Hématologique (GFCH)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38016423/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired clonal chromosomal abnormalities (CAs) are usually considered to be disease-related. However, when a CA of this type is the only abnormality present (and especially in small clones), the clinical significance is unclear. Here, we review the literature on recurrent CAs whose significance is regularly subject to debate. Our objective was to help with their interpretation and develop guidelines for sex chromosome loss, trisomy 15, trisomy 8, deletion 20q and other isolated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38016423"
                        open
                      >
                        <summary class="article-expander__title">Cytogenetics in the management of clonal chromosomal abnormalities of undetermined significance and persistent polyclonal B-cell lymphocytosis: Guidelines from the Groupe Francophone de Cytogénétique Hématologique (GFCH)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38016423/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired clonal chromosomal abnormalities (CAs) are usually considered to be disease-related. However, when a CA of this type is the only abnormality present (and especially in small clones), the clinical significance is unclear. Here, we review the literature on recurrent CAs whose significance is regularly subject to debate. Our objective was to help with their interpretation and develop guidelines for sex chromosome loss, trisomy 15, trisomy 8, deletion 20q and other isolated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38016423"
                        open
                      >
                        <summary class="article-expander__title">Cytogenetics in the management of clonal chromosomal abnormalities of undetermined significance and persistent polyclonal B-cell lymphocytosis: Guidelines from the Groupe Francophone de Cytogénétique Hématologique (GFCH)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38016423/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired clonal chromosomal abnormalities (CAs) are usually considered to be disease-related. However, when a CA of this type is the only abnormality present (and especially in small clones), the clinical significance is unclear. Here, we review the literature on recurrent CAs whose significance is regularly subject to debate. Our objective was to help with their interpretation and develop guidelines for sex chromosome loss, trisomy 15, trisomy 8, deletion 20q and other isolated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38016423"
                        open
                      >
                        <summary class="article-expander__title">Cytogenetics in the management of clonal chromosomal abnormalities of undetermined significance and persistent polyclonal B-cell lymphocytosis: Guidelines from the Groupe Francophone de Cytogénétique Hématologique (GFCH)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38016423/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired clonal chromosomal abnormalities (CAs) are usually considered to be disease-related. However, when a CA of this type is the only abnormality present (and especially in small clones), the clinical significance is unclear. Here, we review the literature on recurrent CAs whose significance is regularly subject to debate. Our objective was to help with their interpretation and develop guidelines for sex chromosome loss, trisomy 15, trisomy 8, deletion 20q and other isolated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38016423"
                        open
                      >
                        <summary class="article-expander__title">Cytogenetics in the management of clonal chromosomal abnormalities of undetermined significance and persistent polyclonal B-cell lymphocytosis: Guidelines from the Groupe Francophone de Cytogénétique Hématologique (GFCH)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38016423/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired clonal chromosomal abnormalities (CAs) are usually considered to be disease-related. However, when a CA of this type is the only abnormality present (and especially in small clones), the clinical significance is unclear. Here, we review the literature on recurrent CAs whose significance is regularly subject to debate. Our objective was to help with their interpretation and develop guidelines for sex chromosome loss, trisomy 15, trisomy 8, deletion 20q and other isolated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38016423"
                        open
                      >
                        <summary class="article-expander__title">Cytogenetics in the management of clonal chromosomal abnormalities of undetermined significance and persistent polyclonal B-cell lymphocytosis: Guidelines from the Groupe Francophone de Cytogénétique Hématologique (GFCH)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38016423/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired clonal chromosomal abnormalities (CAs) are usually considered to be disease-related. However, when a CA of this type is the only abnormality present (and especially in small clones), the clinical significance is unclear. Here, we review the literature on recurrent CAs whose significance is regularly subject to debate. Our objective was to help with their interpretation and develop guidelines for sex chromosome loss, trisomy 15, trisomy 8, deletion 20q and other isolated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38016423"
                        open
                      >
                        <summary class="article-expander__title">Cytogenetics in the management of clonal chromosomal abnormalities of undetermined significance and persistent polyclonal B-cell lymphocytosis: Guidelines from the Groupe Francophone de Cytogénétique Hématologique (GFCH)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38016423/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired clonal chromosomal abnormalities (CAs) are usually considered to be disease-related. However, when a CA of this type is the only abnormality present (and especially in small clones), the clinical significance is unclear. Here, we review the literature on recurrent CAs whose significance is regularly subject to debate. Our objective was to help with their interpretation and develop guidelines for sex chromosome loss, trisomy 15, trisomy 8, deletion 20q and other isolated...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017136"
                        open
                      >
                        <summary class="article-expander__title">CONIPHER: a computational framework for scalable phylogenetic reconstruction with error correction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017136/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity provides the fuel for the evolution and selection of subclonal tumor cell populations. However, accurate inference of tumor subclonal architecture and reconstruction of tumor evolutionary histories from bulk DNA sequencing data remains challenging. Frequently, sequencing and alignment artifacts are not fully filtered out from cancer somatic mutations, and errors in the identification of copy number alterations or complex evolutionary events (e.g., mutation losses) affect...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017105"
                        open
                      >
                        <summary class="article-expander__title">Chronic lymphocytic leukemia patient-derived xenografts recapitulate clonal evolution to Richter transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic lymphocytic leukemia (CLL) is a B-cell neoplasm with a heterogeneous clinical behavior. In 5-10% of patients the disease transforms into a diffuse large-B cell lymphoma known as Richter transformation (RT), which is associated with dismal prognosis. Here, we aimed to establish patient-derived xenograft (PDX) models to study the molecular features and evolution of CLL and RT. We generated two PDXs by injecting CLL (PDX12) and RT (PDX19) cells into immunocompromised NSG mice. Both PDXs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012682"
                        open
                      >
                        <summary class="article-expander__title">Long-term decitabine/retinoic acid maintenance treatment in an elderly sAML patient with high-risk genetics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231206195312&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Elderly patients with AML ineligible for induction have a dismal prognosis; hence disease stabilization is a primary treatment goal. This case of a 75-year-old patient with secondary AML receiving the combination of decitabine and ATRA (within the DECIDER trial, NCT00867672) demonstrates an above-average survival. The therapy administered over 52 cycles led to complete molecular and hematological remission and resulted in 5.3 years overall survival. Clonal evolution of the leukemic clone could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017136"
                        open
                      >
                        <summary class="article-expander__title">CONIPHER: a computational framework for scalable phylogenetic reconstruction with error correction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017136/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity provides the fuel for the evolution and selection of subclonal tumor cell populations. However, accurate inference of tumor subclonal architecture and reconstruction of tumor evolutionary histories from bulk DNA sequencing data remains challenging. Frequently, sequencing and alignment artifacts are not fully filtered out from cancer somatic mutations, and errors in the identification of copy number alterations or complex evolutionary events (e.g., mutation losses) affect...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017105"
                        open
                      >
                        <summary class="article-expander__title">Chronic lymphocytic leukemia patient-derived xenografts recapitulate clonal evolution to Richter transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic lymphocytic leukemia (CLL) is a B-cell neoplasm with a heterogeneous clinical behavior. In 5-10% of patients the disease transforms into a diffuse large-B cell lymphoma known as Richter transformation (RT), which is associated with dismal prognosis. Here, we aimed to establish patient-derived xenograft (PDX) models to study the molecular features and evolution of CLL and RT. We generated two PDXs by injecting CLL (PDX12) and RT (PDX19) cells into immunocompromised NSG mice. Both PDXs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012682"
                        open
                      >
                        <summary class="article-expander__title">Long-term decitabine/retinoic acid maintenance treatment in an elderly sAML patient with high-risk genetics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Elderly patients with AML ineligible for induction have a dismal prognosis; hence disease stabilization is a primary treatment goal. This case of a 75-year-old patient with secondary AML receiving the combination of decitabine and ATRA (within the DECIDER trial, NCT00867672) demonstrates an above-average survival. The therapy administered over 52 cycles led to complete molecular and hematological remission and resulted in 5.3 years overall survival. Clonal evolution of the leukemic clone could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017136"
                        open
                      >
                        <summary class="article-expander__title">CONIPHER: a computational framework for scalable phylogenetic reconstruction with error correction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017136/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity provides the fuel for the evolution and selection of subclonal tumor cell populations. However, accurate inference of tumor subclonal architecture and reconstruction of tumor evolutionary histories from bulk DNA sequencing data remains challenging. Frequently, sequencing and alignment artifacts are not fully filtered out from cancer somatic mutations, and errors in the identification of copy number alterations or complex evolutionary events (e.g., mutation losses) affect...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017105"
                        open
                      >
                        <summary class="article-expander__title">Chronic lymphocytic leukemia patient-derived xenografts recapitulate clonal evolution to Richter transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic lymphocytic leukemia (CLL) is a B-cell neoplasm with a heterogeneous clinical behavior. In 5-10% of patients the disease transforms into a diffuse large-B cell lymphoma known as Richter transformation (RT), which is associated with dismal prognosis. Here, we aimed to establish patient-derived xenograft (PDX) models to study the molecular features and evolution of CLL and RT. We generated two PDXs by injecting CLL (PDX12) and RT (PDX19) cells into immunocompromised NSG mice. Both PDXs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012682"
                        open
                      >
                        <summary class="article-expander__title">Long-term decitabine/retinoic acid maintenance treatment in an elderly sAML patient with high-risk genetics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Elderly patients with AML ineligible for induction have a dismal prognosis; hence disease stabilization is a primary treatment goal. This case of a 75-year-old patient with secondary AML receiving the combination of decitabine and ATRA (within the DECIDER trial, NCT00867672) demonstrates an above-average survival. The therapy administered over 52 cycles led to complete molecular and hematological remission and resulted in 5.3 years overall survival. Clonal evolution of the leukemic clone could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017136"
                        open
                      >
                        <summary class="article-expander__title">CONIPHER: a computational framework for scalable phylogenetic reconstruction with error correction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017136/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity provides the fuel for the evolution and selection of subclonal tumor cell populations. However, accurate inference of tumor subclonal architecture and reconstruction of tumor evolutionary histories from bulk DNA sequencing data remains challenging. Frequently, sequencing and alignment artifacts are not fully filtered out from cancer somatic mutations, and errors in the identification of copy number alterations or complex evolutionary events (e.g., mutation losses) affect...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017105"
                        open
                      >
                        <summary class="article-expander__title">Chronic lymphocytic leukemia patient-derived xenografts recapitulate clonal evolution to Richter transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic lymphocytic leukemia (CLL) is a B-cell neoplasm with a heterogeneous clinical behavior. In 5-10% of patients the disease transforms into a diffuse large-B cell lymphoma known as Richter transformation (RT), which is associated with dismal prognosis. Here, we aimed to establish patient-derived xenograft (PDX) models to study the molecular features and evolution of CLL and RT. We generated two PDXs by injecting CLL (PDX12) and RT (PDX19) cells into immunocompromised NSG mice. Both PDXs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012682"
                        open
                      >
                        <summary class="article-expander__title">Long-term decitabine/retinoic acid maintenance treatment in an elderly sAML patient with high-risk genetics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Elderly patients with AML ineligible for induction have a dismal prognosis; hence disease stabilization is a primary treatment goal. This case of a 75-year-old patient with secondary AML receiving the combination of decitabine and ATRA (within the DECIDER trial, NCT00867672) demonstrates an above-average survival. The therapy administered over 52 cycles led to complete molecular and hematological remission and resulted in 5.3 years overall survival. Clonal evolution of the leukemic clone could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017136"
                        open
                      >
                        <summary class="article-expander__title">CONIPHER: a computational framework for scalable phylogenetic reconstruction with error correction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017136/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity provides the fuel for the evolution and selection of subclonal tumor cell populations. However, accurate inference of tumor subclonal architecture and reconstruction of tumor evolutionary histories from bulk DNA sequencing data remains challenging. Frequently, sequencing and alignment artifacts are not fully filtered out from cancer somatic mutations, and errors in the identification of copy number alterations or complex evolutionary events (e.g., mutation losses) affect...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017105"
                        open
                      >
                        <summary class="article-expander__title">Chronic lymphocytic leukemia patient-derived xenografts recapitulate clonal evolution to Richter transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic lymphocytic leukemia (CLL) is a B-cell neoplasm with a heterogeneous clinical behavior. In 5-10% of patients the disease transforms into a diffuse large-B cell lymphoma known as Richter transformation (RT), which is associated with dismal prognosis. Here, we aimed to establish patient-derived xenograft (PDX) models to study the molecular features and evolution of CLL and RT. We generated two PDXs by injecting CLL (PDX12) and RT (PDX19) cells into immunocompromised NSG mice. Both PDXs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012682"
                        open
                      >
                        <summary class="article-expander__title">Long-term decitabine/retinoic acid maintenance treatment in an elderly sAML patient with high-risk genetics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Elderly patients with AML ineligible for induction have a dismal prognosis; hence disease stabilization is a primary treatment goal. This case of a 75-year-old patient with secondary AML receiving the combination of decitabine and ATRA (within the DECIDER trial, NCT00867672) demonstrates an above-average survival. The therapy administered over 52 cycles led to complete molecular and hematological remission and resulted in 5.3 years overall survival. Clonal evolution of the leukemic clone could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017136"
                        open
                      >
                        <summary class="article-expander__title">CONIPHER: a computational framework for scalable phylogenetic reconstruction with error correction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017136/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity provides the fuel for the evolution and selection of subclonal tumor cell populations. However, accurate inference of tumor subclonal architecture and reconstruction of tumor evolutionary histories from bulk DNA sequencing data remains challenging. Frequently, sequencing and alignment artifacts are not fully filtered out from cancer somatic mutations, and errors in the identification of copy number alterations or complex evolutionary events (e.g., mutation losses) affect...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017105"
                        open
                      >
                        <summary class="article-expander__title">Chronic lymphocytic leukemia patient-derived xenografts recapitulate clonal evolution to Richter transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic lymphocytic leukemia (CLL) is a B-cell neoplasm with a heterogeneous clinical behavior. In 5-10% of patients the disease transforms into a diffuse large-B cell lymphoma known as Richter transformation (RT), which is associated with dismal prognosis. Here, we aimed to establish patient-derived xenograft (PDX) models to study the molecular features and evolution of CLL and RT. We generated two PDXs by injecting CLL (PDX12) and RT (PDX19) cells into immunocompromised NSG mice. Both PDXs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012682"
                        open
                      >
                        <summary class="article-expander__title">Long-term decitabine/retinoic acid maintenance treatment in an elderly sAML patient with high-risk genetics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Elderly patients with AML ineligible for induction have a dismal prognosis; hence disease stabilization is a primary treatment goal. This case of a 75-year-old patient with secondary AML receiving the combination of decitabine and ATRA (within the DECIDER trial, NCT00867672) demonstrates an above-average survival. The therapy administered over 52 cycles led to complete molecular and hematological remission and resulted in 5.3 years overall survival. Clonal evolution of the leukemic clone could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017136"
                        open
                      >
                        <summary class="article-expander__title">CONIPHER: a computational framework for scalable phylogenetic reconstruction with error correction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017136/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity provides the fuel for the evolution and selection of subclonal tumor cell populations. However, accurate inference of tumor subclonal architecture and reconstruction of tumor evolutionary histories from bulk DNA sequencing data remains challenging. Frequently, sequencing and alignment artifacts are not fully filtered out from cancer somatic mutations, and errors in the identification of copy number alterations or complex evolutionary events (e.g., mutation losses) affect...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017105"
                        open
                      >
                        <summary class="article-expander__title">Chronic lymphocytic leukemia patient-derived xenografts recapitulate clonal evolution to Richter transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic lymphocytic leukemia (CLL) is a B-cell neoplasm with a heterogeneous clinical behavior. In 5-10% of patients the disease transforms into a diffuse large-B cell lymphoma known as Richter transformation (RT), which is associated with dismal prognosis. Here, we aimed to establish patient-derived xenograft (PDX) models to study the molecular features and evolution of CLL and RT. We generated two PDXs by injecting CLL (PDX12) and RT (PDX19) cells into immunocompromised NSG mice. Both PDXs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012682"
                        open
                      >
                        <summary class="article-expander__title">Long-term decitabine/retinoic acid maintenance treatment in an elderly sAML patient with high-risk genetics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Elderly patients with AML ineligible for induction have a dismal prognosis; hence disease stabilization is a primary treatment goal. This case of a 75-year-old patient with secondary AML receiving the combination of decitabine and ATRA (within the DECIDER trial, NCT00867672) demonstrates an above-average survival. The therapy administered over 52 cycles led to complete molecular and hematological remission and resulted in 5.3 years overall survival. Clonal evolution of the leukemic clone could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017136"
                        open
                      >
                        <summary class="article-expander__title">CONIPHER: a computational framework for scalable phylogenetic reconstruction with error correction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017136/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumor heterogeneity provides the fuel for the evolution and selection of subclonal tumor cell populations. However, accurate inference of tumor subclonal architecture and reconstruction of tumor evolutionary histories from bulk DNA sequencing data remains challenging. Frequently, sequencing and alignment artifacts are not fully filtered out from cancer somatic mutations, and errors in the identification of copy number alterations or complex evolutionary events (e.g., mutation losses) affect...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38017105"
                        open
                      >
                        <summary class="article-expander__title">Chronic lymphocytic leukemia patient-derived xenografts recapitulate clonal evolution to Richter transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38017105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic lymphocytic leukemia (CLL) is a B-cell neoplasm with a heterogeneous clinical behavior. In 5-10% of patients the disease transforms into a diffuse large-B cell lymphoma known as Richter transformation (RT), which is associated with dismal prognosis. Here, we aimed to establish patient-derived xenograft (PDX) models to study the molecular features and evolution of CLL and RT. We generated two PDXs by injecting CLL (PDX12) and RT (PDX19) cells into immunocompromised NSG mice. Both PDXs...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012682"
                        open
                      >
                        <summary class="article-expander__title">Long-term decitabine/retinoic acid maintenance treatment in an elderly sAML patient with high-risk genetics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012682/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Elderly patients with AML ineligible for induction have a dismal prognosis; hence disease stabilization is a primary treatment goal. This case of a 75-year-old patient with secondary AML receiving the combination of decitabine and ATRA (within the DECIDER trial, NCT00867672) demonstrates an above-average survival. The therapy administered over 52 cycles led to complete molecular and hematological remission and resulted in 5.3 years overall survival. Clonal evolution of the leukemic clone could...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-27 datetime="2023-11-27T11:00:00.000Z">2023-11-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-27 datetime="2023-11-27T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012392"
                        open
                      >
                        <summary class="article-expander__title">Tumor heterogeneity and immune-evasive T follicular cell lymphoma phenotypes at single-cell resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012392/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T follicular helper (T(FH)) cell lymphomas (TFHLs) are characterized by T(FH)-like properties and accompanied by substantial immune-cell infiltration into tumor tissues. Nevertheless, the comprehensive understanding of tumor-cell heterogeneity and immune profiles of TFHL remains elusive. To address this, we conducted single-cell transcriptomic analysis on 9 lymph node (LN) and 16 peripheral blood (PB) samples from TFHL patients. Tumor cells were divided into 5 distinct subclusters, with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012392"
                        open
                      >
                        <summary class="article-expander__title">Tumor heterogeneity and immune-evasive T follicular cell lymphoma phenotypes at single-cell resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012392/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T follicular helper (T(FH)) cell lymphomas (TFHLs) are characterized by T(FH)-like properties and accompanied by substantial immune-cell infiltration into tumor tissues. Nevertheless, the comprehensive understanding of tumor-cell heterogeneity and immune profiles of TFHL remains elusive. To address this, we conducted single-cell transcriptomic analysis on 9 lymph node (LN) and 16 peripheral blood (PB) samples from TFHL patients. Tumor cells were divided into 5 distinct subclusters, with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012392"
                        open
                      >
                        <summary class="article-expander__title">Tumor heterogeneity and immune-evasive T follicular cell lymphoma phenotypes at single-cell resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012392/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T follicular helper (T(FH)) cell lymphomas (TFHLs) are characterized by T(FH)-like properties and accompanied by substantial immune-cell infiltration into tumor tissues. Nevertheless, the comprehensive understanding of tumor-cell heterogeneity and immune profiles of TFHL remains elusive. To address this, we conducted single-cell transcriptomic analysis on 9 lymph node (LN) and 16 peripheral blood (PB) samples from TFHL patients. Tumor cells were divided into 5 distinct subclusters, with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012392"
                        open
                      >
                        <summary class="article-expander__title">Tumor heterogeneity and immune-evasive T follicular cell lymphoma phenotypes at single-cell resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012392/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T follicular helper (T(FH)) cell lymphomas (TFHLs) are characterized by T(FH)-like properties and accompanied by substantial immune-cell infiltration into tumor tissues. Nevertheless, the comprehensive understanding of tumor-cell heterogeneity and immune profiles of TFHL remains elusive. To address this, we conducted single-cell transcriptomic analysis on 9 lymph node (LN) and 16 peripheral blood (PB) samples from TFHL patients. Tumor cells were divided into 5 distinct subclusters, with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012392"
                        open
                      >
                        <summary class="article-expander__title">Tumor heterogeneity and immune-evasive T follicular cell lymphoma phenotypes at single-cell resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012392/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T follicular helper (T(FH)) cell lymphomas (TFHLs) are characterized by T(FH)-like properties and accompanied by substantial immune-cell infiltration into tumor tissues. Nevertheless, the comprehensive understanding of tumor-cell heterogeneity and immune profiles of TFHL remains elusive. To address this, we conducted single-cell transcriptomic analysis on 9 lymph node (LN) and 16 peripheral blood (PB) samples from TFHL patients. Tumor cells were divided into 5 distinct subclusters, with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012392"
                        open
                      >
                        <summary class="article-expander__title">Tumor heterogeneity and immune-evasive T follicular cell lymphoma phenotypes at single-cell resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012392/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T follicular helper (T(FH)) cell lymphomas (TFHLs) are characterized by T(FH)-like properties and accompanied by substantial immune-cell infiltration into tumor tissues. Nevertheless, the comprehensive understanding of tumor-cell heterogeneity and immune profiles of TFHL remains elusive. To address this, we conducted single-cell transcriptomic analysis on 9 lymph node (LN) and 16 peripheral blood (PB) samples from TFHL patients. Tumor cells were divided into 5 distinct subclusters, with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012392"
                        open
                      >
                        <summary class="article-expander__title">Tumor heterogeneity and immune-evasive T follicular cell lymphoma phenotypes at single-cell resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012392/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T follicular helper (T(FH)) cell lymphomas (TFHLs) are characterized by T(FH)-like properties and accompanied by substantial immune-cell infiltration into tumor tissues. Nevertheless, the comprehensive understanding of tumor-cell heterogeneity and immune profiles of TFHL remains elusive. To address this, we conducted single-cell transcriptomic analysis on 9 lymph node (LN) and 16 peripheral blood (PB) samples from TFHL patients. Tumor cells were divided into 5 distinct subclusters, with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38012392"
                        open
                      >
                        <summary class="article-expander__title">Tumor heterogeneity and immune-evasive T follicular cell lymphoma phenotypes at single-cell resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38012392/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T follicular helper (T(FH)) cell lymphomas (TFHLs) are characterized by T(FH)-like properties and accompanied by substantial immune-cell infiltration into tumor tissues. Nevertheless, the comprehensive understanding of tumor-cell heterogeneity and immune profiles of TFHL remains elusive. To address this, we conducted single-cell transcriptomic analysis on 9 lymph node (LN) and 16 peripheral blood (PB) samples from TFHL patients. Tumor cells were divided into 5 distinct subclusters, with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-24 datetime="2023-11-24T11:00:00.000Z">2023-11-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-24 datetime="2023-11-24T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38000729"
                        open
                      >
                        <summary class="article-expander__title">Transcriptional and Functional Consequences of Oncostatin M Signaling on Young Dnmt3a-Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38000729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective growth advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identified Oncostatin M (OSM) signaling as a candidate contributor to age-related Dnmt3a-mutant CH. We found that Dnmt3a-mutant HSCs from young adult mice (3-6 months...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999991"
                        open
                      >
                        <summary class="article-expander__title">Molecular diagnostic criteria of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999991/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasms (MPN) are a heterogeneous group of clonal hematopoietic stem cell neoplasms characterized by the driver mutations JAK2, CALR, and MPL. These mutations cause constitutive activation of JAK-STAT signaling, which is central to pathogenesis of MPNs. Next-generation sequencing has further expanded the molecular landscape allowing for improved diagnostics, prognostication, and targeted therapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38000729"
                        open
                      >
                        <summary class="article-expander__title">Transcriptional and Functional Consequences of Oncostatin M Signaling on Young Dnmt3a-Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38000729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective growth advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identified Oncostatin M (OSM) signaling as a candidate contributor to age-related Dnmt3a-mutant CH. We found that Dnmt3a-mutant HSCs from young adult mice (3-6 months...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999991"
                        open
                      >
                        <summary class="article-expander__title">Molecular diagnostic criteria of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999991/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasms (MPN) are a heterogeneous group of clonal hematopoietic stem cell neoplasms characterized by the driver mutations JAK2, CALR, and MPL. These mutations cause constitutive activation of JAK-STAT signaling, which is central to pathogenesis of MPNs. Next-generation sequencing has further expanded the molecular landscape allowing for improved diagnostics, prognostication, and targeted therapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38000729"
                        open
                      >
                        <summary class="article-expander__title">Transcriptional and Functional Consequences of Oncostatin M Signaling on Young Dnmt3a-Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38000729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective growth advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identified Oncostatin M (OSM) signaling as a candidate contributor to age-related Dnmt3a-mutant CH. We found that Dnmt3a-mutant HSCs from young adult mice (3-6 months...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999991"
                        open
                      >
                        <summary class="article-expander__title">Molecular diagnostic criteria of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999991/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasms (MPN) are a heterogeneous group of clonal hematopoietic stem cell neoplasms characterized by the driver mutations JAK2, CALR, and MPL. These mutations cause constitutive activation of JAK-STAT signaling, which is central to pathogenesis of MPNs. Next-generation sequencing has further expanded the molecular landscape allowing for improved diagnostics, prognostication, and targeted therapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38000729"
                        open
                      >
                        <summary class="article-expander__title">Transcriptional and Functional Consequences of Oncostatin M Signaling on Young Dnmt3a-Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38000729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective growth advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identified Oncostatin M (OSM) signaling as a candidate contributor to age-related Dnmt3a-mutant CH. We found that Dnmt3a-mutant HSCs from young adult mice (3-6 months...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999991"
                        open
                      >
                        <summary class="article-expander__title">Molecular diagnostic criteria of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999991/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasms (MPN) are a heterogeneous group of clonal hematopoietic stem cell neoplasms characterized by the driver mutations JAK2, CALR, and MPL. These mutations cause constitutive activation of JAK-STAT signaling, which is central to pathogenesis of MPNs. Next-generation sequencing has further expanded the molecular landscape allowing for improved diagnostics, prognostication, and targeted therapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38000729"
                        open
                      >
                        <summary class="article-expander__title">Transcriptional and Functional Consequences of Oncostatin M Signaling on Young Dnmt3a-Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38000729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective growth advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identified Oncostatin M (OSM) signaling as a candidate contributor to age-related Dnmt3a-mutant CH. We found that Dnmt3a-mutant HSCs from young adult mice (3-6 months...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999991"
                        open
                      >
                        <summary class="article-expander__title">Molecular diagnostic criteria of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999991/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasms (MPN) are a heterogeneous group of clonal hematopoietic stem cell neoplasms characterized by the driver mutations JAK2, CALR, and MPL. These mutations cause constitutive activation of JAK-STAT signaling, which is central to pathogenesis of MPNs. Next-generation sequencing has further expanded the molecular landscape allowing for improved diagnostics, prognostication, and targeted therapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38000729"
                        open
                      >
                        <summary class="article-expander__title">Transcriptional and Functional Consequences of Oncostatin M Signaling on Young Dnmt3a-Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38000729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective growth advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identified Oncostatin M (OSM) signaling as a candidate contributor to age-related Dnmt3a-mutant CH. We found that Dnmt3a-mutant HSCs from young adult mice (3-6 months...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999991"
                        open
                      >
                        <summary class="article-expander__title">Molecular diagnostic criteria of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999991/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasms (MPN) are a heterogeneous group of clonal hematopoietic stem cell neoplasms characterized by the driver mutations JAK2, CALR, and MPL. These mutations cause constitutive activation of JAK-STAT signaling, which is central to pathogenesis of MPNs. Next-generation sequencing has further expanded the molecular landscape allowing for improved diagnostics, prognostication, and targeted therapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38000729"
                        open
                      >
                        <summary class="article-expander__title">Transcriptional and Functional Consequences of Oncostatin M Signaling on Young Dnmt3a-Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38000729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective growth advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identified Oncostatin M (OSM) signaling as a candidate contributor to age-related Dnmt3a-mutant CH. We found that Dnmt3a-mutant HSCs from young adult mice (3-6 months...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999991"
                        open
                      >
                        <summary class="article-expander__title">Molecular diagnostic criteria of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999991/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasms (MPN) are a heterogeneous group of clonal hematopoietic stem cell neoplasms characterized by the driver mutations JAK2, CALR, and MPL. These mutations cause constitutive activation of JAK-STAT signaling, which is central to pathogenesis of MPNs. Next-generation sequencing has further expanded the molecular landscape allowing for improved diagnostics, prognostication, and targeted therapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38000729"
                        open
                      >
                        <summary class="article-expander__title">Transcriptional and Functional Consequences of Oncostatin M Signaling on Young Dnmt3a-Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38000729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective growth advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identified Oncostatin M (OSM) signaling as a candidate contributor to age-related Dnmt3a-mutant CH. We found that Dnmt3a-mutant HSCs from young adult mice (3-6 months...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999991"
                        open
                      >
                        <summary class="article-expander__title">Molecular diagnostic criteria of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999991/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasms (MPN) are a heterogeneous group of clonal hematopoietic stem cell neoplasms characterized by the driver mutations JAK2, CALR, and MPL. These mutations cause constitutive activation of JAK-STAT signaling, which is central to pathogenesis of MPNs. Next-generation sequencing has further expanded the molecular landscape allowing for improved diagnostics, prognostication, and targeted therapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38000729"
                        open
                      >
                        <summary class="article-expander__title">Transcriptional and Functional Consequences of Oncostatin M Signaling on Young Dnmt3a-Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38000729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective growth advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identified Oncostatin M (OSM) signaling as a candidate contributor to age-related Dnmt3a-mutant CH. We found that Dnmt3a-mutant HSCs from young adult mice (3-6 months...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999991"
                        open
                      >
                        <summary class="article-expander__title">Molecular diagnostic criteria of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999991/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasms (MPN) are a heterogeneous group of clonal hematopoietic stem cell neoplasms characterized by the driver mutations JAK2, CALR, and MPL. These mutations cause constitutive activation of JAK-STAT signaling, which is central to pathogenesis of MPNs. Next-generation sequencing has further expanded the molecular landscape allowing for improved diagnostics, prognostication, and targeted therapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38000729"
                        open
                      >
                        <summary class="article-expander__title">Transcriptional and Functional Consequences of Oncostatin M Signaling on Young Dnmt3a-Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38000729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective growth advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identified Oncostatin M (OSM) signaling as a candidate contributor to age-related Dnmt3a-mutant CH. We found that Dnmt3a-mutant HSCs from young adult mice (3-6 months...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999991"
                        open
                      >
                        <summary class="article-expander__title">Molecular diagnostic criteria of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999991/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasms (MPN) are a heterogeneous group of clonal hematopoietic stem cell neoplasms characterized by the driver mutations JAK2, CALR, and MPL. These mutations cause constitutive activation of JAK-STAT signaling, which is central to pathogenesis of MPNs. Next-generation sequencing has further expanded the molecular landscape allowing for improved diagnostics, prognostication, and targeted therapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38000729"
                        open
                      >
                        <summary class="article-expander__title">Transcriptional and Functional Consequences of Oncostatin M Signaling on Young Dnmt3a-Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38000729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective growth advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identified Oncostatin M (OSM) signaling as a candidate contributor to age-related Dnmt3a-mutant CH. We found that Dnmt3a-mutant HSCs from young adult mice (3-6 months...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999991"
                        open
                      >
                        <summary class="article-expander__title">Molecular diagnostic criteria of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999991/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasms (MPN) are a heterogeneous group of clonal hematopoietic stem cell neoplasms characterized by the driver mutations JAK2, CALR, and MPL. These mutations cause constitutive activation of JAK-STAT signaling, which is central to pathogenesis of MPNs. Next-generation sequencing has further expanded the molecular landscape allowing for improved diagnostics, prognostication, and targeted therapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:38000729"
                        open
                      >
                        <summary class="article-expander__title">Transcriptional and Functional Consequences of Oncostatin M Signaling on Young Dnmt3a-Mutant Hematopoietic Stem Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/38000729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-associated clonal hematopoiesis (CH) occurs due to somatic mutations accrued in hematopoietic stem cells (HSCs) that confer a selective growth advantage in the context of aging. The mechanisms by which CH-mutant HSCs gain this advantage with aging are not comprehensively understood. Using unbiased transcriptomic approaches, we identified Oncostatin M (OSM) signaling as a candidate contributor to age-related Dnmt3a-mutant CH. We found that Dnmt3a-mutant HSCs from young adult mice (3-6 months...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999991"
                        open
                      >
                        <summary class="article-expander__title">Molecular diagnostic criteria of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999991/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasms (MPN) are a heterogeneous group of clonal hematopoietic stem cell neoplasms characterized by the driver mutations JAK2, CALR, and MPL. These mutations cause constitutive activation of JAK-STAT signaling, which is central to pathogenesis of MPNs. Next-generation sequencing has further expanded the molecular landscape allowing for improved diagnostics, prognostication, and targeted therapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999991"
                        open
                      >
                        <summary class="article-expander__title">Molecular diagnostic criteria of myeloproliferative neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999991/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myeloproliferative neoplasms (MPN) are a heterogeneous group of clonal hematopoietic stem cell neoplasms characterized by the driver mutations JAK2, CALR, and MPL. These mutations cause constitutive activation of JAK-STAT signaling, which is central to pathogenesis of MPNs. Next-generation sequencing has further expanded the molecular landscape allowing for improved diagnostics, prognostication, and targeted therapy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999800"
                        open
                      >
                        <summary class="article-expander__title">What can we learn about acid-base transporters in cancer from studying somatic mutations in their genes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acidosis is a chemical signature of the tumour microenvironment that challenges intracellular pH homeostasis. The orchestrated activity of acid-base transporters of the solute-linked carrier (SLC) family is critical for removing the end-products of fermentative metabolism (lactate/H^(+)) and maintaining a favourably alkaline cytoplasm. Given the critical role of pH homeostasis in enabling cellular activities, mutations in relevant SLC genes may impact the oncogenic process, emerging as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999800"
                        open
                      >
                        <summary class="article-expander__title">What can we learn about acid-base transporters in cancer from studying somatic mutations in their genes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acidosis is a chemical signature of the tumour microenvironment that challenges intracellular pH homeostasis. The orchestrated activity of acid-base transporters of the solute-linked carrier (SLC) family is critical for removing the end-products of fermentative metabolism (lactate/H^(+)) and maintaining a favourably alkaline cytoplasm. Given the critical role of pH homeostasis in enabling cellular activities, mutations in relevant SLC genes may impact the oncogenic process, emerging as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999800"
                        open
                      >
                        <summary class="article-expander__title">What can we learn about acid-base transporters in cancer from studying somatic mutations in their genes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acidosis is a chemical signature of the tumour microenvironment that challenges intracellular pH homeostasis. The orchestrated activity of acid-base transporters of the solute-linked carrier (SLC) family is critical for removing the end-products of fermentative metabolism (lactate/H^(+)) and maintaining a favourably alkaline cytoplasm. Given the critical role of pH homeostasis in enabling cellular activities, mutations in relevant SLC genes may impact the oncogenic process, emerging as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999800"
                        open
                      >
                        <summary class="article-expander__title">What can we learn about acid-base transporters in cancer from studying somatic mutations in their genes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acidosis is a chemical signature of the tumour microenvironment that challenges intracellular pH homeostasis. The orchestrated activity of acid-base transporters of the solute-linked carrier (SLC) family is critical for removing the end-products of fermentative metabolism (lactate/H^(+)) and maintaining a favourably alkaline cytoplasm. Given the critical role of pH homeostasis in enabling cellular activities, mutations in relevant SLC genes may impact the oncogenic process, emerging as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999800"
                        open
                      >
                        <summary class="article-expander__title">What can we learn about acid-base transporters in cancer from studying somatic mutations in their genes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acidosis is a chemical signature of the tumour microenvironment that challenges intracellular pH homeostasis. The orchestrated activity of acid-base transporters of the solute-linked carrier (SLC) family is critical for removing the end-products of fermentative metabolism (lactate/H^(+)) and maintaining a favourably alkaline cytoplasm. Given the critical role of pH homeostasis in enabling cellular activities, mutations in relevant SLC genes may impact the oncogenic process, emerging as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999800"
                        open
                      >
                        <summary class="article-expander__title">What can we learn about acid-base transporters in cancer from studying somatic mutations in their genes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acidosis is a chemical signature of the tumour microenvironment that challenges intracellular pH homeostasis. The orchestrated activity of acid-base transporters of the solute-linked carrier (SLC) family is critical for removing the end-products of fermentative metabolism (lactate/H^(+)) and maintaining a favourably alkaline cytoplasm. Given the critical role of pH homeostasis in enabling cellular activities, mutations in relevant SLC genes may impact the oncogenic process, emerging as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999800"
                        open
                      >
                        <summary class="article-expander__title">What can we learn about acid-base transporters in cancer from studying somatic mutations in their genes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acidosis is a chemical signature of the tumour microenvironment that challenges intracellular pH homeostasis. The orchestrated activity of acid-base transporters of the solute-linked carrier (SLC) family is critical for removing the end-products of fermentative metabolism (lactate/H^(+)) and maintaining a favourably alkaline cytoplasm. Given the critical role of pH homeostasis in enabling cellular activities, mutations in relevant SLC genes may impact the oncogenic process, emerging as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999800"
                        open
                      >
                        <summary class="article-expander__title">What can we learn about acid-base transporters in cancer from studying somatic mutations in their genes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acidosis is a chemical signature of the tumour microenvironment that challenges intracellular pH homeostasis. The orchestrated activity of acid-base transporters of the solute-linked carrier (SLC) family is critical for removing the end-products of fermentative metabolism (lactate/H^(+)) and maintaining a favourably alkaline cytoplasm. Given the critical role of pH homeostasis in enabling cellular activities, mutations in relevant SLC genes may impact the oncogenic process, emerging as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999800"
                        open
                      >
                        <summary class="article-expander__title">What can we learn about acid-base transporters in cancer from studying somatic mutations in their genes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acidosis is a chemical signature of the tumour microenvironment that challenges intracellular pH homeostasis. The orchestrated activity of acid-base transporters of the solute-linked carrier (SLC) family is critical for removing the end-products of fermentative metabolism (lactate/H^(+)) and maintaining a favourably alkaline cytoplasm. Given the critical role of pH homeostasis in enabling cellular activities, mutations in relevant SLC genes may impact the oncogenic process, emerging as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999800"
                        open
                      >
                        <summary class="article-expander__title">What can we learn about acid-base transporters in cancer from studying somatic mutations in their genes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acidosis is a chemical signature of the tumour microenvironment that challenges intracellular pH homeostasis. The orchestrated activity of acid-base transporters of the solute-linked carrier (SLC) family is critical for removing the end-products of fermentative metabolism (lactate/H^(+)) and maintaining a favourably alkaline cytoplasm. Given the critical role of pH homeostasis in enabling cellular activities, mutations in relevant SLC genes may impact the oncogenic process, emerging as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999800"
                        open
                      >
                        <summary class="article-expander__title">What can we learn about acid-base transporters in cancer from studying somatic mutations in their genes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acidosis is a chemical signature of the tumour microenvironment that challenges intracellular pH homeostasis. The orchestrated activity of acid-base transporters of the solute-linked carrier (SLC) family is critical for removing the end-products of fermentative metabolism (lactate/H^(+)) and maintaining a favourably alkaline cytoplasm. Given the critical role of pH homeostasis in enabling cellular activities, mutations in relevant SLC genes may impact the oncogenic process, emerging as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37999800"
                        open
                      >
                        <summary class="article-expander__title">What can we learn about acid-base transporters in cancer from studying somatic mutations in their genes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37999800/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acidosis is a chemical signature of the tumour microenvironment that challenges intracellular pH homeostasis. The orchestrated activity of acid-base transporters of the solute-linked carrier (SLC) family is critical for removing the end-products of fermentative metabolism (lactate/H^(+)) and maintaining a favourably alkaline cytoplasm. Given the critical role of pH homeostasis in enabling cellular activities, mutations in relevant SLC genes may impact the oncogenic process, emerging as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-23 datetime="2023-11-23T11:00:00.000Z">2023-11-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-23 datetime="2023-11-23T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37994105"
                        open
                      >
                        <summary class="article-expander__title">Massive parallel sequencing unveils homologous recombination deficiency in follicular dendritic cell sarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37994105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231206195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Standardized treatment options are lacking for patients with unresectable or multifocal follicular dendritic cell sarcoma (FDCS) and disease-related mortality is as high as 20%. Applying whole genome sequencing (WGS) in one case and whole exome sequencing (WES) in additional twelve, this study adds information on the molecular landscape of FDCS, expanding knowledge on pathobiological mechanisms and identifying novel markers of potential theragnostic significance. Massive parallel sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37994105"
                        open
                      >
                        <summary class="article-expander__title">Massive parallel sequencing unveils homologous recombination deficiency in follicular dendritic cell sarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37994105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Standardized treatment options are lacking for patients with unresectable or multifocal follicular dendritic cell sarcoma (FDCS) and disease-related mortality is as high as 20%. Applying whole genome sequencing (WGS) in one case and whole exome sequencing (WES) in additional twelve, this study adds information on the molecular landscape of FDCS, expanding knowledge on pathobiological mechanisms and identifying novel markers of potential theragnostic significance. Massive parallel sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37994105"
                        open
                      >
                        <summary class="article-expander__title">Massive parallel sequencing unveils homologous recombination deficiency in follicular dendritic cell sarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37994105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Standardized treatment options are lacking for patients with unresectable or multifocal follicular dendritic cell sarcoma (FDCS) and disease-related mortality is as high as 20%. Applying whole genome sequencing (WGS) in one case and whole exome sequencing (WES) in additional twelve, this study adds information on the molecular landscape of FDCS, expanding knowledge on pathobiological mechanisms and identifying novel markers of potential theragnostic significance. Massive parallel sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37994105"
                        open
                      >
                        <summary class="article-expander__title">Massive parallel sequencing unveils homologous recombination deficiency in follicular dendritic cell sarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37994105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Standardized treatment options are lacking for patients with unresectable or multifocal follicular dendritic cell sarcoma (FDCS) and disease-related mortality is as high as 20%. Applying whole genome sequencing (WGS) in one case and whole exome sequencing (WES) in additional twelve, this study adds information on the molecular landscape of FDCS, expanding knowledge on pathobiological mechanisms and identifying novel markers of potential theragnostic significance. Massive parallel sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37994105"
                        open
                      >
                        <summary class="article-expander__title">Massive parallel sequencing unveils homologous recombination deficiency in follicular dendritic cell sarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37994105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Standardized treatment options are lacking for patients with unresectable or multifocal follicular dendritic cell sarcoma (FDCS) and disease-related mortality is as high as 20%. Applying whole genome sequencing (WGS) in one case and whole exome sequencing (WES) in additional twelve, this study adds information on the molecular landscape of FDCS, expanding knowledge on pathobiological mechanisms and identifying novel markers of potential theragnostic significance. Massive parallel sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37994105"
                        open
                      >
                        <summary class="article-expander__title">Massive parallel sequencing unveils homologous recombination deficiency in follicular dendritic cell sarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37994105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Standardized treatment options are lacking for patients with unresectable or multifocal follicular dendritic cell sarcoma (FDCS) and disease-related mortality is as high as 20%. Applying whole genome sequencing (WGS) in one case and whole exome sequencing (WES) in additional twelve, this study adds information on the molecular landscape of FDCS, expanding knowledge on pathobiological mechanisms and identifying novel markers of potential theragnostic significance. Massive parallel sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37994105"
                        open
                      >
                        <summary class="article-expander__title">Massive parallel sequencing unveils homologous recombination deficiency in follicular dendritic cell sarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37994105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Standardized treatment options are lacking for patients with unresectable or multifocal follicular dendritic cell sarcoma (FDCS) and disease-related mortality is as high as 20%. Applying whole genome sequencing (WGS) in one case and whole exome sequencing (WES) in additional twelve, this study adds information on the molecular landscape of FDCS, expanding knowledge on pathobiological mechanisms and identifying novel markers of potential theragnostic significance. Massive parallel sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37994105"
                        open
                      >
                        <summary class="article-expander__title">Massive parallel sequencing unveils homologous recombination deficiency in follicular dendritic cell sarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37994105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Standardized treatment options are lacking for patients with unresectable or multifocal follicular dendritic cell sarcoma (FDCS) and disease-related mortality is as high as 20%. Applying whole genome sequencing (WGS) in one case and whole exome sequencing (WES) in additional twelve, this study adds information on the molecular landscape of FDCS, expanding knowledge on pathobiological mechanisms and identifying novel markers of potential theragnostic significance. Massive parallel sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37994105"
                        open
                      >
                        <summary class="article-expander__title">Massive parallel sequencing unveils homologous recombination deficiency in follicular dendritic cell sarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37994105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Standardized treatment options are lacking for patients with unresectable or multifocal follicular dendritic cell sarcoma (FDCS) and disease-related mortality is as high as 20%. Applying whole genome sequencing (WGS) in one case and whole exome sequencing (WES) in additional twelve, this study adds information on the molecular landscape of FDCS, expanding knowledge on pathobiological mechanisms and identifying novel markers of potential theragnostic significance. Massive parallel sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37994105"
                        open
                      >
                        <summary class="article-expander__title">Massive parallel sequencing unveils homologous recombination deficiency in follicular dendritic cell sarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37994105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Standardized treatment options are lacking for patients with unresectable or multifocal follicular dendritic cell sarcoma (FDCS) and disease-related mortality is as high as 20%. Applying whole genome sequencing (WGS) in one case and whole exome sequencing (WES) in additional twelve, this study adds information on the molecular landscape of FDCS, expanding knowledge on pathobiological mechanisms and identifying novel markers of potential theragnostic significance. Massive parallel sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37994105"
                        open
                      >
                        <summary class="article-expander__title">Massive parallel sequencing unveils homologous recombination deficiency in follicular dendritic cell sarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37994105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Standardized treatment options are lacking for patients with unresectable or multifocal follicular dendritic cell sarcoma (FDCS) and disease-related mortality is as high as 20%. Applying whole genome sequencing (WGS) in one case and whole exome sequencing (WES) in additional twelve, this study adds information on the molecular landscape of FDCS, expanding knowledge on pathobiological mechanisms and identifying novel markers of potential theragnostic significance. Massive parallel sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37994105"
                        open
                      >
                        <summary class="article-expander__title">Massive parallel sequencing unveils homologous recombination deficiency in follicular dendritic cell sarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37994105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Standardized treatment options are lacking for patients with unresectable or multifocal follicular dendritic cell sarcoma (FDCS) and disease-related mortality is as high as 20%. Applying whole genome sequencing (WGS) in one case and whole exome sequencing (WES) in additional twelve, this study adds information on the molecular landscape of FDCS, expanding knowledge on pathobiological mechanisms and identifying novel markers of potential theragnostic significance. Massive parallel sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37994105"
                        open
                      >
                        <summary class="article-expander__title">Massive parallel sequencing unveils homologous recombination deficiency in follicular dendritic cell sarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37994105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Standardized treatment options are lacking for patients with unresectable or multifocal follicular dendritic cell sarcoma (FDCS) and disease-related mortality is as high as 20%. Applying whole genome sequencing (WGS) in one case and whole exome sequencing (WES) in additional twelve, this study adds information on the molecular landscape of FDCS, expanding knowledge on pathobiological mechanisms and identifying novel markers of potential theragnostic significance. Massive parallel sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37994105"
                        open
                      >
                        <summary class="article-expander__title">Massive parallel sequencing unveils homologous recombination deficiency in follicular dendritic cell sarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37994105/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Standardized treatment options are lacking for patients with unresectable or multifocal follicular dendritic cell sarcoma (FDCS) and disease-related mortality is as high as 20%. Applying whole genome sequencing (WGS) in one case and whole exome sequencing (WES) in additional twelve, this study adds information on the molecular landscape of FDCS, expanding knowledge on pathobiological mechanisms and identifying novel markers of potential theragnostic significance. Massive parallel sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37996680"
                        open
                      >
                        <summary class="article-expander__title">Somatic Cell Fusion in Host Defense and Adaptation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37996680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Evidence of fusion of somatic cells has been noted in health and in disease for more than a century. The most obvious but uncertain hallmark has been the presence of multiple nuclei in cells. Although multinucleated cells are found in normal and diseased tissues, the benefit or harm of such cells can be difficult to elucidate. Still more difficult however is the identification of mononuclear cells previously formed by fusion of somatic cells with one or more nuclei disposed. The later process...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37996680"
                        open
                      >
                        <summary class="article-expander__title">Somatic Cell Fusion in Host Defense and Adaptation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37996680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Evidence of fusion of somatic cells has been noted in health and in disease for more than a century. The most obvious but uncertain hallmark has been the presence of multiple nuclei in cells. Although multinucleated cells are found in normal and diseased tissues, the benefit or harm of such cells can be difficult to elucidate. Still more difficult however is the identification of mononuclear cells previously formed by fusion of somatic cells with one or more nuclei disposed. The later process...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37996680"
                        open
                      >
                        <summary class="article-expander__title">Somatic Cell Fusion in Host Defense and Adaptation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37996680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Evidence of fusion of somatic cells has been noted in health and in disease for more than a century. The most obvious but uncertain hallmark has been the presence of multiple nuclei in cells. Although multinucleated cells are found in normal and diseased tissues, the benefit or harm of such cells can be difficult to elucidate. Still more difficult however is the identification of mononuclear cells previously formed by fusion of somatic cells with one or more nuclei disposed. The later process...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37996680"
                        open
                      >
                        <summary class="article-expander__title">Somatic Cell Fusion in Host Defense and Adaptation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37996680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Evidence of fusion of somatic cells has been noted in health and in disease for more than a century. The most obvious but uncertain hallmark has been the presence of multiple nuclei in cells. Although multinucleated cells are found in normal and diseased tissues, the benefit or harm of such cells can be difficult to elucidate. Still more difficult however is the identification of mononuclear cells previously formed by fusion of somatic cells with one or more nuclei disposed. The later process...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37996680"
                        open
                      >
                        <summary class="article-expander__title">Somatic Cell Fusion in Host Defense and Adaptation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37996680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Evidence of fusion of somatic cells has been noted in health and in disease for more than a century. The most obvious but uncertain hallmark has been the presence of multiple nuclei in cells. Although multinucleated cells are found in normal and diseased tissues, the benefit or harm of such cells can be difficult to elucidate. Still more difficult however is the identification of mononuclear cells previously formed by fusion of somatic cells with one or more nuclei disposed. The later process...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37996680"
                        open
                      >
                        <summary class="article-expander__title">Somatic Cell Fusion in Host Defense and Adaptation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37996680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Evidence of fusion of somatic cells has been noted in health and in disease for more than a century. The most obvious but uncertain hallmark has been the presence of multiple nuclei in cells. Although multinucleated cells are found in normal and diseased tissues, the benefit or harm of such cells can be difficult to elucidate. Still more difficult however is the identification of mononuclear cells previously formed by fusion of somatic cells with one or more nuclei disposed. The later process...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37996680"
                        open
                      >
                        <summary class="article-expander__title">Somatic Cell Fusion in Host Defense and Adaptation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37996680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Evidence of fusion of somatic cells has been noted in health and in disease for more than a century. The most obvious but uncertain hallmark has been the presence of multiple nuclei in cells. Although multinucleated cells are found in normal and diseased tissues, the benefit or harm of such cells can be difficult to elucidate. Still more difficult however is the identification of mononuclear cells previously formed by fusion of somatic cells with one or more nuclei disposed. The later process...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37996680"
                        open
                      >
                        <summary class="article-expander__title">Somatic Cell Fusion in Host Defense and Adaptation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37996680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Evidence of fusion of somatic cells has been noted in health and in disease for more than a century. The most obvious but uncertain hallmark has been the presence of multiple nuclei in cells. Although multinucleated cells are found in normal and diseased tissues, the benefit or harm of such cells can be difficult to elucidate. Still more difficult however is the identification of mononuclear cells previously formed by fusion of somatic cells with one or more nuclei disposed. The later process...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37996680"
                        open
                      >
                        <summary class="article-expander__title">Somatic Cell Fusion in Host Defense and Adaptation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37996680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Evidence of fusion of somatic cells has been noted in health and in disease for more than a century. The most obvious but uncertain hallmark has been the presence of multiple nuclei in cells. Although multinucleated cells are found in normal and diseased tissues, the benefit or harm of such cells can be difficult to elucidate. Still more difficult however is the identification of mononuclear cells previously formed by fusion of somatic cells with one or more nuclei disposed. The later process...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37996680"
                        open
                      >
                        <summary class="article-expander__title">Somatic Cell Fusion in Host Defense and Adaptation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37996680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Evidence of fusion of somatic cells has been noted in health and in disease for more than a century. The most obvious but uncertain hallmark has been the presence of multiple nuclei in cells. Although multinucleated cells are found in normal and diseased tissues, the benefit or harm of such cells can be difficult to elucidate. Still more difficult however is the identification of mononuclear cells previously formed by fusion of somatic cells with one or more nuclei disposed. The later process...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37996680"
                        open
                      >
                        <summary class="article-expander__title">Somatic Cell Fusion in Host Defense and Adaptation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37996680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Evidence of fusion of somatic cells has been noted in health and in disease for more than a century. The most obvious but uncertain hallmark has been the presence of multiple nuclei in cells. Although multinucleated cells are found in normal and diseased tissues, the benefit or harm of such cells can be difficult to elucidate. Still more difficult however is the identification of mononuclear cells previously formed by fusion of somatic cells with one or more nuclei disposed. The later process...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37996680"
                        open
                      >
                        <summary class="article-expander__title">Somatic Cell Fusion in Host Defense and Adaptation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37996680/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Evidence of fusion of somatic cells has been noted in health and in disease for more than a century. The most obvious but uncertain hallmark has been the presence of multiple nuclei in cells. Although multinucleated cells are found in normal and diseased tissues, the benefit or harm of such cells can be difficult to elucidate. Still more difficult however is the identification of mononuclear cells previously formed by fusion of somatic cells with one or more nuclei disposed. The later process...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-22 datetime="2023-11-22T11:00:00.000Z">2023-11-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-22 datetime="2023-11-22T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37992214"
                        open
                      >
                        <summary class="article-expander__title">The lifelong natural history of clonal hematopoiesis and its links to myeloid malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37992214/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231205195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The study of somatic mutations and the associated clonal mosaicism across the human body has transformed our understanding of ageing and its links to cancer. In proliferative human tissues, stem cells compete for dominance and those with an advantage expand clonally in relation to their peers. In the hematopoietic system, such expansion is termed clonal hematopoiesis (CH). The forces driving competition, namely heterogeneity of the hematopoietic stem cell (HSC) pool and attrition of the HSC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37992214"
                        open
                      >
                        <summary class="article-expander__title">The lifelong natural history of clonal hematopoiesis and its links to myeloid malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37992214/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The study of somatic mutations and the associated clonal mosaicism across the human body has transformed our understanding of ageing and its links to cancer. In proliferative human tissues, stem cells compete for dominance and those with an advantage expand clonally in relation to their peers. In the hematopoietic system, such expansion is termed clonal hematopoiesis (CH). The forces driving competition, namely heterogeneity of the hematopoietic stem cell (HSC) pool and attrition of the HSC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37992214"
                        open
                      >
                        <summary class="article-expander__title">The lifelong natural history of clonal hematopoiesis and its links to myeloid malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37992214/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The study of somatic mutations and the associated clonal mosaicism across the human body has transformed our understanding of ageing and its links to cancer. In proliferative human tissues, stem cells compete for dominance and those with an advantage expand clonally in relation to their peers. In the hematopoietic system, such expansion is termed clonal hematopoiesis (CH). The forces driving competition, namely heterogeneity of the hematopoietic stem cell (HSC) pool and attrition of the HSC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37992214"
                        open
                      >
                        <summary class="article-expander__title">The lifelong natural history of clonal hematopoiesis and its links to myeloid malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37992214/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The study of somatic mutations and the associated clonal mosaicism across the human body has transformed our understanding of ageing and its links to cancer. In proliferative human tissues, stem cells compete for dominance and those with an advantage expand clonally in relation to their peers. In the hematopoietic system, such expansion is termed clonal hematopoiesis (CH). The forces driving competition, namely heterogeneity of the hematopoietic stem cell (HSC) pool and attrition of the HSC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37992214"
                        open
                      >
                        <summary class="article-expander__title">The lifelong natural history of clonal hematopoiesis and its links to myeloid malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37992214/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The study of somatic mutations and the associated clonal mosaicism across the human body has transformed our understanding of ageing and its links to cancer. In proliferative human tissues, stem cells compete for dominance and those with an advantage expand clonally in relation to their peers. In the hematopoietic system, such expansion is termed clonal hematopoiesis (CH). The forces driving competition, namely heterogeneity of the hematopoietic stem cell (HSC) pool and attrition of the HSC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37992214"
                        open
                      >
                        <summary class="article-expander__title">The lifelong natural history of clonal hematopoiesis and its links to myeloid malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37992214/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The study of somatic mutations and the associated clonal mosaicism across the human body has transformed our understanding of ageing and its links to cancer. In proliferative human tissues, stem cells compete for dominance and those with an advantage expand clonally in relation to their peers. In the hematopoietic system, such expansion is termed clonal hematopoiesis (CH). The forces driving competition, namely heterogeneity of the hematopoietic stem cell (HSC) pool and attrition of the HSC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37992214"
                        open
                      >
                        <summary class="article-expander__title">The lifelong natural history of clonal hematopoiesis and its links to myeloid malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37992214/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The study of somatic mutations and the associated clonal mosaicism across the human body has transformed our understanding of ageing and its links to cancer. In proliferative human tissues, stem cells compete for dominance and those with an advantage expand clonally in relation to their peers. In the hematopoietic system, such expansion is termed clonal hematopoiesis (CH). The forces driving competition, namely heterogeneity of the hematopoietic stem cell (HSC) pool and attrition of the HSC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37992214"
                        open
                      >
                        <summary class="article-expander__title">The lifelong natural history of clonal hematopoiesis and its links to myeloid malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37992214/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The study of somatic mutations and the associated clonal mosaicism across the human body has transformed our understanding of ageing and its links to cancer. In proliferative human tissues, stem cells compete for dominance and those with an advantage expand clonally in relation to their peers. In the hematopoietic system, such expansion is termed clonal hematopoiesis (CH). The forces driving competition, namely heterogeneity of the hematopoietic stem cell (HSC) pool and attrition of the HSC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37992214"
                        open
                      >
                        <summary class="article-expander__title">The lifelong natural history of clonal hematopoiesis and its links to myeloid malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37992214/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The study of somatic mutations and the associated clonal mosaicism across the human body has transformed our understanding of ageing and its links to cancer. In proliferative human tissues, stem cells compete for dominance and those with an advantage expand clonally in relation to their peers. In the hematopoietic system, such expansion is termed clonal hematopoiesis (CH). The forces driving competition, namely heterogeneity of the hematopoietic stem cell (HSC) pool and attrition of the HSC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37992214"
                        open
                      >
                        <summary class="article-expander__title">The lifelong natural history of clonal hematopoiesis and its links to myeloid malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37992214/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The study of somatic mutations and the associated clonal mosaicism across the human body has transformed our understanding of ageing and its links to cancer. In proliferative human tissues, stem cells compete for dominance and those with an advantage expand clonally in relation to their peers. In the hematopoietic system, such expansion is termed clonal hematopoiesis (CH). The forces driving competition, namely heterogeneity of the hematopoietic stem cell (HSC) pool and attrition of the HSC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37992214"
                        open
                      >
                        <summary class="article-expander__title">The lifelong natural history of clonal hematopoiesis and its links to myeloid malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37992214/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The study of somatic mutations and the associated clonal mosaicism across the human body has transformed our understanding of ageing and its links to cancer. In proliferative human tissues, stem cells compete for dominance and those with an advantage expand clonally in relation to their peers. In the hematopoietic system, such expansion is termed clonal hematopoiesis (CH). The forces driving competition, namely heterogeneity of the hematopoietic stem cell (HSC) pool and attrition of the HSC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37992214"
                        open
                      >
                        <summary class="article-expander__title">The lifelong natural history of clonal hematopoiesis and its links to myeloid malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37992214/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The study of somatic mutations and the associated clonal mosaicism across the human body has transformed our understanding of ageing and its links to cancer. In proliferative human tissues, stem cells compete for dominance and those with an advantage expand clonally in relation to their peers. In the hematopoietic system, such expansion is termed clonal hematopoiesis (CH). The forces driving competition, namely heterogeneity of the hematopoietic stem cell (HSC) pool and attrition of the HSC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37992214"
                        open
                      >
                        <summary class="article-expander__title">The lifelong natural history of clonal hematopoiesis and its links to myeloid malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37992214/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The study of somatic mutations and the associated clonal mosaicism across the human body has transformed our understanding of ageing and its links to cancer. In proliferative human tissues, stem cells compete for dominance and those with an advantage expand clonally in relation to their peers. In the hematopoietic system, such expansion is termed clonal hematopoiesis (CH). The forces driving competition, namely heterogeneity of the hematopoietic stem cell (HSC) pool and attrition of the HSC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37992214"
                        open
                      >
                        <summary class="article-expander__title">The lifelong natural history of clonal hematopoiesis and its links to myeloid malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37992214/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The study of somatic mutations and the associated clonal mosaicism across the human body has transformed our understanding of ageing and its links to cancer. In proliferative human tissues, stem cells compete for dominance and those with an advantage expand clonally in relation to their peers. In the hematopoietic system, such expansion is termed clonal hematopoiesis (CH). The forces driving competition, namely heterogeneity of the hematopoietic stem cell (HSC) pool and attrition of the HSC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37993585"
                        open
                      >
                        <summary class="article-expander__title">Research progress of additional pathogenic mutations in chronic neutrophilic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37993585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic neutrophilic leukemia (CNL) is a rare type of myeloproliferative neoplasm (MPN). Due to its nonspecific clinical symptoms and lack of specific molecular markers, it was previously difficult to distinguish it from other diseases with increased neutrophils. However, the discovery of the CSF3R mutation in CNL 10 years ago and the update of the diagnostic criteria by the World Health Organization (WHO) in 2016 brought CNL into a new era of molecular diagnosis. Next-generation sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37993585"
                        open
                      >
                        <summary class="article-expander__title">Research progress of additional pathogenic mutations in chronic neutrophilic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37993585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic neutrophilic leukemia (CNL) is a rare type of myeloproliferative neoplasm (MPN). Due to its nonspecific clinical symptoms and lack of specific molecular markers, it was previously difficult to distinguish it from other diseases with increased neutrophils. However, the discovery of the CSF3R mutation in CNL 10 years ago and the update of the diagnostic criteria by the World Health Organization (WHO) in 2016 brought CNL into a new era of molecular diagnosis. Next-generation sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37990009"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of subcutaneous patient-derived xenografts reveals immune constraints on tumor evolution in childhood solid cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37990009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Subcutaneous patient-derived xenografts (PDXs) are an important tool for childhood cancer research. Here, we describe a resource of 68 early passage PDXs established from 65 pediatric solid tumor patients. Through genomic profiling of paired PDXs and patient tumors (PTs), we observe low mutational similarity in about 30% of the PT/PDX pairs. Clonal analysis in these pairs show an aggressive PT minor subclone seeds the major clone in the PDX. We show evidence that this subclone is more...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37993585"
                        open
                      >
                        <summary class="article-expander__title">Research progress of additional pathogenic mutations in chronic neutrophilic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37993585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic neutrophilic leukemia (CNL) is a rare type of myeloproliferative neoplasm (MPN). Due to its nonspecific clinical symptoms and lack of specific molecular markers, it was previously difficult to distinguish it from other diseases with increased neutrophils. However, the discovery of the CSF3R mutation in CNL 10 years ago and the update of the diagnostic criteria by the World Health Organization (WHO) in 2016 brought CNL into a new era of molecular diagnosis. Next-generation sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37990009"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of subcutaneous patient-derived xenografts reveals immune constraints on tumor evolution in childhood solid cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37990009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Subcutaneous patient-derived xenografts (PDXs) are an important tool for childhood cancer research. Here, we describe a resource of 68 early passage PDXs established from 65 pediatric solid tumor patients. Through genomic profiling of paired PDXs and patient tumors (PTs), we observe low mutational similarity in about 30% of the PT/PDX pairs. Clonal analysis in these pairs show an aggressive PT minor subclone seeds the major clone in the PDX. We show evidence that this subclone is more...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37993585"
                        open
                      >
                        <summary class="article-expander__title">Research progress of additional pathogenic mutations in chronic neutrophilic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37993585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic neutrophilic leukemia (CNL) is a rare type of myeloproliferative neoplasm (MPN). Due to its nonspecific clinical symptoms and lack of specific molecular markers, it was previously difficult to distinguish it from other diseases with increased neutrophils. However, the discovery of the CSF3R mutation in CNL 10 years ago and the update of the diagnostic criteria by the World Health Organization (WHO) in 2016 brought CNL into a new era of molecular diagnosis. Next-generation sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37990009"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of subcutaneous patient-derived xenografts reveals immune constraints on tumor evolution in childhood solid cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37990009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Subcutaneous patient-derived xenografts (PDXs) are an important tool for childhood cancer research. Here, we describe a resource of 68 early passage PDXs established from 65 pediatric solid tumor patients. Through genomic profiling of paired PDXs and patient tumors (PTs), we observe low mutational similarity in about 30% of the PT/PDX pairs. Clonal analysis in these pairs show an aggressive PT minor subclone seeds the major clone in the PDX. We show evidence that this subclone is more...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37993585"
                        open
                      >
                        <summary class="article-expander__title">Research progress of additional pathogenic mutations in chronic neutrophilic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37993585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic neutrophilic leukemia (CNL) is a rare type of myeloproliferative neoplasm (MPN). Due to its nonspecific clinical symptoms and lack of specific molecular markers, it was previously difficult to distinguish it from other diseases with increased neutrophils. However, the discovery of the CSF3R mutation in CNL 10 years ago and the update of the diagnostic criteria by the World Health Organization (WHO) in 2016 brought CNL into a new era of molecular diagnosis. Next-generation sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37990009"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of subcutaneous patient-derived xenografts reveals immune constraints on tumor evolution in childhood solid cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37990009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Subcutaneous patient-derived xenografts (PDXs) are an important tool for childhood cancer research. Here, we describe a resource of 68 early passage PDXs established from 65 pediatric solid tumor patients. Through genomic profiling of paired PDXs and patient tumors (PTs), we observe low mutational similarity in about 30% of the PT/PDX pairs. Clonal analysis in these pairs show an aggressive PT minor subclone seeds the major clone in the PDX. We show evidence that this subclone is more...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37993585"
                        open
                      >
                        <summary class="article-expander__title">Research progress of additional pathogenic mutations in chronic neutrophilic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37993585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic neutrophilic leukemia (CNL) is a rare type of myeloproliferative neoplasm (MPN). Due to its nonspecific clinical symptoms and lack of specific molecular markers, it was previously difficult to distinguish it from other diseases with increased neutrophils. However, the discovery of the CSF3R mutation in CNL 10 years ago and the update of the diagnostic criteria by the World Health Organization (WHO) in 2016 brought CNL into a new era of molecular diagnosis. Next-generation sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37990009"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of subcutaneous patient-derived xenografts reveals immune constraints on tumor evolution in childhood solid cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37990009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Subcutaneous patient-derived xenografts (PDXs) are an important tool for childhood cancer research. Here, we describe a resource of 68 early passage PDXs established from 65 pediatric solid tumor patients. Through genomic profiling of paired PDXs and patient tumors (PTs), we observe low mutational similarity in about 30% of the PT/PDX pairs. Clonal analysis in these pairs show an aggressive PT minor subclone seeds the major clone in the PDX. We show evidence that this subclone is more...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37993585"
                        open
                      >
                        <summary class="article-expander__title">Research progress of additional pathogenic mutations in chronic neutrophilic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37993585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic neutrophilic leukemia (CNL) is a rare type of myeloproliferative neoplasm (MPN). Due to its nonspecific clinical symptoms and lack of specific molecular markers, it was previously difficult to distinguish it from other diseases with increased neutrophils. However, the discovery of the CSF3R mutation in CNL 10 years ago and the update of the diagnostic criteria by the World Health Organization (WHO) in 2016 brought CNL into a new era of molecular diagnosis. Next-generation sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37990009"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of subcutaneous patient-derived xenografts reveals immune constraints on tumor evolution in childhood solid cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37990009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Subcutaneous patient-derived xenografts (PDXs) are an important tool for childhood cancer research. Here, we describe a resource of 68 early passage PDXs established from 65 pediatric solid tumor patients. Through genomic profiling of paired PDXs and patient tumors (PTs), we observe low mutational similarity in about 30% of the PT/PDX pairs. Clonal analysis in these pairs show an aggressive PT minor subclone seeds the major clone in the PDX. We show evidence that this subclone is more...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37993585"
                        open
                      >
                        <summary class="article-expander__title">Research progress of additional pathogenic mutations in chronic neutrophilic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37993585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic neutrophilic leukemia (CNL) is a rare type of myeloproliferative neoplasm (MPN). Due to its nonspecific clinical symptoms and lack of specific molecular markers, it was previously difficult to distinguish it from other diseases with increased neutrophils. However, the discovery of the CSF3R mutation in CNL 10 years ago and the update of the diagnostic criteria by the World Health Organization (WHO) in 2016 brought CNL into a new era of molecular diagnosis. Next-generation sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37990009"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of subcutaneous patient-derived xenografts reveals immune constraints on tumor evolution in childhood solid cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37990009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Subcutaneous patient-derived xenografts (PDXs) are an important tool for childhood cancer research. Here, we describe a resource of 68 early passage PDXs established from 65 pediatric solid tumor patients. Through genomic profiling of paired PDXs and patient tumors (PTs), we observe low mutational similarity in about 30% of the PT/PDX pairs. Clonal analysis in these pairs show an aggressive PT minor subclone seeds the major clone in the PDX. We show evidence that this subclone is more...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37993585"
                        open
                      >
                        <summary class="article-expander__title">Research progress of additional pathogenic mutations in chronic neutrophilic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37993585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic neutrophilic leukemia (CNL) is a rare type of myeloproliferative neoplasm (MPN). Due to its nonspecific clinical symptoms and lack of specific molecular markers, it was previously difficult to distinguish it from other diseases with increased neutrophils. However, the discovery of the CSF3R mutation in CNL 10 years ago and the update of the diagnostic criteria by the World Health Organization (WHO) in 2016 brought CNL into a new era of molecular diagnosis. Next-generation sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37990009"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of subcutaneous patient-derived xenografts reveals immune constraints on tumor evolution in childhood solid cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37990009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Subcutaneous patient-derived xenografts (PDXs) are an important tool for childhood cancer research. Here, we describe a resource of 68 early passage PDXs established from 65 pediatric solid tumor patients. Through genomic profiling of paired PDXs and patient tumors (PTs), we observe low mutational similarity in about 30% of the PT/PDX pairs. Clonal analysis in these pairs show an aggressive PT minor subclone seeds the major clone in the PDX. We show evidence that this subclone is more...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37993585"
                        open
                      >
                        <summary class="article-expander__title">Research progress of additional pathogenic mutations in chronic neutrophilic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37993585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic neutrophilic leukemia (CNL) is a rare type of myeloproliferative neoplasm (MPN). Due to its nonspecific clinical symptoms and lack of specific molecular markers, it was previously difficult to distinguish it from other diseases with increased neutrophils. However, the discovery of the CSF3R mutation in CNL 10 years ago and the update of the diagnostic criteria by the World Health Organization (WHO) in 2016 brought CNL into a new era of molecular diagnosis. Next-generation sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37990009"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of subcutaneous patient-derived xenografts reveals immune constraints on tumor evolution in childhood solid cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37990009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Subcutaneous patient-derived xenografts (PDXs) are an important tool for childhood cancer research. Here, we describe a resource of 68 early passage PDXs established from 65 pediatric solid tumor patients. Through genomic profiling of paired PDXs and patient tumors (PTs), we observe low mutational similarity in about 30% of the PT/PDX pairs. Clonal analysis in these pairs show an aggressive PT minor subclone seeds the major clone in the PDX. We show evidence that this subclone is more...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37993585"
                        open
                      >
                        <summary class="article-expander__title">Research progress of additional pathogenic mutations in chronic neutrophilic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37993585/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic neutrophilic leukemia (CNL) is a rare type of myeloproliferative neoplasm (MPN). Due to its nonspecific clinical symptoms and lack of specific molecular markers, it was previously difficult to distinguish it from other diseases with increased neutrophils. However, the discovery of the CSF3R mutation in CNL 10 years ago and the update of the diagnostic criteria by the World Health Organization (WHO) in 2016 brought CNL into a new era of molecular diagnosis. Next-generation sequencing...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37990009"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of subcutaneous patient-derived xenografts reveals immune constraints on tumor evolution in childhood solid cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37990009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Subcutaneous patient-derived xenografts (PDXs) are an important tool for childhood cancer research. Here, we describe a resource of 68 early passage PDXs established from 65 pediatric solid tumor patients. Through genomic profiling of paired PDXs and patient tumors (PTs), we observe low mutational similarity in about 30% of the PT/PDX pairs. Clonal analysis in these pairs show an aggressive PT minor subclone seeds the major clone in the PDX. We show evidence that this subclone is more...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37990009"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling of subcutaneous patient-derived xenografts reveals immune constraints on tumor evolution in childhood solid cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37990009/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Subcutaneous patient-derived xenografts (PDXs) are an important tool for childhood cancer research. Here, we describe a resource of 68 early passage PDXs established from 65 pediatric solid tumor patients. Through genomic profiling of paired PDXs and patient tumors (PTs), we observe low mutational similarity in about 30% of the PT/PDX pairs. Clonal analysis in these pairs show an aggressive PT minor subclone seeds the major clone in the PDX. We show evidence that this subclone is more...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-21 datetime="2023-11-21T11:00:00.000Z">2023-11-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-21 datetime="2023-11-21T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986782"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231204195351&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986782"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231203195336&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986782"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231202195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986782"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986782"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986782"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986782"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986782"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986782"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986782"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986782"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986782"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986782"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986782"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective and scalable clonal hematopoiesis assay provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986782/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) is a common age-related phenomenon that occurs when hematopoietic stem cells acquire mutations in a select set of genes commonly mutated in myeloid neoplasia which then expand clonally. Current sequencing assays to detect CHIP are not optimized for the detection of these variants and can be cost-prohibitive when applied to large cohorts or serial sequencing. Here, we present and validate a CHIP targeted sequencing assay that is affordable...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37989729"
                        open
                      >
                        <summary class="article-expander__title">Targetable lesions and proteomes predict therapy sensitivity through disease evolution in pediatric acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37989729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Childhood acute lymphoblastic leukemia (ALL) genomes show that relapses often arise from subclonal outgrowths. However, the impact of clonal evolution on the actionable proteome and response to targeted therapy is not known. Here, we present a comprehensive retrospective analysis of paired ALL diagnosis and relapsed specimen. Targeted next generation sequencing and proteome analysis indicate persistence of actionable genome variants and stable proteomes through disease progression. Paired...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37989729"
                        open
                      >
                        <summary class="article-expander__title">Targetable lesions and proteomes predict therapy sensitivity through disease evolution in pediatric acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37989729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Childhood acute lymphoblastic leukemia (ALL) genomes show that relapses often arise from subclonal outgrowths. However, the impact of clonal evolution on the actionable proteome and response to targeted therapy is not known. Here, we present a comprehensive retrospective analysis of paired ALL diagnosis and relapsed specimen. Targeted next generation sequencing and proteome analysis indicate persistence of actionable genome variants and stable proteomes through disease progression. Paired...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986965"
                        open
                      >
                        <summary class="article-expander__title">Modeling the effect of spatial structure on solid tumor evolution and ctDNA composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform on cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37989729"
                        open
                      >
                        <summary class="article-expander__title">Targetable lesions and proteomes predict therapy sensitivity through disease evolution in pediatric acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37989729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Childhood acute lymphoblastic leukemia (ALL) genomes show that relapses often arise from subclonal outgrowths. However, the impact of clonal evolution on the actionable proteome and response to targeted therapy is not known. Here, we present a comprehensive retrospective analysis of paired ALL diagnosis and relapsed specimen. Targeted next generation sequencing and proteome analysis indicate persistence of actionable genome variants and stable proteomes through disease progression. Paired...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986965"
                        open
                      >
                        <summary class="article-expander__title">Modeling the effect of spatial structure on solid tumor evolution and ctDNA composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform on cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37989729"
                        open
                      >
                        <summary class="article-expander__title">Targetable lesions and proteomes predict therapy sensitivity through disease evolution in pediatric acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37989729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Childhood acute lymphoblastic leukemia (ALL) genomes show that relapses often arise from subclonal outgrowths. However, the impact of clonal evolution on the actionable proteome and response to targeted therapy is not known. Here, we present a comprehensive retrospective analysis of paired ALL diagnosis and relapsed specimen. Targeted next generation sequencing and proteome analysis indicate persistence of actionable genome variants and stable proteomes through disease progression. Paired...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986965"
                        open
                      >
                        <summary class="article-expander__title">Modeling the effect of spatial structure on solid tumor evolution and ctDNA composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform on cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37989729"
                        open
                      >
                        <summary class="article-expander__title">Targetable lesions and proteomes predict therapy sensitivity through disease evolution in pediatric acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37989729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Childhood acute lymphoblastic leukemia (ALL) genomes show that relapses often arise from subclonal outgrowths. However, the impact of clonal evolution on the actionable proteome and response to targeted therapy is not known. Here, we present a comprehensive retrospective analysis of paired ALL diagnosis and relapsed specimen. Targeted next generation sequencing and proteome analysis indicate persistence of actionable genome variants and stable proteomes through disease progression. Paired...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986965"
                        open
                      >
                        <summary class="article-expander__title">Modeling the effect of spatial structure on solid tumor evolution and ctDNA composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform on cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37989729"
                        open
                      >
                        <summary class="article-expander__title">Targetable lesions and proteomes predict therapy sensitivity through disease evolution in pediatric acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37989729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Childhood acute lymphoblastic leukemia (ALL) genomes show that relapses often arise from subclonal outgrowths. However, the impact of clonal evolution on the actionable proteome and response to targeted therapy is not known. Here, we present a comprehensive retrospective analysis of paired ALL diagnosis and relapsed specimen. Targeted next generation sequencing and proteome analysis indicate persistence of actionable genome variants and stable proteomes through disease progression. Paired...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986965"
                        open
                      >
                        <summary class="article-expander__title">Modeling the effect of spatial structure on solid tumor evolution and ctDNA composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform on cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37989729"
                        open
                      >
                        <summary class="article-expander__title">Targetable lesions and proteomes predict therapy sensitivity through disease evolution in pediatric acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37989729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Childhood acute lymphoblastic leukemia (ALL) genomes show that relapses often arise from subclonal outgrowths. However, the impact of clonal evolution on the actionable proteome and response to targeted therapy is not known. Here, we present a comprehensive retrospective analysis of paired ALL diagnosis and relapsed specimen. Targeted next generation sequencing and proteome analysis indicate persistence of actionable genome variants and stable proteomes through disease progression. Paired...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986965"
                        open
                      >
                        <summary class="article-expander__title">Modeling the effect of spatial structure on solid tumor evolution and ctDNA composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform on cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37989729"
                        open
                      >
                        <summary class="article-expander__title">Targetable lesions and proteomes predict therapy sensitivity through disease evolution in pediatric acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37989729/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Childhood acute lymphoblastic leukemia (ALL) genomes show that relapses often arise from subclonal outgrowths. However, the impact of clonal evolution on the actionable proteome and response to targeted therapy is not known. Here, we present a comprehensive retrospective analysis of paired ALL diagnosis and relapsed specimen. Targeted next generation sequencing and proteome analysis indicate persistence of actionable genome variants and stable proteomes through disease progression. Paired...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986965"
                        open
                      >
                        <summary class="article-expander__title">Modeling the effect of spatial structure on solid tumor evolution and ctDNA composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform on cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37986965"
                        open
                      >
                        <summary class="article-expander__title">Modeling the effect of spatial structure on solid tumor evolution and ctDNA composition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37986965/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform on cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-20 datetime="2023-11-20T11:00:00.000Z">2023-11-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-20 datetime="2023-11-20T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37982261"
                        open
                      >
                        <summary class="article-expander__title">Transfusion avoidance in myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37982261/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231201195140&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myelodysplastic neoplasms (MDS) are diseases of stem cell aging associated with complications from inadequate hematopoiesis (red cells, neutrophils and platelets) and variable risk for transformation to acute myeloid leukemia. Those with low-risk disease also suffer and die from MDS-related complications. Among the most challenging is development of anemia and transfusion dependence, which impacts quality of life and is associated with reduced survival. Appreciating and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37982261"
                        open
                      >
                        <summary class="article-expander__title">Transfusion avoidance in myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37982261/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myelodysplastic neoplasms (MDS) are diseases of stem cell aging associated with complications from inadequate hematopoiesis (red cells, neutrophils and platelets) and variable risk for transformation to acute myeloid leukemia. Those with low-risk disease also suffer and die from MDS-related complications. Among the most challenging is development of anemia and transfusion dependence, which impacts quality of life and is associated with reduced survival. Appreciating and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37982261"
                        open
                      >
                        <summary class="article-expander__title">Transfusion avoidance in myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37982261/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myelodysplastic neoplasms (MDS) are diseases of stem cell aging associated with complications from inadequate hematopoiesis (red cells, neutrophils and platelets) and variable risk for transformation to acute myeloid leukemia. Those with low-risk disease also suffer and die from MDS-related complications. Among the most challenging is development of anemia and transfusion dependence, which impacts quality of life and is associated with reduced survival. Appreciating and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37982261"
                        open
                      >
                        <summary class="article-expander__title">Transfusion avoidance in myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37982261/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myelodysplastic neoplasms (MDS) are diseases of stem cell aging associated with complications from inadequate hematopoiesis (red cells, neutrophils and platelets) and variable risk for transformation to acute myeloid leukemia. Those with low-risk disease also suffer and die from MDS-related complications. Among the most challenging is development of anemia and transfusion dependence, which impacts quality of life and is associated with reduced survival. Appreciating and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37982261"
                        open
                      >
                        <summary class="article-expander__title">Transfusion avoidance in myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37982261/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myelodysplastic neoplasms (MDS) are diseases of stem cell aging associated with complications from inadequate hematopoiesis (red cells, neutrophils and platelets) and variable risk for transformation to acute myeloid leukemia. Those with low-risk disease also suffer and die from MDS-related complications. Among the most challenging is development of anemia and transfusion dependence, which impacts quality of life and is associated with reduced survival. Appreciating and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37982261"
                        open
                      >
                        <summary class="article-expander__title">Transfusion avoidance in myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37982261/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myelodysplastic neoplasms (MDS) are diseases of stem cell aging associated with complications from inadequate hematopoiesis (red cells, neutrophils and platelets) and variable risk for transformation to acute myeloid leukemia. Those with low-risk disease also suffer and die from MDS-related complications. Among the most challenging is development of anemia and transfusion dependence, which impacts quality of life and is associated with reduced survival. Appreciating and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37982261"
                        open
                      >
                        <summary class="article-expander__title">Transfusion avoidance in myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37982261/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myelodysplastic neoplasms (MDS) are diseases of stem cell aging associated with complications from inadequate hematopoiesis (red cells, neutrophils and platelets) and variable risk for transformation to acute myeloid leukemia. Those with low-risk disease also suffer and die from MDS-related complications. Among the most challenging is development of anemia and transfusion dependence, which impacts quality of life and is associated with reduced survival. Appreciating and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37982261"
                        open
                      >
                        <summary class="article-expander__title">Transfusion avoidance in myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37982261/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myelodysplastic neoplasms (MDS) are diseases of stem cell aging associated with complications from inadequate hematopoiesis (red cells, neutrophils and platelets) and variable risk for transformation to acute myeloid leukemia. Those with low-risk disease also suffer and die from MDS-related complications. Among the most challenging is development of anemia and transfusion dependence, which impacts quality of life and is associated with reduced survival. Appreciating and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37982261"
                        open
                      >
                        <summary class="article-expander__title">Transfusion avoidance in myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37982261/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myelodysplastic neoplasms (MDS) are diseases of stem cell aging associated with complications from inadequate hematopoiesis (red cells, neutrophils and platelets) and variable risk for transformation to acute myeloid leukemia. Those with low-risk disease also suffer and die from MDS-related complications. Among the most challenging is development of anemia and transfusion dependence, which impacts quality of life and is associated with reduced survival. Appreciating and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37982261"
                        open
                      >
                        <summary class="article-expander__title">Transfusion avoidance in myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37982261/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myelodysplastic neoplasms (MDS) are diseases of stem cell aging associated with complications from inadequate hematopoiesis (red cells, neutrophils and platelets) and variable risk for transformation to acute myeloid leukemia. Those with low-risk disease also suffer and die from MDS-related complications. Among the most challenging is development of anemia and transfusion dependence, which impacts quality of life and is associated with reduced survival. Appreciating and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37982261"
                        open
                      >
                        <summary class="article-expander__title">Transfusion avoidance in myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37982261/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myelodysplastic neoplasms (MDS) are diseases of stem cell aging associated with complications from inadequate hematopoiesis (red cells, neutrophils and platelets) and variable risk for transformation to acute myeloid leukemia. Those with low-risk disease also suffer and die from MDS-related complications. Among the most challenging is development of anemia and transfusion dependence, which impacts quality of life and is associated with reduced survival. Appreciating and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37982261"
                        open
                      >
                        <summary class="article-expander__title">Transfusion avoidance in myelodysplastic neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37982261/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Myelodysplastic neoplasms (MDS) are diseases of stem cell aging associated with complications from inadequate hematopoiesis (red cells, neutrophils and platelets) and variable risk for transformation to acute myeloid leukemia. Those with low-risk disease also suffer and die from MDS-related complications. Among the most challenging is development of anemia and transfusion dependence, which impacts quality of life and is associated with reduced survival. Appreciating and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-18 datetime="2023-11-18T11:00:00.000Z">2023-11-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-18 datetime="2023-11-18T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37980303"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of a novel dic(18;20) clone following allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37980303/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 55-year-old man in first complete remission of acute myeloid leukemia with a normal karyotype underwent allogeneic hematopoietic stem cell transplantation from a human-leukocyte-antigen-matched sibling. Bone marrow examination on day 28 confirmed complete remission, but G-banding analysis revealed a novel chromosomal abnormality, including dic(18;20)(p11.2;q11.2). The patient developed moderate chronic graft-versus-host disease on day 174, and the abnormal clones identified by dic(18;20)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37980303"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of a novel dic(18;20) clone following allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37980303/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 55-year-old man in first complete remission of acute myeloid leukemia with a normal karyotype underwent allogeneic hematopoietic stem cell transplantation from a human-leukocyte-antigen-matched sibling. Bone marrow examination on day 28 confirmed complete remission, but G-banding analysis revealed a novel chromosomal abnormality, including dic(18;20)(p11.2;q11.2). The patient developed moderate chronic graft-versus-host disease on day 174, and the abnormal clones identified by dic(18;20)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37980303"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of a novel dic(18;20) clone following allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37980303/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 55-year-old man in first complete remission of acute myeloid leukemia with a normal karyotype underwent allogeneic hematopoietic stem cell transplantation from a human-leukocyte-antigen-matched sibling. Bone marrow examination on day 28 confirmed complete remission, but G-banding analysis revealed a novel chromosomal abnormality, including dic(18;20)(p11.2;q11.2). The patient developed moderate chronic graft-versus-host disease on day 174, and the abnormal clones identified by dic(18;20)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37980303"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of a novel dic(18;20) clone following allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37980303/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 55-year-old man in first complete remission of acute myeloid leukemia with a normal karyotype underwent allogeneic hematopoietic stem cell transplantation from a human-leukocyte-antigen-matched sibling. Bone marrow examination on day 28 confirmed complete remission, but G-banding analysis revealed a novel chromosomal abnormality, including dic(18;20)(p11.2;q11.2). The patient developed moderate chronic graft-versus-host disease on day 174, and the abnormal clones identified by dic(18;20)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37980303"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of a novel dic(18;20) clone following allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37980303/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 55-year-old man in first complete remission of acute myeloid leukemia with a normal karyotype underwent allogeneic hematopoietic stem cell transplantation from a human-leukocyte-antigen-matched sibling. Bone marrow examination on day 28 confirmed complete remission, but G-banding analysis revealed a novel chromosomal abnormality, including dic(18;20)(p11.2;q11.2). The patient developed moderate chronic graft-versus-host disease on day 174, and the abnormal clones identified by dic(18;20)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37980303"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of a novel dic(18;20) clone following allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37980303/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 55-year-old man in first complete remission of acute myeloid leukemia with a normal karyotype underwent allogeneic hematopoietic stem cell transplantation from a human-leukocyte-antigen-matched sibling. Bone marrow examination on day 28 confirmed complete remission, but G-banding analysis revealed a novel chromosomal abnormality, including dic(18;20)(p11.2;q11.2). The patient developed moderate chronic graft-versus-host disease on day 174, and the abnormal clones identified by dic(18;20)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37980303"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of a novel dic(18;20) clone following allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37980303/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 55-year-old man in first complete remission of acute myeloid leukemia with a normal karyotype underwent allogeneic hematopoietic stem cell transplantation from a human-leukocyte-antigen-matched sibling. Bone marrow examination on day 28 confirmed complete remission, but G-banding analysis revealed a novel chromosomal abnormality, including dic(18;20)(p11.2;q11.2). The patient developed moderate chronic graft-versus-host disease on day 174, and the abnormal clones identified by dic(18;20)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37980303"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of a novel dic(18;20) clone following allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37980303/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 55-year-old man in first complete remission of acute myeloid leukemia with a normal karyotype underwent allogeneic hematopoietic stem cell transplantation from a human-leukocyte-antigen-matched sibling. Bone marrow examination on day 28 confirmed complete remission, but G-banding analysis revealed a novel chromosomal abnormality, including dic(18;20)(p11.2;q11.2). The patient developed moderate chronic graft-versus-host disease on day 174, and the abnormal clones identified by dic(18;20)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37980303"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of a novel dic(18;20) clone following allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37980303/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 55-year-old man in first complete remission of acute myeloid leukemia with a normal karyotype underwent allogeneic hematopoietic stem cell transplantation from a human-leukocyte-antigen-matched sibling. Bone marrow examination on day 28 confirmed complete remission, but G-banding analysis revealed a novel chromosomal abnormality, including dic(18;20)(p11.2;q11.2). The patient developed moderate chronic graft-versus-host disease on day 174, and the abnormal clones identified by dic(18;20)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37980303"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of a novel dic(18;20) clone following allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37980303/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 55-year-old man in first complete remission of acute myeloid leukemia with a normal karyotype underwent allogeneic hematopoietic stem cell transplantation from a human-leukocyte-antigen-matched sibling. Bone marrow examination on day 28 confirmed complete remission, but G-banding analysis revealed a novel chromosomal abnormality, including dic(18;20)(p11.2;q11.2). The patient developed moderate chronic graft-versus-host disease on day 174, and the abnormal clones identified by dic(18;20)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37980303"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of a novel dic(18;20) clone following allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37980303/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 55-year-old man in first complete remission of acute myeloid leukemia with a normal karyotype underwent allogeneic hematopoietic stem cell transplantation from a human-leukocyte-antigen-matched sibling. Bone marrow examination on day 28 confirmed complete remission, but G-banding analysis revealed a novel chromosomal abnormality, including dic(18;20)(p11.2;q11.2). The patient developed moderate chronic graft-versus-host disease on day 174, and the abnormal clones identified by dic(18;20)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37980303"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of a novel dic(18;20) clone following allogeneic hematopoietic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37980303/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A 55-year-old man in first complete remission of acute myeloid leukemia with a normal karyotype underwent allogeneic hematopoietic stem cell transplantation from a human-leukocyte-antigen-matched sibling. Bone marrow examination on day 28 confirmed complete remission, but G-banding analysis revealed a novel chromosomal abnormality, including dic(18;20)(p11.2;q11.2). The patient developed moderate chronic graft-versus-host disease on day 174, and the abnormal clones identified by dic(18;20)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-17 datetime="2023-11-17T11:00:00.000Z">2023-11-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-17 datetime="2023-11-17T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37976039"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in Persons living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37976039/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231130195732&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In older, well-treated, Scandinavian PLWH, more than one in four had at least one CHIP mutation. We did not find evidence of an association between CHIP and inflammatory markers or between CHIP and CAD. CHIP is an unlikely underlying mechanism to explain the association between inflammation and CAD in treated HIV disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37976039"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in Persons living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37976039/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231129195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In older, well-treated, Scandinavian PLWH, more than one in four had at least one CHIP mutation. We did not find evidence of an association between CHIP and inflammatory markers or between CHIP and CAD. CHIP is an unlikely underlying mechanism to explain the association between inflammation and CAD in treated HIV disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37976039"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in Persons living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37976039/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In older, well-treated, Scandinavian PLWH, more than one in four had at least one CHIP mutation. We did not find evidence of an association between CHIP and inflammatory markers or between CHIP and CAD. CHIP is an unlikely underlying mechanism to explain the association between inflammation and CAD in treated HIV disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973947"
                        open
                      >
                        <summary class="article-expander__title">Clonal selection of hematopoietic stem cells after gene therapy for sickle cell disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gene therapy (GT) provides a potentially curative treatment option for patients with sickle cell disease (SCD); however, the occurrence of myeloid malignancies in GT clinical trials has prompted concern, with several postulated mechanisms. Here, we used whole-genome sequencing to track hematopoietic stem cells (HSCs) from six patients with SCD at pre- and post-GT time points to map the somatic mutation and clonal landscape of gene-modified and unmodified HSCs. Pre-GT, phylogenetic trees were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37976039"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in Persons living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37976039/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In older, well-treated, Scandinavian PLWH, more than one in four had at least one CHIP mutation. We did not find evidence of an association between CHIP and inflammatory markers or between CHIP and CAD. CHIP is an unlikely underlying mechanism to explain the association between inflammation and CAD in treated HIV disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973947"
                        open
                      >
                        <summary class="article-expander__title">Clonal selection of hematopoietic stem cells after gene therapy for sickle cell disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gene therapy (GT) provides a potentially curative treatment option for patients with sickle cell disease (SCD); however, the occurrence of myeloid malignancies in GT clinical trials has prompted concern, with several postulated mechanisms. Here, we used whole-genome sequencing to track hematopoietic stem cells (HSCs) from six patients with SCD at pre- and post-GT time points to map the somatic mutation and clonal landscape of gene-modified and unmodified HSCs. Pre-GT, phylogenetic trees were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37976039"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in Persons living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37976039/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In older, well-treated, Scandinavian PLWH, more than one in four had at least one CHIP mutation. We did not find evidence of an association between CHIP and inflammatory markers or between CHIP and CAD. CHIP is an unlikely underlying mechanism to explain the association between inflammation and CAD in treated HIV disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973947"
                        open
                      >
                        <summary class="article-expander__title">Clonal selection of hematopoietic stem cells after gene therapy for sickle cell disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gene therapy (GT) provides a potentially curative treatment option for patients with sickle cell disease (SCD); however, the occurrence of myeloid malignancies in GT clinical trials has prompted concern, with several postulated mechanisms. Here, we used whole-genome sequencing to track hematopoietic stem cells (HSCs) from six patients with SCD at pre- and post-GT time points to map the somatic mutation and clonal landscape of gene-modified and unmodified HSCs. Pre-GT, phylogenetic trees were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37976039"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in Persons living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37976039/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In older, well-treated, Scandinavian PLWH, more than one in four had at least one CHIP mutation. We did not find evidence of an association between CHIP and inflammatory markers or between CHIP and CAD. CHIP is an unlikely underlying mechanism to explain the association between inflammation and CAD in treated HIV disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973947"
                        open
                      >
                        <summary class="article-expander__title">Clonal selection of hematopoietic stem cells after gene therapy for sickle cell disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gene therapy (GT) provides a potentially curative treatment option for patients with sickle cell disease (SCD); however, the occurrence of myeloid malignancies in GT clinical trials has prompted concern, with several postulated mechanisms. Here, we used whole-genome sequencing to track hematopoietic stem cells (HSCs) from six patients with SCD at pre- and post-GT time points to map the somatic mutation and clonal landscape of gene-modified and unmodified HSCs. Pre-GT, phylogenetic trees were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37976039"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in Persons living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37976039/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In older, well-treated, Scandinavian PLWH, more than one in four had at least one CHIP mutation. We did not find evidence of an association between CHIP and inflammatory markers or between CHIP and CAD. CHIP is an unlikely underlying mechanism to explain the association between inflammation and CAD in treated HIV disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973947"
                        open
                      >
                        <summary class="article-expander__title">Clonal selection of hematopoietic stem cells after gene therapy for sickle cell disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gene therapy (GT) provides a potentially curative treatment option for patients with sickle cell disease (SCD); however, the occurrence of myeloid malignancies in GT clinical trials has prompted concern, with several postulated mechanisms. Here, we used whole-genome sequencing to track hematopoietic stem cells (HSCs) from six patients with SCD at pre- and post-GT time points to map the somatic mutation and clonal landscape of gene-modified and unmodified HSCs. Pre-GT, phylogenetic trees were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37976039"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in Persons living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37976039/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In older, well-treated, Scandinavian PLWH, more than one in four had at least one CHIP mutation. We did not find evidence of an association between CHIP and inflammatory markers or between CHIP and CAD. CHIP is an unlikely underlying mechanism to explain the association between inflammation and CAD in treated HIV disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973947"
                        open
                      >
                        <summary class="article-expander__title">Clonal selection of hematopoietic stem cells after gene therapy for sickle cell disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gene therapy (GT) provides a potentially curative treatment option for patients with sickle cell disease (SCD); however, the occurrence of myeloid malignancies in GT clinical trials has prompted concern, with several postulated mechanisms. Here, we used whole-genome sequencing to track hematopoietic stem cells (HSCs) from six patients with SCD at pre- and post-GT time points to map the somatic mutation and clonal landscape of gene-modified and unmodified HSCs. Pre-GT, phylogenetic trees were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37976039"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in Persons living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37976039/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In older, well-treated, Scandinavian PLWH, more than one in four had at least one CHIP mutation. We did not find evidence of an association between CHIP and inflammatory markers or between CHIP and CAD. CHIP is an unlikely underlying mechanism to explain the association between inflammation and CAD in treated HIV disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973947"
                        open
                      >
                        <summary class="article-expander__title">Clonal selection of hematopoietic stem cells after gene therapy for sickle cell disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gene therapy (GT) provides a potentially curative treatment option for patients with sickle cell disease (SCD); however, the occurrence of myeloid malignancies in GT clinical trials has prompted concern, with several postulated mechanisms. Here, we used whole-genome sequencing to track hematopoietic stem cells (HSCs) from six patients with SCD at pre- and post-GT time points to map the somatic mutation and clonal landscape of gene-modified and unmodified HSCs. Pre-GT, phylogenetic trees were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37976039"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in Persons living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37976039/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In older, well-treated, Scandinavian PLWH, more than one in four had at least one CHIP mutation. We did not find evidence of an association between CHIP and inflammatory markers or between CHIP and CAD. CHIP is an unlikely underlying mechanism to explain the association between inflammation and CAD in treated HIV disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973947"
                        open
                      >
                        <summary class="article-expander__title">Clonal selection of hematopoietic stem cells after gene therapy for sickle cell disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gene therapy (GT) provides a potentially curative treatment option for patients with sickle cell disease (SCD); however, the occurrence of myeloid malignancies in GT clinical trials has prompted concern, with several postulated mechanisms. Here, we used whole-genome sequencing to track hematopoietic stem cells (HSCs) from six patients with SCD at pre- and post-GT time points to map the somatic mutation and clonal landscape of gene-modified and unmodified HSCs. Pre-GT, phylogenetic trees were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37976039"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in Persons living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37976039/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In older, well-treated, Scandinavian PLWH, more than one in four had at least one CHIP mutation. We did not find evidence of an association between CHIP and inflammatory markers or between CHIP and CAD. CHIP is an unlikely underlying mechanism to explain the association between inflammation and CAD in treated HIV disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973947"
                        open
                      >
                        <summary class="article-expander__title">Clonal selection of hematopoietic stem cells after gene therapy for sickle cell disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gene therapy (GT) provides a potentially curative treatment option for patients with sickle cell disease (SCD); however, the occurrence of myeloid malignancies in GT clinical trials has prompted concern, with several postulated mechanisms. Here, we used whole-genome sequencing to track hematopoietic stem cells (HSCs) from six patients with SCD at pre- and post-GT time points to map the somatic mutation and clonal landscape of gene-modified and unmodified HSCs. Pre-GT, phylogenetic trees were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37976039"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in Persons living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37976039/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In older, well-treated, Scandinavian PLWH, more than one in four had at least one CHIP mutation. We did not find evidence of an association between CHIP and inflammatory markers or between CHIP and CAD. CHIP is an unlikely underlying mechanism to explain the association between inflammation and CAD in treated HIV disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973947"
                        open
                      >
                        <summary class="article-expander__title">Clonal selection of hematopoietic stem cells after gene therapy for sickle cell disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gene therapy (GT) provides a potentially curative treatment option for patients with sickle cell disease (SCD); however, the occurrence of myeloid malignancies in GT clinical trials has prompted concern, with several postulated mechanisms. Here, we used whole-genome sequencing to track hematopoietic stem cells (HSCs) from six patients with SCD at pre- and post-GT time points to map the somatic mutation and clonal landscape of gene-modified and unmodified HSCs. Pre-GT, phylogenetic trees were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37976039"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in Persons living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37976039/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In older, well-treated, Scandinavian PLWH, more than one in four had at least one CHIP mutation. We did not find evidence of an association between CHIP and inflammatory markers or between CHIP and CAD. CHIP is an unlikely underlying mechanism to explain the association between inflammation and CAD in treated HIV disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973947"
                        open
                      >
                        <summary class="article-expander__title">Clonal selection of hematopoietic stem cells after gene therapy for sickle cell disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gene therapy (GT) provides a potentially curative treatment option for patients with sickle cell disease (SCD); however, the occurrence of myeloid malignancies in GT clinical trials has prompted concern, with several postulated mechanisms. Here, we used whole-genome sequencing to track hematopoietic stem cells (HSCs) from six patients with SCD at pre- and post-GT time points to map the somatic mutation and clonal landscape of gene-modified and unmodified HSCs. Pre-GT, phylogenetic trees were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37976039"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in Persons living with HIV</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37976039/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In older, well-treated, Scandinavian PLWH, more than one in four had at least one CHIP mutation. We did not find evidence of an association between CHIP and inflammatory markers or between CHIP and CAD. CHIP is an unlikely underlying mechanism to explain the association between inflammation and CAD in treated HIV disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973947"
                        open
                      >
                        <summary class="article-expander__title">Clonal selection of hematopoietic stem cells after gene therapy for sickle cell disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973947/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gene therapy (GT) provides a potentially curative treatment option for patients with sickle cell disease (SCD); however, the occurrence of myeloid malignancies in GT clinical trials has prompted concern, with several postulated mechanisms. Here, we used whole-genome sequencing to track hematopoietic stem cells (HSCs) from six patients with SCD at pre- and post-GT time points to map the somatic mutation and clonal landscape of gene-modified and unmodified HSCs. Pre-GT, phylogenetic trees were...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973922"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling and pre-clinical modelling of breast cancer leptomeningeal metastasis reveals acquisition of a lobular-like phenotype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Breast cancer leptomeningeal metastasis (BCLM), where tumour cells grow along the lining of the brain and spinal cord, is a devastating development for patients. Investigating this metastatic site is hampered by difficulty in accessing tumour material. Here, we utilise cerebrospinal fluid (CSF) cell-free DNA (cfDNA) and CSF disseminated tumour cells (DTCs) to explore the clonal evolution of BCLM and heterogeneity between leptomeningeal and extracranial metastatic sites. Somatic alterations with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973922"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling and pre-clinical modelling of breast cancer leptomeningeal metastasis reveals acquisition of a lobular-like phenotype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Breast cancer leptomeningeal metastasis (BCLM), where tumour cells grow along the lining of the brain and spinal cord, is a devastating development for patients. Investigating this metastatic site is hampered by difficulty in accessing tumour material. Here, we utilise cerebrospinal fluid (CSF) cell-free DNA (cfDNA) and CSF disseminated tumour cells (DTCs) to explore the clonal evolution of BCLM and heterogeneity between leptomeningeal and extracranial metastatic sites. Somatic alterations with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973922"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling and pre-clinical modelling of breast cancer leptomeningeal metastasis reveals acquisition of a lobular-like phenotype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Breast cancer leptomeningeal metastasis (BCLM), where tumour cells grow along the lining of the brain and spinal cord, is a devastating development for patients. Investigating this metastatic site is hampered by difficulty in accessing tumour material. Here, we utilise cerebrospinal fluid (CSF) cell-free DNA (cfDNA) and CSF disseminated tumour cells (DTCs) to explore the clonal evolution of BCLM and heterogeneity between leptomeningeal and extracranial metastatic sites. Somatic alterations with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973922"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling and pre-clinical modelling of breast cancer leptomeningeal metastasis reveals acquisition of a lobular-like phenotype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Breast cancer leptomeningeal metastasis (BCLM), where tumour cells grow along the lining of the brain and spinal cord, is a devastating development for patients. Investigating this metastatic site is hampered by difficulty in accessing tumour material. Here, we utilise cerebrospinal fluid (CSF) cell-free DNA (cfDNA) and CSF disseminated tumour cells (DTCs) to explore the clonal evolution of BCLM and heterogeneity between leptomeningeal and extracranial metastatic sites. Somatic alterations with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973922"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling and pre-clinical modelling of breast cancer leptomeningeal metastasis reveals acquisition of a lobular-like phenotype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Breast cancer leptomeningeal metastasis (BCLM), where tumour cells grow along the lining of the brain and spinal cord, is a devastating development for patients. Investigating this metastatic site is hampered by difficulty in accessing tumour material. Here, we utilise cerebrospinal fluid (CSF) cell-free DNA (cfDNA) and CSF disseminated tumour cells (DTCs) to explore the clonal evolution of BCLM and heterogeneity between leptomeningeal and extracranial metastatic sites. Somatic alterations with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973922"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling and pre-clinical modelling of breast cancer leptomeningeal metastasis reveals acquisition of a lobular-like phenotype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Breast cancer leptomeningeal metastasis (BCLM), where tumour cells grow along the lining of the brain and spinal cord, is a devastating development for patients. Investigating this metastatic site is hampered by difficulty in accessing tumour material. Here, we utilise cerebrospinal fluid (CSF) cell-free DNA (cfDNA) and CSF disseminated tumour cells (DTCs) to explore the clonal evolution of BCLM and heterogeneity between leptomeningeal and extracranial metastatic sites. Somatic alterations with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973922"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling and pre-clinical modelling of breast cancer leptomeningeal metastasis reveals acquisition of a lobular-like phenotype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Breast cancer leptomeningeal metastasis (BCLM), where tumour cells grow along the lining of the brain and spinal cord, is a devastating development for patients. Investigating this metastatic site is hampered by difficulty in accessing tumour material. Here, we utilise cerebrospinal fluid (CSF) cell-free DNA (cfDNA) and CSF disseminated tumour cells (DTCs) to explore the clonal evolution of BCLM and heterogeneity between leptomeningeal and extracranial metastatic sites. Somatic alterations with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973922"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling and pre-clinical modelling of breast cancer leptomeningeal metastasis reveals acquisition of a lobular-like phenotype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Breast cancer leptomeningeal metastasis (BCLM), where tumour cells grow along the lining of the brain and spinal cord, is a devastating development for patients. Investigating this metastatic site is hampered by difficulty in accessing tumour material. Here, we utilise cerebrospinal fluid (CSF) cell-free DNA (cfDNA) and CSF disseminated tumour cells (DTCs) to explore the clonal evolution of BCLM and heterogeneity between leptomeningeal and extracranial metastatic sites. Somatic alterations with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973922"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling and pre-clinical modelling of breast cancer leptomeningeal metastasis reveals acquisition of a lobular-like phenotype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Breast cancer leptomeningeal metastasis (BCLM), where tumour cells grow along the lining of the brain and spinal cord, is a devastating development for patients. Investigating this metastatic site is hampered by difficulty in accessing tumour material. Here, we utilise cerebrospinal fluid (CSF) cell-free DNA (cfDNA) and CSF disseminated tumour cells (DTCs) to explore the clonal evolution of BCLM and heterogeneity between leptomeningeal and extracranial metastatic sites. Somatic alterations with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973922"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling and pre-clinical modelling of breast cancer leptomeningeal metastasis reveals acquisition of a lobular-like phenotype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Breast cancer leptomeningeal metastasis (BCLM), where tumour cells grow along the lining of the brain and spinal cord, is a devastating development for patients. Investigating this metastatic site is hampered by difficulty in accessing tumour material. Here, we utilise cerebrospinal fluid (CSF) cell-free DNA (cfDNA) and CSF disseminated tumour cells (DTCs) to explore the clonal evolution of BCLM and heterogeneity between leptomeningeal and extracranial metastatic sites. Somatic alterations with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973922"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling and pre-clinical modelling of breast cancer leptomeningeal metastasis reveals acquisition of a lobular-like phenotype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Breast cancer leptomeningeal metastasis (BCLM), where tumour cells grow along the lining of the brain and spinal cord, is a devastating development for patients. Investigating this metastatic site is hampered by difficulty in accessing tumour material. Here, we utilise cerebrospinal fluid (CSF) cell-free DNA (cfDNA) and CSF disseminated tumour cells (DTCs) to explore the clonal evolution of BCLM and heterogeneity between leptomeningeal and extracranial metastatic sites. Somatic alterations with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37973922"
                        open
                      >
                        <summary class="article-expander__title">Genomic profiling and pre-clinical modelling of breast cancer leptomeningeal metastasis reveals acquisition of a lobular-like phenotype</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37973922/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Breast cancer leptomeningeal metastasis (BCLM), where tumour cells grow along the lining of the brain and spinal cord, is a devastating development for patients. Investigating this metastatic site is hampered by difficulty in accessing tumour material. Here, we utilise cerebrospinal fluid (CSF) cell-free DNA (cfDNA) and CSF disseminated tumour cells (DTCs) to explore the clonal evolution of BCLM and heterogeneity between leptomeningeal and extracranial metastatic sites. Somatic alterations with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-16 datetime="2023-11-16T11:00:00.000Z">2023-11-16</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-16 datetime="2023-11-16T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37972368"
                        open
                      >
                        <summary class="article-expander__title">Cardiac and autologous stem cell transplantation hematopoietic in patients with AL amyloidosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37972368/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: OHT represents a treatment option in patients with severe heart failure due to amyloidosis, allowing later intensive treatment with induction chemotherapy and ASCT. Although studies are lacking, immunosuppressive therapy after OHT might have some effect on clonal plasma cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37972368"
                        open
                      >
                        <summary class="article-expander__title">Cardiac and autologous stem cell transplantation hematopoietic in patients with AL amyloidosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37972368/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: OHT represents a treatment option in patients with severe heart failure due to amyloidosis, allowing later intensive treatment with induction chemotherapy and ASCT. Although studies are lacking, immunosuppressive therapy after OHT might have some effect on clonal plasma cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37972368"
                        open
                      >
                        <summary class="article-expander__title">Cardiac and autologous stem cell transplantation hematopoietic in patients with AL amyloidosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37972368/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: OHT represents a treatment option in patients with severe heart failure due to amyloidosis, allowing later intensive treatment with induction chemotherapy and ASCT. Although studies are lacking, immunosuppressive therapy after OHT might have some effect on clonal plasma cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37972368"
                        open
                      >
                        <summary class="article-expander__title">Cardiac and autologous stem cell transplantation hematopoietic in patients with AL amyloidosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37972368/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: OHT represents a treatment option in patients with severe heart failure due to amyloidosis, allowing later intensive treatment with induction chemotherapy and ASCT. Although studies are lacking, immunosuppressive therapy after OHT might have some effect on clonal plasma cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37972368"
                        open
                      >
                        <summary class="article-expander__title">Cardiac and autologous stem cell transplantation hematopoietic in patients with AL amyloidosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37972368/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: OHT represents a treatment option in patients with severe heart failure due to amyloidosis, allowing later intensive treatment with induction chemotherapy and ASCT. Although studies are lacking, immunosuppressive therapy after OHT might have some effect on clonal plasma cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37972368"
                        open
                      >
                        <summary class="article-expander__title">Cardiac and autologous stem cell transplantation hematopoietic in patients with AL amyloidosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37972368/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: OHT represents a treatment option in patients with severe heart failure due to amyloidosis, allowing later intensive treatment with induction chemotherapy and ASCT. Although studies are lacking, immunosuppressive therapy after OHT might have some effect on clonal plasma cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37972368"
                        open
                      >
                        <summary class="article-expander__title">Cardiac and autologous stem cell transplantation hematopoietic in patients with AL amyloidosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37972368/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: OHT represents a treatment option in patients with severe heart failure due to amyloidosis, allowing later intensive treatment with induction chemotherapy and ASCT. Although studies are lacking, immunosuppressive therapy after OHT might have some effect on clonal plasma cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37972368"
                        open
                      >
                        <summary class="article-expander__title">Cardiac and autologous stem cell transplantation hematopoietic in patients with AL amyloidosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37972368/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: OHT represents a treatment option in patients with severe heart failure due to amyloidosis, allowing later intensive treatment with induction chemotherapy and ASCT. Although studies are lacking, immunosuppressive therapy after OHT might have some effect on clonal plasma cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37972368"
                        open
                      >
                        <summary class="article-expander__title">Cardiac and autologous stem cell transplantation hematopoietic in patients with AL amyloidosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37972368/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: OHT represents a treatment option in patients with severe heart failure due to amyloidosis, allowing later intensive treatment with induction chemotherapy and ASCT. Although studies are lacking, immunosuppressive therapy after OHT might have some effect on clonal plasma cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-15 datetime="2023-11-15T11:00:00.000Z">2023-11-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-15 datetime="2023-11-15T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37965337"
                        open
                      >
                        <summary class="article-expander__title">Clonal redemption of B cells in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37965337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Potentially self-reactive B cells constitute a large portion of the peripheral B cell repertoire in both mice and humans. Maintenance of autoreactive B cell populations could conceivably be detrimental to the host but their conservation throughout evolution suggests performance of a critical and beneficial immune function. We discuss herein how the process of clonal redemption may provide insight to preservation of an autoreactive B cell pool in the context of infection and autoimmunity. Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37965337"
                        open
                      >
                        <summary class="article-expander__title">Clonal redemption of B cells in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37965337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Potentially self-reactive B cells constitute a large portion of the peripheral B cell repertoire in both mice and humans. Maintenance of autoreactive B cell populations could conceivably be detrimental to the host but their conservation throughout evolution suggests performance of a critical and beneficial immune function. We discuss herein how the process of clonal redemption may provide insight to preservation of an autoreactive B cell pool in the context of infection and autoimmunity. Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37965337"
                        open
                      >
                        <summary class="article-expander__title">Clonal redemption of B cells in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37965337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Potentially self-reactive B cells constitute a large portion of the peripheral B cell repertoire in both mice and humans. Maintenance of autoreactive B cell populations could conceivably be detrimental to the host but their conservation throughout evolution suggests performance of a critical and beneficial immune function. We discuss herein how the process of clonal redemption may provide insight to preservation of an autoreactive B cell pool in the context of infection and autoimmunity. Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37965337"
                        open
                      >
                        <summary class="article-expander__title">Clonal redemption of B cells in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37965337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Potentially self-reactive B cells constitute a large portion of the peripheral B cell repertoire in both mice and humans. Maintenance of autoreactive B cell populations could conceivably be detrimental to the host but their conservation throughout evolution suggests performance of a critical and beneficial immune function. We discuss herein how the process of clonal redemption may provide insight to preservation of an autoreactive B cell pool in the context of infection and autoimmunity. Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37965337"
                        open
                      >
                        <summary class="article-expander__title">Clonal redemption of B cells in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37965337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Potentially self-reactive B cells constitute a large portion of the peripheral B cell repertoire in both mice and humans. Maintenance of autoreactive B cell populations could conceivably be detrimental to the host but their conservation throughout evolution suggests performance of a critical and beneficial immune function. We discuss herein how the process of clonal redemption may provide insight to preservation of an autoreactive B cell pool in the context of infection and autoimmunity. Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37965337"
                        open
                      >
                        <summary class="article-expander__title">Clonal redemption of B cells in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37965337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Potentially self-reactive B cells constitute a large portion of the peripheral B cell repertoire in both mice and humans. Maintenance of autoreactive B cell populations could conceivably be detrimental to the host but their conservation throughout evolution suggests performance of a critical and beneficial immune function. We discuss herein how the process of clonal redemption may provide insight to preservation of an autoreactive B cell pool in the context of infection and autoimmunity. Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37965337"
                        open
                      >
                        <summary class="article-expander__title">Clonal redemption of B cells in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37965337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Potentially self-reactive B cells constitute a large portion of the peripheral B cell repertoire in both mice and humans. Maintenance of autoreactive B cell populations could conceivably be detrimental to the host but their conservation throughout evolution suggests performance of a critical and beneficial immune function. We discuss herein how the process of clonal redemption may provide insight to preservation of an autoreactive B cell pool in the context of infection and autoimmunity. Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37965337"
                        open
                      >
                        <summary class="article-expander__title">Clonal redemption of B cells in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37965337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Potentially self-reactive B cells constitute a large portion of the peripheral B cell repertoire in both mice and humans. Maintenance of autoreactive B cell populations could conceivably be detrimental to the host but their conservation throughout evolution suggests performance of a critical and beneficial immune function. We discuss herein how the process of clonal redemption may provide insight to preservation of an autoreactive B cell pool in the context of infection and autoimmunity. Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37965337"
                        open
                      >
                        <summary class="article-expander__title">Clonal redemption of B cells in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37965337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Potentially self-reactive B cells constitute a large portion of the peripheral B cell repertoire in both mice and humans. Maintenance of autoreactive B cell populations could conceivably be detrimental to the host but their conservation throughout evolution suggests performance of a critical and beneficial immune function. We discuss herein how the process of clonal redemption may provide insight to preservation of an autoreactive B cell pool in the context of infection and autoimmunity. Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37965337"
                        open
                      >
                        <summary class="article-expander__title">Clonal redemption of B cells in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37965337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Potentially self-reactive B cells constitute a large portion of the peripheral B cell repertoire in both mice and humans. Maintenance of autoreactive B cell populations could conceivably be detrimental to the host but their conservation throughout evolution suggests performance of a critical and beneficial immune function. We discuss herein how the process of clonal redemption may provide insight to preservation of an autoreactive B cell pool in the context of infection and autoimmunity. Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37965337"
                        open
                      >
                        <summary class="article-expander__title">Clonal redemption of B cells in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37965337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Potentially self-reactive B cells constitute a large portion of the peripheral B cell repertoire in both mice and humans. Maintenance of autoreactive B cell populations could conceivably be detrimental to the host but their conservation throughout evolution suggests performance of a critical and beneficial immune function. We discuss herein how the process of clonal redemption may provide insight to preservation of an autoreactive B cell pool in the context of infection and autoimmunity. Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37965337"
                        open
                      >
                        <summary class="article-expander__title">Clonal redemption of B cells in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37965337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Potentially self-reactive B cells constitute a large portion of the peripheral B cell repertoire in both mice and humans. Maintenance of autoreactive B cell populations could conceivably be detrimental to the host but their conservation throughout evolution suggests performance of a critical and beneficial immune function. We discuss herein how the process of clonal redemption may provide insight to preservation of an autoreactive B cell pool in the context of infection and autoimmunity. Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37965337"
                        open
                      >
                        <summary class="article-expander__title">Clonal redemption of B cells in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37965337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Potentially self-reactive B cells constitute a large portion of the peripheral B cell repertoire in both mice and humans. Maintenance of autoreactive B cell populations could conceivably be detrimental to the host but their conservation throughout evolution suggests performance of a critical and beneficial immune function. We discuss herein how the process of clonal redemption may provide insight to preservation of an autoreactive B cell pool in the context of infection and autoimmunity. Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37965337"
                        open
                      >
                        <summary class="article-expander__title">Clonal redemption of B cells in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37965337/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Potentially self-reactive B cells constitute a large portion of the peripheral B cell repertoire in both mice and humans. Maintenance of autoreactive B cell populations could conceivably be detrimental to the host but their conservation throughout evolution suggests performance of a critical and beneficial immune function. We discuss herein how the process of clonal redemption may provide insight to preservation of an autoreactive B cell pool in the context of infection and autoimmunity. Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-14 datetime="2023-11-14T11:00:00.000Z">2023-11-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-14 datetime="2023-11-14T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37958591"
                        open
                      >
                        <summary class="article-expander__title">Interdependence of Molecular Lesions That Drive Uveal Melanoma Metastasis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37958591/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The metastatic risk of uveal melanoma (UM) is defined by a limited number of molecular lesions, somatic mutations (SF3B1 and BAP1), and copy number alterations (CNA): monosomy of chromosome 3 (M3), chr8q gain (8q), chr6p gain (6p), yet the sequence of events is not clear. We analyzed data from three datasets (TCGA-UVM, GSE27831, GSE51880) with information regarding M3, 8q, 6p, SF3B1, and BAP1 status. We confirm that BAP1 mutations are always associated with M3 in high-risk patients. All other...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37957851"
                        open
                      >
                        <summary class="article-expander__title">Exploring Novel Genome Instability-associated lncRNAs and their Potential Function in Pan-Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37957851/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: In this study, we discovered four novel GI-lncRNAs and developed a novel signature that effectively predicted clinical outcomes in pan-RCC. The findings provide valuable insights for pan-RCC immunotherapy and shed light on potential underlying mechanisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-13 datetime="2023-11-13T11:00:00.000Z">2023-11-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-13 datetime="2023-11-13T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37956474"
                        open
                      >
                        <summary class="article-expander__title">How low can you go?: Methodologic considerations in clonal hematopoiesis variant calling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37956474/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of an expanded clonal hematopoietic cell population due to an acquired mutation conferring a selective growth advantage and is known to predispose to hematologic malignancy. In this review, we discuss sequencing methods for CH detection in bulk sequencing data and corresponding bioinformatic approaches for variant calling, filtering, and curation. We detail practical recommendations for CH calling. Finally, we discuss how improvements in CH...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-12 datetime="2023-11-12T11:00:00.000Z">2023-11-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-12 datetime="2023-11-12T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952978"
                        open
                      >
                        <summary class="article-expander__title">Cellular carcinogenesis in preleukemic conditions:drivers and defenses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952978/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) arises from preleukemic conditions. We have investigated the pathogenesis of typical preleukemia, myeloproliferative neoplasms, and clonal hematopoiesis. Hematopoietic stem cells in both preleukemic conditions harbor recurrent driver mutations; additional mutation provokes further malignant transformation, leading to AML onset. Although genetic alterations are defined as the main cause of malignant transformation, non-genetic factors are also involved in disease...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37952610"
                        open
                      >
                        <summary class="article-expander__title">Phenotype-structured model of intra-clonal heterogeneity and drug resistance in multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37952610/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple myeloma (MM) is a genetically complex hematological cancer characterized by the abnormal proliferation of malignant plasma cells in the bone marrow. This disease progresses from a premalignant condition known as monoclonal gammopathy of unknown significance (MGUS) through sequential genetic alterations involving various genes. These genetic changes contribute to the uncontrolled growth of multiple clones of plasma cells. In this study, we present a phenotype-structured model that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-11 datetime="2023-11-11T11:00:00.000Z">2023-11-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-11 datetime="2023-11-11T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951089"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and acquired genetic abnormalities of the red cell: An historical review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951089/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Several syndromes affecting the red cell that mimic those induced by germline mutations may result from a somatic mutation that accompanies a myeloid malignancy. These syndromes are most notable in cases of myelodysplastic syndrome, but they are not limited to any one category of myeloid neoplasm. Their occurrence in males exceed the male predominance that is evident in myeloid neoplasms. The syndromes include disorders of globin chain synthesis (α- and β-thalassemia), heme synthesis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37951129"
                        open
                      >
                        <summary class="article-expander__title">Sequencing paired tumor DNA and white blood cells improves circulating tumor DNA tracking and detects pathogenic germline variants in localized colon cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37951129/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The germline testing method not only enhanced sequencing results and raised the proportion of patients eligible for plasma monitoring, but also uncovered the existence of pathogenic germline variations, thereby aiding in the identification of patients at a higher risk of hereditary cancer syndromes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-10 datetime="2023-11-10T11:00:00.000Z">2023-11-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-10 datetime="2023-11-10T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37947606"
                        open
                      >
                        <summary class="article-expander__title">The Role of Clonal Hematopoiesis of Indeterminant Potential and DNA (Cytosine-5)-Methyltransferase Dysregulation in Pulmonary Arterial Hypertension and Other Cardiovascular Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37947606/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA methylation is an epigenetic mechanism that regulates gene expression without altering gene sequences in health and disease. DNA methyltransferases (DNMTs) are enzymes responsible for DNA methylation, and their dysregulation is both a pathogenic mechanism of disease and a therapeutic target. DNMTs change gene expression by methylating CpG islands within exonic and intergenic DNA regions, which typically reduces gene transcription. Initially, mutations in the DNMT genes and pathologic DNMT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-08 datetime="2023-11-08T11:00:00.000Z">2023-11-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-08 datetime="2023-11-08T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231128195411&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231127195330&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37938837"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Cardiovascular Disease in Patients With Multiple Myeloma Undergoing Hematopoietic Cell Transplant</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37938837/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION AND RELEVANCE: CHIP was significantly and independently associated with risk of CVD in patients with MM undergoing HCT and may serve as a novel biologically plausible biomarker for CVD in this cohort. Patients with MM and both CHIP and cardiovascular risk factors had an exceptionally high risk of CVD. Additional studies are warranted to determine if cardiovascular preventive measures can reduce CHIP-associated CVD risk.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37937078"
                        open
                      >
                        <summary class="article-expander__title">Depletion of tet2 results in age-dependent changes in DNA methylation and gene expression in a zebrafish model of myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37937078/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelodysplastic syndrome (MDS) is a heterogeneous group of clonal hematopoietic disorders characterized by ineffective hematopoiesis, cytopenias, and dysplasia. The gene encoding ten-eleven translocation 2 (tet2), a dioxygenase enzyme that catalyzes the conversion of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine, is a recurrently mutated tumor suppressor gene in MDS and other myeloid malignancies. Previously, we reported a stable zebrafish line with a loss-of-function mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-07 datetime="2023-11-07T11:00:00.000Z">2023-11-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-07 datetime="2023-11-07T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37934948"
                        open
                      >
                        <summary class="article-expander__title">Effects of IL-1β inhibition on anemia and clonal hematopoiesis in the randomized CANTOS trial</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37934948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Canakinumab, a monoclonal antibody targeting pro-inflammatory cytokine interleukin (IL)-1β, improved hemoglobin levels while preventing recurrent cardiovascular events in the Canakinumab Anti-inflammatory Thrombosis Outcomes Study (CANTOS). This cardiovascular (CV) preventive effect was greater in patients with TET2 mutations associated with clonal hematopoiesis (CH). The current proteogenomic analysis aimed to understand the clinical response to canakinumab and underlying proteomic profiles in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-06 datetime="2023-11-06T11:00:00.000Z">2023-11-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-06 datetime="2023-11-06T11:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37930339"
                        open
                      >
                        <summary class="article-expander__title">Impacted second molars, which surgical treatment choosing?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37930339/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231120195346&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This article presents the different therapeutic options in the treatment of second molar inclusions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37930339"
                        open
                      >
                        <summary class="article-expander__title">Impacted second molars, which surgical treatment choosing?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37930339/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This article presents the different therapeutic options in the treatment of second molar inclusions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37928907"
                        open
                      >
                        <summary class="article-expander__title">Epigenetic roles in clonal hematopoiesis and aging kidney-related chronic kidney disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37928907/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231119195337&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accumulation of somatic hematopoietic stem cell mutations with aging has been revealed by the recent genome-wide analysis. Clonal expansion, known as clonal hematopoiesis of indeterminate potential (CHIP), is a premalignant condition of hematological cancers. It is defined as the absence of definitive morphological evidence of a hematological neoplasm and occurrence of ≥2% of mutant allele fraction in the peripheral blood. In CHIP, the most frequently mutated genes are epigenetic regulators such...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37930339"
                        open
                      >
                        <summary class="article-expander__title">Impacted second molars, which surgical treatment choosing?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37930339/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This article presents the different therapeutic options in the treatment of second molar inclusions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37928907"
                        open
                      >
                        <summary class="article-expander__title">Epigenetic roles in clonal hematopoiesis and aging kidney-related chronic kidney disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37928907/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accumulation of somatic hematopoietic stem cell mutations with aging has been revealed by the recent genome-wide analysis. Clonal expansion, known as clonal hematopoiesis of indeterminate potential (CHIP), is a premalignant condition of hematological cancers. It is defined as the absence of definitive morphological evidence of a hematological neoplasm and occurrence of ≥2% of mutant allele fraction in the peripheral blood. In CHIP, the most frequently mutated genes are epigenetic regulators such...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37930339"
                        open
                      >
                        <summary class="article-expander__title">Impacted second molars, which surgical treatment choosing?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37930339/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This article presents the different therapeutic options in the treatment of second molar inclusions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37928907"
                        open
                      >
                        <summary class="article-expander__title">Epigenetic roles in clonal hematopoiesis and aging kidney-related chronic kidney disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37928907/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accumulation of somatic hematopoietic stem cell mutations with aging has been revealed by the recent genome-wide analysis. Clonal expansion, known as clonal hematopoiesis of indeterminate potential (CHIP), is a premalignant condition of hematological cancers. It is defined as the absence of definitive morphological evidence of a hematological neoplasm and occurrence of ≥2% of mutant allele fraction in the peripheral blood. In CHIP, the most frequently mutated genes are epigenetic regulators such...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37930339"
                        open
                      >
                        <summary class="article-expander__title">Impacted second molars, which surgical treatment choosing?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37930339/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This article presents the different therapeutic options in the treatment of second molar inclusions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37928907"
                        open
                      >
                        <summary class="article-expander__title">Epigenetic roles in clonal hematopoiesis and aging kidney-related chronic kidney disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37928907/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accumulation of somatic hematopoietic stem cell mutations with aging has been revealed by the recent genome-wide analysis. Clonal expansion, known as clonal hematopoiesis of indeterminate potential (CHIP), is a premalignant condition of hematological cancers. It is defined as the absence of definitive morphological evidence of a hematological neoplasm and occurrence of ≥2% of mutant allele fraction in the peripheral blood. In CHIP, the most frequently mutated genes are epigenetic regulators such...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37930339"
                        open
                      >
                        <summary class="article-expander__title">Impacted second molars, which surgical treatment choosing?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37930339/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This article presents the different therapeutic options in the treatment of second molar inclusions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37928907"
                        open
                      >
                        <summary class="article-expander__title">Epigenetic roles in clonal hematopoiesis and aging kidney-related chronic kidney disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37928907/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accumulation of somatic hematopoietic stem cell mutations with aging has been revealed by the recent genome-wide analysis. Clonal expansion, known as clonal hematopoiesis of indeterminate potential (CHIP), is a premalignant condition of hematological cancers. It is defined as the absence of definitive morphological evidence of a hematological neoplasm and occurrence of ≥2% of mutant allele fraction in the peripheral blood. In CHIP, the most frequently mutated genes are epigenetic regulators such...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37930339"
                        open
                      >
                        <summary class="article-expander__title">Impacted second molars, which surgical treatment choosing?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37930339/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This article presents the different therapeutic options in the treatment of second molar inclusions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37928907"
                        open
                      >
                        <summary class="article-expander__title">Epigenetic roles in clonal hematopoiesis and aging kidney-related chronic kidney disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37928907/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accumulation of somatic hematopoietic stem cell mutations with aging has been revealed by the recent genome-wide analysis. Clonal expansion, known as clonal hematopoiesis of indeterminate potential (CHIP), is a premalignant condition of hematological cancers. It is defined as the absence of definitive morphological evidence of a hematological neoplasm and occurrence of ≥2% of mutant allele fraction in the peripheral blood. In CHIP, the most frequently mutated genes are epigenetic regulators such...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37930339"
                        open
                      >
                        <summary class="article-expander__title">Impacted second molars, which surgical treatment choosing?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37930339/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This article presents the different therapeutic options in the treatment of second molar inclusions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37928907"
                        open
                      >
                        <summary class="article-expander__title">Epigenetic roles in clonal hematopoiesis and aging kidney-related chronic kidney disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37928907/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accumulation of somatic hematopoietic stem cell mutations with aging has been revealed by the recent genome-wide analysis. Clonal expansion, known as clonal hematopoiesis of indeterminate potential (CHIP), is a premalignant condition of hematological cancers. It is defined as the absence of definitive morphological evidence of a hematological neoplasm and occurrence of ≥2% of mutant allele fraction in the peripheral blood. In CHIP, the most frequently mutated genes are epigenetic regulators such...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37930339"
                        open
                      >
                        <summary class="article-expander__title">Impacted second molars, which surgical treatment choosing?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37930339/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This article presents the different therapeutic options in the treatment of second molar inclusions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37928907"
                        open
                      >
                        <summary class="article-expander__title">Epigenetic roles in clonal hematopoiesis and aging kidney-related chronic kidney disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37928907/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accumulation of somatic hematopoietic stem cell mutations with aging has been revealed by the recent genome-wide analysis. Clonal expansion, known as clonal hematopoiesis of indeterminate potential (CHIP), is a premalignant condition of hematological cancers. It is defined as the absence of definitive morphological evidence of a hematological neoplasm and occurrence of ≥2% of mutant allele fraction in the peripheral blood. In CHIP, the most frequently mutated genes are epigenetic regulators such...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37930339"
                        open
                      >
                        <summary class="article-expander__title">Impacted second molars, which surgical treatment choosing?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37930339/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This article presents the different therapeutic options in the treatment of second molar inclusions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37928907"
                        open
                      >
                        <summary class="article-expander__title">Epigenetic roles in clonal hematopoiesis and aging kidney-related chronic kidney disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37928907/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accumulation of somatic hematopoietic stem cell mutations with aging has been revealed by the recent genome-wide analysis. Clonal expansion, known as clonal hematopoiesis of indeterminate potential (CHIP), is a premalignant condition of hematological cancers. It is defined as the absence of definitive morphological evidence of a hematological neoplasm and occurrence of ≥2% of mutant allele fraction in the peripheral blood. In CHIP, the most frequently mutated genes are epigenetic regulators such...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37930339"
                        open
                      >
                        <summary class="article-expander__title">Impacted second molars, which surgical treatment choosing?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37930339/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This article presents the different therapeutic options in the treatment of second molar inclusions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37928907"
                        open
                      >
                        <summary class="article-expander__title">Epigenetic roles in clonal hematopoiesis and aging kidney-related chronic kidney disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37928907/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accumulation of somatic hematopoietic stem cell mutations with aging has been revealed by the recent genome-wide analysis. Clonal expansion, known as clonal hematopoiesis of indeterminate potential (CHIP), is a premalignant condition of hematological cancers. It is defined as the absence of definitive morphological evidence of a hematological neoplasm and occurrence of ≥2% of mutant allele fraction in the peripheral blood. In CHIP, the most frequently mutated genes are epigenetic regulators such...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37930339"
                        open
                      >
                        <summary class="article-expander__title">Impacted second molars, which surgical treatment choosing?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37930339/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This article presents the different therapeutic options in the treatment of second molar inclusions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37928907"
                        open
                      >
                        <summary class="article-expander__title">Epigenetic roles in clonal hematopoiesis and aging kidney-related chronic kidney disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37928907/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accumulation of somatic hematopoietic stem cell mutations with aging has been revealed by the recent genome-wide analysis. Clonal expansion, known as clonal hematopoiesis of indeterminate potential (CHIP), is a premalignant condition of hematological cancers. It is defined as the absence of definitive morphological evidence of a hematological neoplasm and occurrence of ≥2% of mutant allele fraction in the peripheral blood. In CHIP, the most frequently mutated genes are epigenetic regulators such...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37930339"
                        open
                      >
                        <summary class="article-expander__title">Impacted second molars, which surgical treatment choosing?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37930339/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This article presents the different therapeutic options in the treatment of second molar inclusions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37928907"
                        open
                      >
                        <summary class="article-expander__title">Epigenetic roles in clonal hematopoiesis and aging kidney-related chronic kidney disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37928907/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accumulation of somatic hematopoietic stem cell mutations with aging has been revealed by the recent genome-wide analysis. Clonal expansion, known as clonal hematopoiesis of indeterminate potential (CHIP), is a premalignant condition of hematological cancers. It is defined as the absence of definitive morphological evidence of a hematological neoplasm and occurrence of ≥2% of mutant allele fraction in the peripheral blood. In CHIP, the most frequently mutated genes are epigenetic regulators such...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932435"
                        open
                      >
                        <summary class="article-expander__title">Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932435/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37930339"
                        open
                      >
                        <summary class="article-expander__title">Impacted second molars, which surgical treatment choosing?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37930339/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This article presents the different therapeutic options in the treatment of second molar inclusions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37928907"
                        open
                      >
                        <summary class="article-expander__title">Epigenetic roles in clonal hematopoiesis and aging kidney-related chronic kidney disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37928907/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accumulation of somatic hematopoietic stem cell mutations with aging has been revealed by the recent genome-wide analysis. Clonal expansion, known as clonal hematopoiesis of indeterminate potential (CHIP), is a premalignant condition of hematological cancers. It is defined as the absence of definitive morphological evidence of a hematological neoplasm and occurrence of ≥2% of mutant allele fraction in the peripheral blood. In CHIP, the most frequently mutated genes are epigenetic regulators such...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37931207"
                        open
                      >
                        <summary class="article-expander__title">Causes and Consequences of Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37931207/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) describes the outsized contribution of expanded clones of hematopoietic stem and progenitor cells (HSPCs) to blood cell production. The prevalence of CH increases dramatically with age. CH can be caused by somatic mutations in individual genes or by gains and or losses of larger chromosomal segments. CH is a premalignant state; the somatic mutations detected in CH are the initiating mutations for hematologic malignancies, and CH is a strong predictor for the development...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37928907"
                        open
                      >
                        <summary class="article-expander__title">Epigenetic roles in clonal hematopoiesis and aging kidney-related chronic kidney disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37928907/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accumulation of somatic hematopoietic stem cell mutations with aging has been revealed by the recent genome-wide analysis. Clonal expansion, known as clonal hematopoiesis of indeterminate potential (CHIP), is a premalignant condition of hematological cancers. It is defined as the absence of definitive morphological evidence of a hematological neoplasm and occurrence of ≥2% of mutant allele fraction in the peripheral blood. In CHIP, the most frequently mutated genes are epigenetic regulators such...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231127195331&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37932252"
                        open
                      >
                        <summary class="article-expander__title">Tracing cancer evolution and heterogeneity using Hi-C</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37932252/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosomal rearrangements can initiate and drive cancer progression, yet it has been challenging to evaluate their impact, especially in genetically heterogeneous solid cancers. To address this problem we developed HiDENSEC, a new computational framework for analyzing chromatin conformation capture in heterogeneous samples that can infer somatic copy number alterations, characterize large-scale chromosomal rearrangements, and estimate cancer cell fractions. After validating HiDENSEC with in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-03 datetime="2023-11-03T10:00:00.000Z">2023-11-03</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-03 datetime="2023-11-03T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921439"
                        open
                      >
                        <summary class="article-expander__title">SRCAP Mutations and Cellular Stressors Promote Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921439/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SRCAP mutations promote clonal expansion of hematopoietic stem cells (HSC) in response to cellular stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231126195311&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231125195551&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231124195037&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37921274"
                        open
                      >
                        <summary class="article-expander__title">Personalised neoantigen-based therapy in colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37921274/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colorectal cancer (CRC) has become one of the most common tumours with high morbidity, mortality and distinctive evolution mechanism. The neoantigens arising from the somatic mutations have become considerable treatment targets in the management of CRC. As cancer-specific aberrant peptides, neoantigens can trigger the robust host immune response and exert anti-tumour effects while minimising the emergence of adverse events commonly associated with alternative therapeutic regimens. In this...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-02 datetime="2023-11-02T10:00:00.000Z">2023-11-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-02 datetime="2023-11-02T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37915324"
                        open
                      >
                        <summary class="article-expander__title">Philadelphia chromosome-negative myeloproliferative chronic neoplasms: is clonal hematopoiesis the main determinant of autoimmune and cardio-vascular manifestations?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37915324/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this article, we reviewed the possible mechanisms linking the clonal hematopoiesis of indeterminate potential (CHIP) to chronic myeloproliferative neoplasms (MPNs), autoimmune diseases (ADs), and cardiovascular diseases (CADs). CHIP is characterized by the presence of clonal mutations with an allelic frequency &gt;2% in the peripheral blood without dysplasia, overt hematological neoplasms, or abnormalities in blood cell count. The prevalence may reach 20% of elderly healthy individuals and is...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-11-01 datetime="2023-11-01T10:00:00.000Z">2023-11-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-11-01 datetime="2023-11-01T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37913908"
                        open
                      >
                        <summary class="article-expander__title">Donor-derived malignancy and transplant morbidity: Risks of patient and donor genetics in allogeneic hematopoietic stem cell transplant: Donor-derived malignancy and genetic risks in allo-HSCT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37913908/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Allogeneic hematopoietic stem cell transplant (allo-HSCT) remains a key treatment option for hematologic malignancies (HMs), although it carries significant risks. Up to 30% of patients relapse after allo-HSCT, of which 2-5% are donor-derived malignancies (DDM). DDM can arise from a germline genetic predisposition allele or clonal hematopoiesis (CH) in the donor. Increasingly, genetic testing reveals that patient and donor genetic factors contribute to the development of DDM and other allo-HSCT...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37909388"
                        open
                      >
                        <summary class="article-expander__title">Somatic TET2 Mutations are Associated with Giant Cell Arteritis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37909388/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH increases risk for development of GCA in a genotype-specific fashion, with greatest risk being conferred by the presence of mutations in TET2. Somatic TET2 mutations likewise increase the risk of GCA-associated vision loss. Integration of somatic genetic testing in GCA diagnostics may be warranted in the future. This article is protected by copyright. All rights reserved.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-31 datetime="2023-10-31T10:00:00.000Z">2023-10-31</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-31 datetime="2023-10-31T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37905118"
                        open
                      >
                        <summary class="article-expander__title">Determinants of mosaic chromosomal alteration fitness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37905118/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is characterized by the acquisition of a somatic mutation in a hematopoietic stem cell that results in a clonal expansion. These driver mutations can be single nucleotide variants in cancer driver genes or larger structural rearrangements called mosaic chromosomal alterations (mCAs). The factors that influence the variations in mCA fitness and ultimately result in different clonal expansion rates are not well-understood. We used the Passenger-Approximated Clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904876"
                        open
                      >
                        <summary class="article-expander__title">Germline and somatic drivers in inherited hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904876/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited hematologic malignancies are linked to a heterogenous group of genes, knowledge of which is rapidly expanding using panel-based next-generation sequencing (NGS) or whole-exome/whole-genome sequencing. Importantly, the penetrance for these syndromes is incomplete, and disease development, progression or transformation has critical clinical implications. With the earlier detection of healthy carriers and sequential monitoring of these patients, clonal hematopoiesis and somatic driver...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904125"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and significance of clonal hematopoiesis of indeterminate potential in lung transplant recipients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904125/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Lung recipients have a higher prevalence of CHIP and a larger variety of genes with CHIP-associated mutations compared with previous reports for the general population. CHIP did not increase the risk of AR, CLAD, or death in lung recipients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37904927"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics Illustrations Decoded by ChatGPT: The Good, The Bad, and The Ugly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37904927/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging studies underscore the promising capabilities of large language model-based chatbots in conducting fundamental bioinformatics data analyses. The recent feature of accepting image-inputs by ChatGPT motivated us to explore its efficacy in deciphering bioinformatics illustrations. Our evaluation with examples in cancer research, including sequencing data analysis, multimodal network-based drug repositioning, and tumor clonal evolution, revealed that ChatGPT can proficiently explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-30 datetime="2023-10-30T10:00:00.000Z">2023-10-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-30 datetime="2023-10-30T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231123195150&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &quot;ChReporter&quot; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as well as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &quot;ChReporter&quot; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as well as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231122195232&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &quot;ChReporter&quot; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as well as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &quot;ChReporter&quot; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as well as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &quot;ChReporter&quot; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as well as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &quot;ChReporter&quot; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as well as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &quot;ChReporter&quot; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as well as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &quot;ChReporter&quot; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as well as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &quot;ChReporter&quot; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as well as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &quot;ChReporter&quot; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as well as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &quot;ChReporter&quot; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as well as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37903225"
                        open
                      >
                        <summary class="article-expander__title">Differences in cell shape, motility, and growth reflect chromosomal number variations that can be visualized with live-cell ChReporters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37903225/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chromosome numbers often change dynamically in tumors and in cultured cells, which complicates therapy as well as understanding genotype-mechanotype relationships. Here we use a live-cell &#x27;ChReporter&#x27; method to identify cells with a single chromosomal loss in efforts to better understand differences in cell shape, motility, and growth. We focus on a standard lung cancer line and first show clonal populations that retain the ChReporter exhibit large differences in cell and nuclear morphology as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37901004"
                        open
                      >
                        <summary class="article-expander__title">Leukemic transformation during anti-tuberculosis treatment in aplastic anemia-paroxysmal nocturnal hemoglobinuria syndrome: A case report and review of literature</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37901004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Aggravated inflammatory stressors can repress normal and leukemic hematopoiesis, and relieved inflammatory stressors can facilitate penetration of neoplastic hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-28 datetime="2023-10-28T10:00:00.000Z">2023-10-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-28 datetime="2023-10-28T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895120"
                        open
                      >
                        <summary class="article-expander__title">Deciphering Potential Molecular Signatures to Differentiate Acute Myeloid Leukemia (AML) with BCR::ABL1 from Chronic Myeloid Leukemia (CML) in Blast Crisis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895120/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute myeloid leukemia (AML) with BCR::ABL1 has recently been recognized as a distinct subtype in international classifications. Distinguishing it from myeloid blast crisis chronic myeloid leukemia (BC-CML) without evidence of a chronic phase (CP), remains challenging. We aimed to better characterize this entity by integrating clonal architecture analysis, mutational landscape assessment, and gene expression profiling. We analyzed a large retrospective cohort study including CML and AML...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37892987"
                        open
                      >
                        <summary class="article-expander__title">Emerging Insights into Molecular Mechanisms of Inflammation in Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37892987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inflammation impacts human hematopoiesis across physiologic and pathologic conditions, as signals derived from the bone marrow microenvironment, such as pro-inflammatory cytokines and chemokines, have been shown to alter hematopoietic stem cell (HSCs) homeostasis. Dysregulated inflammation can skew HSC fate-related decisions, leading to aberrant hematopoiesis and potentially contributing to the pathogenesis of hematological disorders such as myelodysplastic syndromes (MDS). Recently, emerging...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231121195454&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231120195347&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231119195338&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231118195624&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231117195203&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37898477"
                        open
                      >
                        <summary class="article-expander__title">Adaptive therapy: a tumor therapy strategy based on Darwinian evolution theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37898477/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer progression is a dynamic process of continuous evolution, in which genetic diversity and heterogeneity are generated by clonal and subclonal amplification based on random mutations. Traditional cancer treatment strategies have a great challenge, which often leads to treatment failure due to drug resistance. Integrating evolutionary dynamics into treatment regimens may be an effective way to overcome the problem of drug resistance. In particular, a potential treatment is adaptive therapy,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895205"
                        open
                      >
                        <summary class="article-expander__title">Decoding Cancer Evolution: Integrating Genetic and Non-Genetic Insights</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895205/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The development of cancer begins with cells transitioning from their multicellular nature to a state akin to unicellular organisms. This shift leads to a breakdown in the crucial regulators inherent to multicellularity, resulting in the emergence of diverse cancer cell subpopulations that have enhanced adaptability. The presence of different cell subpopulations within a tumour, known as intratumoural heterogeneity (ITH), poses challenges for cancer treatment. In this review, we delve into the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37895004"
                        open
                      >
                        <summary class="article-expander__title">Hematologic Neoplasms Associated with Down Syndrome: Cellular and Molecular Heterogeneity of the Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37895004/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The molecular basis of Down syndrome (DS) predisposition to leukemia is not fully understood but involves various factors such as chromosomal abnormalities, oncogenic mutations, epigenetic alterations, and changes in selection dynamics. Myeloid leukemia associated with DS (ML-DS) is preceded by a preleukemic phase called transient abnormal myelopoiesis driven by GATA1 gene mutations and progresses to ML-DS via additional mutations in cohesin genes, CTCF, RAS, or JAK/STAT pathway genes....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37894756"
                        open
                      >
                        <summary class="article-expander__title">Overview of Tumor Heterogeneity in High-Grade Serous Ovarian Cancers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37894756/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancers encompass a group of neoplasms originating from germinal tissues and exhibiting distinct clinical, pathological, and molecular features. Among these, epithelial ovarian cancers (EOCs) are the most prevalent, comprising five distinct tumor histotypes. Notably, high-grade serous ovarian cancers (HGSOCs) represent the majority, accounting for over 70% of EOC cases. Due to their silent and asymptomatic behavior, HGSOCs are generally diagnosed in advanced stages with an evolved and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-26 datetime="2023-10-26T10:00:00.000Z">2023-10-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-26 datetime="2023-10-26T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37884893"
                        open
                      >
                        <summary class="article-expander__title">Alterations of the expression of TET2 and DNA 5-hmC predict poor prognosis in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37884893/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: IPSS-R higher-risk MDS categories and AML-MR showed a reduction in TET2 expression, which was not apparent in lower-risk MDS. DNA 5-hmC levels followed a similar pattern. Overall, a decreased TET2 expression and a low DNA 5-hmC level are predictors of advanced disease and adverse outcome in MDS in the population studied, i.e., MDS patients from India.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-25 datetime="2023-10-25T10:00:00.000Z">2023-10-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-25 datetime="2023-10-25T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and inflammation: A review of mechanisms and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and inflammation: A review of mechanisms and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and inflammation: A review of mechanisms and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and inflammation: A review of mechanisms and clinical implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37879493"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and Inflammation: A Review of Mechanisms and Clinical Implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37879493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) is defined by the presence of somatic mutations in hematopoietic stem and progenitor cells (HSPC). CH is associated primarily with advancing age and confers an elevated risk of progression to overt hematologic malignancy and cardiovascular disease. Increasingly, CH is associated with a wide range of diseases driven by, and sequelae of, inflammation. Accordingly, there is great interest in better understanding the pathophysiologic and clinical relationship between CH,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231116195259&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231115195301&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37877809"
                        open
                      >
                        <summary class="article-expander__title">Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37877809/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic infection of hepatitis B virus (HBV) is the major cause of hepatocellular carcinoma (HCC). Notably, 90% of HBV-positive HCC cases exhibit detectable HBV integrations, hinting at the potential early entanglement of these viral integrations in tumorigenesis and their subsequent oncogenic implications. Nevertheless, the precise chronology of integration events during HCC tumorigenesis, alongside their sequential structural patterns, has remained elusive thus far. In this study, we applied...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-24 datetime="2023-10-24T10:00:00.000Z">2023-10-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-24 datetime="2023-10-24T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231114195221&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231113195114&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231112195243&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37874744"
                        open
                      >
                        <summary class="article-expander__title">SETD2 mutations do not contribute to clonal fitness in response to chemotherapy in childhood B cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37874744/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in genes encoding epigenetic regulators are commonly observed at relapse in B cell acute lymphoblastic leukemia (B-ALL). Loss-of-function mutations in SETD2, an H3K36 methyltransferase, have been observed in B-ALL and other cancers. Previous studies on mutated SETD2 in solid tumors and acute myelogenous leukemia support a role in promoting resistance to DNA damaging agents. We did not observe chemoresistance, an impaired DNA damage response, nor increased mutation frequency in response...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873488"
                        open
                      >
                        <summary class="article-expander__title">Growth Dynamics of Ductal Carcinoma in Situ Recapitulate Normal Breast Development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873488/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ductal carcinoma in situ (DCIS) and invasive breast cancer share many morphologic, proteomic, and genomic alterations. Yet in contrast to invasive cancer, many DCIS tumors do not progress and may remain indolent over decades. To better understand the heterogenous nature of this disease, we reconstructed the growth dynamics of 18 DCIS tumors based on the geo-spatial distribution of their somatic mutations. The somatic mutation topographies revealed that DCIS is multiclonal and consists of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37873375"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Cancer Evolution Landscape Based on Accurate Detection of Somatic Mutations in Single Tumor Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37873375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in single tumor cells is greatly desired as it allows us to quantify the single-cell mutation burden and construct the mutation-based phylogenetic tree. Here we developed scNanoSeq chemistry and profiled 842 single cells from 21 human breast cancer samples. The majority of the mutation-based phylogenetic trees comprise a characteristic stem evolution followed by the clonal sweep. We observed the subtype-dependent lengths in the stem evolution. To explain...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37872885"
                        open
                      >
                        <summary class="article-expander__title">A guide to epigenetics in leukaemia stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37872885/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Leukaemia stem cells (LSCs) are the critical seed for the growth of haematological malignancies, driving the clonal expansion that enables disease initiation, relapse and often resistance. Specifically, they display inherent phenotypic and epigenetic plasticity resulting in complex heterogenic diseases. In this review, we discuss the key principles of deregulation of epigenetic processes that shape this disease evolution. We consider measures to define and quantify clonal heterogeneity,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-23 datetime="2023-10-23T10:00:00.000Z">2023-10-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-23 datetime="2023-10-23T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231111195446&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231110195009&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231109195049&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231108195217&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231107195058&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231106195136&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231103204955&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37871126"
                        open
                      >
                        <summary class="article-expander__title">Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37871126/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most computational methods that infer somatic copy number alterations (SCNAs) from bulk sequencing of DNA analyse tumour samples individually. However, the sequencing of multiple tumour samples from a patient&#x27;s disease is an increasingly common practice. We introduce Refphase, an algorithm that leverages this multi-sampling approach to infer haplotype-specific copy numbers through multi-sample phasing. We demonstrate Refphase&#x27;s ability to infer haplotype-specific SCNAs and characterise their...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37870568"
                        open
                      >
                        <summary class="article-expander__title">TFH lymphoma and associated clonal hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37870568/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>T‑follicular helper (TFH) cell lymphoma (TFHL) is a lymphoma of mature T cells with phenotypic characteristics and gene expression signature of TFH cells. The lymphoma harbors recurrent mutations of RHOA^(G17V), IDH2^(R172), TET2 and DNMT3A. Whereas RHOA^(G17V) and IDH2^(R172) are almost exclusively found in this entity, TET2 and DNMT3A mutations occur in a broad variety of hematological neoplasms and are the most frequently affected genes in clonal hematopoiesis (CH). CH in humans shows a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-21 datetime="2023-10-21T10:00:00.000Z">2023-10-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-21 datetime="2023-10-21T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231105195156&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231104205408&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231103204956&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865509"
                        open
                      >
                        <summary class="article-expander__title">Acute Myeloid Leukemia Arising from Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a group of myeloid neoplasms characterized by clonal hematopoiesis and abnormal maturation of hematopoietic cells, resulting in cytopenias. The transformation of MDS to acute myeloid leukemia (AML) reflects a progressive increase in blasts due to impaired maturation of the malignant clone, and thus MDS and many AML subtypes form a biological continuum rather than representing two distinct diseases. Recent data suggest that, in addition to previously described...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37865503"
                        open
                      >
                        <summary class="article-expander__title">Premalignant Clonal Hematopoiesis (Clonal Hematopoiesis of Indeterminate Potential and Clonal Cytopenia of Undetermined Significance)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37865503/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Premalignant clonal hematopoiesis is the presence of somatic alterations in the blood of otherwise healthy individuals. Although the condition is not considered as a cancer, it carries an increased risk of developing a hematologic malignancy, particularly in those with large neoplastic clones, multiple pathogenic mutations, and high-risk mutations. In addition to the increased risk of malignancy, clonal hematopoiesis carries a markedly increased risk of cardiovascular events and death....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-20 datetime="2023-10-20T10:00:00.000Z">2023-10-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-20 datetime="2023-10-20T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231102205033&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37863054"
                        open
                      >
                        <summary class="article-expander__title">SRCAP mutations drive clonal hematopoiesis through epigenetic and DNA repair dysregulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37863054/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations accumulate in all cells with age and can confer a selective advantage, leading to clonal expansion over time. In hematopoietic cells, mutations in a subset of genes regulating DNA repair or epigenetics frequently lead to clonal hematopoiesis (CH). Here, we describe the context and mechanisms that lead to enrichment of hematopoietic stem cells (HSCs) with mutations in SRCAP, which encodes a chromatin remodeler that also influences DNA repair. We show that SRCAP mutations confer...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858853"
                        open
                      >
                        <summary class="article-expander__title">A pan-cancer landscape of pathogenic somatic copy number variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858853/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study contributes to elucidate the functional impact of pathogenic CNVs in tumor progression and sheds light on their potential as prognostic markers in specific cancer types.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-19 datetime="2023-10-19T10:00:00.000Z">2023-10-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-19 datetime="2023-10-19T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37858132"
                        open
                      >
                        <summary class="article-expander__title">Oligo-metastatic neoPlasms from the gastro-intestinal tract: iDentIfiCaTIon of cliNical and molecular drivers: the PREDICTION study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37858132/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Metastatic disease in tumors originating from the gastrointestinal tract can exhibit varying degrees of tumor burden at presentation. Some patients follow a less aggressive disease course, characterized by a limited number of metastatic sites, referred to as &quot;oligo-metastatic disease&quot; (OMD). The precise biological characteristics that define the oligometastatic behavior remain uncertain. In this study, we present a protocol designed to prospectively identify OMD, with the aim of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-18 datetime="2023-10-18T10:00:00.000Z">2023-10-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-18 datetime="2023-10-18T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37852258"
                        open
                      >
                        <summary class="article-expander__title">A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37852258/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cellular lineage histories and their molecular states encode fundamental principles of tissue development and homeostasis. Current lineage-recording mouse models have insufficient barcode diversity and single-cell lineage coverage for profiling tissues composed of millions of cells. Here, we developed DARLIN, an inducible Cas9 barcoding mouse line that utilizes terminal deoxynucleotidyl transferase (TdT) and 30 CRISPR target sites. DARLIN is inducible, generates massive lineage barcodes across...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-14 datetime="2023-10-14T10:00:00.000Z">2023-10-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-14 datetime="2023-10-14T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231101205012&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37835504"
                        open
                      >
                        <summary class="article-expander__title">The Role of Non-Coding RNAs in Myelodysplastic Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37835504/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes or neoplasms (MDS) are a heterogeneous group of myeloid clonal disorders characterized by peripheral blood cytopenias, blood and marrow cell dysplasia, and increased risk of evolution to acute myeloid leukemia (AML). Non-coding RNAs, especially microRNAs and long non-coding RNAs, serve as regulators of normal and malignant hematopoiesis and have been implicated in carcinogenesis. This review presents a comprehensive summary of the biology and role of non-coding RNAs,...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-13 datetime="2023-10-13T10:00:00.000Z">2023-10-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-13 datetime="2023-10-13T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231031205323&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37833013"
                        open
                      >
                        <summary class="article-expander__title">Technology of genomic balancing of chromatin of autologous hematopoietic stem cells for gene therapy of fatal immune-mediated diseases of civilization, extended life expectancy and sudden human death prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37833013/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A biotechnology for personalized ex vivo gene therapy based on molecular genomic balancing of hematopoietic stem cell (HSC) chromatin with nucleosome monomers of human genomic DNA (hDNA^(nmr)) has been developed and implemented in the clinic to change (to &quot;correct&quot;) mutant chromosome loci genomes of dominant HSC clones that form mono- and oligoclonal hematopoiesis during aging and major (oncological, cardiovascular, neurodegenerative and autoimmune) fatal immune-mediated diseases of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37832948"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis, Inflammation, and Hematologic Malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37832948/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic or acquired mutations are postzygotic genetic variations that can occur within any tissue. These mutations accumulate during aging and have classically been linked to malignant processes. Tremendous advancements over the past years have led to a deeper understanding of the role of somatic mutations in benign and malignant age-related diseases. Here, we review the somatic mutations that accumulate in the blood and their connection to disease states, with a particular focus on inflammatory...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-12 datetime="2023-10-12T10:00:00.000Z">2023-10-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-12 datetime="2023-10-12T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37827894"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in the application of induced pluripotent stem cell technology to the study of myeloid malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37827894/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acquired myeloid malignancies are a spectrum of clonal disorders known to be caused by sequential acquisition of genetic lesions in hematopoietic stem and progenitor cells, leading to their aberrant self-renewal and differentiation. The increasing use of induced pluripotent stem cell (iPSC) technology to study myeloid malignancies has helped usher a paradigm shift in approaches to disease modeling and drug discovery, especially when combined with gene-editing technology. The process of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37823726"
                        open
                      >
                        <summary class="article-expander__title">Targeted single-cell RNA sequencing analysis reveals metabolic reprogramming and the ferroptosis-resistant state in hematologic malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37823726/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Hematologic malignancies are the most common hematopoietic diseases and a major public health concern. However, the mechanisms underlying myeloid tumors remain unknown owing to the intricate interplay between mutations and diverse clonal evolution patterns, as evidenced by the analysis of bulk cell-derived omics data. Several single-cell omics techniques have been used to characterize the hierarchies and altered immune microenvironments of hematologic malignancies. The comprehensive single-cell...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-11 datetime="2023-10-11T10:00:00.000Z">2023-10-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-11 datetime="2023-10-11T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37819942"
                        open
                      >
                        <summary class="article-expander__title">Phertilizer: Growing a clonal tree from ultra-low coverage single-cell DNA sequencing of tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37819942/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Emerging ultra-low coverage single-cell DNA sequencing (scDNA-seq) technologies have enabled high resolution evolutionary studies of copy number aberrations (CNAs) within tumors. While these sequencing technologies are well suited for identifying CNAs due to the uniformity of sequencing coverage, the sparsity of coverage poses challenges for the study of single-nucleotide variants (SNVs). In order to maximize the utility of increasingly available ultra-low coverage scDNA-seq data and obtain a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-10 datetime="2023-10-10T10:00:00.000Z">2023-10-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-10 datetime="2023-10-10T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231030205036&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37816954"
                        open
                      >
                        <summary class="article-expander__title">STING activation in TET2-mutated hematopoietic stem/progenitor cells contributes to the increased self-renewal and neoplastic transformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37816954/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic loss-of-function mutations of the dioxygenase Ten-eleven translocation-2 (TET2) occur frequently in individuals with clonal hematopoiesis (CH) and acute myeloid leukemia (AML). These common hematopoietic disorders can be recapitulated in mouse models. However, the underlying mechanisms by which the deficiency in TET2 promotes these disorders remain unclear. Here we show that the cyclic guanosine monophosphate-adenosine monophosphate synthase (cGAS)-stimulator of interferon genes (STING)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2023-10-09 datetime="2023-10-09T10:00:00.000Z">2023-10-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2023-10-09 datetime="2023-10-09T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20231206195311&amp;utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231029205030&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231028205254&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231027204803&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231023204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231012205120&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37809221"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic Syndrome After Anti-CD19 Chimeric Antigen Receptor T-cell Therapy: A Case Series</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37809221/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The utility of CD19-targeted chimeric antigen receptor T-cell (CAR-T cell) therapy in the management of refractory/relapsed B-cell malignancies has increased tremendously in recent times. In addition to cytokine release syndrome (CRS), neurotoxicity, and infections, CAR-T cell patients develop cytopenias, with about 15% of the patients continuing to have severe cytopenias up to three months after treatment. Retrospective reviews have reported the development of myelodysplastic syndrome (MDS) in...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;utm_campaign&#x3D;pubmed-2&amp;utm_source&#x3D;axios&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;v&#x3D;2.17.9.post6+86293ac&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20231206195312">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231027204802&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231026204906&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231025204855&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231024204954&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231023204959&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231022205027&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231021205333&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231020204820&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231019204958&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231018205013&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231017205017&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231016205028&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231015205105&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231014205339&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231013204816&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231012205121&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231011204744&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231010204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:37812025"
                        open
                      >
                        <summary class="article-expander__title">Subclonal Somatic Copy-Number Alterations Emerge and Dominate in Recurrent Osteosarcoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/37812025/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20231009204941&amp;v&#x3D;2.17.9.post6+86293ac">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple large-scale genomic profiling efforts have been undertaken in osteosarcoma to define the genomic drivers of tumorigenesis, therapeutic response, and disease recurrence. The spatial and temporal intratumor heterogeneity could also play a role in promoting tumor growth and treatment resistance. We conducted longitudinal whole-genome sequencing of 37 tumor samples from 8 patients with relapsed or refractory osteosarcoma. Each patient had at least one sample from a primary site and a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z">2022-06-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="72a3a11d39c217b1a80cbb551444e6f6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nodal interferon exposure drives tumour-specific tolerance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1533878403463405569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nodal interferon exposure drives tumour-specific tolerance https://t.co/FT4NeBcFTg pic.twitter.com/NRHkVDUsOr
— NatureReviewsCancer (@NatureRevCancer) June 6, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z">2022-05-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="bb6bc242bcbc356c4a38db850e74d812"
                        open
                      >
                        <summary class="article-expander__title">@VIBConferences: #Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.⏰Registration Deadline: Thursday 2 June. Read more</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VIBConferences/status/1531132720952266752">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>#Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.
⏰Registration Deadline: Thursday 2 June. Read more https://t.co/cc0HC5cj6W pic.twitter.com/u8LFuzBbsV
— VIB Conferences (@VIBConferences) May 30, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z">2022-05-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="ed6c996dc1a718afb523de6229a53131"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530198403686072320">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy https://t.co/wjuHplkpof pic.twitter.com/1SoKYhRsPO
— NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="d15a3a4b24853812d7fa21fd56de5815"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content 👇🏻💌</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530174481313972224">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content 👇🏻
💌 https://t.co/Fo6XRWV3OZ@natrescancer pic.twitter.com/rbxRPqLV6g
— NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-26 datetime="2022-05-26T08:53:50.000Z">2022-05-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-26 datetime="2022-05-26T08:53:50.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fe13755a2693ea65459e5e21df097d2a"
                        open
                      >
                        <summary class="article-expander__title">@UlrikeHarjes: This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer:  do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest☺️</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/UlrikeHarjes/status/1529747629936193537">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer: https://t.co/p3Y2cdNhco
I do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest☺️ https://t.co/qWH2dTv8j9
— Ulrike Harjes (@UlrikeHarjes) May 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z">2022-05-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9ba5279424b31b0122a4ae20c18e6d03"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Developing dietary interventions as therapy for cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1529516411059437568">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Developing dietary interventions as therapy for cancer https://t.co/ijuqctk1wJ pic.twitter.com/KHPpMKnCFa
— NatureReviewsCancer (@NatureRevCancer) May 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z">2022-05-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="611d40f2e6ea2796b028bebda41a86dd"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Digging for treasures in the tumour interactome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526928478490095616">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Digging for treasures in the tumour interactome https://t.co/NhHYAwUgLE
— NatureReviewsCancer (@NatureRevCancer) May 18, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z">2022-05-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9590a971b00448cd312078183965b21d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: As part of our overall commitment to diversity, equity and inclusion at Springer Nature, we’d like to better understand what DEI means within the global research community. Take the survey here</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526560254091313156">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>As part of our overall commitment to diversity, equity and inclusion at Springer Nature, we’d like to better understand what DEI means within the global research community. Take the survey here https://t.co/03yLiUxFTq
— NatureReviewsCancer (@NatureRevCancer) May 17, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z">2022-05-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="aa38ff4245b93e964773f191844ddc7a"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT 🧰 🛠️ where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial 👇🏻</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524772831904509953">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT 🧰 🛠️ where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial 👇🏻 https://t.co/VbyoUJmTRJ
— NatureReviewsCancer (@NatureRevCancer) May 12, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z">2022-05-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="65aca76b9f794aaa668d84ad32dca4a0"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tools for the next generation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524068259712036864">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tools for the next generation https://t.co/lEzTPFYDf5
— NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="afa9b6784b4ccc7c05ed4efd26ddfb2d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524022709386190856">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon https://t.co/XXT0c9SAYX
— NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z">2022-04-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="43e2d5ccebc9030f737d656f47ac9d98"
                        open
                      >
                        <summary class="article-expander__title">@VarunVenkatara2: Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VarunVenkatara2/status/1520050291986604033">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter https://t.co/S55cKTGNFd
— Varun Venkataramani (@VarunVenkatara2) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9f6b8644378881f4c2e6b6833c93fc2f"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! An inhospitable site</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020349068316673">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! An inhospitable site https://t.co/0dvqIKCWxh pic.twitter.com/PtY8J9Fz7r
— NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="f6d91efcd60dae769e2722585bdfa8df"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Disconnecting multicellular networks in brain tumours</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020340201598978">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Disconnecting multicellular networks in brain tumours https://t.co/LJrzy8bRma pic.twitter.com/CO7BnooP0r
— NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z">2022-04-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="76e3b436bc8e43ad21366a80c120e395"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.💌</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519678964364623872">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.
💌 https://t.co/vQwHFJhpwv@natrescancer pic.twitter.com/6kYgjRYhdT
— NatureReviewsCancer (@NatureRevCancer) April 28, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fdf224ea293cdf039086d16e5699a59a"
                        open
                      >
                        <summary class="article-expander__title">@tmurraystewart: Our latest publication on #polyamines is online!@HopkinsMedicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/tmurraystewart/status/1519413460978704384">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our latest publication on #polyamines is online!@HopkinsMedicine https://t.co/ehRGgN1w9R
— Tracy Murray Stewart (@tmurraystewart) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z">2022-04-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="4f43226ec4dc8e531ff6e64a9c5ab8e6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519381375874265088">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity https://t.co/YCJX1hRQLO pic.twitter.com/zRUCRNXyxH
— NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="a44ed79b9f76aad377b9272fb392f553"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.📯</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519317944995569664">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.
📯 https://t.co/JTUhX0DHQ1@natrescancer pic.twitter.com/2PdCmsDu2p
— NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z">2022-04-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="06ff7641d8f5fcc0dff91f835ac3bbc1"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino  @natrescancer @Columbia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518967636561956865">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino
 https://t.co/GeTVCMJQsp @natrescancer
 @Columbia pic.twitter.com/ZXW6A0Fe0c
— NatureReviewsCancer (@NatureRevCancer) April 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z">2022-04-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="96b2ddff4eb269cf343213c6a1b1ce95"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tracking dormant disseminated tumour cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518563607440621569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tracking dormant disseminated tumour cells https://t.co/kSXyWAvCxF
— NatureReviewsCancer (@NatureRevCancer) April 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>

    <footer>
      <a class="footer-link" href="https://github.com/MacwanEric/cancer.rss.feed/actions/runs/7122254570">
        <time id="build-timestamp" datetime="2023-12-07T00:53:15.291Z">2023-12-07T00:53:15.291Z</time>
      </a>
      <a class="footer-link" href="https://github.com/osmoscraft/osmosfeed">osmosfeed 1.15.1</a>
    </footer>
    <script src="index.js?v1.14.4"></script>
    <!-- %before-body-end.html% -->
  </body>

</html>