<!DOCTYPE html>
<html lang="en">

  <head>
    <title>CancerFeed</title>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="robots" content="noindex, nofollow" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="alternate" type="application/rss+xml" title="CancerFeed" href="feed.atom" />
    <link href="index.css?v1.14.4" rel="stylesheet" />
    <style>
  :root {
    /**
     * Solarized Light
     * Ethan Schoonover (modified by aramisgithub)
     */
    --base00: #fdf6e3;
    --base01: #eee8d5;
    --base02: #93a1a1;
    --base03: #839496;
    --base04: #657b83;
    --base05: #586e75;
    --base06: #073642;
    --base07: #002b36;
    --base08: #dc322f;
    --base09: #cb4b16;
    --base0A: #b58900;
    --base0B: #859900;
    --base0C: #2aa198;
    --base0D: #268bd2;
    --base0E: #6c71c4;
    --base0F: #d33682;

    /**
    * Components
    */
    --source-name-color: var(--base0C);
    /**
   * Typograph
   */
    --font-family-default: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell,
      "Helvetica Neue", sans-serif;
    --font-size-L: 3.6rem;
    --font-size-m: 2.2rem;
    --font-size-s: 1.8rem;
    --font-size-scaler: 52.5%; /* 1rem = 10px */
    --line-height-body: 1.4;
    --line-height-heading: 1.2;

  }
</style>

  </head>

  <body>
    <!-- %after-body-begin.html% -->
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-14 datetime="2022-07-14T10:00:00.000Z">2022-07-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-14 datetime="2022-07-14T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832562"
                        open
                      >
                        <summary class="article-expander__title">A Novel Prognostic Scoring Model for Myelodysplastic Syndrome Patients With SF3B1 Mutation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832562/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The outcomes of myelodysplastic syndrome (MDS) patients with SF3B1 mutation, despite identified as a favorable prognostic biomarker, are variable. To comprehend the heterogeneity in clinical characteristics and outcomes, we reviewed 140 MDS patients with SF3B1 mutation in Zhejiang province of China. Seventy-three (52.1%) patients diagnosed as MDS with ring sideroblasts (MDS-RS) following the 2016 World Health Organization (WHO) classification and 118 (84.3%) patients belonged to lower risk...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35832612"
                        open
                      >
                        <summary class="article-expander__title">Gene utility recapitulates chromosomal aberrancies in advanced stage neuroblastoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35832612/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Neuroblastoma (NB) is the most common extracranial solid tumor in children. Although only a few recurrent somatic mutations have been identified, chromosomal abnormalities, including the loss of heterozygosity (LOH) at the chromosome 1p and gains of chromosome 17q, are often seen in the high-risk cases. The biological basis and evolutionary forces that drive such genetic abnormalities remain enigmatic. Here, we conceptualize the Gene Utility Model (GUM) that seeks to identify genes driving...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-13 datetime="2022-07-13T10:00:00.000Z">2022-07-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-13 datetime="2022-07-13T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831189"
                        open
                      >
                        <summary class="article-expander__title">Pathophysiology of hematological malignancies associated with ASXL1 mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831189/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Somatic mutations in the epigenetic regulator ASXL1 are considered a poor prognostic factor in myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). ASXL1 mutations coexist with other mutations in majority of patients, suggesting that its mutation alone is insufficient to cause cancer. ASXL1 mutations have been detected in age-related clonal hematopoiesis (CH), which has been linked to an increased risk of hematological malignancies. Therefore, ASXL1...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35821803"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis Analyses in Clinical, Epidemiologic, and Genetic Aging Studies to Unravel Underlying Mechanisms of Age-Related Dysfunction in Humans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35821803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging is characterized by increased mortality, functional decline, and exponential increases in the incidence of diseases such as cancer, stroke, cardiovascular disease, neurological disease, respiratory disease, etc. Though the role of aging in these diseases is widely accepted and considered to be a common denominator, the underlying mechanisms are largely unknown. A significant age-related feature observed in many population cohorts is somatic mosaicism, the detectable accumulation of somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35822030"
                        open
                      >
                        <summary class="article-expander__title">Stem Cells in the Myelodysplastic Syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35822030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The myelodysplastic syndromes (MDS) represent a group of clonal disorders characterized by ineffective hematopoiesis, resulting in peripheral cytopenias and frequent transformation to acute myeloid leukemia (AML). We and others have demonstrated that MDS arises in, and is propagated by malignant stem cells (MDS-SCs), that arise due to the sequential acquisition of genetic and epigenetic alterations in normal hematopoietic stem cells (HSCs). This review focuses on recent advancements in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35821803"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis Analyses in Clinical, Epidemiologic, and Genetic Aging Studies to Unravel Underlying Mechanisms of Age-Related Dysfunction in Humans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35821803/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging is characterized by increased mortality, functional decline, and exponential increases in the incidence of diseases such as cancer, stroke, cardiovascular disease, neurological disease, respiratory disease, etc. Though the role of aging in these diseases is widely accepted and considered to be a common denominator, the underlying mechanisms are largely unknown. A significant age-related feature observed in many population cohorts is somatic mosaicism, the detectable accumulation of somatic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35831448"
                        open
                      >
                        <summary class="article-expander__title">Single-cell DNA sequencing identifies risk-associated clonal complexity and evolutionary trajectories in childhood medulloblastoma development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35831448/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reconstructed the natural history and temporal evolution of the most common childhood brain malignancy, medulloblastoma, by single-cell whole-genome sequencing (sc-WGS) of tumours representing its major molecular sub-classes and clinical risk groups. Favourable-risk disease sub-types assessed (MB(WNT) and infant desmoplastic/nodular MB(SHH)) typically comprised a single clone with no evidence of further evolution. In contrast, highest risk sub-classes (MYC-amplified MB(Group3) and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-12 datetime="2022-07-12T10:00:00.000Z">2022-07-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-12 datetime="2022-07-12T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35818539"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular Disease in Myeloproliferative Neoplasms: JACC: CardioOncology State-of-the-Art Review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35818539/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloproliferative neoplasms are associated with increased risk for thrombotic complications. These conditions most commonly involve somatic mutations in genes that lead to constitutive activation of the Janus-associated kinase signaling pathway (eg, Janus kinase 2, calreticulin, myeloproliferative leukemia protein). Acquired gain-of-function mutations in these genes, particularly Janus kinase 2, can cause a spectrum of disorders, ranging from clonal hematopoiesis of indeterminate potential, a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35820396"
                        open
                      >
                        <summary class="article-expander__title">Immune pressure sculps tumor cells and trims high-quality mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35820396/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Immunoediting, the loss of tumor (neo)antigens due to T cell-dependent selection, sculpts tumor immunogenicity. In Nature, Łuksza et al. conceive a model to score neoantigens&#x27; immunogenicity and predict tumor clonal evolution. With this model, they demonstrate that durable tumor control associates with selective limited acquisition of high-quality mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-11 datetime="2022-07-11T10:00:00.000Z">2022-07-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-11 datetime="2022-07-11T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815881"
                        open
                      >
                        <summary class="article-expander__title">Targeted sequencing of candidate gene regions for myelofibrosis in dogs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815881/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS AND CLINICAL IMPORTANCE: Somatic variants might initiate or perpetuate myelofibrosis in dogs. Our findings suggest the occurrence of clonal hematopoiesis in dogs, with increasing incidence with age, as observed in humans.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35815661"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive genetic analysis of histological components of combined small cell carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35815661/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: We conclude that the morphological evolution of heterogeneous histological components in cSCLC may be associated with differences in ASCL1 expression levels, but not in acquired somatic gene mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-09 datetime="2022-07-09T10:00:00.000Z">2022-07-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-09 datetime="2022-07-09T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35805204"
                        open
                      >
                        <summary class="article-expander__title">Association of Clonal Hematopoiesis of Indeterminate Potential with Inflammatory Gene Expression in Patients with COPD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35805204/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic obstructive pulmonary disease (COPD) is a disease with an inflammatory phenotype with increasing prevalence in the elderly. Expanded population of mutant blood cells carrying somatic mutations is termed clonal hematopoiesis of indeterminate potential (CHIP). The association between CHIP and COPD and its relevant effects on DNA methylation in aging are mainly unknown. Analyzing the deep-targeted amplicon sequencing from 125 COPD patients, we found enhanced incidence of CHIP mutations...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-08 datetime="2022-07-08T10:00:00.000Z">2022-07-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-08 datetime="2022-07-08T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35802878"
                        open
                      >
                        <summary class="article-expander__title">JAK inhibitors in the treatment of myelofibrosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35802878/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelofibrosis (MF) is a myeloproliferative neoplasm driven by constitutive activation of the JAK/STAT pathway, resulting in clonal hematopoiesis, fibrotic replacement of the bone marrow, extramedullary hematopoiesis, splenomegaly, and debilitating constitutional symptoms. The advent of JAK inhibitors has changed the landscape of treatment options for patients with MF, providing relatively tolerable drug options that control symptoms, reduce splenomegaly, and improve quality of life, but often at...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35803921"
                        open
                      >
                        <summary class="article-expander__title">Therapy-related clonal cytopenia as a precursor to therapy-related myeloid neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35803921/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Therapy-related myeloid neoplasms (t-MN) are aggressive leukemia that develops as a complication of prior exposure to DNA-damaging agents. Clonal cytopenia of undetermined significance (CCUS) is a precursor of de novo myeloid neoplasms. Characteristics of CCUS that develop following cytotoxic therapies (therapy-related clonal cytopenia, t-CC) and outcomes following t-CC have not been described. We identified 33 patients with t-CC and compared to a cohort of the WHO-defined t-MN (n &#x3D; 309). t-CC...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-07 datetime="2022-07-07T10:00:00.000Z">2022-07-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-07 datetime="2022-07-07T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797508"
                        open
                      >
                        <summary class="article-expander__title">Matched Molecular Profiling of Cell-Free DNA and Tumor Tissue in Patients With Advanced Clear Cell Renal Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797508/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: cfDNA-based NGS profiling yielded low detection rates in this metastatic ccRCC cohort. Concordance with tumor profiling was low, even for truncal mutations such as VHL, and some findings in peripheral blood may represent clonal hematopoiesis. Routine cfDNA panel testing is not supported, and its application in biomarker efforts must account for these limitations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35795468"
                        open
                      >
                        <summary class="article-expander__title">CSF3R T618I mutant myelodysplastic/myeloproliferative neoplasm in the elderly: An age-related disease with unfavorable prognosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35795468/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We reported two MDS/MPN-U and one CMML patients with CSF3R T618I mutation. There were two males and one female, with a median age of 84 years. Three patients presented with leukocytosis and anemia, two with thrombocytopenia, and one with monocytosis. In all the patients, bone marrow showed hypercellularity with myeloid or erythroid predominant trilineage hematopoiesis and dysplasia. Two cases carried -7/-7q abnormalities. In addition to CSF3R T618/I mutation, each case carried 3-5 additional...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35797509"
                        open
                      >
                        <summary class="article-expander__title">Multidimensional Molecular Profiling of Metastatic Triple-Negative Breast Cancer and Immune Checkpoint Inhibitor Benefit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35797509/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This study identified potential biomarkers of response to ICIs among patients with mTNBC: high tumor mutational burden; presence of CD8+, CD4 memory T cells, follicular helper T cells, and M1 macrophages; and inflammatory gene expression pathways. Pretreatment deficiencies in the homologous recombination DNA damage repair pathway and the absence of or minimal clonal evolution post-treatment may be associated with worse outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-07-05 datetime="2022-07-05T10:00:00.000Z">2022-07-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-07-05 datetime="2022-07-05T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications, and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease-related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications, and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease-related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications, and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease-related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications, and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease-related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Introduction: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Introduction: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Introduction: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Introduction: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788175"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788175/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Introduction: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787548"
                        open
                      >
                        <summary class="article-expander__title">Role of p53 in regulation of hematopoiesis in health and disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787548/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Human aging is associated with an exponential increase in the occurrence of clonal hematopoiesis of indeterminate potential (CHIP). CHIP is associated with increased risks of de novo and therapy-related hematologic neoplasms and serves as a reservoir for leukemic relapse. Somatic mutations in the TP53 gene, which encodes the tumor suppressor protein p53, rank in the top five among genes that were mutated in CHIP. TP53 mutations in CHIP are associated with an increased...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787546"
                        open
                      >
                        <summary class="article-expander__title">No time to die? Intrinsic apoptosis signaling in hematopoietic stem and progenitor cells and therapeutic implications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787546/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Dysregulated apoptosis contributes to the pathogenesis of many hematologic malignancies. BH3-mimetics, antagonists of antiapoptotic BCL-2 proteins, represent novel, and promising cancer drugs. While the acute myelosuppressive effects of Venetoclax, the first Food and Drug Administration approved BCL-2 inhibitor, are fairly well described, little is known about side effects of novel BH3-mimetics and effects of chronic Venetoclax treatment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35787092"
                        open
                      >
                        <summary class="article-expander__title">Fedratinib, the first selective JAK2 inhibitor approved for treatment of myelofibrosis - an option beyond ruxolitinib</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35787092/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Introduction: Myelofibrosis, a life shortening clonal disorder, presents with a constellation of features: bone marrow fibrosis, abnormal blood counts, extramedullary hematopoiesis, splenomegaly, thrombohemorrhagic complications and constitutional symptoms. Until recently Ruxolitinib, a JAK1 and 2 inhibitor, has been the only targeted therapy available for transplant-ineligible patients requiring treatment for splenomegaly and disease related symptoms. However, most patients discontinue...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35786159"
                        open
                      >
                        <summary class="article-expander__title">Current and emerging drugs for the treatment of atherosclerosis: the evidence to date</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35786159/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>INTRODUCTION: Atherosclerosis can be considered a chronic inflammatory process that stands out as a dominant cause of cardiovascular disease (CVD). Since blood lipids are the leading risk factor for atherosclerosis development, lowering low-density lipoprotein cholesterol (LDL-C) and other apolipoprotein B-containing lipoproteins reduces the risk of future cardiovascular events. However, there has been significant progress in developing lipid-lowering drugs for aggressive management of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35788590"
                        open
                      >
                        <summary class="article-expander__title">Clonal barcoding with qPCR detection enables live cell functional analyses for cancer research</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35788590/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Single-cell analysis methods are valuable tools; however, current approaches do not easily enable live cell retrieval. That is a particular issue when further study of cells that were eliminated during experimentation could provide critical information. We report a clonal molecular barcoding method, called SunCatcher, that enables longitudinal tracking and live cell functional analysis. From complex cell populations, we generate single cell-derived clonal populations, infect each with a unique...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35785618"
                        open
                      >
                        <summary class="article-expander__title">Single-cell sequencing reveals CD133+CD44--originating evolution and novel stemness related variants in human colorectal cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35785618/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: Tumor heterogeneity of human colorectal cancer (CRC)-initiating cells (CRCICs) in cancer tissues often represents aggressive features of cancer progression. For high-resolution examination of CRCICs, we performed single-cell whole-exome sequencing (scWES) and bulk cell targeted exome sequencing (TES) of CRCICs to investigate stemness-specific somatic alterations or clonal evolution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35781188"
                        open
                      >
                        <summary class="article-expander__title">Germline and Somatic Defects in DDX41 and its Impact on Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35781188/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: While DDX41 mutation (m) is one of the most prevalent predisposition genes in adult myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML), most patients do not always present with a family history of MDS/AML. In this review, we will be highlighting epidemiological data on DDX41m, roles of DDX41 in oncogenesis, mechanisms of clonal evolution with somatic DDX41m, and clinical phenotypes and management of MDS/AML in patients harboring DDX41m.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-30 datetime="2022-06-30T10:00:00.000Z">2022-06-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-30 datetime="2022-06-30T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772431"
                        open
                      >
                        <summary class="article-expander__title">Myelodysplastic syndrome and autoimmune disorders: two sides of the same coin?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772431/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Systemic inflammatory and autoimmune diseases and myelodysplastic syndromes have been linked in individual patients and in larger case series for at least 25 years. These associations frequently include thyroid disease, neutrophilic dermatoses, polyarthritis, connective tissue diseases, vasculitis, and autoimmune cytopenias. Studies have found that autoimmune disease (or its therapy) is a risk factor for the development of myelodysplastic syndromes, but such syndromes might also be an instigator...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35772351"
                        open
                      >
                        <summary class="article-expander__title">Langerhans cell histiocytosis and associated malignancies: A retrospective analysis of 270 patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35772351/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The largest collection of data on LCH-AM to date clearly indicates inherent relationships between specific types of AM and LCH, which may be due to therapy effects, clonal evolution, and germ-line predisposition, respectively. Prospective thorough genetic analysis is warranted and will hopefully shed light on the association of LCH and second neoplasms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35771847"
                        open
                      >
                        <summary class="article-expander__title">Interferon-alpha2 treatment of patients with polycythemia vera and related neoplasms favorably impacts deregulation of oxidative stress genes and antioxidative defense mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35771847/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chronic inflammation is considered a major driving force for clonal expansion and evolution in the Philadelphia-negative myeloproliferative neoplasms, which include essential thrombocythemia, polycythemia vera and primary myelofibrosis (MPNs). One of the key mutation drivers is the JAK2V617F mutation, which has been shown to induce the generation of reactive oxygen species (ROS). Using whole blood gene expression profiling, deregulation of several oxidative stress and anti-oxidative defense...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35770493"
                        open
                      >
                        <summary class="article-expander__title">To B- or not to B-: A review of lineage switched acute leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35770493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute leukemia is a heterogeneous disorder of hematologic malignancies composed primarily of hematopoietic precursors that have acquired unregulated self-renewal and proliferation. Hematology classification systems typically divide these neoplasms into lymphoid (B- or T-) and myeloid-lineage subtypes, with therapy dependent upon this distinction. Infrequently, certain acute leukemias may undergo a complete lineage switch at relapse, subsequently complicating the diagnosis and treatment of these...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-29 datetime="2022-06-29T10:00:00.000Z">2022-06-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-29 datetime="2022-06-29T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive analysis of intratumoural heterogeneity of somatic copy number alterations in diffuse glioma reveals clonality-dependent prognostic patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive Analysis of Intratumoural Heterogeneity of Somatic Copy Number Alterations in Diffuse Glioma Reveals Clonality-dependent Prognostic Patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive Analysis of Intratumoural Heterogeneity of Somatic Copy Number Alterations in Diffuse Glioma Reveals Clonality-dependent Prognostic Patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive Analysis of Intratumoural Heterogeneity of Somatic Copy Number Alterations in Diffuse Glioma Reveals Clonality-dependent Prognostic Patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive Analysis of Intratumoural Heterogeneity of Somatic Copy Number Alterations in Diffuse Glioma Reveals Clonality-dependent Prognostic Patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive Analysis of Intratumoural Heterogeneity of Somatic Copy Number Alterations in Diffuse Glioma Reveals Clonality-dependent Prognostic Patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive Analysis of Intratumoural Heterogeneity of Somatic Copy Number Alterations in Diffuse Glioma Reveals Clonality-dependent Prognostic Patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35767937"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive Analysis of Intratumoural Heterogeneity of Somatic Copy Number Alterations in Diffuse Glioma Reveals Clonality-dependent Prognostic Patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35767937/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: The systematic analysis of ITH-SCNAs in large cohorts of DGs highlighted the importance of considering the clonality of SCNA in discovery of tumour prognostic markers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-28 datetime="2022-06-28T10:00:00.000Z">2022-06-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-28 datetime="2022-06-28T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35764632"
                        open
                      >
                        <summary class="article-expander__title">Clonal reconstruction from co-occurrence of vector integration sites accurately quantifies expanding clones in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35764632/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>High transduction rates of viral vectors in gene therapies (GT) and experimental hematopoiesis ensure a high frequency of gene delivery, although multiple integration events can occur in the same cell. Therefore, tracing of integration sites (IS) leads to mis-quantification of the true clonal spectrum and limits safety considerations in GT. Hence, we use correlations between repeated measurements of IS abundances to estimate their mutual similarity and identify clusters of co-occurring IS, for...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-27 datetime="2022-06-27T10:00:00.000Z">2022-06-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-27 datetime="2022-06-27T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220714212604&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35758777"
                        open
                      >
                        <summary class="article-expander__title">Reconstructing tumor clonal lineage trees incorporating single-nucleotide variants, copy number alterations and structural variations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35758777/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MOTIVATION: Cancer develops through a process of clonal evolution in which an initially healthy cell gives rise to progeny gradually differentiating through the accumulation of genetic and epigenetic mutations. These mutations can take various forms, including single-nucleotide variants (SNVs), copy number alterations (CNAs) or structural variations (SVs), with each variant type providing complementary insights into tumor evolution as well as offering distinct challenges to phylogenetic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-26 datetime="2022-06-26T10:00:00.000Z">2022-06-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-26 datetime="2022-06-26T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35753868"
                        open
                      >
                        <summary class="article-expander__title">Game of clones: Diverse implications for clonal hematopoiesis in lymphoma and multiple myeloma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35753868/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis (CH) refers to the disproportionate expansion of hematopoietic stem cell clones and their corresponding progeny following the acquisition of somatic mutations. CH is common at the time of diagnosis in patients with blood cancers, including multiple myeloma (MM) and lymphoma. The presence of CH mutations correlates with IL-6 mediated inflammation and may result in lymphoma or MM modulation through microenvironment effects or by manifestations of the mutations themselves...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-25 datetime="2022-06-25T10:00:00.000Z">2022-06-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-25 datetime="2022-06-25T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35752705"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis is not prevalent in Hutchinson-Gilford progeria syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35752705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP), defined as the presence of somatic mutations in cancer-related genes in blood cells in the absence of hematological cancer, has recently emerged as an important risk factor for several age-related conditions, especially cardiovascular disease. CHIP is strongly associated with normal aging, but its role in premature aging syndromes is unknown. Hutchinson-Gilford progeria syndrome (HGPS) is an ultra-rare genetic condition driven by the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220713211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751806"
                        open
                      >
                        <summary class="article-expander__title">MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751806/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate detection of somatic mutations in genetically heterogeneous tumor cell populations using next-generation sequencing remains challenging. We have developed MuSE, Mutation calling using a Markov Substitution model for Evolution, a novel approach for modeling the evolution of the allelic composition of tumor and normal tissue at each reference base. It adopts a sample-specific error model to depict inter-tumor heterogeneity, which greatly improves the overall accuracy. Here, we describe...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35751752"
                        open
                      >
                        <summary class="article-expander__title">ANKRD26-Related Thrombocytopenia and Predisposition to Myeloid Neoplasms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35751752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: This review describes ANKRD26-related thrombocytopenia (RT) from a molecular, clinical, and laboratory perspective, with a focus on the clinical decision-making that takes place in the diagnosis and management of families with ANKRD26-RT.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-24 datetime="2022-06-24T10:00:00.000Z">2022-06-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-24 datetime="2022-06-24T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741030"
                        open
                      >
                        <summary class="article-expander__title">Next-Generation Sequencing on Circulating Tumor DNA in Advanced Solid Cancer: Swiss Army Knife for the Molecular Tumor Board? A Review of the Literature Focused on FDA Approved Test</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741030/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>FDA-approved next-generation sequencing assays based on cell-free DNA offers new opportunities in a molecular-tumor-board context thanks to the noninvasiveness of liquid biopsy, the diversity of analyzed parameters and the short turnaround time. It gives the opportunity to study the heterogeneity of the tumor, to elucidate complex resistance mechanisms and to adapt treatment strategies. However, lowering the limit of detection and increasing the panels&#x27; size raise new questions in terms of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220712211231&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35741115"
                        open
                      >
                        <summary class="article-expander__title">Digital Droplet PCR in Hematologic Malignancies: A New Useful Molecular Tool</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35741115/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Digital droplet PCR (ddPCR) is a recent version of quantitative PCR (QT-PCR), useful for measuring gene expression, doing clonality assays and detecting hot spot mutations. In respect of QT-PCR, ddPCR is more sensitive, does not need any reference curve and can quantify one quarter of samples already defined as &quot;positive but not quantifiable&quot;. In the IgH and TCR clonality assessment, ddPCR recapitulates the allele-specific oligonucleotide PCR (ASO-PCR), being not adapt for detecting clonal...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740616"
                        open
                      >
                        <summary class="article-expander__title">The Homologous Recombination Deficiency Scar in Advanced Cancer: Agnostic Targeting of Damaged DNA Repair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740616/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BRCA1 and BRCA2 are the most recognized tumor-suppressor genes involved in double-strand DNA break repair through the homologous recombination (HR) system. Widely known for its role in hereditary cancer, HR deficiency (HRD) has turned out to be critical beyond breast and ovarian cancer: for prostate and pancreatic cancer also. The relevance for the identification of these patients exceeds diagnostic purposes, since results published from clinical trials with poly-ADP ribose polymerase (PARP)...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35740343"
                        open
                      >
                        <summary class="article-expander__title">Stratification of Oligometastatic Prostate Cancer Patients by Liquid Biopsy: Clinical Insights from a Pilot Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35740343/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We propose a pilot, prospective, translational study with the aim of identifying possible molecular markers underlying metastatic prostate cancer (PC) evolution with the use of liquid biopsy. Twenty-eight castrate sensitive, oligometastatic PC patients undergoing bone and/or nodal stereotactic body radiotherapy (SBRT) were recruited. Peripheral blood samples were collected before the commencement of SBRT, then they were processed for circulating cell free DNA (cfDNA) extraction. Deep targeted...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-23 datetime="2022-06-23T10:00:00.000Z">2022-06-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-23 datetime="2022-06-23T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35738887"
                        open
                      >
                        <summary class="article-expander__title">Predicting the likelihood of a BRCA1/2 pathogenic variant being somatic by testing only tumour DNA in non-mucinous high-grade epithelial ovarian cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35738887/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We predict the likelihood of a tumour BRCA1/2 PV being somatic is 99.8% in non-familial cases of NMEOC diagnosed aged ≥75, where the VAF is ≤30% and there is no regional germline commonality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35734926"
                        open
                      >
                        <summary class="article-expander__title">In-depth characterization of intratumoral heterogeneity in refractory B-cell non-Hodgkin lymphoma through the lens of a Research Autopsy Program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35734926/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity (ITH) provides the substrate for tumor evolution and treatment resistance, yet is remarkably understudied in lymphoma, due to the often limited amount of tissue that gets sampled during the routine diagnostic process, generally from a single nodal or extranodal site. Furthermore, the trajectory of how lymphoma, and especially non-Hodgkin lymphoma, spreads throughout the human body remains poorly understood. Here, we present a detailed characterization of ITH by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-22 datetime="2022-06-22T10:00:00.000Z">2022-06-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-22 datetime="2022-06-22T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220711211933&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730391"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in men living with HIV and association with subclinical atherosclerosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730391/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: CH was more common in PLWH and among PLWH was associated with the extent of coronary artery disease. Larger studies are needed to further examine the biological and clinical consequences of CH in PLWH.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35732905"
                        open
                      >
                        <summary class="article-expander__title">A phylogenetic approach to study the evolution of somatic mutational processes in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35732905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer cell genomes change continuously due to mutations, and mutational processes change over time in patients, leaving dynamic signatures in the accumulated genomic variation in tumors. Many computational methods detect the relative activities of known mutation signatures. However, these methods may produce erroneous signatures when applied to individual branches in cancer cell phylogenies. Here, we show that the inference of branch-specific mutational signatures can be improved through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35732905"
                        open
                      >
                        <summary class="article-expander__title">A phylogenetic approach to study the evolution of somatic mutational processes in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35732905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer cell genomes change continuously due to mutations, and mutational processes change over time in patients, leaving dynamic signatures in the accumulated genomic variation in tumors. Many computational methods detect the relative activities of known mutation signatures. However, these methods may produce erroneous signatures when applied to individual branches in cancer cell phylogenies. Here, we show that the inference of branch-specific mutational signatures can be improved through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35732905"
                        open
                      >
                        <summary class="article-expander__title">A phylogenetic approach to study the evolution of somatic mutational processes in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35732905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer cell genomes change continuously due to mutations, and mutational processes change over time in patients, leaving dynamic signatures in the accumulated genomic variation in tumors. Many computational methods detect the relative activities of known mutation signatures. However, these methods may produce erroneous signatures when applied to individual branches in cancer cell phylogenies. Here, we show that the inference of branch-specific mutational signatures can be improved through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35732905"
                        open
                      >
                        <summary class="article-expander__title">A phylogenetic approach to study the evolution of somatic mutational processes in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35732905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer cell genomes change continuously due to mutations, and mutational processes change over time in patients, leaving dynamic signatures in the accumulated genomic variation in tumors. Many computational methods detect the relative activities of known mutation signatures. However, these methods may produce erroneous signatures when applied to individual branches in cancer cell phylogenies. Here, we show that the inference of branch-specific mutational signatures can be improved through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35732905"
                        open
                      >
                        <summary class="article-expander__title">A phylogenetic approach to study the evolution of somatic mutational processes in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35732905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer cell genomes change continuously due to mutations, and mutational processes change over time in patients, leaving dynamic signatures in the accumulated genomic variation in tumors. Many computational methods detect the relative activities of known mutation signatures. However, these methods may produce erroneous signatures when applied to individual branches in cancer cell phylogenies. Here, we show that the inference of branch-specific mutational signatures can be improved through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730597"
                        open
                      >
                        <summary class="article-expander__title">Genetic analysis of pheochromocytoma and paraganglioma complicating cyanotic congenital heart disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730597/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results highlight a key role of activated HIF2α due to mutated EPAS1 in positive selection under hypoxic environments, although hypoxemia itself may not necessarily be required for the EPAS1-mutated clones to progress to PPGL.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35732905"
                        open
                      >
                        <summary class="article-expander__title">A phylogenetic approach to study the evolution of somatic mutational processes in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35732905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer cell genomes change continuously due to mutations, and mutational processes change over time in patients, leaving dynamic signatures in the accumulated genomic variation in tumors. Many computational methods detect the relative activities of known mutation signatures. However, these methods may produce erroneous signatures when applied to individual branches in cancer cell phylogenies. Here, we show that the inference of branch-specific mutational signatures can be improved through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730597"
                        open
                      >
                        <summary class="article-expander__title">Genetic analysis of pheochromocytoma and paraganglioma complicating cyanotic congenital heart disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730597/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results highlight a key role of activated HIF2α due to mutated EPAS1 in positive selection under hypoxic environments, although hypoxemia itself may not necessarily be required for the EPAS1-mutated clones to progress to PPGL.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35732905"
                        open
                      >
                        <summary class="article-expander__title">A phylogenetic approach to study the evolution of somatic mutational processes in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35732905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer cell genomes change continuously due to mutations, and mutational processes change over time in patients, leaving dynamic signatures in the accumulated genomic variation in tumors. Many computational methods detect the relative activities of known mutation signatures. However, these methods may produce erroneous signatures when applied to individual branches in cancer cell phylogenies. Here, we show that the inference of branch-specific mutational signatures can be improved through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730597"
                        open
                      >
                        <summary class="article-expander__title">Genetic analysis of pheochromocytoma and paraganglioma complicating cyanotic congenital heart disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730597/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results highlight a key role of activated HIF2α due to mutated EPAS1 in positive selection under hypoxic environments, although hypoxemia itself may not necessarily be required for the EPAS1-mutated clones to progress to PPGL.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35732905"
                        open
                      >
                        <summary class="article-expander__title">A phylogenetic approach to study the evolution of somatic mutational processes in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35732905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer cell genomes change continuously due to mutations, and mutational processes change over time in patients, leaving dynamic signatures in the accumulated genomic variation in tumors. Many computational methods detect the relative activities of known mutation signatures. However, these methods may produce erroneous signatures when applied to individual branches in cancer cell phylogenies. Here, we show that the inference of branch-specific mutational signatures can be improved through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730597"
                        open
                      >
                        <summary class="article-expander__title">Genetic analysis of pheochromocytoma and paraganglioma complicating cyanotic congenital heart disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730597/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results highlight a key role of activated HIF2α due to mutated EPAS1 in positive selection under hypoxic environments, although hypoxemia itself may not necessarily be required for the EPAS1-mutated clones to progress to PPGL.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35732905"
                        open
                      >
                        <summary class="article-expander__title">A phylogenetic approach to study the evolution of somatic mutational processes in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35732905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer cell genomes change continuously due to mutations, and mutational processes change over time in patients, leaving dynamic signatures in the accumulated genomic variation in tumors. Many computational methods detect the relative activities of known mutation signatures. However, these methods may produce erroneous signatures when applied to individual branches in cancer cell phylogenies. Here, we show that the inference of branch-specific mutational signatures can be improved through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730597"
                        open
                      >
                        <summary class="article-expander__title">Genetic analysis of pheochromocytoma and paraganglioma complicating cyanotic congenital heart disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730597/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results highlight a key role of activated HIF2α due to mutated EPAS1 in positive selection under hypoxic environments, although hypoxemia itself may not necessarily be required for the EPAS1-mutated clones to progress to PPGL.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35732905"
                        open
                      >
                        <summary class="article-expander__title">A phylogenetic approach to study the evolution of somatic mutational processes in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35732905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer cell genomes change continuously due to mutations, and mutational processes change over time in patients, leaving dynamic signatures in the accumulated genomic variation in tumors. Many computational methods detect the relative activities of known mutation signatures. However, these methods may produce erroneous signatures when applied to individual branches in cancer cell phylogenies. Here, we show that the inference of branch-specific mutational signatures can be improved through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730597"
                        open
                      >
                        <summary class="article-expander__title">Genetic analysis of pheochromocytoma and paraganglioma complicating cyanotic congenital heart disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730597/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results highlight a key role of activated HIF2α due to mutated EPAS1 in positive selection under hypoxic environments, although hypoxemia itself may not necessarily be required for the EPAS1-mutated clones to progress to PPGL.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35732905"
                        open
                      >
                        <summary class="article-expander__title">A phylogenetic approach to study the evolution of somatic mutational processes in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35732905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer cell genomes change continuously due to mutations, and mutational processes change over time in patients, leaving dynamic signatures in the accumulated genomic variation in tumors. Many computational methods detect the relative activities of known mutation signatures. However, these methods may produce erroneous signatures when applied to individual branches in cancer cell phylogenies. Here, we show that the inference of branch-specific mutational signatures can be improved through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730597"
                        open
                      >
                        <summary class="article-expander__title">Genetic analysis of pheochromocytoma and paraganglioma complicating cyanotic congenital heart disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730597/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results highlight a key role of activated HIF2α due to mutated EPAS1 in positive selection under hypoxic environments, although hypoxemia itself may not necessarily be required for the EPAS1-mutated clones to progress to PPGL.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35732905"
                        open
                      >
                        <summary class="article-expander__title">A phylogenetic approach to study the evolution of somatic mutational processes in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35732905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer cell genomes change continuously due to mutations, and mutational processes change over time in patients, leaving dynamic signatures in the accumulated genomic variation in tumors. Many computational methods detect the relative activities of known mutation signatures. However, these methods may produce erroneous signatures when applied to individual branches in cancer cell phylogenies. Here, we show that the inference of branch-specific mutational signatures can be improved through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730597"
                        open
                      >
                        <summary class="article-expander__title">Genetic analysis of pheochromocytoma and paraganglioma complicating cyanotic congenital heart disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730597/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results highlight a key role of activated HIF2α due to mutated EPAS1 in positive selection under hypoxic environments, although hypoxemia itself may not necessarily be required for the EPAS1-mutated clones to progress to PPGL.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35732905"
                        open
                      >
                        <summary class="article-expander__title">A phylogenetic approach to study the evolution of somatic mutational processes in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35732905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer cell genomes change continuously due to mutations, and mutational processes change over time in patients, leaving dynamic signatures in the accumulated genomic variation in tumors. Many computational methods detect the relative activities of known mutation signatures. However, these methods may produce erroneous signatures when applied to individual branches in cancer cell phylogenies. Here, we show that the inference of branch-specific mutational signatures can be improved through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730597"
                        open
                      >
                        <summary class="article-expander__title">Genetic analysis of pheochromocytoma and paraganglioma complicating cyanotic congenital heart disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730597/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results highlight a key role of activated HIF2α due to mutated EPAS1 in positive selection under hypoxic environments, although hypoxemia itself may not necessarily be required for the EPAS1-mutated clones to progress to PPGL.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35732905"
                        open
                      >
                        <summary class="article-expander__title">A phylogenetic approach to study the evolution of somatic mutational processes in cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35732905/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cancer cell genomes change continuously due to mutations, and mutational processes change over time in patients, leaving dynamic signatures in the accumulated genomic variation in tumors. Many computational methods detect the relative activities of known mutation signatures. However, these methods may produce erroneous signatures when applied to individual branches in cancer cell phylogenies. Here, we show that the inference of branch-specific mutational signatures can be improved through a...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730597"
                        open
                      >
                        <summary class="article-expander__title">Genetic analysis of pheochromocytoma and paraganglioma complicating cyanotic congenital heart disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730597/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results highlight a key role of activated HIF2α due to mutated EPAS1 in positive selection under hypoxic environments, although hypoxemia itself may not necessarily be required for the EPAS1-mutated clones to progress to PPGL.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35731928"
                        open
                      >
                        <summary class="article-expander__title">Treatment represents a key driver of metastatic cancer evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35731928/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Metastasis is the main cause of cancer death, yet the evolutionary processes behind it remain largely unknown. Here, through analysis of large panel-based genomic datasets from the AACR GENIE project, including 40,979 primary and metastatic tumors across 25 distinct cancer types, we explore how the evolutionary pressure of cancer metastasis shapes the selection of genomic drivers of cancer. The most commonly affected genes were TP53, MYC, and CDKN2A, with no specific pattern associated with...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35730597"
                        open
                      >
                        <summary class="article-expander__title">Genetic analysis of pheochromocytoma and paraganglioma complicating cyanotic congenital heart disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35730597/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results highlight a key role of activated HIF2α due to mutated EPAS1 in positive selection under hypoxic environments, although hypoxemia itself may not necessarily be required for the EPAS1-mutated clones to progress to PPGL.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-21 datetime="2022-06-21T10:00:00.000Z">2022-06-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-21 datetime="2022-06-21T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220710211548&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220709212311&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726685"
                        open
                      >
                        <summary class="article-expander__title">The sculpting of somatic mutational landscapes by evolutionary forces and their impacts on aging-related disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726685/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aging represents the major risk factor for the development of cancer and many other diseases. Recent findings show that normal tissues become riddled with expanded clones that are frequently driven by cancer-associated mutations in an aging-dependent fashion. Additional studies show how aged tissue microenvironments promote the initiation and progression of malignancies, while young healthy tissues actively suppress the outgrowth of malignant clones. Here, we discuss conserved mechanisms that...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726693"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity and rates of specific chromosome gains are risk predictors in childhood high-hyperdiploid B-cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726693/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell acute lymphoblastic leukemia (B-ALL) is the commonest childhood cancer. High hyperdiploidy (HHD) identifies the most frequent cytogenetic subgroup in childhood B-ALL. Although hyperdiploidy represents an important prognostic factor in childhood B-ALL, the specific chromosome gains with prognostic value in HHD-B-ALL remain controversial, and the current knowledge about the hierarchy of chromosome gains, clonal heterogeneity and chromosomal instability in HHD-B-ALL remains very limited. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726693"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity and rates of specific chromosome gains are risk predictors in childhood high-hyperdiploid B-cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726693/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell acute lymphoblastic leukemia (B-ALL) is the commonest childhood cancer. High hyperdiploidy (HHD) identifies the most frequent cytogenetic subgroup in childhood B-ALL. Although hyperdiploidy represents an important prognostic factor in childhood B-ALL, the specific chromosome gains with prognostic value in HHD-B-ALL remain controversial, and the current knowledge about the hierarchy of chromosome gains, clonal heterogeneity and chromosomal instability in HHD-B-ALL remains very limited. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35725896"
                        open
                      >
                        <summary class="article-expander__title">A clinically annotated post-mortem approach to study multi-organ somatic mutational clonality in normal tissues</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35725896/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent research on normal human tissues identified omnipresent clones of cells, driven by somatic mutations known to be responsible for carcinogenesis (e.g., in TP53 or NOTCH1). These new insights are fundamentally changing current tumor evolution models, with broad oncological implications. Most studies are based on surgical remnant tissues, which are not available for many organs and rarely in a pan-organ setting (multiple organs from the same individual). Here, we describe an approach based...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726693"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity and rates of specific chromosome gains are risk predictors in childhood high-hyperdiploid B-cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726693/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell acute lymphoblastic leukemia (B-ALL) is the commonest childhood cancer. High hyperdiploidy (HHD) identifies the most frequent cytogenetic subgroup in childhood B-ALL. Although hyperdiploidy represents an important prognostic factor in childhood B-ALL, the specific chromosome gains with prognostic value in HHD-B-ALL remain controversial, and the current knowledge about the hierarchy of chromosome gains, clonal heterogeneity and chromosomal instability in HHD-B-ALL remains very limited. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35725896"
                        open
                      >
                        <summary class="article-expander__title">A clinically annotated post-mortem approach to study multi-organ somatic mutational clonality in normal tissues</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35725896/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent research on normal human tissues identified omnipresent clones of cells, driven by somatic mutations known to be responsible for carcinogenesis (e.g., in TP53 or NOTCH1). These new insights are fundamentally changing current tumor evolution models, with broad oncological implications. Most studies are based on surgical remnant tissues, which are not available for many organs and rarely in a pan-organ setting (multiple organs from the same individual). Here, we describe an approach based...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726693"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity and rates of specific chromosome gains are risk predictors in childhood high-hyperdiploid B-cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726693/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell acute lymphoblastic leukemia (B-ALL) is the commonest childhood cancer. High hyperdiploidy (HHD) identifies the most frequent cytogenetic subgroup in childhood B-ALL. Although hyperdiploidy represents an important prognostic factor in childhood B-ALL, the specific chromosome gains with prognostic value in HHD-B-ALL remain controversial, and the current knowledge about the hierarchy of chromosome gains, clonal heterogeneity and chromosomal instability in HHD-B-ALL remains very limited. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35725896"
                        open
                      >
                        <summary class="article-expander__title">A clinically annotated post-mortem approach to study multi-organ somatic mutational clonality in normal tissues</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35725896/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent research on normal human tissues identified omnipresent clones of cells, driven by somatic mutations known to be responsible for carcinogenesis (e.g., in TP53 or NOTCH1). These new insights are fundamentally changing current tumor evolution models, with broad oncological implications. Most studies are based on surgical remnant tissues, which are not available for many organs and rarely in a pan-organ setting (multiple organs from the same individual). Here, we describe an approach based...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726693"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity and rates of specific chromosome gains are risk predictors in childhood high-hyperdiploid B-cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726693/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell acute lymphoblastic leukemia (B-ALL) is the commonest childhood cancer. High hyperdiploidy (HHD) identifies the most frequent cytogenetic subgroup in childhood B-ALL. Although hyperdiploidy represents an important prognostic factor in childhood B-ALL, the specific chromosome gains with prognostic value in HHD-B-ALL remain controversial, and the current knowledge about the hierarchy of chromosome gains, clonal heterogeneity and chromosomal instability in HHD-B-ALL remains very limited. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35725896"
                        open
                      >
                        <summary class="article-expander__title">A clinically annotated post-mortem approach to study multi-organ somatic mutational clonality in normal tissues</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35725896/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent research on normal human tissues identified omnipresent clones of cells, driven by somatic mutations known to be responsible for carcinogenesis (e.g., in TP53 or NOTCH1). These new insights are fundamentally changing current tumor evolution models, with broad oncological implications. Most studies are based on surgical remnant tissues, which are not available for many organs and rarely in a pan-organ setting (multiple organs from the same individual). Here, we describe an approach based...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726693"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity and rates of specific chromosome gains are risk predictors in childhood high-hyperdiploid B-cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726693/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell acute lymphoblastic leukemia (B-ALL) is the commonest childhood cancer. High hyperdiploidy (HHD) identifies the most frequent cytogenetic subgroup in childhood B-ALL. Although hyperdiploidy represents an important prognostic factor in childhood B-ALL, the specific chromosome gains with prognostic value in HHD-B-ALL remain controversial, and the current knowledge about the hierarchy of chromosome gains, clonal heterogeneity and chromosomal instability in HHD-B-ALL remains very limited. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35725896"
                        open
                      >
                        <summary class="article-expander__title">A clinically annotated post-mortem approach to study multi-organ somatic mutational clonality in normal tissues</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35725896/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent research on normal human tissues identified omnipresent clones of cells, driven by somatic mutations known to be responsible for carcinogenesis (e.g., in TP53 or NOTCH1). These new insights are fundamentally changing current tumor evolution models, with broad oncological implications. Most studies are based on surgical remnant tissues, which are not available for many organs and rarely in a pan-organ setting (multiple organs from the same individual). Here, we describe an approach based...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726693"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity and rates of specific chromosome gains are risk predictors in childhood high-hyperdiploid B-cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726693/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell acute lymphoblastic leukemia (B-ALL) is the commonest childhood cancer. High hyperdiploidy (HHD) identifies the most frequent cytogenetic subgroup in childhood B-ALL. Although hyperdiploidy represents an important prognostic factor in childhood B-ALL, the specific chromosome gains with prognostic value in HHD-B-ALL remain controversial, and the current knowledge about the hierarchy of chromosome gains, clonal heterogeneity and chromosomal instability in HHD-B-ALL remains very limited. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35725896"
                        open
                      >
                        <summary class="article-expander__title">A clinically annotated post-mortem approach to study multi-organ somatic mutational clonality in normal tissues</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35725896/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent research on normal human tissues identified omnipresent clones of cells, driven by somatic mutations known to be responsible for carcinogenesis (e.g., in TP53 or NOTCH1). These new insights are fundamentally changing current tumor evolution models, with broad oncological implications. Most studies are based on surgical remnant tissues, which are not available for many organs and rarely in a pan-organ setting (multiple organs from the same individual). Here, we describe an approach based...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726693"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity and rates of specific chromosome gains are risk predictors in childhood high-hyperdiploid B-cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726693/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell acute lymphoblastic leukemia (B-ALL) is the commonest childhood cancer. High hyperdiploidy (HHD) identifies the most frequent cytogenetic subgroup in childhood B-ALL. Although hyperdiploidy represents an important prognostic factor in childhood B-ALL, the specific chromosome gains with prognostic value in HHD-B-ALL remain controversial, and the current knowledge about the hierarchy of chromosome gains, clonal heterogeneity and chromosomal instability in HHD-B-ALL remains very limited. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35725896"
                        open
                      >
                        <summary class="article-expander__title">A clinically annotated post-mortem approach to study multi-organ somatic mutational clonality in normal tissues</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35725896/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent research on normal human tissues identified omnipresent clones of cells, driven by somatic mutations known to be responsible for carcinogenesis (e.g., in TP53 or NOTCH1). These new insights are fundamentally changing current tumor evolution models, with broad oncological implications. Most studies are based on surgical remnant tissues, which are not available for many organs and rarely in a pan-organ setting (multiple organs from the same individual). Here, we describe an approach based...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726693"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity and rates of specific chromosome gains are risk predictors in childhood high-hyperdiploid B-cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726693/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell acute lymphoblastic leukemia (B-ALL) is the commonest childhood cancer. High hyperdiploidy (HHD) identifies the most frequent cytogenetic subgroup in childhood B-ALL. Although hyperdiploidy represents an important prognostic factor in childhood B-ALL, the specific chromosome gains with prognostic value in HHD-B-ALL remain controversial, and the current knowledge about the hierarchy of chromosome gains, clonal heterogeneity and chromosomal instability in HHD-B-ALL remains very limited. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35725896"
                        open
                      >
                        <summary class="article-expander__title">A clinically annotated post-mortem approach to study multi-organ somatic mutational clonality in normal tissues</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35725896/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent research on normal human tissues identified omnipresent clones of cells, driven by somatic mutations known to be responsible for carcinogenesis (e.g., in TP53 or NOTCH1). These new insights are fundamentally changing current tumor evolution models, with broad oncological implications. Most studies are based on surgical remnant tissues, which are not available for many organs and rarely in a pan-organ setting (multiple organs from the same individual). Here, we describe an approach based...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726693"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity and rates of specific chromosome gains are risk predictors in childhood high-hyperdiploid B-cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726693/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell acute lymphoblastic leukemia (B-ALL) is the commonest childhood cancer. High hyperdiploidy (HHD) identifies the most frequent cytogenetic subgroup in childhood B-ALL. Although hyperdiploidy represents an important prognostic factor in childhood B-ALL, the specific chromosome gains with prognostic value in HHD-B-ALL remain controversial, and the current knowledge about the hierarchy of chromosome gains, clonal heterogeneity and chromosomal instability in HHD-B-ALL remains very limited. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35725896"
                        open
                      >
                        <summary class="article-expander__title">A clinically annotated post-mortem approach to study multi-organ somatic mutational clonality in normal tissues</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35725896/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent research on normal human tissues identified omnipresent clones of cells, driven by somatic mutations known to be responsible for carcinogenesis (e.g., in TP53 or NOTCH1). These new insights are fundamentally changing current tumor evolution models, with broad oncological implications. Most studies are based on surgical remnant tissues, which are not available for many organs and rarely in a pan-organ setting (multiple organs from the same individual). Here, we describe an approach based...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726693"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity and rates of specific chromosome gains are risk predictors in childhood high-hyperdiploid B-cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726693/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell acute lymphoblastic leukemia (B-ALL) is the commonest childhood cancer. High hyperdiploidy (HHD) identifies the most frequent cytogenetic subgroup in childhood B-ALL. Although hyperdiploidy represents an important prognostic factor in childhood B-ALL, the specific chromosome gains with prognostic value in HHD-B-ALL remain controversial, and the current knowledge about the hierarchy of chromosome gains, clonal heterogeneity and chromosomal instability in HHD-B-ALL remains very limited. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35725896"
                        open
                      >
                        <summary class="article-expander__title">A clinically annotated post-mortem approach to study multi-organ somatic mutational clonality in normal tissues</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35725896/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent research on normal human tissues identified omnipresent clones of cells, driven by somatic mutations known to be responsible for carcinogenesis (e.g., in TP53 or NOTCH1). These new insights are fundamentally changing current tumor evolution models, with broad oncological implications. Most studies are based on surgical remnant tissues, which are not available for many organs and rarely in a pan-organ setting (multiple organs from the same individual). Here, we describe an approach based...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35726693"
                        open
                      >
                        <summary class="article-expander__title">Clonal heterogeneity and rates of specific chromosome gains are risk predictors in childhood high-hyperdiploid B-cell acute lymphoblastic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35726693/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>B-cell acute lymphoblastic leukemia (B-ALL) is the commonest childhood cancer. High hyperdiploidy (HHD) identifies the most frequent cytogenetic subgroup in childhood B-ALL. Although hyperdiploidy represents an important prognostic factor in childhood B-ALL, the specific chromosome gains with prognostic value in HHD-B-ALL remain controversial, and the current knowledge about the hierarchy of chromosome gains, clonal heterogeneity and chromosomal instability in HHD-B-ALL remains very limited. We...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35725896"
                        open
                      >
                        <summary class="article-expander__title">A clinically annotated post-mortem approach to study multi-organ somatic mutational clonality in normal tissues</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35725896/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent research on normal human tissues identified omnipresent clones of cells, driven by somatic mutations known to be responsible for carcinogenesis (e.g., in TP53 or NOTCH1). These new insights are fundamentally changing current tumor evolution models, with broad oncological implications. Most studies are based on surgical remnant tissues, which are not available for many organs and rarely in a pan-organ setting (multiple organs from the same individual). Here, we describe an approach based...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-20 datetime="2022-06-20T10:00:00.000Z">2022-06-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-20 datetime="2022-06-20T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220708210406&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35724849"
                        open
                      >
                        <summary class="article-expander__title">Increased late non-cardiac non-relapse mortality in patients with atrial fibrillation diagnosed during hospital stay for allogeneic stem cell transplantation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35724849/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Although the underlying cellular and molecular mechanisms remain to be characterized in further detail, these data clearly demonstrate the impact of in-patient AF manifestation/AFiH on long-term outcome of SCT patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-19 datetime="2022-06-19T10:00:00.000Z">2022-06-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-19 datetime="2022-06-19T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220707211901&amp;v&#x3D;2.17.7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35718411"
                        open
                      >
                        <summary class="article-expander__title">Therapy-Related Clonal Hematopoiesis: A New Link Between Cancer and Cardiovascular Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35718411/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis is a precancerous state that is recognized as a new causal risk factor for cardiovascular disease. Therapy-related clonal hematopoiesis is a condition that is often found in cancer survivors. These clonal expansions are caused by mutations in DNA damage-response pathway genes that allow hematopoietic stem cells to undergo positive selection in response to the genotoxic stress. These mutant cells increasingly give rise to progeny leukocytes that display enhanced...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-17 datetime="2022-06-17T10:00:00.000Z">2022-06-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-17 datetime="2022-06-17T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220706212003&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714308"
                        open
                      >
                        <summary class="article-expander__title">TET2-mutant clonal hematopoiesis and risk of gout</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714308/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gout is a common inflammatory arthritis caused by precipitation of monosodium urate (MSU) crystals in individuals with hyperuricemia. Acute flares are accompanied by secretion of pro-inflammatory cytokines, including interleukin-1 beta (IL-1B). Clonal hematopoiesis of indeterminate potential (CHIP) is an age-related condition predisposing to hematologic cancers and cardiovascular disease. CHIP is associated with elevated IL-1B, thus we investigated CHIP as a risk factor for gout. To test the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35714307"
                        open
                      >
                        <summary class="article-expander__title">Macaque Clonal Hematopoiesis Model Demonstrates Expansion of TET2-Disrupted Clones and Utility for Testing Interventions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35714307/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Individuals with age-related clonal hematopoiesis (CH) are at greater risk for hematologic malignancies and cardiovascular diseases. However, predictive preclinical animal models to recapitulate the spectrum of human CH are lacking. Through error-corrected sequencing of 56 human CH/myeloid malignancy genes we identified natural CH driver mutations in aged rhesus macaques matching genes somatically mutated in human CH, with DNMT3A mutations being the most frequent. A CH model in young adult...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-16 datetime="2022-06-16T10:00:00.000Z">2022-06-16</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-16 datetime="2022-06-16T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220705212255&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35709748"
                        open
                      >
                        <summary class="article-expander__title">Bone marrow hematopoiesis drives multiple sclerosis progression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35709748/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiple sclerosis (MS) is a T cell-mediated autoimmune disease of the central nervous system (CNS). Bone marrow hematopoietic stem and progenitor cells (HSPCs) rapidly sense immune activation, yet their potential interplay with autoreactive T cells in MS is unknown. Here, we report that bone marrow HSPCs are skewed toward myeloid lineage concomitant with the clonal expansion of T cells in MS patients. Lineage tracing in experimental autoimmune encephalomyelitis, a mouse model of MS, reveals...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35710421"
                        open
                      >
                        <summary class="article-expander__title">HBV genome-enriched single cell sequencing revealed heterogeneity in HBV-driven hepatocellular carcinoma (HCC)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35710421/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: Our HGE-scSeq data reveals high heterogeneity of HCC tumor cells in terms of both HBV integrations and CNVs. There were two HBV integration hotspots across cells, and cells from multiple tumor sites shared some HBV integration and CNV patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-15 datetime="2022-06-15T10:00:00.000Z">2022-06-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-15 datetime="2022-06-15T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705805"
                        open
                      >
                        <summary class="article-expander__title">Lifelong multilineage contribution by embryonic-born blood progenitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705805/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Haematopoietic stem cells (HSCs) arise in the embryo from the arterial endothelium through a process known as the endothelial-to-haematopoietic transition (EHT)^(1-4). This process generates hundreds of blood progenitors, of which a fraction go on to become definitive HSCs. It is generally thought that most adult blood is derived from those HSCs, but to what extent other progenitors contribute to adult haematopoiesis is not known. Here we use in situ barcoding and classical fate mapping to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705805"
                        open
                      >
                        <summary class="article-expander__title">Lifelong multilineage contribution by embryonic-born blood progenitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705805/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Haematopoietic stem cells (HSCs) arise in the embryo from the arterial endothelium through a process known as the endothelial-to-haematopoietic transition (EHT)^(1-4). This process generates hundreds of blood progenitors, of which a fraction go on to become definitive HSCs. It is generally thought that most adult blood is derived from those HSCs, but to what extent other progenitors contribute to adult haematopoiesis is not known. Here we use in situ barcoding and classical fate mapping to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705805"
                        open
                      >
                        <summary class="article-expander__title">Lifelong multilineage contribution by embryonic-born blood progenitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705805/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Haematopoietic stem cells (HSCs) arise in the embryo from the arterial endothelium through a process known as the endothelial-to-haematopoietic transition (EHT)^(1-4). This process generates hundreds of blood progenitors, of which a fraction go on to become definitive HSCs. It is generally thought that most adult blood is derived from those HSCs, but to what extent other progenitors contribute to adult haematopoiesis is not known. Here we use in situ barcoding and classical fate mapping to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705805"
                        open
                      >
                        <summary class="article-expander__title">Lifelong multilineage contribution by embryonic-born blood progenitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705805/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Haematopoietic stem cells (HSCs) arise in the embryo from the arterial endothelium through a process known as the endothelial-to-haematopoietic transition (EHT)^(1-4). This process generates hundreds of blood progenitors, of which a fraction go on to become definitive HSCs. It is generally thought that most adult blood is derived from those HSCs, but to what extent other progenitors contribute to adult haematopoiesis is not known. Here we use in situ barcoding and classical fate mapping to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705805"
                        open
                      >
                        <summary class="article-expander__title">Lifelong multilineage contribution by embryonic-born blood progenitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705805/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Haematopoietic stem cells (HSCs) arise in the embryo from the arterial endothelium through a process known as the endothelial-to-haematopoietic transition (EHT)^(1-4). This process generates hundreds of blood progenitors, of which a fraction go on to become definitive HSCs. It is generally thought that most adult blood is derived from those HSCs, but to what extent other progenitors contribute to adult haematopoiesis is not known. Here we use in situ barcoding and classical fate mapping to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705805"
                        open
                      >
                        <summary class="article-expander__title">Lifelong multilineage contribution by embryonic-born blood progenitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705805/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Haematopoietic stem cells (HSCs) arise in the embryo from the arterial endothelium through a process known as the endothelial-to-haematopoietic transition (EHT)^(1-4). This process generates hundreds of blood progenitors, of which a fraction go on to become definitive HSCs. It is generally thought that most adult blood is derived from those HSCs, but to what extent other progenitors contribute to adult haematopoiesis is not known. Here we use in situ barcoding and classical fate mapping to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705805"
                        open
                      >
                        <summary class="article-expander__title">Lifelong multilineage contribution by embryonic-born blood progenitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705805/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Haematopoietic stem cells (HSCs) arise in the embryo from the arterial endothelium through a process known as the endothelial-to-haematopoietic transition (EHT)^(1-4). This process generates hundreds of blood progenitors, of which a fraction go on to become definitive HSCs. It is generally thought that most adult blood is derived from those HSCs, but to what extent other progenitors contribute to adult haematopoiesis is not known. Here we use in situ barcoding and classical fate mapping to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705805"
                        open
                      >
                        <summary class="article-expander__title">Lifelong multilineage contribution by embryonic-born blood progenitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705805/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Haematopoietic stem cells (HSCs) arise in the embryo from the arterial endothelium through a process known as the endothelial-to-haematopoietic transition (EHT)^(1-4). This process generates hundreds of blood progenitors, of which a fraction go on to become definitive HSCs. It is generally thought that most adult blood is derived from those HSCs, but to what extent other progenitors contribute to adult haematopoiesis is not known. Here we use in situ barcoding and classical fate mapping to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705805"
                        open
                      >
                        <summary class="article-expander__title">Lifelong multilineage contribution by embryonic-born blood progenitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705805/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Haematopoietic stem cells (HSCs) arise in the embryo from the arterial endothelium through a process known as the endothelial-to-haematopoietic transition (EHT)^(1-4). This process generates hundreds of blood progenitors, of which a fraction go on to become definitive HSCs. It is generally thought that most adult blood is derived from those HSCs, but to what extent other progenitors contribute to adult haematopoiesis is not known. Here we use in situ barcoding and classical fate mapping to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705805"
                        open
                      >
                        <summary class="article-expander__title">Lifelong multilineage contribution by embryonic-born blood progenitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705805/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Haematopoietic stem cells (HSCs) arise in the embryo from the arterial endothelium through a process known as the endothelial-to-haematopoietic transition (EHT)^(1-4). This process generates hundreds of blood progenitors, of which a fraction go on to become definitive HSCs. It is generally thought that most adult blood is derived from those HSCs, but to what extent other progenitors contribute to adult haematopoiesis is not known. Here we use in situ barcoding and classical fate mapping to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705805"
                        open
                      >
                        <summary class="article-expander__title">Lifelong multilineage contribution by embryonic-born blood progenitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705805/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Haematopoietic stem cells (HSCs) arise in the embryo from the arterial endothelium through a process known as the endothelial-to-haematopoietic transition (EHT)^(1-4). This process generates hundreds of blood progenitors, of which a fraction go on to become definitive HSCs. It is generally thought that most adult blood is derived from those HSCs, but to what extent other progenitors contribute to adult haematopoiesis is not known. Here we use in situ barcoding and classical fate mapping to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705805"
                        open
                      >
                        <summary class="article-expander__title">Lifelong multilineage contribution by embryonic-born blood progenitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705805/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Haematopoietic stem cells (HSCs) arise in the embryo from the arterial endothelium through a process known as the endothelial-to-haematopoietic transition (EHT)^(1-4). This process generates hundreds of blood progenitors, of which a fraction go on to become definitive HSCs. It is generally thought that most adult blood is derived from those HSCs, but to what extent other progenitors contribute to adult haematopoiesis is not known. Here we use in situ barcoding and classical fate mapping to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705805"
                        open
                      >
                        <summary class="article-expander__title">Lifelong multilineage contribution by embryonic-born blood progenitors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705805/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Haematopoietic stem cells (HSCs) arise in the embryo from the arterial endothelium through a process known as the endothelial-to-haematopoietic transition (EHT)^(1-4). This process generates hundreds of blood progenitors, of which a fraction go on to become definitive HSCs. It is generally thought that most adult blood is derived from those HSCs, but to what extent other progenitors contribute to adult haematopoiesis is not known. Here we use in situ barcoding and classical fate mapping to...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35705990"
                        open
                      >
                        <summary class="article-expander__title">Hematopoietic differentiation persists in human iPSCs defective in de novo DNA methylation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35705990/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Our results demonstrate that iPSCs with homozygous knockout of different exons of DNMT3A remain capable of mesenchymal and hematopoietic differentiation-and exon 23^(-/-) iHPCs even gained growth advantage-despite loss of almost the entire de novo DNA methylation. Partial recapitulation of DNA methylation patterns of AML with DNMT3A mutations by our DNMT3A knockout iHPCs indicates that our model system can help to elucidate mechanisms of clonal hematopoiesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35704087"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis of indeterminate potential in patients with acute coronary syndrome undergoing percutaneous coronary intervention in the absence of traditional risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35704087/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The most frequent CHIP-related mutations, DNMT3A, TET2, and ASXL1 are significantly associated with increased risk of recurrent cardiovascular events. Our study may be valuable target to reduce residual risk in patients with ACS carrying specific mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35703914"
                        open
                      >
                        <summary class="article-expander__title">Targeted Next-Generation Sequencing Reveals Divergent Clonal Evolution in Components of Composite Pleomorphic Xanthoastrocytoma-Ganglioglioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35703914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Composite pleomorphic xanthoastrocytoma-ganglioglioma (PXA-GG) is an extremely rare central nervous system neoplasm with 2 distinct but intermingled components. Whether this tumor represents a &quot;collision tumor&quot; of separate neoplasms or a monoclonal neoplasm with divergent evolution is poorly understood. Clinicopathologic studies and capture-based next generation sequencing were performed on extracted DNA from all available PXA-GG at 2 medical centers. Five PXA-GG were diagnosed in 1 male and 4...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35703914"
                        open
                      >
                        <summary class="article-expander__title">Targeted Next-Generation Sequencing Reveals Divergent Clonal Evolution in Components of Composite Pleomorphic Xanthoastrocytoma-Ganglioglioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35703914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Composite pleomorphic xanthoastrocytoma-ganglioglioma (PXA-GG) is an extremely rare central nervous system neoplasm with 2 distinct but intermingled components. Whether this tumor represents a &quot;collision tumor&quot; of separate neoplasms or a monoclonal neoplasm with divergent evolution is poorly understood. Clinicopathologic studies and capture-based next generation sequencing were performed on extracted DNA from all available PXA-GG at 2 medical centers. Five PXA-GG were diagnosed in 1 male and 4...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35703914"
                        open
                      >
                        <summary class="article-expander__title">Targeted Next-Generation Sequencing Reveals Divergent Clonal Evolution in Components of Composite Pleomorphic Xanthoastrocytoma-Ganglioglioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35703914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Composite pleomorphic xanthoastrocytoma-ganglioglioma (PXA-GG) is an extremely rare central nervous system neoplasm with 2 distinct but intermingled components. Whether this tumor represents a &quot;collision tumor&quot; of separate neoplasms or a monoclonal neoplasm with divergent evolution is poorly understood. Clinicopathologic studies and capture-based next generation sequencing were performed on extracted DNA from all available PXA-GG at 2 medical centers. Five PXA-GG were diagnosed in 1 male and 4...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35703914"
                        open
                      >
                        <summary class="article-expander__title">Targeted Next-Generation Sequencing Reveals Divergent Clonal Evolution in Components of Composite Pleomorphic Xanthoastrocytoma-Ganglioglioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35703914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Composite pleomorphic xanthoastrocytoma-ganglioglioma (PXA-GG) is an extremely rare central nervous system neoplasm with 2 distinct but intermingled components. Whether this tumor represents a &quot;collision tumor&quot; of separate neoplasms or a monoclonal neoplasm with divergent evolution is poorly understood. Clinicopathologic studies and capture-based next generation sequencing were performed on extracted DNA from all available PXA-GG at 2 medical centers. Five PXA-GG were diagnosed in 1 male and 4...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35703914"
                        open
                      >
                        <summary class="article-expander__title">Targeted Next-Generation Sequencing Reveals Divergent Clonal Evolution in Components of Composite Pleomorphic Xanthoastrocytoma-Ganglioglioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35703914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Composite pleomorphic xanthoastrocytoma-ganglioglioma (PXA-GG) is an extremely rare central nervous system neoplasm with 2 distinct but intermingled components. Whether this tumor represents a &quot;collision tumor&quot; of separate neoplasms or a monoclonal neoplasm with divergent evolution is poorly understood. Clinicopathologic studies and capture-based next generation sequencing were performed on extracted DNA from all available PXA-GG at 2 medical centers. Five PXA-GG were diagnosed in 1 male and 4...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35703914"
                        open
                      >
                        <summary class="article-expander__title">Targeted Next-Generation Sequencing Reveals Divergent Clonal Evolution in Components of Composite Pleomorphic Xanthoastrocytoma-Ganglioglioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35703914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Composite pleomorphic xanthoastrocytoma-ganglioglioma (PXA-GG) is an extremely rare central nervous system neoplasm with 2 distinct but intermingled components. Whether this tumor represents a &quot;collision tumor&quot; of separate neoplasms or a monoclonal neoplasm with divergent evolution is poorly understood. Clinicopathologic studies and capture-based next generation sequencing were performed on extracted DNA from all available PXA-GG at 2 medical centers. Five PXA-GG were diagnosed in 1 male and 4...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35703914"
                        open
                      >
                        <summary class="article-expander__title">Targeted Next-Generation Sequencing Reveals Divergent Clonal Evolution in Components of Composite Pleomorphic Xanthoastrocytoma-Ganglioglioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35703914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Composite pleomorphic xanthoastrocytoma-ganglioglioma (PXA-GG) is an extremely rare central nervous system neoplasm with 2 distinct but intermingled components. Whether this tumor represents a &quot;collision tumor&quot; of separate neoplasms or a monoclonal neoplasm with divergent evolution is poorly understood. Clinicopathologic studies and capture-based next generation sequencing were performed on extracted DNA from all available PXA-GG at 2 medical centers. Five PXA-GG were diagnosed in 1 male and 4...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35703914"
                        open
                      >
                        <summary class="article-expander__title">Targeted Next-Generation Sequencing Reveals Divergent Clonal Evolution in Components of Composite Pleomorphic Xanthoastrocytoma-Ganglioglioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35703914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Composite pleomorphic xanthoastrocytoma-ganglioglioma (PXA-GG) is an extremely rare central nervous system neoplasm with 2 distinct but intermingled components. Whether this tumor represents a &quot;collision tumor&quot; of separate neoplasms or a monoclonal neoplasm with divergent evolution is poorly understood. Clinicopathologic studies and capture-based next generation sequencing were performed on extracted DNA from all available PXA-GG at 2 medical centers. Five PXA-GG were diagnosed in 1 male and 4...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35703914"
                        open
                      >
                        <summary class="article-expander__title">Targeted Next-Generation Sequencing Reveals Divergent Clonal Evolution in Components of Composite Pleomorphic Xanthoastrocytoma-Ganglioglioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35703914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Composite pleomorphic xanthoastrocytoma-ganglioglioma (PXA-GG) is an extremely rare central nervous system neoplasm with 2 distinct but intermingled components. Whether this tumor represents a &quot;collision tumor&quot; of separate neoplasms or a monoclonal neoplasm with divergent evolution is poorly understood. Clinicopathologic studies and capture-based next generation sequencing were performed on extracted DNA from all available PXA-GG at 2 medical centers. Five PXA-GG were diagnosed in 1 male and 4...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35703914"
                        open
                      >
                        <summary class="article-expander__title">Targeted Next-Generation Sequencing Reveals Divergent Clonal Evolution in Components of Composite Pleomorphic Xanthoastrocytoma-Ganglioglioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35703914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Composite pleomorphic xanthoastrocytoma-ganglioglioma (PXA-GG) is an extremely rare central nervous system neoplasm with 2 distinct but intermingled components. Whether this tumor represents a &quot;collision tumor&quot; of separate neoplasms or a monoclonal neoplasm with divergent evolution is poorly understood. Clinicopathologic studies and capture-based next generation sequencing were performed on extracted DNA from all available PXA-GG at 2 medical centers. Five PXA-GG were diagnosed in 1 male and 4...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35703914"
                        open
                      >
                        <summary class="article-expander__title">Targeted Next-Generation Sequencing Reveals Divergent Clonal Evolution in Components of Composite Pleomorphic Xanthoastrocytoma-Ganglioglioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35703914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Composite pleomorphic xanthoastrocytoma-ganglioglioma (PXA-GG) is an extremely rare central nervous system neoplasm with 2 distinct but intermingled components. Whether this tumor represents a &quot;collision tumor&quot; of separate neoplasms or a monoclonal neoplasm with divergent evolution is poorly understood. Clinicopathologic studies and capture-based next generation sequencing were performed on extracted DNA from all available PXA-GG at 2 medical centers. Five PXA-GG were diagnosed in 1 male and 4...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35703914"
                        open
                      >
                        <summary class="article-expander__title">Targeted Next-Generation Sequencing Reveals Divergent Clonal Evolution in Components of Composite Pleomorphic Xanthoastrocytoma-Ganglioglioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35703914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Composite pleomorphic xanthoastrocytoma-ganglioglioma (PXA-GG) is an extremely rare central nervous system neoplasm with 2 distinct but intermingled components. Whether this tumor represents a &quot;collision tumor&quot; of separate neoplasms or a monoclonal neoplasm with divergent evolution is poorly understood. Clinicopathologic studies and capture-based next generation sequencing were performed on extracted DNA from all available PXA-GG at 2 medical centers. Five PXA-GG were diagnosed in 1 male and 4...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35703914"
                        open
                      >
                        <summary class="article-expander__title">Targeted Next-Generation Sequencing Reveals Divergent Clonal Evolution in Components of Composite Pleomorphic Xanthoastrocytoma-Ganglioglioma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35703914/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Composite pleomorphic xanthoastrocytoma-ganglioglioma (PXA-GG) is an extremely rare central nervous system neoplasm with 2 distinct but intermingled components. Whether this tumor represents a &quot;collision tumor&quot; of separate neoplasms or a monoclonal neoplasm with divergent evolution is poorly understood. Clinicopathologic studies and capture-based next generation sequencing were performed on extracted DNA from all available PXA-GG at 2 medical centers. Five PXA-GG were diagnosed in 1 male and 4...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-14 datetime="2022-06-14T10:00:00.000Z">2022-06-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-14 datetime="2022-06-14T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220704211122&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220703212221&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220702211519&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220701211233&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220630212102&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629210920&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220629090955&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220628211035&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220627211523&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35699987"
                        open
                      >
                        <summary class="article-expander__title">What constitutes meaningful improvement in myelodysplastic syndromes?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35699987/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndromes (MDS) are a heterogeneous spectrum of clonal stem cell disorders characterized by ineffective hematopoiesis and resulting peripheral blood cytopenias. Clinical manifestations of the disorder are varied and range from mild asymptomatic anemia to a markedly truncated life expectancy, underscoring the importance of risk-adapted therapy. MDS therapy for both lower- and higher-risk patients is an area of immense therapeutic opportunity. This is partly due to the diversity of...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700772"
                        open
                      >
                        <summary class="article-expander__title">Dysplastic stem cell plasticity functions as a driving force for neoplastic transformation of pre-cancerous gastric mucosa</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700772/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These findings indicate that the DSCs are de novo gastric cancer-initiating cells responsible for neoplastic transformation and a promising target for intervention in early induction of gastric cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700772"
                        open
                      >
                        <summary class="article-expander__title">Dysplastic stem cell plasticity functions as a driving force for neoplastic transformation of pre-cancerous gastric mucosa</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700772/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These findings indicate that the DSCs are de novo gastric cancer-initiating cells responsible for neoplastic transformation and a promising target for intervention in early induction of gastric cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700772"
                        open
                      >
                        <summary class="article-expander__title">Dysplastic stem cell plasticity functions as a driving force for neoplastic transformation of pre-cancerous gastric mucosa</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700772/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These findings indicate that the DSCs are de novo gastric cancer-initiating cells responsible for neoplastic transformation and a promising target for intervention in early induction of gastric cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700772"
                        open
                      >
                        <summary class="article-expander__title">Dysplastic stem cell plasticity functions as a driving force for neoplastic transformation of pre-cancerous gastric mucosa</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700772/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These findings indicate that the DSCs are de novo gastric cancer-initiating cells responsible for neoplastic transformation and a promising target for intervention in early induction of gastric cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700772"
                        open
                      >
                        <summary class="article-expander__title">Dysplastic stem cell plasticity functions as a driving force for neoplastic transformation of pre-cancerous gastric mucosa</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700772/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These findings indicate that the DSCs are de novo gastric cancer-initiating cells responsible for neoplastic transformation and a promising target for intervention in early induction of gastric cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700772"
                        open
                      >
                        <summary class="article-expander__title">Dysplastic stem cell plasticity functions as a driving force for neoplastic transformation of pre-cancerous gastric mucosa</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700772/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These findings indicate that the DSCs are de novo gastric cancer-initiating cells responsible for neoplastic transformation and a promising target for intervention in early induction of gastric cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700772"
                        open
                      >
                        <summary class="article-expander__title">Dysplastic stem cell plasticity functions as a driving force for neoplastic transformation of pre-cancerous gastric mucosa</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700772/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These findings indicate that the DSCs are de novo gastric cancer-initiating cells responsible for neoplastic transformation and a promising target for intervention in early induction of gastric cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700772"
                        open
                      >
                        <summary class="article-expander__title">Dysplastic stem cell plasticity functions as a driving force for neoplastic transformation of pre-cancerous gastric mucosa</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700772/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These findings indicate that the DSCs are de novo gastric cancer-initiating cells responsible for neoplastic transformation and a promising target for intervention in early induction of gastric cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700772"
                        open
                      >
                        <summary class="article-expander__title">Dysplastic stem cell plasticity functions as a driving force for neoplastic transformation of pre-cancerous gastric mucosa</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700772/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These findings indicate that the DSCs are de novo gastric cancer-initiating cells responsible for neoplastic transformation and a promising target for intervention in early induction of gastric cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700150"
                        open
                      >
                        <summary class="article-expander__title">TP53 Variant in the Blood of a Patient with Gastric Cancer Undergoing Tumor Profiling Tests Diagnosed as Clonal Hematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700150/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND Clonal hematopoiesis is the production of a specific single clonal type of cell in the blood and is often found in cancer genomic profiling tests. When the clone carries a pathogenic variant, it may be important to differentiate between somatic or germline origin. The variant in the blood that has a lower minor allele frequency could reflect heterozygous germline origin, somatic mosaicism, and clonal hematopoiesis. It is important to evaluate suspected variants to determine the course...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35700772"
                        open
                      >
                        <summary class="article-expander__title">Dysplastic stem cell plasticity functions as a driving force for neoplastic transformation of pre-cancerous gastric mucosa</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35700772/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: These findings indicate that the DSCs are de novo gastric cancer-initiating cells responsible for neoplastic transformation and a promising target for intervention in early induction of gastric cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-13 datetime="2022-06-13T10:00:00.000Z">2022-06-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-13 datetime="2022-06-13T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626212149&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220626131729&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35694869"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in clinical practice: walking a tightrope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35694869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220625211446&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The understanding of clonal hematopoiesis (CH) and its features is rapidly evolving in step with the spread of sequencing techniques. Indeed, CH detection is now an emerging aspect in clinical practice. The awareness of CH intersects with consolidated diagnostic paths, thus exposing &#x27;grey zone&#x27; circumstances under the magnifying lens of clinicians. The interpretation of genomic data poses, in some cases, a true clinical challenge, sometimes further complicating the route to diagnosis. The line...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35694869"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in clinical practice: walking a tightrope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35694869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220624210923&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The understanding of clonal hematopoiesis (CH) and its features is rapidly evolving in step with the spread of sequencing techniques. Indeed, CH detection is now an emerging aspect in clinical practice. The awareness of CH intersects with consolidated diagnostic paths, thus exposing &#x27;grey zone&#x27; circumstances under the magnifying lens of clinicians. The interpretation of genomic data poses, in some cases, a true clinical challenge, sometimes further complicating the route to diagnosis. The line...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35694869"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in clinical practice: walking a tightrope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35694869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The understanding of clonal hematopoiesis (CH) and its features is rapidly evolving in step with the spread of sequencing techniques. Indeed, CH detection is now an emerging aspect in clinical practice. The awareness of CH intersects with consolidated diagnostic paths, thus exposing &#x27;grey zone&#x27; circumstances under the magnifying lens of clinicians. The interpretation of genomic data poses, in some cases, a true clinical challenge, sometimes further complicating the route to diagnosis. The line...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35694869"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in clinical practice: walking a tightrope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35694869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The understanding of clonal hematopoiesis (CH) and its features is rapidly evolving in step with the spread of sequencing techniques. Indeed, CH detection is now an emerging aspect in clinical practice. The awareness of CH intersects with consolidated diagnostic paths, thus exposing &#x27;grey zone&#x27; circumstances under the magnifying lens of clinicians. The interpretation of genomic data poses, in some cases, a true clinical challenge, sometimes further complicating the route to diagnosis. The line...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35694869"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in clinical practice: walking a tightrope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35694869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The understanding of clonal hematopoiesis (CH) and its features is rapidly evolving in step with the spread of sequencing techniques. Indeed, CH detection is now an emerging aspect in clinical practice. The awareness of CH intersects with consolidated diagnostic paths, thus exposing &#x27;grey zone&#x27; circumstances under the magnifying lens of clinicians. The interpretation of genomic data poses, in some cases, a true clinical challenge, sometimes further complicating the route to diagnosis. The line...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35694869"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in clinical practice: walking a tightrope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35694869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The understanding of clonal hematopoiesis (CH) and its features is rapidly evolving in step with the spread of sequencing techniques. Indeed, CH detection is now an emerging aspect in clinical practice. The awareness of CH intersects with consolidated diagnostic paths, thus exposing &#x27;grey zone&#x27; circumstances under the magnifying lens of clinicians. The interpretation of genomic data poses, in some cases, a true clinical challenge, sometimes further complicating the route to diagnosis. The line...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35694869"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in clinical practice: walking a tightrope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35694869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The understanding of clonal hematopoiesis (CH) and its features is rapidly evolving in step with the spread of sequencing techniques. Indeed, CH detection is now an emerging aspect in clinical practice. The awareness of CH intersects with consolidated diagnostic paths, thus exposing &#x27;grey zone&#x27; circumstances under the magnifying lens of clinicians. The interpretation of genomic data poses, in some cases, a true clinical challenge, sometimes further complicating the route to diagnosis. The line...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35694869"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in clinical practice: walking a tightrope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35694869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The understanding of clonal hematopoiesis (CH) and its features is rapidly evolving in step with the spread of sequencing techniques. Indeed, CH detection is now an emerging aspect in clinical practice. The awareness of CH intersects with consolidated diagnostic paths, thus exposing &#x27;grey zone&#x27; circumstances under the magnifying lens of clinicians. The interpretation of genomic data poses, in some cases, a true clinical challenge, sometimes further complicating the route to diagnosis. The line...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35694869"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in clinical practice: walking a tightrope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35694869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The understanding of clonal hematopoiesis (CH) and its features is rapidly evolving in step with the spread of sequencing techniques. Indeed, CH detection is now an emerging aspect in clinical practice. The awareness of CH intersects with consolidated diagnostic paths, thus exposing &#x27;grey zone&#x27; circumstances under the magnifying lens of clinicians. The interpretation of genomic data poses, in some cases, a true clinical challenge, sometimes further complicating the route to diagnosis. The line...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35694869"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in clinical practice: walking a tightrope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35694869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The understanding of clonal hematopoiesis (CH) and its features is rapidly evolving in step with the spread of sequencing techniques. Indeed, CH detection is now an emerging aspect in clinical practice. The awareness of CH intersects with consolidated diagnostic paths, thus exposing &#x27;grey zone&#x27; circumstances under the magnifying lens of clinicians. The interpretation of genomic data poses, in some cases, a true clinical challenge, sometimes further complicating the route to diagnosis. The line...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35694869"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in clinical practice: walking a tightrope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35694869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The understanding of clonal hematopoiesis (CH) and its features is rapidly evolving in step with the spread of sequencing techniques. Indeed, CH detection is now an emerging aspect in clinical practice. The awareness of CH intersects with consolidated diagnostic paths, thus exposing &#x27;grey zone&#x27; circumstances under the magnifying lens of clinicians. The interpretation of genomic data poses, in some cases, a true clinical challenge, sometimes further complicating the route to diagnosis. The line...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35694869"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in clinical practice: walking a tightrope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35694869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The understanding of clonal hematopoiesis (CH) and its features is rapidly evolving in step with the spread of sequencing techniques. Indeed, CH detection is now an emerging aspect in clinical practice. The awareness of CH intersects with consolidated diagnostic paths, thus exposing &#x27;grey zone&#x27; circumstances under the magnifying lens of clinicians. The interpretation of genomic data poses, in some cases, a true clinical challenge, sometimes further complicating the route to diagnosis. The line...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35695283"
                        open
                      >
                        <summary class="article-expander__title">Hematological disorders after salvage PARPi treatment for ovarian cancer: cytogenetic and molecular defects and clinical outcomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35695283/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inhibitors of poly (ADP-ribose) polymerase (PARPi) are increasingly employed as salvage therapy in epithelial ovarian cancer (EOC), but cytotoxic drug exposure along with PARP inhibition may favor development of hematological disorders. In this study, of 182 women with EOC treated with PARPi, 16 (8.7%) developed therapy-related myeloid neoplasms (t-MNs), with 12 cases of myelodysplasia and 4 of acute myeloid leukemia. All experienced persistent cytopenia after PARPi discontinuation. Seven...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35694869"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis in clinical practice: walking a tightrope</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35694869/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The understanding of clonal hematopoiesis (CH) and its features is rapidly evolving in step with the spread of sequencing techniques. Indeed, CH detection is now an emerging aspect in clinical practice. The awareness of CH intersects with consolidated diagnostic paths, thus exposing &#x27;grey zone&#x27; circumstances under the magnifying lens of clinicians. The interpretation of genomic data poses, in some cases, a true clinical challenge, sometimes further complicating the route to diagnosis. The line...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35697558"
                        open
                      >
                        <summary class="article-expander__title">Clinical-Molecular Prospective Cohort Study in Non-Small Cell Lung Cancer (PROMOLE study): A Comprehensive Approach to Identify New Predictive Markers of Pharmacological Response</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35697558/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lung cancers account for over 90% of thoracic malignancies and the rapid development of specific cytotoxic drugs and molecular therapies requires a detailed identification of the different histologies, gene drivers or immune microenvironment biomarkers. Nevertheless, the heterogeneous clonal evolution, the emergency of drug-induced resistance and the limited occurrence of genetic alterations claim the need of a deep integration of the tumor&#x27;s and the patient&#x27;s biological features. The aim of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35697558"
                        open
                      >
                        <summary class="article-expander__title">Clinical-Molecular Prospective Cohort Study in Non-Small Cell Lung Cancer (PROMOLE study): A Comprehensive Approach to Identify New Predictive Markers of Pharmacological Response</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35697558/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lung cancers account for over 90% of thoracic malignancies and the rapid development of specific cytotoxic drugs and molecular therapies requires a detailed identification of the different histologies, gene drivers or immune microenvironment biomarkers. Nevertheless, the heterogeneous clonal evolution, the emergency of drug-induced resistance and the limited occurrence of genetic alterations claim the need of a deep integration of the tumor&#x27;s and the patient&#x27;s biological features. The aim of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35697558"
                        open
                      >
                        <summary class="article-expander__title">Clinical-Molecular Prospective Cohort Study in Non-Small Cell Lung Cancer (PROMOLE study): A Comprehensive Approach to Identify New Predictive Markers of Pharmacological Response</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35697558/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lung cancers account for over 90% of thoracic malignancies and the rapid development of specific cytotoxic drugs and molecular therapies requires a detailed identification of the different histologies, gene drivers or immune microenvironment biomarkers. Nevertheless, the heterogeneous clonal evolution, the emergency of drug-induced resistance and the limited occurrence of genetic alterations claim the need of a deep integration of the tumor&#x27;s and the patient&#x27;s biological features. The aim of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35697558"
                        open
                      >
                        <summary class="article-expander__title">Clinical-Molecular Prospective Cohort Study in Non-Small Cell Lung Cancer (PROMOLE study): A Comprehensive Approach to Identify New Predictive Markers of Pharmacological Response</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35697558/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lung cancers account for over 90% of thoracic malignancies and the rapid development of specific cytotoxic drugs and molecular therapies requires a detailed identification of the different histologies, gene drivers or immune microenvironment biomarkers. Nevertheless, the heterogeneous clonal evolution, the emergency of drug-induced resistance and the limited occurrence of genetic alterations claim the need of a deep integration of the tumor&#x27;s and the patient&#x27;s biological features. The aim of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35697558"
                        open
                      >
                        <summary class="article-expander__title">Clinical-Molecular Prospective Cohort Study in Non-Small Cell Lung Cancer (PROMOLE study): A Comprehensive Approach to Identify New Predictive Markers of Pharmacological Response</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35697558/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lung cancers account for over 90% of thoracic malignancies and the rapid development of specific cytotoxic drugs and molecular therapies requires a detailed identification of the different histologies, gene drivers or immune microenvironment biomarkers. Nevertheless, the heterogeneous clonal evolution, the emergency of drug-induced resistance and the limited occurrence of genetic alterations claim the need of a deep integration of the tumor&#x27;s and the patient&#x27;s biological features. The aim of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35697558"
                        open
                      >
                        <summary class="article-expander__title">Clinical-Molecular Prospective Cohort Study in Non-Small Cell Lung Cancer (PROMOLE study): A Comprehensive Approach to Identify New Predictive Markers of Pharmacological Response</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35697558/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lung cancers account for over 90% of thoracic malignancies and the rapid development of specific cytotoxic drugs and molecular therapies requires a detailed identification of the different histologies, gene drivers or immune microenvironment biomarkers. Nevertheless, the heterogeneous clonal evolution, the emergency of drug-induced resistance and the limited occurrence of genetic alterations claim the need of a deep integration of the tumor&#x27;s and the patient&#x27;s biological features. The aim of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35697558"
                        open
                      >
                        <summary class="article-expander__title">Clinical-Molecular Prospective Cohort Study in Non-Small Cell Lung Cancer (PROMOLE study): A Comprehensive Approach to Identify New Predictive Markers of Pharmacological Response</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35697558/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lung cancers account for over 90% of thoracic malignancies and the rapid development of specific cytotoxic drugs and molecular therapies requires a detailed identification of the different histologies, gene drivers or immune microenvironment biomarkers. Nevertheless, the heterogeneous clonal evolution, the emergency of drug-induced resistance and the limited occurrence of genetic alterations claim the need of a deep integration of the tumor&#x27;s and the patient&#x27;s biological features. The aim of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35697558"
                        open
                      >
                        <summary class="article-expander__title">Clinical-Molecular Prospective Cohort Study in Non-Small Cell Lung Cancer (PROMOLE study): A Comprehensive Approach to Identify New Predictive Markers of Pharmacological Response</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35697558/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lung cancers account for over 90% of thoracic malignancies and the rapid development of specific cytotoxic drugs and molecular therapies requires a detailed identification of the different histologies, gene drivers or immune microenvironment biomarkers. Nevertheless, the heterogeneous clonal evolution, the emergency of drug-induced resistance and the limited occurrence of genetic alterations claim the need of a deep integration of the tumor&#x27;s and the patient&#x27;s biological features. The aim of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35697558"
                        open
                      >
                        <summary class="article-expander__title">Clinical-Molecular Prospective Cohort Study in Non-Small Cell Lung Cancer (PROMOLE study): A Comprehensive Approach to Identify New Predictive Markers of Pharmacological Response</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35697558/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lung cancers account for over 90% of thoracic malignancies and the rapid development of specific cytotoxic drugs and molecular therapies requires a detailed identification of the different histologies, gene drivers or immune microenvironment biomarkers. Nevertheless, the heterogeneous clonal evolution, the emergency of drug-induced resistance and the limited occurrence of genetic alterations claim the need of a deep integration of the tumor&#x27;s and the patient&#x27;s biological features. The aim of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35697558"
                        open
                      >
                        <summary class="article-expander__title">Clinical-Molecular Prospective Cohort Study in Non-Small Cell Lung Cancer (PROMOLE study): A Comprehensive Approach to Identify New Predictive Markers of Pharmacological Response</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35697558/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Lung cancers account for over 90% of thoracic malignancies and the rapid development of specific cytotoxic drugs and molecular therapies requires a detailed identification of the different histologies, gene drivers or immune microenvironment biomarkers. Nevertheless, the heterogeneous clonal evolution, the emergency of drug-induced resistance and the limited occurrence of genetic alterations claim the need of a deep integration of the tumor&#x27;s and the patient&#x27;s biological features. The aim of the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-10 datetime="2022-06-10T10:00:00.000Z">2022-06-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-10 datetime="2022-06-10T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35683547"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and the Risk of Hematologic Malignancies after Curative Therapies for Sickle Cell Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35683547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sickle cell disease (SCD) is associated with severe morbidity and early mortality. Two large population studies found an increased risk for leukemia in individuals with SCD. Notably, while the relative risk of leukemia development is high, the absolute risk is low in individuals with SCD who do not receive cell-based therapies. However, the risk of leukemia in SCD is high after graft rejection and with gene therapy. Clonal hematopoiesis (CH) is a well-recognized premalignant condition in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35683547"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and the Risk of Hematologic Malignancies after Curative Therapies for Sickle Cell Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35683547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sickle cell disease (SCD) is associated with severe morbidity and early mortality. Two large population studies found an increased risk for leukemia in individuals with SCD. Notably, while the relative risk of leukemia development is high, the absolute risk is low in individuals with SCD who do not receive cell-based therapies. However, the risk of leukemia in SCD is high after graft rejection and with gene therapy. Clonal hematopoiesis (CH) is a well-recognized premalignant condition in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35683547"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and the Risk of Hematologic Malignancies after Curative Therapies for Sickle Cell Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35683547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sickle cell disease (SCD) is associated with severe morbidity and early mortality. Two large population studies found an increased risk for leukemia in individuals with SCD. Notably, while the relative risk of leukemia development is high, the absolute risk is low in individuals with SCD who do not receive cell-based therapies. However, the risk of leukemia in SCD is high after graft rejection and with gene therapy. Clonal hematopoiesis (CH) is a well-recognized premalignant condition in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35683547"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and the Risk of Hematologic Malignancies after Curative Therapies for Sickle Cell Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35683547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sickle cell disease (SCD) is associated with severe morbidity and early mortality. Two large population studies found an increased risk for leukemia in individuals with SCD. Notably, while the relative risk of leukemia development is high, the absolute risk is low in individuals with SCD who do not receive cell-based therapies. However, the risk of leukemia in SCD is high after graft rejection and with gene therapy. Clonal hematopoiesis (CH) is a well-recognized premalignant condition in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35683547"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and the Risk of Hematologic Malignancies after Curative Therapies for Sickle Cell Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35683547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sickle cell disease (SCD) is associated with severe morbidity and early mortality. Two large population studies found an increased risk for leukemia in individuals with SCD. Notably, while the relative risk of leukemia development is high, the absolute risk is low in individuals with SCD who do not receive cell-based therapies. However, the risk of leukemia in SCD is high after graft rejection and with gene therapy. Clonal hematopoiesis (CH) is a well-recognized premalignant condition in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35683547"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and the Risk of Hematologic Malignancies after Curative Therapies for Sickle Cell Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35683547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sickle cell disease (SCD) is associated with severe morbidity and early mortality. Two large population studies found an increased risk for leukemia in individuals with SCD. Notably, while the relative risk of leukemia development is high, the absolute risk is low in individuals with SCD who do not receive cell-based therapies. However, the risk of leukemia in SCD is high after graft rejection and with gene therapy. Clonal hematopoiesis (CH) is a well-recognized premalignant condition in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35683547"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and the Risk of Hematologic Malignancies after Curative Therapies for Sickle Cell Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35683547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sickle cell disease (SCD) is associated with severe morbidity and early mortality. Two large population studies found an increased risk for leukemia in individuals with SCD. Notably, while the relative risk of leukemia development is high, the absolute risk is low in individuals with SCD who do not receive cell-based therapies. However, the risk of leukemia in SCD is high after graft rejection and with gene therapy. Clonal hematopoiesis (CH) is a well-recognized premalignant condition in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35683547"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and the Risk of Hematologic Malignancies after Curative Therapies for Sickle Cell Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35683547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sickle cell disease (SCD) is associated with severe morbidity and early mortality. Two large population studies found an increased risk for leukemia in individuals with SCD. Notably, while the relative risk of leukemia development is high, the absolute risk is low in individuals with SCD who do not receive cell-based therapies. However, the risk of leukemia in SCD is high after graft rejection and with gene therapy. Clonal hematopoiesis (CH) is a well-recognized premalignant condition in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35683547"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and the Risk of Hematologic Malignancies after Curative Therapies for Sickle Cell Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35683547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sickle cell disease (SCD) is associated with severe morbidity and early mortality. Two large population studies found an increased risk for leukemia in individuals with SCD. Notably, while the relative risk of leukemia development is high, the absolute risk is low in individuals with SCD who do not receive cell-based therapies. However, the risk of leukemia in SCD is high after graft rejection and with gene therapy. Clonal hematopoiesis (CH) is a well-recognized premalignant condition in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35683547"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and the Risk of Hematologic Malignancies after Curative Therapies for Sickle Cell Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35683547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sickle cell disease (SCD) is associated with severe morbidity and early mortality. Two large population studies found an increased risk for leukemia in individuals with SCD. Notably, while the relative risk of leukemia development is high, the absolute risk is low in individuals with SCD who do not receive cell-based therapies. However, the risk of leukemia in SCD is high after graft rejection and with gene therapy. Clonal hematopoiesis (CH) is a well-recognized premalignant condition in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35683547"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and the Risk of Hematologic Malignancies after Curative Therapies for Sickle Cell Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35683547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sickle cell disease (SCD) is associated with severe morbidity and early mortality. Two large population studies found an increased risk for leukemia in individuals with SCD. Notably, while the relative risk of leukemia development is high, the absolute risk is low in individuals with SCD who do not receive cell-based therapies. However, the risk of leukemia in SCD is high after graft rejection and with gene therapy. Clonal hematopoiesis (CH) is a well-recognized premalignant condition in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35683547"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and the Risk of Hematologic Malignancies after Curative Therapies for Sickle Cell Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35683547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sickle cell disease (SCD) is associated with severe morbidity and early mortality. Two large population studies found an increased risk for leukemia in individuals with SCD. Notably, while the relative risk of leukemia development is high, the absolute risk is low in individuals with SCD who do not receive cell-based therapies. However, the risk of leukemia in SCD is high after graft rejection and with gene therapy. Clonal hematopoiesis (CH) is a well-recognized premalignant condition in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35683547"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and the Risk of Hematologic Malignancies after Curative Therapies for Sickle Cell Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35683547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sickle cell disease (SCD) is associated with severe morbidity and early mortality. Two large population studies found an increased risk for leukemia in individuals with SCD. Notably, while the relative risk of leukemia development is high, the absolute risk is low in individuals with SCD who do not receive cell-based therapies. However, the risk of leukemia in SCD is high after graft rejection and with gene therapy. Clonal hematopoiesis (CH) is a well-recognized premalignant condition in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35683547"
                        open
                      >
                        <summary class="article-expander__title">Clonal Hematopoiesis and the Risk of Hematologic Malignancies after Curative Therapies for Sickle Cell Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35683547/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sickle cell disease (SCD) is associated with severe morbidity and early mortality. Two large population studies found an increased risk for leukemia in individuals with SCD. Notably, while the relative risk of leukemia development is high, the absolute risk is low in individuals with SCD who do not receive cell-based therapies. However, the risk of leukemia in SCD is high after graft rejection and with gene therapy. Clonal hematopoiesis (CH) is a well-recognized premalignant condition in the...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35682989"
                        open
                      >
                        <summary class="article-expander__title">Complex Elucidation of Cells-of-Origin in Pediatric Soft Tissue Sarcoma: From Concepts to Real Life, Hide-and-Seek through Epigenetic and Transcriptional Reprogramming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35682989/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Soft tissue sarcoma (STS) comprise a large group of mesenchymal malignant tumors with heterogeneous cellular morphology, proliferative index, genetic lesions and, more importantly, clinical features. Full elucidation of this wide diversity remains a central question to improve their therapeutic management and the identity of cell(s)-of-origin from which these tumors arise is part of this enigma. Cellular reprogramming allows transitions of a mature cell between phenotypes, or identities, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35682989"
                        open
                      >
                        <summary class="article-expander__title">Complex Elucidation of Cells-of-Origin in Pediatric Soft Tissue Sarcoma: From Concepts to Real Life, Hide-and-Seek through Epigenetic and Transcriptional Reprogramming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35682989/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Soft tissue sarcoma (STS) comprise a large group of mesenchymal malignant tumors with heterogeneous cellular morphology, proliferative index, genetic lesions and, more importantly, clinical features. Full elucidation of this wide diversity remains a central question to improve their therapeutic management and the identity of cell(s)-of-origin from which these tumors arise is part of this enigma. Cellular reprogramming allows transitions of a mature cell between phenotypes, or identities, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35682989"
                        open
                      >
                        <summary class="article-expander__title">Complex Elucidation of Cells-of-Origin in Pediatric Soft Tissue Sarcoma: From Concepts to Real Life, Hide-and-Seek through Epigenetic and Transcriptional Reprogramming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35682989/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Soft tissue sarcoma (STS) comprise a large group of mesenchymal malignant tumors with heterogeneous cellular morphology, proliferative index, genetic lesions and, more importantly, clinical features. Full elucidation of this wide diversity remains a central question to improve their therapeutic management and the identity of cell(s)-of-origin from which these tumors arise is part of this enigma. Cellular reprogramming allows transitions of a mature cell between phenotypes, or identities, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35682989"
                        open
                      >
                        <summary class="article-expander__title">Complex Elucidation of Cells-of-Origin in Pediatric Soft Tissue Sarcoma: From Concepts to Real Life, Hide-and-Seek through Epigenetic and Transcriptional Reprogramming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35682989/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Soft tissue sarcoma (STS) comprise a large group of mesenchymal malignant tumors with heterogeneous cellular morphology, proliferative index, genetic lesions and, more importantly, clinical features. Full elucidation of this wide diversity remains a central question to improve their therapeutic management and the identity of cell(s)-of-origin from which these tumors arise is part of this enigma. Cellular reprogramming allows transitions of a mature cell between phenotypes, or identities, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35682989"
                        open
                      >
                        <summary class="article-expander__title">Complex Elucidation of Cells-of-Origin in Pediatric Soft Tissue Sarcoma: From Concepts to Real Life, Hide-and-Seek through Epigenetic and Transcriptional Reprogramming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35682989/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Soft tissue sarcoma (STS) comprise a large group of mesenchymal malignant tumors with heterogeneous cellular morphology, proliferative index, genetic lesions and, more importantly, clinical features. Full elucidation of this wide diversity remains a central question to improve their therapeutic management and the identity of cell(s)-of-origin from which these tumors arise is part of this enigma. Cellular reprogramming allows transitions of a mature cell between phenotypes, or identities, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35682989"
                        open
                      >
                        <summary class="article-expander__title">Complex Elucidation of Cells-of-Origin in Pediatric Soft Tissue Sarcoma: From Concepts to Real Life, Hide-and-Seek through Epigenetic and Transcriptional Reprogramming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35682989/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Soft tissue sarcoma (STS) comprise a large group of mesenchymal malignant tumors with heterogeneous cellular morphology, proliferative index, genetic lesions and, more importantly, clinical features. Full elucidation of this wide diversity remains a central question to improve their therapeutic management and the identity of cell(s)-of-origin from which these tumors arise is part of this enigma. Cellular reprogramming allows transitions of a mature cell between phenotypes, or identities, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35682989"
                        open
                      >
                        <summary class="article-expander__title">Complex Elucidation of Cells-of-Origin in Pediatric Soft Tissue Sarcoma: From Concepts to Real Life, Hide-and-Seek through Epigenetic and Transcriptional Reprogramming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35682989/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Soft tissue sarcoma (STS) comprise a large group of mesenchymal malignant tumors with heterogeneous cellular morphology, proliferative index, genetic lesions and, more importantly, clinical features. Full elucidation of this wide diversity remains a central question to improve their therapeutic management and the identity of cell(s)-of-origin from which these tumors arise is part of this enigma. Cellular reprogramming allows transitions of a mature cell between phenotypes, or identities, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35682989"
                        open
                      >
                        <summary class="article-expander__title">Complex Elucidation of Cells-of-Origin in Pediatric Soft Tissue Sarcoma: From Concepts to Real Life, Hide-and-Seek through Epigenetic and Transcriptional Reprogramming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35682989/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Soft tissue sarcoma (STS) comprise a large group of mesenchymal malignant tumors with heterogeneous cellular morphology, proliferative index, genetic lesions and, more importantly, clinical features. Full elucidation of this wide diversity remains a central question to improve their therapeutic management and the identity of cell(s)-of-origin from which these tumors arise is part of this enigma. Cellular reprogramming allows transitions of a mature cell between phenotypes, or identities, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35682989"
                        open
                      >
                        <summary class="article-expander__title">Complex Elucidation of Cells-of-Origin in Pediatric Soft Tissue Sarcoma: From Concepts to Real Life, Hide-and-Seek through Epigenetic and Transcriptional Reprogramming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35682989/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Soft tissue sarcoma (STS) comprise a large group of mesenchymal malignant tumors with heterogeneous cellular morphology, proliferative index, genetic lesions and, more importantly, clinical features. Full elucidation of this wide diversity remains a central question to improve their therapeutic management and the identity of cell(s)-of-origin from which these tumors arise is part of this enigma. Cellular reprogramming allows transitions of a mature cell between phenotypes, or identities, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35682989"
                        open
                      >
                        <summary class="article-expander__title">Complex Elucidation of Cells-of-Origin in Pediatric Soft Tissue Sarcoma: From Concepts to Real Life, Hide-and-Seek through Epigenetic and Transcriptional Reprogramming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35682989/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Soft tissue sarcoma (STS) comprise a large group of mesenchymal malignant tumors with heterogeneous cellular morphology, proliferative index, genetic lesions and, more importantly, clinical features. Full elucidation of this wide diversity remains a central question to improve their therapeutic management and the identity of cell(s)-of-origin from which these tumors arise is part of this enigma. Cellular reprogramming allows transitions of a mature cell between phenotypes, or identities, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35682989"
                        open
                      >
                        <summary class="article-expander__title">Complex Elucidation of Cells-of-Origin in Pediatric Soft Tissue Sarcoma: From Concepts to Real Life, Hide-and-Seek through Epigenetic and Transcriptional Reprogramming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35682989/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Soft tissue sarcoma (STS) comprise a large group of mesenchymal malignant tumors with heterogeneous cellular morphology, proliferative index, genetic lesions and, more importantly, clinical features. Full elucidation of this wide diversity remains a central question to improve their therapeutic management and the identity of cell(s)-of-origin from which these tumors arise is part of this enigma. Cellular reprogramming allows transitions of a mature cell between phenotypes, or identities, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35682989"
                        open
                      >
                        <summary class="article-expander__title">Complex Elucidation of Cells-of-Origin in Pediatric Soft Tissue Sarcoma: From Concepts to Real Life, Hide-and-Seek through Epigenetic and Transcriptional Reprogramming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35682989/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Soft tissue sarcoma (STS) comprise a large group of mesenchymal malignant tumors with heterogeneous cellular morphology, proliferative index, genetic lesions and, more importantly, clinical features. Full elucidation of this wide diversity remains a central question to improve their therapeutic management and the identity of cell(s)-of-origin from which these tumors arise is part of this enigma. Cellular reprogramming allows transitions of a mature cell between phenotypes, or identities, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35682989"
                        open
                      >
                        <summary class="article-expander__title">Complex Elucidation of Cells-of-Origin in Pediatric Soft Tissue Sarcoma: From Concepts to Real Life, Hide-and-Seek through Epigenetic and Transcriptional Reprogramming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35682989/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Soft tissue sarcoma (STS) comprise a large group of mesenchymal malignant tumors with heterogeneous cellular morphology, proliferative index, genetic lesions and, more importantly, clinical features. Full elucidation of this wide diversity remains a central question to improve their therapeutic management and the identity of cell(s)-of-origin from which these tumors arise is part of this enigma. Cellular reprogramming allows transitions of a mature cell between phenotypes, or identities, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35682989"
                        open
                      >
                        <summary class="article-expander__title">Complex Elucidation of Cells-of-Origin in Pediatric Soft Tissue Sarcoma: From Concepts to Real Life, Hide-and-Seek through Epigenetic and Transcriptional Reprogramming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35682989/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Soft tissue sarcoma (STS) comprise a large group of mesenchymal malignant tumors with heterogeneous cellular morphology, proliferative index, genetic lesions and, more importantly, clinical features. Full elucidation of this wide diversity remains a central question to improve their therapeutic management and the identity of cell(s)-of-origin from which these tumors arise is part of this enigma. Cellular reprogramming allows transitions of a mature cell between phenotypes, or identities, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-08 datetime="2022-06-08T10:00:00.000Z">2022-06-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-08 datetime="2022-06-08T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35674998"
                        open
                      >
                        <summary class="article-expander__title">Germline CHEK2 and ATM Variants in Myeloid and Other Hematopoietic Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35674998/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: An intact DNA damage response is crucial to preventing cancer development, including in myeloid and lymphoid malignancies. Deficiencies in the homologous recombination (HR) pathway can lead to defective DNA damage responses, and this can occur through inherited germline mutations in HR pathway genes, such as CHEK2 and ATM. We now understand that germline mutations can be identified frequently (~ 5-10%) in patients with myeloid and lymphoid malignancies, and among the most...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35674998"
                        open
                      >
                        <summary class="article-expander__title">Germline CHEK2 and ATM Variants in Myeloid and Other Hematopoietic Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35674998/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: An intact DNA damage response is crucial to preventing cancer development, including in myeloid and lymphoid malignancies. Deficiencies in the homologous recombination (HR) pathway can lead to defective DNA damage responses, and this can occur through inherited germline mutations in HR pathway genes, such as CHEK2 and ATM. We now understand that germline mutations can be identified frequently (~ 5-10%) in patients with myeloid and lymphoid malignancies, and among the most...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35674998"
                        open
                      >
                        <summary class="article-expander__title">Germline CHEK2 and ATM Variants in Myeloid and Other Hematopoietic Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35674998/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: An intact DNA damage response is crucial to preventing cancer development, including in myeloid and lymphoid malignancies. Deficiencies in the homologous recombination (HR) pathway can lead to defective DNA damage responses, and this can occur through inherited germline mutations in HR pathway genes, such as CHEK2 and ATM. We now understand that germline mutations can be identified frequently (~ 5-10%) in patients with myeloid and lymphoid malignancies, and among the most...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35674998"
                        open
                      >
                        <summary class="article-expander__title">Germline CHEK2 and ATM Variants in Myeloid and Other Hematopoietic Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35674998/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: An intact DNA damage response is crucial to preventing cancer development, including in myeloid and lymphoid malignancies. Deficiencies in the homologous recombination (HR) pathway can lead to defective DNA damage responses, and this can occur through inherited germline mutations in HR pathway genes, such as CHEK2 and ATM. We now understand that germline mutations can be identified frequently (~ 5-10%) in patients with myeloid and lymphoid malignancies, and among the most...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35674998"
                        open
                      >
                        <summary class="article-expander__title">Germline CHEK2 and ATM Variants in Myeloid and Other Hematopoietic Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35674998/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: An intact DNA damage response is crucial to preventing cancer development, including in myeloid and lymphoid malignancies. Deficiencies in the homologous recombination (HR) pathway can lead to defective DNA damage responses, and this can occur through inherited germline mutations in HR pathway genes, such as CHEK2 and ATM. We now understand that germline mutations can be identified frequently (~ 5-10%) in patients with myeloid and lymphoid malignancies, and among the most...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35674998"
                        open
                      >
                        <summary class="article-expander__title">Germline CHEK2 and ATM Variants in Myeloid and Other Hematopoietic Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35674998/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: An intact DNA damage response is crucial to preventing cancer development, including in myeloid and lymphoid malignancies. Deficiencies in the homologous recombination (HR) pathway can lead to defective DNA damage responses, and this can occur through inherited germline mutations in HR pathway genes, such as CHEK2 and ATM. We now understand that germline mutations can be identified frequently (~ 5-10%) in patients with myeloid and lymphoid malignancies, and among the most...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35674998"
                        open
                      >
                        <summary class="article-expander__title">Germline CHEK2 and ATM Variants in Myeloid and Other Hematopoietic Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35674998/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: An intact DNA damage response is crucial to preventing cancer development, including in myeloid and lymphoid malignancies. Deficiencies in the homologous recombination (HR) pathway can lead to defective DNA damage responses, and this can occur through inherited germline mutations in HR pathway genes, such as CHEK2 and ATM. We now understand that germline mutations can be identified frequently (~ 5-10%) in patients with myeloid and lymphoid malignancies, and among the most...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35674998"
                        open
                      >
                        <summary class="article-expander__title">Germline CHEK2 and ATM Variants in Myeloid and Other Hematopoietic Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35674998/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: An intact DNA damage response is crucial to preventing cancer development, including in myeloid and lymphoid malignancies. Deficiencies in the homologous recombination (HR) pathway can lead to defective DNA damage responses, and this can occur through inherited germline mutations in HR pathway genes, such as CHEK2 and ATM. We now understand that germline mutations can be identified frequently (~ 5-10%) in patients with myeloid and lymphoid malignancies, and among the most...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35674998"
                        open
                      >
                        <summary class="article-expander__title">Germline CHEK2 and ATM Variants in Myeloid and Other Hematopoietic Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35674998/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: An intact DNA damage response is crucial to preventing cancer development, including in myeloid and lymphoid malignancies. Deficiencies in the homologous recombination (HR) pathway can lead to defective DNA damage responses, and this can occur through inherited germline mutations in HR pathway genes, such as CHEK2 and ATM. We now understand that germline mutations can be identified frequently (~ 5-10%) in patients with myeloid and lymphoid malignancies, and among the most...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35674998"
                        open
                      >
                        <summary class="article-expander__title">Germline CHEK2 and ATM Variants in Myeloid and Other Hematopoietic Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35674998/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: An intact DNA damage response is crucial to preventing cancer development, including in myeloid and lymphoid malignancies. Deficiencies in the homologous recombination (HR) pathway can lead to defective DNA damage responses, and this can occur through inherited germline mutations in HR pathway genes, such as CHEK2 and ATM. We now understand that germline mutations can be identified frequently (~ 5-10%) in patients with myeloid and lymphoid malignancies, and among the most...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35674998"
                        open
                      >
                        <summary class="article-expander__title">Germline CHEK2 and ATM Variants in Myeloid and Other Hematopoietic Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35674998/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: An intact DNA damage response is crucial to preventing cancer development, including in myeloid and lymphoid malignancies. Deficiencies in the homologous recombination (HR) pathway can lead to defective DNA damage responses, and this can occur through inherited germline mutations in HR pathway genes, such as CHEK2 and ATM. We now understand that germline mutations can be identified frequently (~ 5-10%) in patients with myeloid and lymphoid malignancies, and among the most...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35674998"
                        open
                      >
                        <summary class="article-expander__title">Germline CHEK2 and ATM Variants in Myeloid and Other Hematopoietic Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35674998/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: An intact DNA damage response is crucial to preventing cancer development, including in myeloid and lymphoid malignancies. Deficiencies in the homologous recombination (HR) pathway can lead to defective DNA damage responses, and this can occur through inherited germline mutations in HR pathway genes, such as CHEK2 and ATM. We now understand that germline mutations can be identified frequently (~ 5-10%) in patients with myeloid and lymphoid malignancies, and among the most...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35674998"
                        open
                      >
                        <summary class="article-expander__title">Germline CHEK2 and ATM Variants in Myeloid and Other Hematopoietic Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35674998/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: An intact DNA damage response is crucial to preventing cancer development, including in myeloid and lymphoid malignancies. Deficiencies in the homologous recombination (HR) pathway can lead to defective DNA damage responses, and this can occur through inherited germline mutations in HR pathway genes, such as CHEK2 and ATM. We now understand that germline mutations can be identified frequently (~ 5-10%) in patients with myeloid and lymphoid malignancies, and among the most...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35675647"
                        open
                      >
                        <summary class="article-expander__title">Emergency Hematopoiesis in the Pathobiology of COVID-19: The Dark Side of an Early Innate Protective Mechanism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35675647/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The recognition of pathogens to which we are constantly exposed induces the immediate replenishment of innate immune cells from the most primitive stages of their development through emergency hematopoiesis, a central mechanism contributing to early infection control. However, as with other protective mechanisms, its functional success is at risk when the excess of inducing signals accelerates immunological catastrophes. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35674998"
                        open
                      >
                        <summary class="article-expander__title">Germline CHEK2 and ATM Variants in Myeloid and Other Hematopoietic Malignancies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35674998/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: An intact DNA damage response is crucial to preventing cancer development, including in myeloid and lymphoid malignancies. Deficiencies in the homologous recombination (HR) pathway can lead to defective DNA damage responses, and this can occur through inherited germline mutations in HR pathway genes, such as CHEK2 and ATM. We now understand that germline mutations can be identified frequently (~ 5-10%) in patients with myeloid and lymphoid malignancies, and among the most...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z">2022-06-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-06 datetime="2022-06-06T18:28:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="72a3a11d39c217b1a80cbb551444e6f6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nodal interferon exposure drives tumour-specific tolerance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1533878403463405569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nodal interferon exposure drives tumour-specific tolerance https://t.co/FT4NeBcFTg pic.twitter.com/NRHkVDUsOr
— NatureReviewsCancer (@NatureRevCancer) June 6, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663100"
                        open
                      >
                        <summary class="article-expander__title">Inflammation in Myeloid Malignancies: From Bench to Bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, stemming from a somatically mutated hematopoietic clone, can cause a wide variety of clinical consequences, including pancytopenia in myelodysplastic syndrome, overproduction of three myeloid lineages in myeloproliferative neoplasm, and the rapid growth of immature hematopoietic cells in acute myeloid leukemia (AML). It is becoming clear that inflammation is a hallmark feature of clonal myeloid conditions, ranging from clonal hematopoiesis of indeterminate potential to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663063"
                        open
                      >
                        <summary class="article-expander__title">Development of plasma cell dyscrasias in a patient with chronic myeloid leukemia: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663063/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This case highlights that gene translocation may contribute to initiation and sustainability of clonal proliferation. Moreover, the treatment with tyrosine kinase inhibitor and cytogenetic change may contribute to progression from monoclonal gammopathy of undetermined significance to smoldering multiple myeloma and eventually to multiple myeloma.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663100"
                        open
                      >
                        <summary class="article-expander__title">Inflammation in Myeloid Malignancies: From Bench to Bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, stemming from a somatically mutated hematopoietic clone, can cause a wide variety of clinical consequences, including pancytopenia in myelodysplastic syndrome, overproduction of three myeloid lineages in myeloproliferative neoplasm, and the rapid growth of immature hematopoietic cells in acute myeloid leukemia (AML). It is becoming clear that inflammation is a hallmark feature of clonal myeloid conditions, ranging from clonal hematopoiesis of indeterminate potential to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663063"
                        open
                      >
                        <summary class="article-expander__title">Development of plasma cell dyscrasias in a patient with chronic myeloid leukemia: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663063/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This case highlights that gene translocation may contribute to initiation and sustainability of clonal proliferation. Moreover, the treatment with tyrosine kinase inhibitor and cytogenetic change may contribute to progression from monoclonal gammopathy of undetermined significance to smoldering multiple myeloma and eventually to multiple myeloma.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659875"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: Mutation-specific adaptation to environmental change</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659875/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes a widespread expansion of genetically variant hematopoietic cells that increases exponentially with age and is associated with increased risks of cancers, cardiovascular disease, and other maladies. Here, we discuss how environmental contexts associated with CHIP, such as old age, infections, chemotherapy, or cigarette smoking, alter tissue microenvironments to facilitate the selection and expansion of specific CHIP mutant clones....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663100"
                        open
                      >
                        <summary class="article-expander__title">Inflammation in Myeloid Malignancies: From Bench to Bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, stemming from a somatically mutated hematopoietic clone, can cause a wide variety of clinical consequences, including pancytopenia in myelodysplastic syndrome, overproduction of three myeloid lineages in myeloproliferative neoplasm, and the rapid growth of immature hematopoietic cells in acute myeloid leukemia (AML). It is becoming clear that inflammation is a hallmark feature of clonal myeloid conditions, ranging from clonal hematopoiesis of indeterminate potential to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663063"
                        open
                      >
                        <summary class="article-expander__title">Development of plasma cell dyscrasias in a patient with chronic myeloid leukemia: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663063/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This case highlights that gene translocation may contribute to initiation and sustainability of clonal proliferation. Moreover, the treatment with tyrosine kinase inhibitor and cytogenetic change may contribute to progression from monoclonal gammopathy of undetermined significance to smoldering multiple myeloma and eventually to multiple myeloma.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659875"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: Mutation-specific adaptation to environmental change</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659875/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes a widespread expansion of genetically variant hematopoietic cells that increases exponentially with age and is associated with increased risks of cancers, cardiovascular disease, and other maladies. Here, we discuss how environmental contexts associated with CHIP, such as old age, infections, chemotherapy, or cigarette smoking, alter tissue microenvironments to facilitate the selection and expansion of specific CHIP mutant clones....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663100"
                        open
                      >
                        <summary class="article-expander__title">Inflammation in Myeloid Malignancies: From Bench to Bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, stemming from a somatically mutated hematopoietic clone, can cause a wide variety of clinical consequences, including pancytopenia in myelodysplastic syndrome, overproduction of three myeloid lineages in myeloproliferative neoplasm, and the rapid growth of immature hematopoietic cells in acute myeloid leukemia (AML). It is becoming clear that inflammation is a hallmark feature of clonal myeloid conditions, ranging from clonal hematopoiesis of indeterminate potential to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663063"
                        open
                      >
                        <summary class="article-expander__title">Development of plasma cell dyscrasias in a patient with chronic myeloid leukemia: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663063/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This case highlights that gene translocation may contribute to initiation and sustainability of clonal proliferation. Moreover, the treatment with tyrosine kinase inhibitor and cytogenetic change may contribute to progression from monoclonal gammopathy of undetermined significance to smoldering multiple myeloma and eventually to multiple myeloma.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659875"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: Mutation-specific adaptation to environmental change</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659875/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes a widespread expansion of genetically variant hematopoietic cells that increases exponentially with age and is associated with increased risks of cancers, cardiovascular disease, and other maladies. Here, we discuss how environmental contexts associated with CHIP, such as old age, infections, chemotherapy, or cigarette smoking, alter tissue microenvironments to facilitate the selection and expansion of specific CHIP mutant clones....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663100"
                        open
                      >
                        <summary class="article-expander__title">Inflammation in Myeloid Malignancies: From Bench to Bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, stemming from a somatically mutated hematopoietic clone, can cause a wide variety of clinical consequences, including pancytopenia in myelodysplastic syndrome, overproduction of three myeloid lineages in myeloproliferative neoplasm, and the rapid growth of immature hematopoietic cells in acute myeloid leukemia (AML). It is becoming clear that inflammation is a hallmark feature of clonal myeloid conditions, ranging from clonal hematopoiesis of indeterminate potential to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663063"
                        open
                      >
                        <summary class="article-expander__title">Development of plasma cell dyscrasias in a patient with chronic myeloid leukemia: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663063/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This case highlights that gene translocation may contribute to initiation and sustainability of clonal proliferation. Moreover, the treatment with tyrosine kinase inhibitor and cytogenetic change may contribute to progression from monoclonal gammopathy of undetermined significance to smoldering multiple myeloma and eventually to multiple myeloma.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659875"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: Mutation-specific adaptation to environmental change</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659875/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes a widespread expansion of genetically variant hematopoietic cells that increases exponentially with age and is associated with increased risks of cancers, cardiovascular disease, and other maladies. Here, we discuss how environmental contexts associated with CHIP, such as old age, infections, chemotherapy, or cigarette smoking, alter tissue microenvironments to facilitate the selection and expansion of specific CHIP mutant clones....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663100"
                        open
                      >
                        <summary class="article-expander__title">Inflammation in Myeloid Malignancies: From Bench to Bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, stemming from a somatically mutated hematopoietic clone, can cause a wide variety of clinical consequences, including pancytopenia in myelodysplastic syndrome, overproduction of three myeloid lineages in myeloproliferative neoplasm, and the rapid growth of immature hematopoietic cells in acute myeloid leukemia (AML). It is becoming clear that inflammation is a hallmark feature of clonal myeloid conditions, ranging from clonal hematopoiesis of indeterminate potential to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663063"
                        open
                      >
                        <summary class="article-expander__title">Development of plasma cell dyscrasias in a patient with chronic myeloid leukemia: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663063/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This case highlights that gene translocation may contribute to initiation and sustainability of clonal proliferation. Moreover, the treatment with tyrosine kinase inhibitor and cytogenetic change may contribute to progression from monoclonal gammopathy of undetermined significance to smoldering multiple myeloma and eventually to multiple myeloma.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659875"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: Mutation-specific adaptation to environmental change</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659875/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes a widespread expansion of genetically variant hematopoietic cells that increases exponentially with age and is associated with increased risks of cancers, cardiovascular disease, and other maladies. Here, we discuss how environmental contexts associated with CHIP, such as old age, infections, chemotherapy, or cigarette smoking, alter tissue microenvironments to facilitate the selection and expansion of specific CHIP mutant clones....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663100"
                        open
                      >
                        <summary class="article-expander__title">Inflammation in Myeloid Malignancies: From Bench to Bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, stemming from a somatically mutated hematopoietic clone, can cause a wide variety of clinical consequences, including pancytopenia in myelodysplastic syndrome, overproduction of three myeloid lineages in myeloproliferative neoplasm, and the rapid growth of immature hematopoietic cells in acute myeloid leukemia (AML). It is becoming clear that inflammation is a hallmark feature of clonal myeloid conditions, ranging from clonal hematopoiesis of indeterminate potential to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663063"
                        open
                      >
                        <summary class="article-expander__title">Development of plasma cell dyscrasias in a patient with chronic myeloid leukemia: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663063/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This case highlights that gene translocation may contribute to initiation and sustainability of clonal proliferation. Moreover, the treatment with tyrosine kinase inhibitor and cytogenetic change may contribute to progression from monoclonal gammopathy of undetermined significance to smoldering multiple myeloma and eventually to multiple myeloma.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659875"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: Mutation-specific adaptation to environmental change</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659875/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes a widespread expansion of genetically variant hematopoietic cells that increases exponentially with age and is associated with increased risks of cancers, cardiovascular disease, and other maladies. Here, we discuss how environmental contexts associated with CHIP, such as old age, infections, chemotherapy, or cigarette smoking, alter tissue microenvironments to facilitate the selection and expansion of specific CHIP mutant clones....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663100"
                        open
                      >
                        <summary class="article-expander__title">Inflammation in Myeloid Malignancies: From Bench to Bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, stemming from a somatically mutated hematopoietic clone, can cause a wide variety of clinical consequences, including pancytopenia in myelodysplastic syndrome, overproduction of three myeloid lineages in myeloproliferative neoplasm, and the rapid growth of immature hematopoietic cells in acute myeloid leukemia (AML). It is becoming clear that inflammation is a hallmark feature of clonal myeloid conditions, ranging from clonal hematopoiesis of indeterminate potential to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663063"
                        open
                      >
                        <summary class="article-expander__title">Development of plasma cell dyscrasias in a patient with chronic myeloid leukemia: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663063/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This case highlights that gene translocation may contribute to initiation and sustainability of clonal proliferation. Moreover, the treatment with tyrosine kinase inhibitor and cytogenetic change may contribute to progression from monoclonal gammopathy of undetermined significance to smoldering multiple myeloma and eventually to multiple myeloma.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659875"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: Mutation-specific adaptation to environmental change</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659875/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes a widespread expansion of genetically variant hematopoietic cells that increases exponentially with age and is associated with increased risks of cancers, cardiovascular disease, and other maladies. Here, we discuss how environmental contexts associated with CHIP, such as old age, infections, chemotherapy, or cigarette smoking, alter tissue microenvironments to facilitate the selection and expansion of specific CHIP mutant clones....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663100"
                        open
                      >
                        <summary class="article-expander__title">Inflammation in Myeloid Malignancies: From Bench to Bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, stemming from a somatically mutated hematopoietic clone, can cause a wide variety of clinical consequences, including pancytopenia in myelodysplastic syndrome, overproduction of three myeloid lineages in myeloproliferative neoplasm, and the rapid growth of immature hematopoietic cells in acute myeloid leukemia (AML). It is becoming clear that inflammation is a hallmark feature of clonal myeloid conditions, ranging from clonal hematopoiesis of indeterminate potential to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663063"
                        open
                      >
                        <summary class="article-expander__title">Development of plasma cell dyscrasias in a patient with chronic myeloid leukemia: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663063/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This case highlights that gene translocation may contribute to initiation and sustainability of clonal proliferation. Moreover, the treatment with tyrosine kinase inhibitor and cytogenetic change may contribute to progression from monoclonal gammopathy of undetermined significance to smoldering multiple myeloma and eventually to multiple myeloma.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659875"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: Mutation-specific adaptation to environmental change</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659875/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes a widespread expansion of genetically variant hematopoietic cells that increases exponentially with age and is associated with increased risks of cancers, cardiovascular disease, and other maladies. Here, we discuss how environmental contexts associated with CHIP, such as old age, infections, chemotherapy, or cigarette smoking, alter tissue microenvironments to facilitate the selection and expansion of specific CHIP mutant clones....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663100"
                        open
                      >
                        <summary class="article-expander__title">Inflammation in Myeloid Malignancies: From Bench to Bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, stemming from a somatically mutated hematopoietic clone, can cause a wide variety of clinical consequences, including pancytopenia in myelodysplastic syndrome, overproduction of three myeloid lineages in myeloproliferative neoplasm, and the rapid growth of immature hematopoietic cells in acute myeloid leukemia (AML). It is becoming clear that inflammation is a hallmark feature of clonal myeloid conditions, ranging from clonal hematopoiesis of indeterminate potential to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663063"
                        open
                      >
                        <summary class="article-expander__title">Development of plasma cell dyscrasias in a patient with chronic myeloid leukemia: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663063/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This case highlights that gene translocation may contribute to initiation and sustainability of clonal proliferation. Moreover, the treatment with tyrosine kinase inhibitor and cytogenetic change may contribute to progression from monoclonal gammopathy of undetermined significance to smoldering multiple myeloma and eventually to multiple myeloma.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659875"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: Mutation-specific adaptation to environmental change</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659875/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes a widespread expansion of genetically variant hematopoietic cells that increases exponentially with age and is associated with increased risks of cancers, cardiovascular disease, and other maladies. Here, we discuss how environmental contexts associated with CHIP, such as old age, infections, chemotherapy, or cigarette smoking, alter tissue microenvironments to facilitate the selection and expansion of specific CHIP mutant clones....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663100"
                        open
                      >
                        <summary class="article-expander__title">Inflammation in Myeloid Malignancies: From Bench to Bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, stemming from a somatically mutated hematopoietic clone, can cause a wide variety of clinical consequences, including pancytopenia in myelodysplastic syndrome, overproduction of three myeloid lineages in myeloproliferative neoplasm, and the rapid growth of immature hematopoietic cells in acute myeloid leukemia (AML). It is becoming clear that inflammation is a hallmark feature of clonal myeloid conditions, ranging from clonal hematopoiesis of indeterminate potential to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663063"
                        open
                      >
                        <summary class="article-expander__title">Development of plasma cell dyscrasias in a patient with chronic myeloid leukemia: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663063/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This case highlights that gene translocation may contribute to initiation and sustainability of clonal proliferation. Moreover, the treatment with tyrosine kinase inhibitor and cytogenetic change may contribute to progression from monoclonal gammopathy of undetermined significance to smoldering multiple myeloma and eventually to multiple myeloma.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659875"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: Mutation-specific adaptation to environmental change</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659875/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes a widespread expansion of genetically variant hematopoietic cells that increases exponentially with age and is associated with increased risks of cancers, cardiovascular disease, and other maladies. Here, we discuss how environmental contexts associated with CHIP, such as old age, infections, chemotherapy, or cigarette smoking, alter tissue microenvironments to facilitate the selection and expansion of specific CHIP mutant clones....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663100"
                        open
                      >
                        <summary class="article-expander__title">Inflammation in Myeloid Malignancies: From Bench to Bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, stemming from a somatically mutated hematopoietic clone, can cause a wide variety of clinical consequences, including pancytopenia in myelodysplastic syndrome, overproduction of three myeloid lineages in myeloproliferative neoplasm, and the rapid growth of immature hematopoietic cells in acute myeloid leukemia (AML). It is becoming clear that inflammation is a hallmark feature of clonal myeloid conditions, ranging from clonal hematopoiesis of indeterminate potential to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663063"
                        open
                      >
                        <summary class="article-expander__title">Development of plasma cell dyscrasias in a patient with chronic myeloid leukemia: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663063/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This case highlights that gene translocation may contribute to initiation and sustainability of clonal proliferation. Moreover, the treatment with tyrosine kinase inhibitor and cytogenetic change may contribute to progression from monoclonal gammopathy of undetermined significance to smoldering multiple myeloma and eventually to multiple myeloma.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659875"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: Mutation-specific adaptation to environmental change</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659875/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes a widespread expansion of genetically variant hematopoietic cells that increases exponentially with age and is associated with increased risks of cancers, cardiovascular disease, and other maladies. Here, we discuss how environmental contexts associated with CHIP, such as old age, infections, chemotherapy, or cigarette smoking, alter tissue microenvironments to facilitate the selection and expansion of specific CHIP mutant clones....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663100"
                        open
                      >
                        <summary class="article-expander__title">Inflammation in Myeloid Malignancies: From Bench to Bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, stemming from a somatically mutated hematopoietic clone, can cause a wide variety of clinical consequences, including pancytopenia in myelodysplastic syndrome, overproduction of three myeloid lineages in myeloproliferative neoplasm, and the rapid growth of immature hematopoietic cells in acute myeloid leukemia (AML). It is becoming clear that inflammation is a hallmark feature of clonal myeloid conditions, ranging from clonal hematopoiesis of indeterminate potential to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663063"
                        open
                      >
                        <summary class="article-expander__title">Development of plasma cell dyscrasias in a patient with chronic myeloid leukemia: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663063/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This case highlights that gene translocation may contribute to initiation and sustainability of clonal proliferation. Moreover, the treatment with tyrosine kinase inhibitor and cytogenetic change may contribute to progression from monoclonal gammopathy of undetermined significance to smoldering multiple myeloma and eventually to multiple myeloma.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659875"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: Mutation-specific adaptation to environmental change</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659875/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes a widespread expansion of genetically variant hematopoietic cells that increases exponentially with age and is associated with increased risks of cancers, cardiovascular disease, and other maladies. Here, we discuss how environmental contexts associated with CHIP, such as old age, infections, chemotherapy, or cigarette smoking, alter tissue microenvironments to facilitate the selection and expansion of specific CHIP mutant clones....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663100"
                        open
                      >
                        <summary class="article-expander__title">Inflammation in Myeloid Malignancies: From Bench to Bedside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663100/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608003655&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myeloid malignancies, stemming from a somatically mutated hematopoietic clone, can cause a wide variety of clinical consequences, including pancytopenia in myelodysplastic syndrome, overproduction of three myeloid lineages in myeloproliferative neoplasm, and the rapid growth of immature hematopoietic cells in acute myeloid leukemia (AML). It is becoming clear that inflammation is a hallmark feature of clonal myeloid conditions, ranging from clonal hematopoiesis of indeterminate potential to AML....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35663063"
                        open
                      >
                        <summary class="article-expander__title">Development of plasma cell dyscrasias in a patient with chronic myeloid leukemia: A case report</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35663063/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608003655&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: This case highlights that gene translocation may contribute to initiation and sustainability of clonal proliferation. Moreover, the treatment with tyrosine kinase inhibitor and cytogenetic change may contribute to progression from monoclonal gammopathy of undetermined significance to smoldering multiple myeloma and eventually to multiple myeloma.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659875"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis: Mutation-specific adaptation to environmental change</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659875/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608003655&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal hematopoiesis of indeterminate potential (CHIP) describes a widespread expansion of genetically variant hematopoietic cells that increases exponentially with age and is associated with increased risks of cancers, cardiovascular disease, and other maladies. Here, we discuss how environmental contexts associated with CHIP, such as old age, infections, chemotherapy, or cigarette smoking, alter tissue microenvironments to facilitate the selection and expansion of specific CHIP mutant clones....</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220623210756&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35659345"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary perspectives, heterogeneity and ovarian cancer: a complicated tale from past to present</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35659345/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ovarian cancer is composed of a complex system of cells best described by features such as clonal evolution, spatial and temporal genetic heterogeneity, and development of drug resistance, thus making it the most lethal gynecologic cancer. Seminal work on cancer as an evolutionary process has a long history; however, recent cost-effective large-scale molecular profiling has started to provide novel insights coupled with the development of mathematical algorithms. In the current review, we have...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35661532"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal genomic alternations and clonal dynamics analysis of primary malignant melanoma of the esophagus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35661532/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Primary malignant melanoma of the esophagus (PMME) is a rare gastrointestinal melanoma with a high rate of recurrence and metastasis. The standard of care for PMME has not been established yet due to a lack of understanding of its clinical and molecular pathogenesis. Thus, we performed genomic profiling on a recurrent PMME case to seek novel opportunities for the management of this rare disease. Between 2013 and 2016, 6 tissue samples including 3 from the primary tumors, 2 from the relapsed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-03 datetime="2022-06-03T10:00:00.000Z">2022-06-03</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-03 datetime="2022-06-03T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657598"
                        open
                      >
                        <summary class="article-expander__title">PPM1D in Solid and Hematologic Malignancies: Friend and Foe?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the face of constant genomic insults, the DNA damage response (DDR) is initiated to preserve genome integrity; its disruption is a classic hallmark of cancer. Protein Phosphatase Mg2+/Mn2+-Dependent 1D (PPM1D) is a central negative regulator of the DDR that is mutated or amplified in many solid cancers. PPM1D overexpression is associated with increased proliferative and metastatic behavior in multiple solid tumor types and patients with PPM1D-mutated malignancies have poorer prognoses. Recent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657494"
                        open
                      >
                        <summary class="article-expander__title">Somatic Mutations and Clonal Hematopoiesis as Drivers of Age-Related Cardiovascular Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Clonal hematopoiesis of indeterminate potential (CHIP) has been identified as a novel cardiovascular risk factor. Here we review the relationship of lifestyle and environmental risk factors predisposing to somatic mutations and CHIP and provide an overview on age-related cardiovascular outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657598"
                        open
                      >
                        <summary class="article-expander__title">PPM1D in Solid and Hematologic Malignancies: Friend and Foe?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the face of constant genomic insults, the DNA damage response (DDR) is initiated to preserve genome integrity; its disruption is a classic hallmark of cancer. Protein Phosphatase Mg2+/Mn2+-Dependent 1D (PPM1D) is a central negative regulator of the DDR that is mutated or amplified in many solid cancers. PPM1D overexpression is associated with increased proliferative and metastatic behavior in multiple solid tumor types and patients with PPM1D-mutated malignancies have poorer prognoses. Recent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657494"
                        open
                      >
                        <summary class="article-expander__title">Somatic Mutations and Clonal Hematopoiesis as Drivers of Age-Related Cardiovascular Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Clonal hematopoiesis of indeterminate potential (CHIP) has been identified as a novel cardiovascular risk factor. Here we review the relationship of lifestyle and environmental risk factors predisposing to somatic mutations and CHIP and provide an overview on age-related cardiovascular outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657598"
                        open
                      >
                        <summary class="article-expander__title">PPM1D in Solid and Hematologic Malignancies: Friend and Foe?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the face of constant genomic insults, the DNA damage response (DDR) is initiated to preserve genome integrity; its disruption is a classic hallmark of cancer. Protein Phosphatase Mg2+/Mn2+-Dependent 1D (PPM1D) is a central negative regulator of the DDR that is mutated or amplified in many solid cancers. PPM1D overexpression is associated with increased proliferative and metastatic behavior in multiple solid tumor types and patients with PPM1D-mutated malignancies have poorer prognoses. Recent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657494"
                        open
                      >
                        <summary class="article-expander__title">Somatic Mutations and Clonal Hematopoiesis as Drivers of Age-Related Cardiovascular Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Clonal hematopoiesis of indeterminate potential (CHIP) has been identified as a novel cardiovascular risk factor. Here we review the relationship of lifestyle and environmental risk factors predisposing to somatic mutations and CHIP and provide an overview on age-related cardiovascular outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657598"
                        open
                      >
                        <summary class="article-expander__title">PPM1D in Solid and Hematologic Malignancies: Friend and Foe?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the face of constant genomic insults, the DNA damage response (DDR) is initiated to preserve genome integrity; its disruption is a classic hallmark of cancer. Protein Phosphatase Mg2+/Mn2+-Dependent 1D (PPM1D) is a central negative regulator of the DDR that is mutated or amplified in many solid cancers. PPM1D overexpression is associated with increased proliferative and metastatic behavior in multiple solid tumor types and patients with PPM1D-mutated malignancies have poorer prognoses. Recent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657494"
                        open
                      >
                        <summary class="article-expander__title">Somatic Mutations and Clonal Hematopoiesis as Drivers of Age-Related Cardiovascular Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Clonal hematopoiesis of indeterminate potential (CHIP) has been identified as a novel cardiovascular risk factor. Here we review the relationship of lifestyle and environmental risk factors predisposing to somatic mutations and CHIP and provide an overview on age-related cardiovascular outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657598"
                        open
                      >
                        <summary class="article-expander__title">PPM1D in Solid and Hematologic Malignancies: Friend and Foe?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the face of constant genomic insults, the DNA damage response (DDR) is initiated to preserve genome integrity; its disruption is a classic hallmark of cancer. Protein Phosphatase Mg2+/Mn2+-Dependent 1D (PPM1D) is a central negative regulator of the DDR that is mutated or amplified in many solid cancers. PPM1D overexpression is associated with increased proliferative and metastatic behavior in multiple solid tumor types and patients with PPM1D-mutated malignancies have poorer prognoses. Recent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657494"
                        open
                      >
                        <summary class="article-expander__title">Somatic Mutations and Clonal Hematopoiesis as Drivers of Age-Related Cardiovascular Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Clonal hematopoiesis of indeterminate potential (CHIP) has been identified as a novel cardiovascular risk factor. Here we review the relationship of lifestyle and environmental risk factors predisposing to somatic mutations and CHIP and provide an overview on age-related cardiovascular outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657598"
                        open
                      >
                        <summary class="article-expander__title">PPM1D in Solid and Hematologic Malignancies: Friend and Foe?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the face of constant genomic insults, the DNA damage response (DDR) is initiated to preserve genome integrity; its disruption is a classic hallmark of cancer. Protein Phosphatase Mg2+/Mn2+-Dependent 1D (PPM1D) is a central negative regulator of the DDR that is mutated or amplified in many solid cancers. PPM1D overexpression is associated with increased proliferative and metastatic behavior in multiple solid tumor types and patients with PPM1D-mutated malignancies have poorer prognoses. Recent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657494"
                        open
                      >
                        <summary class="article-expander__title">Somatic Mutations and Clonal Hematopoiesis as Drivers of Age-Related Cardiovascular Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Clonal hematopoiesis of indeterminate potential (CHIP) has been identified as a novel cardiovascular risk factor. Here we review the relationship of lifestyle and environmental risk factors predisposing to somatic mutations and CHIP and provide an overview on age-related cardiovascular outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657598"
                        open
                      >
                        <summary class="article-expander__title">PPM1D in Solid and Hematologic Malignancies: Friend and Foe?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the face of constant genomic insults, the DNA damage response (DDR) is initiated to preserve genome integrity; its disruption is a classic hallmark of cancer. Protein Phosphatase Mg2+/Mn2+-Dependent 1D (PPM1D) is a central negative regulator of the DDR that is mutated or amplified in many solid cancers. PPM1D overexpression is associated with increased proliferative and metastatic behavior in multiple solid tumor types and patients with PPM1D-mutated malignancies have poorer prognoses. Recent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657494"
                        open
                      >
                        <summary class="article-expander__title">Somatic Mutations and Clonal Hematopoiesis as Drivers of Age-Related Cardiovascular Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Clonal hematopoiesis of indeterminate potential (CHIP) has been identified as a novel cardiovascular risk factor. Here we review the relationship of lifestyle and environmental risk factors predisposing to somatic mutations and CHIP and provide an overview on age-related cardiovascular outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657598"
                        open
                      >
                        <summary class="article-expander__title">PPM1D in Solid and Hematologic Malignancies: Friend and Foe?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the face of constant genomic insults, the DNA damage response (DDR) is initiated to preserve genome integrity; its disruption is a classic hallmark of cancer. Protein Phosphatase Mg2+/Mn2+-Dependent 1D (PPM1D) is a central negative regulator of the DDR that is mutated or amplified in many solid cancers. PPM1D overexpression is associated with increased proliferative and metastatic behavior in multiple solid tumor types and patients with PPM1D-mutated malignancies have poorer prognoses. Recent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657494"
                        open
                      >
                        <summary class="article-expander__title">Somatic Mutations and Clonal Hematopoiesis as Drivers of Age-Related Cardiovascular Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Clonal hematopoiesis of indeterminate potential (CHIP) has been identified as a novel cardiovascular risk factor. Here we review the relationship of lifestyle and environmental risk factors predisposing to somatic mutations and CHIP and provide an overview on age-related cardiovascular outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657598"
                        open
                      >
                        <summary class="article-expander__title">PPM1D in Solid and Hematologic Malignancies: Friend and Foe?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the face of constant genomic insults, the DNA damage response (DDR) is initiated to preserve genome integrity; its disruption is a classic hallmark of cancer. Protein Phosphatase Mg2+/Mn2+-Dependent 1D (PPM1D) is a central negative regulator of the DDR that is mutated or amplified in many solid cancers. PPM1D overexpression is associated with increased proliferative and metastatic behavior in multiple solid tumor types and patients with PPM1D-mutated malignancies have poorer prognoses. Recent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657494"
                        open
                      >
                        <summary class="article-expander__title">Somatic Mutations and Clonal Hematopoiesis as Drivers of Age-Related Cardiovascular Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Clonal hematopoiesis of indeterminate potential (CHIP) has been identified as a novel cardiovascular risk factor. Here we review the relationship of lifestyle and environmental risk factors predisposing to somatic mutations and CHIP and provide an overview on age-related cardiovascular outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657598"
                        open
                      >
                        <summary class="article-expander__title">PPM1D in Solid and Hematologic Malignancies: Friend and Foe?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657598/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608003655&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In the face of constant genomic insults, the DNA damage response (DDR) is initiated to preserve genome integrity; its disruption is a classic hallmark of cancer. Protein Phosphatase Mg2+/Mn2+-Dependent 1D (PPM1D) is a central negative regulator of the DDR that is mutated or amplified in many solid cancers. PPM1D overexpression is associated with increased proliferative and metastatic behavior in multiple solid tumor types and patients with PPM1D-mutated malignancies have poorer prognoses. Recent...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657494"
                        open
                      >
                        <summary class="article-expander__title">Somatic Mutations and Clonal Hematopoiesis as Drivers of Age-Related Cardiovascular Risk</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657494/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608003655&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>PURPOSE OF REVIEW: Clonal hematopoiesis of indeterminate potential (CHIP) has been identified as a novel cardiovascular risk factor. Here we review the relationship of lifestyle and environmental risk factors predisposing to somatic mutations and CHIP and provide an overview on age-related cardiovascular outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35657155"
                        open
                      >
                        <summary class="article-expander__title">The evolving landscape of prostate cancer somatic mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35657155/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BACKGROUND: The landscape of somatic mutations in prostate cancer (PCa) has quickly evolved over the past years.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-02 datetime="2022-06-02T10:00:00.000Z">2022-06-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-02 datetime="2022-06-02T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654360"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and mosaicism revealed by a multi-tissue analysis of constitutional TP53 Status</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654360/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We demonstrated an effective approach and tools for discerning germline TP53 status.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654360"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and mosaicism revealed by a multi-tissue analysis of constitutional TP53 Status</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654360/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We demonstrated an effective approach and tools for discerning germline TP53 status.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654360"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and mosaicism revealed by a multi-tissue analysis of constitutional TP53 Status</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654360/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We demonstrated an effective approach and tools for discerning germline TP53 status.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654360"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and mosaicism revealed by a multi-tissue analysis of constitutional TP53 Status</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654360/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We demonstrated an effective approach and tools for discerning germline TP53 status.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654360"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and mosaicism revealed by a multi-tissue analysis of constitutional TP53 Status</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654360/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We demonstrated an effective approach and tools for discerning germline TP53 status.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654360"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and mosaicism revealed by a multi-tissue analysis of constitutional TP53 Status</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654360/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We demonstrated an effective approach and tools for discerning germline TP53 status.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654360"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and mosaicism revealed by a multi-tissue analysis of constitutional TP53 Status</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654360/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We demonstrated an effective approach and tools for discerning germline TP53 status.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654360"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and mosaicism revealed by a multi-tissue analysis of constitutional TP53 Status</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654360/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We demonstrated an effective approach and tools for discerning germline TP53 status.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654360"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and mosaicism revealed by a multi-tissue analysis of constitutional TP53 Status</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654360/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We demonstrated an effective approach and tools for discerning germline TP53 status.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654360"
                        open
                      >
                        <summary class="article-expander__title">Clonal hematopoiesis and mosaicism revealed by a multi-tissue analysis of constitutional TP53 Status</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654360/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608003655&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: We demonstrated an effective approach and tools for discerning germline TP53 status.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220622210352&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35654752"
                        open
                      >
                        <summary class="article-expander__title">Intratumoral resolution of driver gene mutation heterogeneity in renal cancer using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35654752/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Intratumoral heterogeneity arising from tumor evolution poses significant challenges biologically and clinically. Dissecting this complexity may benefit from deep learning (DL) algorithms, which can infer molecular features from ubiquitous H&amp;E stained tissue sections. While DL algorithms have been developed to predict some driver mutations from H&amp;E images, the ability of these DL algorithms to resolve intratumoral mutation heterogeneity at sub-clonal spatial resolution is unexplored. Here, we...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35651714"
                        open
                      >
                        <summary class="article-expander__title">The Promise of Single-cell Technology in Providing New Insights Into the Molecular Heterogeneity and Management of Acute Lymphoblastic Leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35651714/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Drug resistance and treatment failure in pediatric acute lymphoblastic leukemia (ALL) are in part driven by tumor heterogeneity and clonal evolution. Although bulk tumor genomic analyses have provided some insight into these processes, single-cell sequencing has emerged as a powerful technique to profile individual cells in unprecedented detail. Since the introduction of single-cell RNA sequencing, we now have the capability to capture not only transcriptomic, but also genomic, epigenetic, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-06-01 datetime="2022-06-01T10:00:00.000Z">2022-06-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-06-01 datetime="2022-06-01T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650444"
                        open
                      >
                        <summary class="article-expander__title">The longitudinal dynamics and natural history of clonal haematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650444/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions driven by somatic mutations become pervasive across human tissues with age, including in the haematopoietic system, where the phenomenon is termed clonal haematopoiesis^(1-4). The understanding of how and when clonal haematopoiesis develops, the factors that govern its behaviour, how it interacts with ageing and how these variables relate to malignant progression remains limited^(5,6). Here we track 697 clonal haematopoiesis clones from 385 individuals 55 years of age or older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650444"
                        open
                      >
                        <summary class="article-expander__title">The longitudinal dynamics and natural history of clonal haematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650444/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions driven by somatic mutations become pervasive across human tissues with age, including in the haematopoietic system, where the phenomenon is termed clonal haematopoiesis^(1-4). The understanding of how and when clonal haematopoiesis develops, the factors that govern its behaviour, how it interacts with ageing and how these variables relate to malignant progression remains limited^(5,6). Here we track 697 clonal haematopoiesis clones from 385 individuals 55 years of age or older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650442"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics of haematopoiesis across the human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650442/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related change in human haematopoiesis causes reduced regenerative capacity¹, cytopenias², immune dysfunction³ and increased risk of blood cancer^(4-6), but the reason for such abrupt functional decline after 70 years of age remains unclear. Here we sequenced 3,579 genomes from single cell-derived colonies of haematopoietic cells across 10 human subjects from 0 to 81 years of age. Haematopoietic stem cells or multipotent progenitors (HSC/MPPs) accumulated a mean of 17 mutations per year...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650444"
                        open
                      >
                        <summary class="article-expander__title">The longitudinal dynamics and natural history of clonal haematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650444/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions driven by somatic mutations become pervasive across human tissues with age, including in the haematopoietic system, where the phenomenon is termed clonal haematopoiesis^(1-4). The understanding of how and when clonal haematopoiesis develops, the factors that govern its behaviour, how it interacts with ageing and how these variables relate to malignant progression remains limited^(5,6). Here we track 697 clonal haematopoiesis clones from 385 individuals 55 years of age or older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650442"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics of haematopoiesis across the human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650442/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related change in human haematopoiesis causes reduced regenerative capacity¹, cytopenias², immune dysfunction³ and increased risk of blood cancer^(4-6), but the reason for such abrupt functional decline after 70 years of age remains unclear. Here we sequenced 3,579 genomes from single cell-derived colonies of haematopoietic cells across 10 human subjects from 0 to 81 years of age. Haematopoietic stem cells or multipotent progenitors (HSC/MPPs) accumulated a mean of 17 mutations per year...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643340"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular risk factors and molecular routes underlying endothelial dysfunction: Novel opportunities for primary prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>One of the major challenges of cardiovascular primary prevention approach is the absence of early biomarkers of endothelial dysfunction which may be useful for identifying at-risk subjects. Endothelial dysfunction is a systemic disorder in which traditional cardiovascular risk factors, such as aging, gender, hypertension, smoking, hyperglycemia, and dyslipidemia, as well as emerging risk determinants, such as fetal factors, gut microbiome alteration, clonal hematopoiesis, air pollution, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642674"
                        open
                      >
                        <summary class="article-expander__title">Relationship of HIF‑1α expression with apoptosis and cell cycle in bone marrow mesenchymal stem cells from patients with myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of abnormal clonal disorders with ineffective hematopoiesis, which are incurable with conventional therapy. Of note, MDS features an abnormal bone marrow microenvironment, which is related to its incidence. The hypoxia‑inducible factor‑1α (HIF‑1α) transcriptional signature is generally activated in bone marrow stem/progenitor cells of patients with MDS. To analyze the expression of HIF‑1α in bone marrow mesenchymal stem cells (BM‑MSCs) and the apoptosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650444"
                        open
                      >
                        <summary class="article-expander__title">The longitudinal dynamics and natural history of clonal haematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650444/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions driven by somatic mutations become pervasive across human tissues with age, including in the haematopoietic system, where the phenomenon is termed clonal haematopoiesis^(1-4). The understanding of how and when clonal haematopoiesis develops, the factors that govern its behaviour, how it interacts with ageing and how these variables relate to malignant progression remains limited^(5,6). Here we track 697 clonal haematopoiesis clones from 385 individuals 55 years of age or older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650442"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics of haematopoiesis across the human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650442/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related change in human haematopoiesis causes reduced regenerative capacity¹, cytopenias², immune dysfunction³ and increased risk of blood cancer^(4-6), but the reason for such abrupt functional decline after 70 years of age remains unclear. Here we sequenced 3,579 genomes from single cell-derived colonies of haematopoietic cells across 10 human subjects from 0 to 81 years of age. Haematopoietic stem cells or multipotent progenitors (HSC/MPPs) accumulated a mean of 17 mutations per year...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643340"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular risk factors and molecular routes underlying endothelial dysfunction: Novel opportunities for primary prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>One of the major challenges of cardiovascular primary prevention approach is the absence of early biomarkers of endothelial dysfunction which may be useful for identifying at-risk subjects. Endothelial dysfunction is a systemic disorder in which traditional cardiovascular risk factors, such as aging, gender, hypertension, smoking, hyperglycemia, and dyslipidemia, as well as emerging risk determinants, such as fetal factors, gut microbiome alteration, clonal hematopoiesis, air pollution, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642674"
                        open
                      >
                        <summary class="article-expander__title">Relationship of HIF‑1α expression with apoptosis and cell cycle in bone marrow mesenchymal stem cells from patients with myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of abnormal clonal disorders with ineffective hematopoiesis, which are incurable with conventional therapy. Of note, MDS features an abnormal bone marrow microenvironment, which is related to its incidence. The hypoxia‑inducible factor‑1α (HIF‑1α) transcriptional signature is generally activated in bone marrow stem/progenitor cells of patients with MDS. To analyze the expression of HIF‑1α in bone marrow mesenchymal stem cells (BM‑MSCs) and the apoptosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650444"
                        open
                      >
                        <summary class="article-expander__title">The longitudinal dynamics and natural history of clonal haematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650444/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions driven by somatic mutations become pervasive across human tissues with age, including in the haematopoietic system, where the phenomenon is termed clonal haematopoiesis^(1-4). The understanding of how and when clonal haematopoiesis develops, the factors that govern its behaviour, how it interacts with ageing and how these variables relate to malignant progression remains limited^(5,6). Here we track 697 clonal haematopoiesis clones from 385 individuals 55 years of age or older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650442"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics of haematopoiesis across the human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650442/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related change in human haematopoiesis causes reduced regenerative capacity¹, cytopenias², immune dysfunction³ and increased risk of blood cancer^(4-6), but the reason for such abrupt functional decline after 70 years of age remains unclear. Here we sequenced 3,579 genomes from single cell-derived colonies of haematopoietic cells across 10 human subjects from 0 to 81 years of age. Haematopoietic stem cells or multipotent progenitors (HSC/MPPs) accumulated a mean of 17 mutations per year...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643340"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular risk factors and molecular routes underlying endothelial dysfunction: Novel opportunities for primary prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>One of the major challenges of cardiovascular primary prevention approach is the absence of early biomarkers of endothelial dysfunction which may be useful for identifying at-risk subjects. Endothelial dysfunction is a systemic disorder in which traditional cardiovascular risk factors, such as aging, gender, hypertension, smoking, hyperglycemia, and dyslipidemia, as well as emerging risk determinants, such as fetal factors, gut microbiome alteration, clonal hematopoiesis, air pollution, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642674"
                        open
                      >
                        <summary class="article-expander__title">Relationship of HIF‑1α expression with apoptosis and cell cycle in bone marrow mesenchymal stem cells from patients with myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of abnormal clonal disorders with ineffective hematopoiesis, which are incurable with conventional therapy. Of note, MDS features an abnormal bone marrow microenvironment, which is related to its incidence. The hypoxia‑inducible factor‑1α (HIF‑1α) transcriptional signature is generally activated in bone marrow stem/progenitor cells of patients with MDS. To analyze the expression of HIF‑1α in bone marrow mesenchymal stem cells (BM‑MSCs) and the apoptosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650444"
                        open
                      >
                        <summary class="article-expander__title">The longitudinal dynamics and natural history of clonal haematopoiesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650444/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Clonal expansions driven by somatic mutations become pervasive across human tissues with age, including in the haematopoietic system, where the phenomenon is termed clonal haematopoiesis^(1-4). The understanding of how and when clonal haematopoiesis develops, the factors that govern its behaviour, how it interacts with ageing and how these variables relate to malignant progression remains limited^(5,6). Here we track 697 clonal haematopoiesis clones from 385 individuals 55 years of age or older...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650442"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics of haematopoiesis across the human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650442/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related change in human haematopoiesis causes reduced regenerative capacity¹, cytopenias², immune dysfunction³ and increased risk of blood cancer^(4-6), but the reason for such abrupt functional decline after 70 years of age remains unclear. Here we sequenced 3,579 genomes from single cell-derived colonies of haematopoietic cells across 10 human subjects from 0 to 81 years of age. Haematopoietic stem cells or multipotent progenitors (HSC/MPPs) accumulated a mean of 17 mutations per year...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643340"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular risk factors and molecular routes underlying endothelial dysfunction: Novel opportunities for primary prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>One of the major challenges of cardiovascular primary prevention approach is the absence of early biomarkers of endothelial dysfunction which may be useful for identifying at-risk subjects. Endothelial dysfunction is a systemic disorder in which traditional cardiovascular risk factors, such as aging, gender, hypertension, smoking, hyperglycemia, and dyslipidemia, as well as emerging risk determinants, such as fetal factors, gut microbiome alteration, clonal hematopoiesis, air pollution, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642674"
                        open
                      >
                        <summary class="article-expander__title">Relationship of HIF‑1α expression with apoptosis and cell cycle in bone marrow mesenchymal stem cells from patients with myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of abnormal clonal disorders with ineffective hematopoiesis, which are incurable with conventional therapy. Of note, MDS features an abnormal bone marrow microenvironment, which is related to its incidence. The hypoxia‑inducible factor‑1α (HIF‑1α) transcriptional signature is generally activated in bone marrow stem/progenitor cells of patients with MDS. To analyze the expression of HIF‑1α in bone marrow mesenchymal stem cells (BM‑MSCs) and the apoptosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643340"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular risk factors and molecular routes underlying endothelial dysfunction: Novel opportunities for primary prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>One of the major challenges of cardiovascular primary prevention approach is the absence of early biomarkers of endothelial dysfunction which may be useful for identifying at-risk subjects. Endothelial dysfunction is a systemic disorder in which traditional cardiovascular risk factors, such as aging, gender, hypertension, smoking, hyperglycemia, and dyslipidemia, as well as emerging risk determinants, such as fetal factors, gut microbiome alteration, clonal hematopoiesis, air pollution, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642674"
                        open
                      >
                        <summary class="article-expander__title">Relationship of HIF‑1α expression with apoptosis and cell cycle in bone marrow mesenchymal stem cells from patients with myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of abnormal clonal disorders with ineffective hematopoiesis, which are incurable with conventional therapy. Of note, MDS features an abnormal bone marrow microenvironment, which is related to its incidence. The hypoxia‑inducible factor‑1α (HIF‑1α) transcriptional signature is generally activated in bone marrow stem/progenitor cells of patients with MDS. To analyze the expression of HIF‑1α in bone marrow mesenchymal stem cells (BM‑MSCs) and the apoptosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643340"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular risk factors and molecular routes underlying endothelial dysfunction: Novel opportunities for primary prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>One of the major challenges of cardiovascular primary prevention approach is the absence of early biomarkers of endothelial dysfunction which may be useful for identifying at-risk subjects. Endothelial dysfunction is a systemic disorder in which traditional cardiovascular risk factors, such as aging, gender, hypertension, smoking, hyperglycemia, and dyslipidemia, as well as emerging risk determinants, such as fetal factors, gut microbiome alteration, clonal hematopoiesis, air pollution, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642674"
                        open
                      >
                        <summary class="article-expander__title">Relationship of HIF‑1α expression with apoptosis and cell cycle in bone marrow mesenchymal stem cells from patients with myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of abnormal clonal disorders with ineffective hematopoiesis, which are incurable with conventional therapy. Of note, MDS features an abnormal bone marrow microenvironment, which is related to its incidence. The hypoxia‑inducible factor‑1α (HIF‑1α) transcriptional signature is generally activated in bone marrow stem/progenitor cells of patients with MDS. To analyze the expression of HIF‑1α in bone marrow mesenchymal stem cells (BM‑MSCs) and the apoptosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643340"
                        open
                      >
                        <summary class="article-expander__title">Cardiovascular risk factors and molecular routes underlying endothelial dysfunction: Novel opportunities for primary prevention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643340/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608003655&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>One of the major challenges of cardiovascular primary prevention approach is the absence of early biomarkers of endothelial dysfunction which may be useful for identifying at-risk subjects. Endothelial dysfunction is a systemic disorder in which traditional cardiovascular risk factors, such as aging, gender, hypertension, smoking, hyperglycemia, and dyslipidemia, as well as emerging risk determinants, such as fetal factors, gut microbiome alteration, clonal hematopoiesis, air pollution, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642674"
                        open
                      >
                        <summary class="article-expander__title">Relationship of HIF‑1α expression with apoptosis and cell cycle in bone marrow mesenchymal stem cells from patients with myelodysplastic syndrome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642674/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608003655&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Myelodysplastic syndrome (MDS) is a group of abnormal clonal disorders with ineffective hematopoiesis, which are incurable with conventional therapy. Of note, MDS features an abnormal bone marrow microenvironment, which is related to its incidence. The hypoxia‑inducible factor‑1α (HIF‑1α) transcriptional signature is generally activated in bone marrow stem/progenitor cells of patients with MDS. To analyze the expression of HIF‑1α in bone marrow mesenchymal stem cells (BM‑MSCs) and the apoptosis...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650442"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics of haematopoiesis across the human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650442/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220621212000&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related change in human haematopoiesis causes reduced regenerative capacity¹, cytopenias², immune dysfunction³ and increased risk of blood cancer^(4-6), but the reason for such abrupt functional decline after 70 years of age remains unclear. Here we sequenced 3,579 genomes from single cell-derived colonies of haematopoietic cells across 10 human subjects from 0 to 81 years of age. Haematopoietic stem cells or multipotent progenitors (HSC/MPPs) accumulated a mean of 17 mutations per year...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650442"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics of haematopoiesis across the human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650442/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related change in human haematopoiesis causes reduced regenerative capacity¹, cytopenias², immune dysfunction³ and increased risk of blood cancer^(4-6), but the reason for such abrupt functional decline after 70 years of age remains unclear. Here we sequenced 3,579 genomes from single cell-derived colonies of haematopoietic cells across 10 human subjects from 0 to 81 years of age. Haematopoietic stem cells or multipotent progenitors (HSC/MPPs) accumulated a mean of 17 mutations per year...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650206"
                        open
                      >
                        <summary class="article-expander__title">Nuclear Vav3 is required for polycomb repression complex-1 activity in B-cell lymphoblastic leukemogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute B-cell lymphoblastic leukemia (B-ALL) results from oligo-clonal evolution of B-cell progenitors endowed with initiating and propagating leukemia properties. The activation of both the Rac guanine nucleotide exchange factor (Rac GEF) Vav3 and Rac GTPases is required for leukemogenesis mediated by the oncogenic fusion protein BCR-ABL. Vav3 expression becomes predominantly nuclear upon expression of BCR-ABL signature. In the nucleus, Vav3 interacts with BCR-ABL, Rac, and the polycomb...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35649412"
                        open
                      >
                        <summary class="article-expander__title">Glioma progression is shaped by genetic evolution and microenvironment interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35649412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The factors driving therapy resistance in diffuse glioma remain poorly understood. To identify treatment-associated cellular and genetic changes, we analyzed RNA and/or DNA sequencing data from the temporally separated tumor pairs of 304 adult patients with isocitrate dehydrogenase (IDH)-wild-type and IDH-mutant glioma. Tumors recurred in distinct manners that were dependent on IDH mutation status and attributable to changes in histological feature composition, somatic alterations, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645619"
                        open
                      >
                        <summary class="article-expander__title">Whole-Exome Sequencing Could Distinguish Primary Pulmonary Squamous Cell Carcinoma From Lung Metastases in Individuals With Cervical Squamous Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645619/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220620211049&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aims: Metastatic cervical carcinoma is hard to cure using traditional treatment and new therapeutic approaches are needed. However, the process of clonal evolution and the molecular alterations that contribute to tumor progression from primary to metastatic carcinoma remain unclear. It is currently difficult to distinguish between the primary pulmonary squamous cell carcinoma (PPSCC) and metastatic cervical squamous cell carcinoma (CSCC). Methods: Paired primary CSCC and lung/lymph nodes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650442"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics of haematopoiesis across the human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650442/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related change in human haematopoiesis causes reduced regenerative capacity¹, cytopenias², immune dysfunction³ and increased risk of blood cancer^(4-6), but the reason for such abrupt functional decline after 70 years of age remains unclear. Here we sequenced 3,579 genomes from single cell-derived colonies of haematopoietic cells across 10 human subjects from 0 to 81 years of age. Haematopoietic stem cells or multipotent progenitors (HSC/MPPs) accumulated a mean of 17 mutations per year...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650206"
                        open
                      >
                        <summary class="article-expander__title">Nuclear Vav3 is required for polycomb repression complex-1 activity in B-cell lymphoblastic leukemogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute B-cell lymphoblastic leukemia (B-ALL) results from oligo-clonal evolution of B-cell progenitors endowed with initiating and propagating leukemia properties. The activation of both the Rac guanine nucleotide exchange factor (Rac GEF) Vav3 and Rac GTPases is required for leukemogenesis mediated by the oncogenic fusion protein BCR-ABL. Vav3 expression becomes predominantly nuclear upon expression of BCR-ABL signature. In the nucleus, Vav3 interacts with BCR-ABL, Rac, and the polycomb...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35649412"
                        open
                      >
                        <summary class="article-expander__title">Glioma progression is shaped by genetic evolution and microenvironment interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35649412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The factors driving therapy resistance in diffuse glioma remain poorly understood. To identify treatment-associated cellular and genetic changes, we analyzed RNA and/or DNA sequencing data from the temporally separated tumor pairs of 304 adult patients with isocitrate dehydrogenase (IDH)-wild-type and IDH-mutant glioma. Tumors recurred in distinct manners that were dependent on IDH mutation status and attributable to changes in histological feature composition, somatic alterations, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645619"
                        open
                      >
                        <summary class="article-expander__title">Whole-Exome Sequencing Could Distinguish Primary Pulmonary Squamous Cell Carcinoma From Lung Metastases in Individuals With Cervical Squamous Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645619/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220619211253&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aims: Metastatic cervical carcinoma is hard to cure using traditional treatment and new therapeutic approaches are needed. However, the process of clonal evolution and the molecular alterations that contribute to tumor progression from primary to metastatic carcinoma remain unclear. It is currently difficult to distinguish between the primary pulmonary squamous cell carcinoma (PPSCC) and metastatic cervical squamous cell carcinoma (CSCC). Methods: Paired primary CSCC and lung/lymph nodes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650442"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics of haematopoiesis across the human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650442/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related change in human haematopoiesis causes reduced regenerative capacity¹, cytopenias², immune dysfunction³ and increased risk of blood cancer^(4-6), but the reason for such abrupt functional decline after 70 years of age remains unclear. Here we sequenced 3,579 genomes from single cell-derived colonies of haematopoietic cells across 10 human subjects from 0 to 81 years of age. Haematopoietic stem cells or multipotent progenitors (HSC/MPPs) accumulated a mean of 17 mutations per year...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650206"
                        open
                      >
                        <summary class="article-expander__title">Nuclear Vav3 is required for polycomb repression complex-1 activity in B-cell lymphoblastic leukemogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute B-cell lymphoblastic leukemia (B-ALL) results from oligo-clonal evolution of B-cell progenitors endowed with initiating and propagating leukemia properties. The activation of both the Rac guanine nucleotide exchange factor (Rac GEF) Vav3 and Rac GTPases is required for leukemogenesis mediated by the oncogenic fusion protein BCR-ABL. Vav3 expression becomes predominantly nuclear upon expression of BCR-ABL signature. In the nucleus, Vav3 interacts with BCR-ABL, Rac, and the polycomb...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35649412"
                        open
                      >
                        <summary class="article-expander__title">Glioma progression is shaped by genetic evolution and microenvironment interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35649412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The factors driving therapy resistance in diffuse glioma remain poorly understood. To identify treatment-associated cellular and genetic changes, we analyzed RNA and/or DNA sequencing data from the temporally separated tumor pairs of 304 adult patients with isocitrate dehydrogenase (IDH)-wild-type and IDH-mutant glioma. Tumors recurred in distinct manners that were dependent on IDH mutation status and attributable to changes in histological feature composition, somatic alterations, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645619"
                        open
                      >
                        <summary class="article-expander__title">Whole-Exome Sequencing Could Distinguish Primary Pulmonary Squamous Cell Carcinoma From Lung Metastases in Individuals With Cervical Squamous Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645619/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220618211120&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aims: Metastatic cervical carcinoma is hard to cure using traditional treatment and new therapeutic approaches are needed. However, the process of clonal evolution and the molecular alterations that contribute to tumor progression from primary to metastatic carcinoma remain unclear. It is currently difficult to distinguish between the primary pulmonary squamous cell carcinoma (PPSCC) and metastatic cervical squamous cell carcinoma (CSCC). Methods: Paired primary CSCC and lung/lymph nodes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650442"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics of haematopoiesis across the human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650442/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related change in human haematopoiesis causes reduced regenerative capacity¹, cytopenias², immune dysfunction³ and increased risk of blood cancer^(4-6), but the reason for such abrupt functional decline after 70 years of age remains unclear. Here we sequenced 3,579 genomes from single cell-derived colonies of haematopoietic cells across 10 human subjects from 0 to 81 years of age. Haematopoietic stem cells or multipotent progenitors (HSC/MPPs) accumulated a mean of 17 mutations per year...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650206"
                        open
                      >
                        <summary class="article-expander__title">Nuclear Vav3 is required for polycomb repression complex-1 activity in B-cell lymphoblastic leukemogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute B-cell lymphoblastic leukemia (B-ALL) results from oligo-clonal evolution of B-cell progenitors endowed with initiating and propagating leukemia properties. The activation of both the Rac guanine nucleotide exchange factor (Rac GEF) Vav3 and Rac GTPases is required for leukemogenesis mediated by the oncogenic fusion protein BCR-ABL. Vav3 expression becomes predominantly nuclear upon expression of BCR-ABL signature. In the nucleus, Vav3 interacts with BCR-ABL, Rac, and the polycomb...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35649412"
                        open
                      >
                        <summary class="article-expander__title">Glioma progression is shaped by genetic evolution and microenvironment interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35649412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The factors driving therapy resistance in diffuse glioma remain poorly understood. To identify treatment-associated cellular and genetic changes, we analyzed RNA and/or DNA sequencing data from the temporally separated tumor pairs of 304 adult patients with isocitrate dehydrogenase (IDH)-wild-type and IDH-mutant glioma. Tumors recurred in distinct manners that were dependent on IDH mutation status and attributable to changes in histological feature composition, somatic alterations, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645619"
                        open
                      >
                        <summary class="article-expander__title">Whole-Exome Sequencing Could Distinguish Primary Pulmonary Squamous Cell Carcinoma From Lung Metastases in Individuals With Cervical Squamous Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645619/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220617210912&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aims: Metastatic cervical carcinoma is hard to cure using traditional treatment and new therapeutic approaches are needed. However, the process of clonal evolution and the molecular alterations that contribute to tumor progression from primary to metastatic carcinoma remain unclear. It is currently difficult to distinguish between the primary pulmonary squamous cell carcinoma (PPSCC) and metastatic cervical squamous cell carcinoma (CSCC). Methods: Paired primary CSCC and lung/lymph nodes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650442"
                        open
                      >
                        <summary class="article-expander__title">Clonal dynamics of haematopoiesis across the human lifespan</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650442/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Age-related change in human haematopoiesis causes reduced regenerative capacity¹, cytopenias², immune dysfunction³ and increased risk of blood cancer^(4-6), but the reason for such abrupt functional decline after 70 years of age remains unclear. Here we sequenced 3,579 genomes from single cell-derived colonies of haematopoietic cells across 10 human subjects from 0 to 81 years of age. Haematopoietic stem cells or multipotent progenitors (HSC/MPPs) accumulated a mean of 17 mutations per year...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650206"
                        open
                      >
                        <summary class="article-expander__title">Nuclear Vav3 is required for polycomb repression complex-1 activity in B-cell lymphoblastic leukemogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute B-cell lymphoblastic leukemia (B-ALL) results from oligo-clonal evolution of B-cell progenitors endowed with initiating and propagating leukemia properties. The activation of both the Rac guanine nucleotide exchange factor (Rac GEF) Vav3 and Rac GTPases is required for leukemogenesis mediated by the oncogenic fusion protein BCR-ABL. Vav3 expression becomes predominantly nuclear upon expression of BCR-ABL signature. In the nucleus, Vav3 interacts with BCR-ABL, Rac, and the polycomb...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35649412"
                        open
                      >
                        <summary class="article-expander__title">Glioma progression is shaped by genetic evolution and microenvironment interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35649412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The factors driving therapy resistance in diffuse glioma remain poorly understood. To identify treatment-associated cellular and genetic changes, we analyzed RNA and/or DNA sequencing data from the temporally separated tumor pairs of 304 adult patients with isocitrate dehydrogenase (IDH)-wild-type and IDH-mutant glioma. Tumors recurred in distinct manners that were dependent on IDH mutation status and attributable to changes in histological feature composition, somatic alterations, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645619"
                        open
                      >
                        <summary class="article-expander__title">Whole-Exome Sequencing Could Distinguish Primary Pulmonary Squamous Cell Carcinoma From Lung Metastases in Individuals With Cervical Squamous Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645619/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aims: Metastatic cervical carcinoma is hard to cure using traditional treatment and new therapeutic approaches are needed. However, the process of clonal evolution and the molecular alterations that contribute to tumor progression from primary to metastatic carcinoma remain unclear. It is currently difficult to distinguish between the primary pulmonary squamous cell carcinoma (PPSCC) and metastatic cervical squamous cell carcinoma (CSCC). Methods: Paired primary CSCC and lung/lymph nodes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645228"
                        open
                      >
                        <summary class="article-expander__title">Conceptual Evolution and Current Approach to Spitz Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645228/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220616210436&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Over the past several decades, the study of Spitz neoplasms has flourished, with expanded conceptualization and refined terminology, providing a framework for the assessment and classification of Spitz nevi, atypical Spitz Tumors, and Spitz melanoma. Cancer genomics have generated concepts such as driver and passenger genes and clonal evolution, which can be applied to Spitz tumors. Herein, we provide a historical perspective, followed by a summary of current knowledge and clinical approaches...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650206"
                        open
                      >
                        <summary class="article-expander__title">Nuclear Vav3 is required for polycomb repression complex-1 activity in B-cell lymphoblastic leukemogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute B-cell lymphoblastic leukemia (B-ALL) results from oligo-clonal evolution of B-cell progenitors endowed with initiating and propagating leukemia properties. The activation of both the Rac guanine nucleotide exchange factor (Rac GEF) Vav3 and Rac GTPases is required for leukemogenesis mediated by the oncogenic fusion protein BCR-ABL. Vav3 expression becomes predominantly nuclear upon expression of BCR-ABL signature. In the nucleus, Vav3 interacts with BCR-ABL, Rac, and the polycomb...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35649412"
                        open
                      >
                        <summary class="article-expander__title">Glioma progression is shaped by genetic evolution and microenvironment interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35649412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The factors driving therapy resistance in diffuse glioma remain poorly understood. To identify treatment-associated cellular and genetic changes, we analyzed RNA and/or DNA sequencing data from the temporally separated tumor pairs of 304 adult patients with isocitrate dehydrogenase (IDH)-wild-type and IDH-mutant glioma. Tumors recurred in distinct manners that were dependent on IDH mutation status and attributable to changes in histological feature composition, somatic alterations, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645619"
                        open
                      >
                        <summary class="article-expander__title">Whole-Exome Sequencing Could Distinguish Primary Pulmonary Squamous Cell Carcinoma From Lung Metastases in Individuals With Cervical Squamous Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645619/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aims: Metastatic cervical carcinoma is hard to cure using traditional treatment and new therapeutic approaches are needed. However, the process of clonal evolution and the molecular alterations that contribute to tumor progression from primary to metastatic carcinoma remain unclear. It is currently difficult to distinguish between the primary pulmonary squamous cell carcinoma (PPSCC) and metastatic cervical squamous cell carcinoma (CSCC). Methods: Paired primary CSCC and lung/lymph nodes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645228"
                        open
                      >
                        <summary class="article-expander__title">Conceptual Evolution and Current Approach to Spitz Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645228/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Over the past several decades, the study of Spitz neoplasms has flourished, with expanded conceptualization and refined terminology, providing a framework for the assessment and classification of Spitz nevi, atypical Spitz Tumors, and Spitz melanoma. Cancer genomics have generated concepts such as driver and passenger genes and clonal evolution, which can be applied to Spitz tumors. Herein, we provide a historical perspective, followed by a summary of current knowledge and clinical approaches...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643867"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution after treatment pressure in multiple myeloma: heterogenous genomic aberrations and transcriptomic convergence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643867/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We investigated genomic and transcriptomic changes in paired tumor samples of 29 in-house multiple myeloma (MM) patients and 28 patients from the MMRF CoMMpass study before and after treatment. A change in clonal composition was found in 46/57 (82%) of patients, and single-nucleotide variants (SNVs) increased from median 67 to 86. The highest increase in prevalence of genetic aberrations was found in RAS genes (60% to 72%), amp1q21 (18% to 35%), and TP53 (9% to 18%). The SBS-MM1 mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642016"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution and multi-wave clonal invasion in H3 K27-altered diffuse midline gliomas treated with a PDGFR inhibitor</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642016/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220615211026&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of diffuse midline gliomas, H3 K27-altered (DMG-H3 K27-a), are infiltrating pediatric brain tumors that arise in the pons with no effective treatment. To understand how clonal evolution contributes to the tumor&#x27;s invasive spread, we performed exome sequencing and SNP array profiling on 49 multi-region autopsy samples from 11 patients with pontine DMG-H3 K27-a enrolled in a phase I clinical trial of PDGFR inhibitor crenolanib. For each patient, a phylogenetic tree was constructed by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650206"
                        open
                      >
                        <summary class="article-expander__title">Nuclear Vav3 is required for polycomb repression complex-1 activity in B-cell lymphoblastic leukemogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute B-cell lymphoblastic leukemia (B-ALL) results from oligo-clonal evolution of B-cell progenitors endowed with initiating and propagating leukemia properties. The activation of both the Rac guanine nucleotide exchange factor (Rac GEF) Vav3 and Rac GTPases is required for leukemogenesis mediated by the oncogenic fusion protein BCR-ABL. Vav3 expression becomes predominantly nuclear upon expression of BCR-ABL signature. In the nucleus, Vav3 interacts with BCR-ABL, Rac, and the polycomb...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35649412"
                        open
                      >
                        <summary class="article-expander__title">Glioma progression is shaped by genetic evolution and microenvironment interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35649412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The factors driving therapy resistance in diffuse glioma remain poorly understood. To identify treatment-associated cellular and genetic changes, we analyzed RNA and/or DNA sequencing data from the temporally separated tumor pairs of 304 adult patients with isocitrate dehydrogenase (IDH)-wild-type and IDH-mutant glioma. Tumors recurred in distinct manners that were dependent on IDH mutation status and attributable to changes in histological feature composition, somatic alterations, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645619"
                        open
                      >
                        <summary class="article-expander__title">Whole-Exome Sequencing Could Distinguish Primary Pulmonary Squamous Cell Carcinoma From Lung Metastases in Individuals With Cervical Squamous Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645619/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aims: Metastatic cervical carcinoma is hard to cure using traditional treatment and new therapeutic approaches are needed. However, the process of clonal evolution and the molecular alterations that contribute to tumor progression from primary to metastatic carcinoma remain unclear. It is currently difficult to distinguish between the primary pulmonary squamous cell carcinoma (PPSCC) and metastatic cervical squamous cell carcinoma (CSCC). Methods: Paired primary CSCC and lung/lymph nodes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645228"
                        open
                      >
                        <summary class="article-expander__title">Conceptual Evolution and Current Approach to Spitz Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645228/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Over the past several decades, the study of Spitz neoplasms has flourished, with expanded conceptualization and refined terminology, providing a framework for the assessment and classification of Spitz nevi, atypical Spitz Tumors, and Spitz melanoma. Cancer genomics have generated concepts such as driver and passenger genes and clonal evolution, which can be applied to Spitz tumors. Herein, we provide a historical perspective, followed by a summary of current knowledge and clinical approaches...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643867"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution after treatment pressure in multiple myeloma: heterogenous genomic aberrations and transcriptomic convergence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643867/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We investigated genomic and transcriptomic changes in paired tumor samples of 29 in-house multiple myeloma (MM) patients and 28 patients from the MMRF CoMMpass study before and after treatment. A change in clonal composition was found in 46/57 (82%) of patients, and single-nucleotide variants (SNVs) increased from median 67 to 86. The highest increase in prevalence of genetic aberrations was found in RAS genes (60% to 72%), amp1q21 (18% to 35%), and TP53 (9% to 18%). The SBS-MM1 mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642016"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution and multi-wave clonal invasion in H3 K27-altered diffuse midline gliomas treated with a PDGFR inhibitor</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642016/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of diffuse midline gliomas, H3 K27-altered (DMG-H3 K27-a), are infiltrating pediatric brain tumors that arise in the pons with no effective treatment. To understand how clonal evolution contributes to the tumor&#x27;s invasive spread, we performed exome sequencing and SNP array profiling on 49 multi-region autopsy samples from 11 patients with pontine DMG-H3 K27-a enrolled in a phase I clinical trial of PDGFR inhibitor crenolanib. For each patient, a phylogenetic tree was constructed by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650206"
                        open
                      >
                        <summary class="article-expander__title">Nuclear Vav3 is required for polycomb repression complex-1 activity in B-cell lymphoblastic leukemogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute B-cell lymphoblastic leukemia (B-ALL) results from oligo-clonal evolution of B-cell progenitors endowed with initiating and propagating leukemia properties. The activation of both the Rac guanine nucleotide exchange factor (Rac GEF) Vav3 and Rac GTPases is required for leukemogenesis mediated by the oncogenic fusion protein BCR-ABL. Vav3 expression becomes predominantly nuclear upon expression of BCR-ABL signature. In the nucleus, Vav3 interacts with BCR-ABL, Rac, and the polycomb...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35649412"
                        open
                      >
                        <summary class="article-expander__title">Glioma progression is shaped by genetic evolution and microenvironment interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35649412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The factors driving therapy resistance in diffuse glioma remain poorly understood. To identify treatment-associated cellular and genetic changes, we analyzed RNA and/or DNA sequencing data from the temporally separated tumor pairs of 304 adult patients with isocitrate dehydrogenase (IDH)-wild-type and IDH-mutant glioma. Tumors recurred in distinct manners that were dependent on IDH mutation status and attributable to changes in histological feature composition, somatic alterations, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645619"
                        open
                      >
                        <summary class="article-expander__title">Whole-Exome Sequencing Could Distinguish Primary Pulmonary Squamous Cell Carcinoma From Lung Metastases in Individuals With Cervical Squamous Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645619/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aims: Metastatic cervical carcinoma is hard to cure using traditional treatment and new therapeutic approaches are needed. However, the process of clonal evolution and the molecular alterations that contribute to tumor progression from primary to metastatic carcinoma remain unclear. It is currently difficult to distinguish between the primary pulmonary squamous cell carcinoma (PPSCC) and metastatic cervical squamous cell carcinoma (CSCC). Methods: Paired primary CSCC and lung/lymph nodes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645228"
                        open
                      >
                        <summary class="article-expander__title">Conceptual Evolution and Current Approach to Spitz Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645228/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Over the past several decades, the study of Spitz neoplasms has flourished, with expanded conceptualization and refined terminology, providing a framework for the assessment and classification of Spitz nevi, atypical Spitz Tumors, and Spitz melanoma. Cancer genomics have generated concepts such as driver and passenger genes and clonal evolution, which can be applied to Spitz tumors. Herein, we provide a historical perspective, followed by a summary of current knowledge and clinical approaches...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643867"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution after treatment pressure in multiple myeloma: heterogenous genomic aberrations and transcriptomic convergence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643867/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We investigated genomic and transcriptomic changes in paired tumor samples of 29 in-house multiple myeloma (MM) patients and 28 patients from the MMRF CoMMpass study before and after treatment. A change in clonal composition was found in 46/57 (82%) of patients, and single-nucleotide variants (SNVs) increased from median 67 to 86. The highest increase in prevalence of genetic aberrations was found in RAS genes (60% to 72%), amp1q21 (18% to 35%), and TP53 (9% to 18%). The SBS-MM1 mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642016"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution and multi-wave clonal invasion in H3 K27-altered diffuse midline gliomas treated with a PDGFR inhibitor</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642016/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of diffuse midline gliomas, H3 K27-altered (DMG-H3 K27-a), are infiltrating pediatric brain tumors that arise in the pons with no effective treatment. To understand how clonal evolution contributes to the tumor&#x27;s invasive spread, we performed exome sequencing and SNP array profiling on 49 multi-region autopsy samples from 11 patients with pontine DMG-H3 K27-a enrolled in a phase I clinical trial of PDGFR inhibitor crenolanib. For each patient, a phylogenetic tree was constructed by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650206"
                        open
                      >
                        <summary class="article-expander__title">Nuclear Vav3 is required for polycomb repression complex-1 activity in B-cell lymphoblastic leukemogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute B-cell lymphoblastic leukemia (B-ALL) results from oligo-clonal evolution of B-cell progenitors endowed with initiating and propagating leukemia properties. The activation of both the Rac guanine nucleotide exchange factor (Rac GEF) Vav3 and Rac GTPases is required for leukemogenesis mediated by the oncogenic fusion protein BCR-ABL. Vav3 expression becomes predominantly nuclear upon expression of BCR-ABL signature. In the nucleus, Vav3 interacts with BCR-ABL, Rac, and the polycomb...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35649412"
                        open
                      >
                        <summary class="article-expander__title">Glioma progression is shaped by genetic evolution and microenvironment interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35649412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The factors driving therapy resistance in diffuse glioma remain poorly understood. To identify treatment-associated cellular and genetic changes, we analyzed RNA and/or DNA sequencing data from the temporally separated tumor pairs of 304 adult patients with isocitrate dehydrogenase (IDH)-wild-type and IDH-mutant glioma. Tumors recurred in distinct manners that were dependent on IDH mutation status and attributable to changes in histological feature composition, somatic alterations, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645619"
                        open
                      >
                        <summary class="article-expander__title">Whole-Exome Sequencing Could Distinguish Primary Pulmonary Squamous Cell Carcinoma From Lung Metastases in Individuals With Cervical Squamous Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645619/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aims: Metastatic cervical carcinoma is hard to cure using traditional treatment and new therapeutic approaches are needed. However, the process of clonal evolution and the molecular alterations that contribute to tumor progression from primary to metastatic carcinoma remain unclear. It is currently difficult to distinguish between the primary pulmonary squamous cell carcinoma (PPSCC) and metastatic cervical squamous cell carcinoma (CSCC). Methods: Paired primary CSCC and lung/lymph nodes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645228"
                        open
                      >
                        <summary class="article-expander__title">Conceptual Evolution and Current Approach to Spitz Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645228/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Over the past several decades, the study of Spitz neoplasms has flourished, with expanded conceptualization and refined terminology, providing a framework for the assessment and classification of Spitz nevi, atypical Spitz Tumors, and Spitz melanoma. Cancer genomics have generated concepts such as driver and passenger genes and clonal evolution, which can be applied to Spitz tumors. Herein, we provide a historical perspective, followed by a summary of current knowledge and clinical approaches...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643867"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution after treatment pressure in multiple myeloma: heterogenous genomic aberrations and transcriptomic convergence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643867/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We investigated genomic and transcriptomic changes in paired tumor samples of 29 in-house multiple myeloma (MM) patients and 28 patients from the MMRF CoMMpass study before and after treatment. A change in clonal composition was found in 46/57 (82%) of patients, and single-nucleotide variants (SNVs) increased from median 67 to 86. The highest increase in prevalence of genetic aberrations was found in RAS genes (60% to 72%), amp1q21 (18% to 35%), and TP53 (9% to 18%). The SBS-MM1 mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642016"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution and multi-wave clonal invasion in H3 K27-altered diffuse midline gliomas treated with a PDGFR inhibitor</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642016/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of diffuse midline gliomas, H3 K27-altered (DMG-H3 K27-a), are infiltrating pediatric brain tumors that arise in the pons with no effective treatment. To understand how clonal evolution contributes to the tumor&#x27;s invasive spread, we performed exome sequencing and SNP array profiling on 49 multi-region autopsy samples from 11 patients with pontine DMG-H3 K27-a enrolled in a phase I clinical trial of PDGFR inhibitor crenolanib. For each patient, a phylogenetic tree was constructed by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650206"
                        open
                      >
                        <summary class="article-expander__title">Nuclear Vav3 is required for polycomb repression complex-1 activity in B-cell lymphoblastic leukemogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute B-cell lymphoblastic leukemia (B-ALL) results from oligo-clonal evolution of B-cell progenitors endowed with initiating and propagating leukemia properties. The activation of both the Rac guanine nucleotide exchange factor (Rac GEF) Vav3 and Rac GTPases is required for leukemogenesis mediated by the oncogenic fusion protein BCR-ABL. Vav3 expression becomes predominantly nuclear upon expression of BCR-ABL signature. In the nucleus, Vav3 interacts with BCR-ABL, Rac, and the polycomb...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35649412"
                        open
                      >
                        <summary class="article-expander__title">Glioma progression is shaped by genetic evolution and microenvironment interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35649412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The factors driving therapy resistance in diffuse glioma remain poorly understood. To identify treatment-associated cellular and genetic changes, we analyzed RNA and/or DNA sequencing data from the temporally separated tumor pairs of 304 adult patients with isocitrate dehydrogenase (IDH)-wild-type and IDH-mutant glioma. Tumors recurred in distinct manners that were dependent on IDH mutation status and attributable to changes in histological feature composition, somatic alterations, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645619"
                        open
                      >
                        <summary class="article-expander__title">Whole-Exome Sequencing Could Distinguish Primary Pulmonary Squamous Cell Carcinoma From Lung Metastases in Individuals With Cervical Squamous Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645619/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aims: Metastatic cervical carcinoma is hard to cure using traditional treatment and new therapeutic approaches are needed. However, the process of clonal evolution and the molecular alterations that contribute to tumor progression from primary to metastatic carcinoma remain unclear. It is currently difficult to distinguish between the primary pulmonary squamous cell carcinoma (PPSCC) and metastatic cervical squamous cell carcinoma (CSCC). Methods: Paired primary CSCC and lung/lymph nodes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645228"
                        open
                      >
                        <summary class="article-expander__title">Conceptual Evolution and Current Approach to Spitz Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645228/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Over the past several decades, the study of Spitz neoplasms has flourished, with expanded conceptualization and refined terminology, providing a framework for the assessment and classification of Spitz nevi, atypical Spitz Tumors, and Spitz melanoma. Cancer genomics have generated concepts such as driver and passenger genes and clonal evolution, which can be applied to Spitz tumors. Herein, we provide a historical perspective, followed by a summary of current knowledge and clinical approaches...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643867"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution after treatment pressure in multiple myeloma: heterogenous genomic aberrations and transcriptomic convergence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643867/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We investigated genomic and transcriptomic changes in paired tumor samples of 29 in-house multiple myeloma (MM) patients and 28 patients from the MMRF CoMMpass study before and after treatment. A change in clonal composition was found in 46/57 (82%) of patients, and single-nucleotide variants (SNVs) increased from median 67 to 86. The highest increase in prevalence of genetic aberrations was found in RAS genes (60% to 72%), amp1q21 (18% to 35%), and TP53 (9% to 18%). The SBS-MM1 mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642016"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution and multi-wave clonal invasion in H3 K27-altered diffuse midline gliomas treated with a PDGFR inhibitor</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642016/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of diffuse midline gliomas, H3 K27-altered (DMG-H3 K27-a), are infiltrating pediatric brain tumors that arise in the pons with no effective treatment. To understand how clonal evolution contributes to the tumor&#x27;s invasive spread, we performed exome sequencing and SNP array profiling on 49 multi-region autopsy samples from 11 patients with pontine DMG-H3 K27-a enrolled in a phase I clinical trial of PDGFR inhibitor crenolanib. For each patient, a phylogenetic tree was constructed by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650206"
                        open
                      >
                        <summary class="article-expander__title">Nuclear Vav3 is required for polycomb repression complex-1 activity in B-cell lymphoblastic leukemogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute B-cell lymphoblastic leukemia (B-ALL) results from oligo-clonal evolution of B-cell progenitors endowed with initiating and propagating leukemia properties. The activation of both the Rac guanine nucleotide exchange factor (Rac GEF) Vav3 and Rac GTPases is required for leukemogenesis mediated by the oncogenic fusion protein BCR-ABL. Vav3 expression becomes predominantly nuclear upon expression of BCR-ABL signature. In the nucleus, Vav3 interacts with BCR-ABL, Rac, and the polycomb...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35649412"
                        open
                      >
                        <summary class="article-expander__title">Glioma progression is shaped by genetic evolution and microenvironment interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35649412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The factors driving therapy resistance in diffuse glioma remain poorly understood. To identify treatment-associated cellular and genetic changes, we analyzed RNA and/or DNA sequencing data from the temporally separated tumor pairs of 304 adult patients with isocitrate dehydrogenase (IDH)-wild-type and IDH-mutant glioma. Tumors recurred in distinct manners that were dependent on IDH mutation status and attributable to changes in histological feature composition, somatic alterations, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645619"
                        open
                      >
                        <summary class="article-expander__title">Whole-Exome Sequencing Could Distinguish Primary Pulmonary Squamous Cell Carcinoma From Lung Metastases in Individuals With Cervical Squamous Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645619/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aims: Metastatic cervical carcinoma is hard to cure using traditional treatment and new therapeutic approaches are needed. However, the process of clonal evolution and the molecular alterations that contribute to tumor progression from primary to metastatic carcinoma remain unclear. It is currently difficult to distinguish between the primary pulmonary squamous cell carcinoma (PPSCC) and metastatic cervical squamous cell carcinoma (CSCC). Methods: Paired primary CSCC and lung/lymph nodes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645228"
                        open
                      >
                        <summary class="article-expander__title">Conceptual Evolution and Current Approach to Spitz Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645228/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Over the past several decades, the study of Spitz neoplasms has flourished, with expanded conceptualization and refined terminology, providing a framework for the assessment and classification of Spitz nevi, atypical Spitz Tumors, and Spitz melanoma. Cancer genomics have generated concepts such as driver and passenger genes and clonal evolution, which can be applied to Spitz tumors. Herein, we provide a historical perspective, followed by a summary of current knowledge and clinical approaches...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643867"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution after treatment pressure in multiple myeloma: heterogenous genomic aberrations and transcriptomic convergence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643867/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We investigated genomic and transcriptomic changes in paired tumor samples of 29 in-house multiple myeloma (MM) patients and 28 patients from the MMRF CoMMpass study before and after treatment. A change in clonal composition was found in 46/57 (82%) of patients, and single-nucleotide variants (SNVs) increased from median 67 to 86. The highest increase in prevalence of genetic aberrations was found in RAS genes (60% to 72%), amp1q21 (18% to 35%), and TP53 (9% to 18%). The SBS-MM1 mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642016"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution and multi-wave clonal invasion in H3 K27-altered diffuse midline gliomas treated with a PDGFR inhibitor</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642016/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of diffuse midline gliomas, H3 K27-altered (DMG-H3 K27-a), are infiltrating pediatric brain tumors that arise in the pons with no effective treatment. To understand how clonal evolution contributes to the tumor&#x27;s invasive spread, we performed exome sequencing and SNP array profiling on 49 multi-region autopsy samples from 11 patients with pontine DMG-H3 K27-a enrolled in a phase I clinical trial of PDGFR inhibitor crenolanib. For each patient, a phylogenetic tree was constructed by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650206"
                        open
                      >
                        <summary class="article-expander__title">Nuclear Vav3 is required for polycomb repression complex-1 activity in B-cell lymphoblastic leukemogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute B-cell lymphoblastic leukemia (B-ALL) results from oligo-clonal evolution of B-cell progenitors endowed with initiating and propagating leukemia properties. The activation of both the Rac guanine nucleotide exchange factor (Rac GEF) Vav3 and Rac GTPases is required for leukemogenesis mediated by the oncogenic fusion protein BCR-ABL. Vav3 expression becomes predominantly nuclear upon expression of BCR-ABL signature. In the nucleus, Vav3 interacts with BCR-ABL, Rac, and the polycomb...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35649412"
                        open
                      >
                        <summary class="article-expander__title">Glioma progression is shaped by genetic evolution and microenvironment interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35649412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The factors driving therapy resistance in diffuse glioma remain poorly understood. To identify treatment-associated cellular and genetic changes, we analyzed RNA and/or DNA sequencing data from the temporally separated tumor pairs of 304 adult patients with isocitrate dehydrogenase (IDH)-wild-type and IDH-mutant glioma. Tumors recurred in distinct manners that were dependent on IDH mutation status and attributable to changes in histological feature composition, somatic alterations, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645619"
                        open
                      >
                        <summary class="article-expander__title">Whole-Exome Sequencing Could Distinguish Primary Pulmonary Squamous Cell Carcinoma From Lung Metastases in Individuals With Cervical Squamous Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645619/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aims: Metastatic cervical carcinoma is hard to cure using traditional treatment and new therapeutic approaches are needed. However, the process of clonal evolution and the molecular alterations that contribute to tumor progression from primary to metastatic carcinoma remain unclear. It is currently difficult to distinguish between the primary pulmonary squamous cell carcinoma (PPSCC) and metastatic cervical squamous cell carcinoma (CSCC). Methods: Paired primary CSCC and lung/lymph nodes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645228"
                        open
                      >
                        <summary class="article-expander__title">Conceptual Evolution and Current Approach to Spitz Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645228/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Over the past several decades, the study of Spitz neoplasms has flourished, with expanded conceptualization and refined terminology, providing a framework for the assessment and classification of Spitz nevi, atypical Spitz Tumors, and Spitz melanoma. Cancer genomics have generated concepts such as driver and passenger genes and clonal evolution, which can be applied to Spitz tumors. Herein, we provide a historical perspective, followed by a summary of current knowledge and clinical approaches...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643867"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution after treatment pressure in multiple myeloma: heterogenous genomic aberrations and transcriptomic convergence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643867/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We investigated genomic and transcriptomic changes in paired tumor samples of 29 in-house multiple myeloma (MM) patients and 28 patients from the MMRF CoMMpass study before and after treatment. A change in clonal composition was found in 46/57 (82%) of patients, and single-nucleotide variants (SNVs) increased from median 67 to 86. The highest increase in prevalence of genetic aberrations was found in RAS genes (60% to 72%), amp1q21 (18% to 35%), and TP53 (9% to 18%). The SBS-MM1 mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642016"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution and multi-wave clonal invasion in H3 K27-altered diffuse midline gliomas treated with a PDGFR inhibitor</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642016/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of diffuse midline gliomas, H3 K27-altered (DMG-H3 K27-a), are infiltrating pediatric brain tumors that arise in the pons with no effective treatment. To understand how clonal evolution contributes to the tumor&#x27;s invasive spread, we performed exome sequencing and SNP array profiling on 49 multi-region autopsy samples from 11 patients with pontine DMG-H3 K27-a enrolled in a phase I clinical trial of PDGFR inhibitor crenolanib. For each patient, a phylogenetic tree was constructed by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650206"
                        open
                      >
                        <summary class="article-expander__title">Nuclear Vav3 is required for polycomb repression complex-1 activity in B-cell lymphoblastic leukemogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute B-cell lymphoblastic leukemia (B-ALL) results from oligo-clonal evolution of B-cell progenitors endowed with initiating and propagating leukemia properties. The activation of both the Rac guanine nucleotide exchange factor (Rac GEF) Vav3 and Rac GTPases is required for leukemogenesis mediated by the oncogenic fusion protein BCR-ABL. Vav3 expression becomes predominantly nuclear upon expression of BCR-ABL signature. In the nucleus, Vav3 interacts with BCR-ABL, Rac, and the polycomb...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35649412"
                        open
                      >
                        <summary class="article-expander__title">Glioma progression is shaped by genetic evolution and microenvironment interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35649412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The factors driving therapy resistance in diffuse glioma remain poorly understood. To identify treatment-associated cellular and genetic changes, we analyzed RNA and/or DNA sequencing data from the temporally separated tumor pairs of 304 adult patients with isocitrate dehydrogenase (IDH)-wild-type and IDH-mutant glioma. Tumors recurred in distinct manners that were dependent on IDH mutation status and attributable to changes in histological feature composition, somatic alterations, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645619"
                        open
                      >
                        <summary class="article-expander__title">Whole-Exome Sequencing Could Distinguish Primary Pulmonary Squamous Cell Carcinoma From Lung Metastases in Individuals With Cervical Squamous Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645619/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aims: Metastatic cervical carcinoma is hard to cure using traditional treatment and new therapeutic approaches are needed. However, the process of clonal evolution and the molecular alterations that contribute to tumor progression from primary to metastatic carcinoma remain unclear. It is currently difficult to distinguish between the primary pulmonary squamous cell carcinoma (PPSCC) and metastatic cervical squamous cell carcinoma (CSCC). Methods: Paired primary CSCC and lung/lymph nodes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645228"
                        open
                      >
                        <summary class="article-expander__title">Conceptual Evolution and Current Approach to Spitz Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645228/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Over the past several decades, the study of Spitz neoplasms has flourished, with expanded conceptualization and refined terminology, providing a framework for the assessment and classification of Spitz nevi, atypical Spitz Tumors, and Spitz melanoma. Cancer genomics have generated concepts such as driver and passenger genes and clonal evolution, which can be applied to Spitz tumors. Herein, we provide a historical perspective, followed by a summary of current knowledge and clinical approaches...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643867"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution after treatment pressure in multiple myeloma: heterogenous genomic aberrations and transcriptomic convergence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643867/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We investigated genomic and transcriptomic changes in paired tumor samples of 29 in-house multiple myeloma (MM) patients and 28 patients from the MMRF CoMMpass study before and after treatment. A change in clonal composition was found in 46/57 (82%) of patients, and single-nucleotide variants (SNVs) increased from median 67 to 86. The highest increase in prevalence of genetic aberrations was found in RAS genes (60% to 72%), amp1q21 (18% to 35%), and TP53 (9% to 18%). The SBS-MM1 mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642016"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution and multi-wave clonal invasion in H3 K27-altered diffuse midline gliomas treated with a PDGFR inhibitor</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642016/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of diffuse midline gliomas, H3 K27-altered (DMG-H3 K27-a), are infiltrating pediatric brain tumors that arise in the pons with no effective treatment. To understand how clonal evolution contributes to the tumor&#x27;s invasive spread, we performed exome sequencing and SNP array profiling on 49 multi-region autopsy samples from 11 patients with pontine DMG-H3 K27-a enrolled in a phase I clinical trial of PDGFR inhibitor crenolanib. For each patient, a phylogenetic tree was constructed by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35650206"
                        open
                      >
                        <summary class="article-expander__title">Nuclear Vav3 is required for polycomb repression complex-1 activity in B-cell lymphoblastic leukemogenesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35650206/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Acute B-cell lymphoblastic leukemia (B-ALL) results from oligo-clonal evolution of B-cell progenitors endowed with initiating and propagating leukemia properties. The activation of both the Rac guanine nucleotide exchange factor (Rac GEF) Vav3 and Rac GTPases is required for leukemogenesis mediated by the oncogenic fusion protein BCR-ABL. Vav3 expression becomes predominantly nuclear upon expression of BCR-ABL signature. In the nucleus, Vav3 interacts with BCR-ABL, Rac, and the polycomb...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35649412"
                        open
                      >
                        <summary class="article-expander__title">Glioma progression is shaped by genetic evolution and microenvironment interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35649412/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The factors driving therapy resistance in diffuse glioma remain poorly understood. To identify treatment-associated cellular and genetic changes, we analyzed RNA and/or DNA sequencing data from the temporally separated tumor pairs of 304 adult patients with isocitrate dehydrogenase (IDH)-wild-type and IDH-mutant glioma. Tumors recurred in distinct manners that were dependent on IDH mutation status and attributable to changes in histological feature composition, somatic alterations, and...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645619"
                        open
                      >
                        <summary class="article-expander__title">Whole-Exome Sequencing Could Distinguish Primary Pulmonary Squamous Cell Carcinoma From Lung Metastases in Individuals With Cervical Squamous Cell Carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645619/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Aims: Metastatic cervical carcinoma is hard to cure using traditional treatment and new therapeutic approaches are needed. However, the process of clonal evolution and the molecular alterations that contribute to tumor progression from primary to metastatic carcinoma remain unclear. It is currently difficult to distinguish between the primary pulmonary squamous cell carcinoma (PPSCC) and metastatic cervical squamous cell carcinoma (CSCC). Methods: Paired primary CSCC and lung/lymph nodes...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35645228"
                        open
                      >
                        <summary class="article-expander__title">Conceptual Evolution and Current Approach to Spitz Tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35645228/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Over the past several decades, the study of Spitz neoplasms has flourished, with expanded conceptualization and refined terminology, providing a framework for the assessment and classification of Spitz nevi, atypical Spitz Tumors, and Spitz melanoma. Cancer genomics have generated concepts such as driver and passenger genes and clonal evolution, which can be applied to Spitz tumors. Herein, we provide a historical perspective, followed by a summary of current knowledge and clinical approaches...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35643867"
                        open
                      >
                        <summary class="article-expander__title">Clonal evolution after treatment pressure in multiple myeloma: heterogenous genomic aberrations and transcriptomic convergence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35643867/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We investigated genomic and transcriptomic changes in paired tumor samples of 29 in-house multiple myeloma (MM) patients and 28 patients from the MMRF CoMMpass study before and after treatment. A change in clonal composition was found in 46/57 (82%) of patients, and single-nucleotide variants (SNVs) increased from median 67 to 86. The highest increase in prevalence of genetic aberrations was found in RAS genes (60% to 72%), amp1q21 (18% to 35%), and TP53 (9% to 18%). The SBS-MM1 mutation...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35642016"
                        open
                      >
                        <summary class="article-expander__title">Convergent evolution and multi-wave clonal invasion in H3 K27-altered diffuse midline gliomas treated with a PDGFR inhibitor</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35642016/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The majority of diffuse midline gliomas, H3 K27-altered (DMG-H3 K27-a), are infiltrating pediatric brain tumors that arise in the pons with no effective treatment. To understand how clonal evolution contributes to the tumor&#x27;s invasive spread, we performed exome sequencing and SNP array profiling on 49 multi-region autopsy samples from 11 patients with pontine DMG-H3 K27-a enrolled in a phase I clinical trial of PDGFR inhibitor crenolanib. For each patient, a phylogenetic tree was constructed by...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-31 datetime="2022-05-31T10:00:00.000Z">2022-05-31</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-31 datetime="2022-05-31T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35635736"
                        open
                      >
                        <summary class="article-expander__title">Recent Advances in Pathology: the 2022 Annual Review Issue of The Journal of Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35635736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The 2022 Annual Review Issue of The Journal of Pathology, Recent Advances in Pathology, contains 15 invited reviews on research areas of growing importance in pathology. This year, the articles include those that focus on digital pathology, employing modern imaging techniques and software to enable improved diagnostic and research applications to study human diseases. This subject area includes the ability to identify specific genetic alterations through the morphological changes they induce, as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35635736"
                        open
                      >
                        <summary class="article-expander__title">Recent Advances in Pathology: the 2022 Annual Review Issue of The Journal of Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35635736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The 2022 Annual Review Issue of The Journal of Pathology, Recent Advances in Pathology, contains 15 invited reviews on research areas of growing importance in pathology. This year, the articles include those that focus on digital pathology, employing modern imaging techniques and software to enable improved diagnostic and research applications to study human diseases. This subject area includes the ability to identify specific genetic alterations through the morphological changes they induce, as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35635736"
                        open
                      >
                        <summary class="article-expander__title">Recent Advances in Pathology: the 2022 Annual Review Issue of The Journal of Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35635736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The 2022 Annual Review Issue of The Journal of Pathology, Recent Advances in Pathology, contains 15 invited reviews on research areas of growing importance in pathology. This year, the articles include those that focus on digital pathology, employing modern imaging techniques and software to enable improved diagnostic and research applications to study human diseases. This subject area includes the ability to identify specific genetic alterations through the morphological changes they induce, as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35635736"
                        open
                      >
                        <summary class="article-expander__title">Recent Advances in Pathology: the 2022 Annual Review Issue of The Journal of Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35635736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The 2022 Annual Review Issue of The Journal of Pathology, Recent Advances in Pathology, contains 15 invited reviews on research areas of growing importance in pathology. This year, the articles include those that focus on digital pathology, employing modern imaging techniques and software to enable improved diagnostic and research applications to study human diseases. This subject area includes the ability to identify specific genetic alterations through the morphological changes they induce, as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35635736"
                        open
                      >
                        <summary class="article-expander__title">Recent Advances in Pathology: the 2022 Annual Review Issue of The Journal of Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35635736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The 2022 Annual Review Issue of The Journal of Pathology, Recent Advances in Pathology, contains 15 invited reviews on research areas of growing importance in pathology. This year, the articles include those that focus on digital pathology, employing modern imaging techniques and software to enable improved diagnostic and research applications to study human diseases. This subject area includes the ability to identify specific genetic alterations through the morphological changes they induce, as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35635736"
                        open
                      >
                        <summary class="article-expander__title">Recent Advances in Pathology: the 2022 Annual Review Issue of The Journal of Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35635736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The 2022 Annual Review Issue of The Journal of Pathology, Recent Advances in Pathology, contains 15 invited reviews on research areas of growing importance in pathology. This year, the articles include those that focus on digital pathology, employing modern imaging techniques and software to enable improved diagnostic and research applications to study human diseases. This subject area includes the ability to identify specific genetic alterations through the morphological changes they induce, as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35635736"
                        open
                      >
                        <summary class="article-expander__title">Recent Advances in Pathology: the 2022 Annual Review Issue of The Journal of Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35635736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The 2022 Annual Review Issue of The Journal of Pathology, Recent Advances in Pathology, contains 15 invited reviews on research areas of growing importance in pathology. This year, the articles include those that focus on digital pathology, employing modern imaging techniques and software to enable improved diagnostic and research applications to study human diseases. This subject area includes the ability to identify specific genetic alterations through the morphological changes they induce, as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35635736"
                        open
                      >
                        <summary class="article-expander__title">Recent Advances in Pathology: the 2022 Annual Review Issue of The Journal of Pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35635736/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The 2022 Annual Review Issue of The Journal of Pathology, Recent Advances in Pathology, contains 15 invited reviews on research areas of growing importance in pathology. This year, the articles include those that focus on digital pathology, employing modern imaging techniques and software to enable improved diagnostic and research applications to study human diseases. This subject area includes the ability to identify specific genetic alterations through the morphological changes they induce, as...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z">2022-05-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-30 datetime="2022-05-30T04:37:42.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="bb6bc242bcbc356c4a38db850e74d812"
                        open
                      >
                        <summary class="article-expander__title">@VIBConferences: #Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.⏰Registration Deadline: Thursday 2 June. Read more</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VIBConferences/status/1531132720952266752">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>#Cancer #immunotherapy has become a crucial pillar of anticancer therapy. 19 leading researchers at #ImmunoOnco22 will give new insights into the immune response to tumors and tackle unresolved questions.
⏰Registration Deadline: Thursday 2 June. Read more https://t.co/cc0HC5cj6W pic.twitter.com/u8LFuzBbsV
— VIB Conferences (@VIBConferences) May 30, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-28 datetime="2022-05-28T10:00:00.000Z">2022-05-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-28 datetime="2022-05-28T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35633493"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in hematopoietic cell transplantation for inherited bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35633493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes (IBMFSs) are a group of rare genetic disorders characterized by bone marrow failure with unique phenotypes and predisposition to cancer. Classical IBMFSs primarily include Fanconi anemia with impaired DNA damage repair, dyskeratosis congenita with telomere maintenance dysfunction, and Diamond-Blackfan anemia with aberrant ribosomal protein biosynthesis. Recently, comprehensive genetic analyses have been implemented for the definitive diagnosis of classic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35633493"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in hematopoietic cell transplantation for inherited bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35633493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes (IBMFSs) are a group of rare genetic disorders characterized by bone marrow failure with unique phenotypes and predisposition to cancer. Classical IBMFSs primarily include Fanconi anemia with impaired DNA damage repair, dyskeratosis congenita with telomere maintenance dysfunction, and Diamond-Blackfan anemia with aberrant ribosomal protein biosynthesis. Recently, comprehensive genetic analyses have been implemented for the definitive diagnosis of classic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35633493"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in hematopoietic cell transplantation for inherited bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35633493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes (IBMFSs) are a group of rare genetic disorders characterized by bone marrow failure with unique phenotypes and predisposition to cancer. Classical IBMFSs primarily include Fanconi anemia with impaired DNA damage repair, dyskeratosis congenita with telomere maintenance dysfunction, and Diamond-Blackfan anemia with aberrant ribosomal protein biosynthesis. Recently, comprehensive genetic analyses have been implemented for the definitive diagnosis of classic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35633493"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in hematopoietic cell transplantation for inherited bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35633493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes (IBMFSs) are a group of rare genetic disorders characterized by bone marrow failure with unique phenotypes and predisposition to cancer. Classical IBMFSs primarily include Fanconi anemia with impaired DNA damage repair, dyskeratosis congenita with telomere maintenance dysfunction, and Diamond-Blackfan anemia with aberrant ribosomal protein biosynthesis. Recently, comprehensive genetic analyses have been implemented for the definitive diagnosis of classic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35633493"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in hematopoietic cell transplantation for inherited bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35633493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes (IBMFSs) are a group of rare genetic disorders characterized by bone marrow failure with unique phenotypes and predisposition to cancer. Classical IBMFSs primarily include Fanconi anemia with impaired DNA damage repair, dyskeratosis congenita with telomere maintenance dysfunction, and Diamond-Blackfan anemia with aberrant ribosomal protein biosynthesis. Recently, comprehensive genetic analyses have been implemented for the definitive diagnosis of classic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35633493"
                        open
                      >
                        <summary class="article-expander__title">Recent advances in hematopoietic cell transplantation for inherited bone marrow failure syndromes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35633493/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608003655&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inherited bone marrow failure syndromes (IBMFSs) are a group of rare genetic disorders characterized by bone marrow failure with unique phenotypes and predisposition to cancer. Classical IBMFSs primarily include Fanconi anemia with impaired DNA damage repair, dyskeratosis congenita with telomere maintenance dysfunction, and Diamond-Blackfan anemia with aberrant ribosomal protein biosynthesis. Recently, comprehensive genetic analyses have been implemented for the definitive diagnosis of classic...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z">2022-05-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-27 datetime="2022-05-27T14:45:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="ed6c996dc1a718afb523de6229a53131"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530198403686072320">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Nonsense-mediated RNA decay: an emerging modulator of malignancy https://t.co/wjuHplkpof pic.twitter.com/1SoKYhRsPO
— NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="d15a3a4b24853812d7fa21fd56de5815"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content 👇🏻💌</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1530174481313972224">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our June issue, out NOW! With articles on #therapyresistance, #senescence &amp; targeting CDK4 &amp; CDK6 in #cancer! Plus, a Comment on cancer mRNA vaccines as well as the launch of our new Tools of the Trade articles #NRCToT. So much content 👇🏻
💌 https://t.co/Fo6XRWV3OZ@natrescancer pic.twitter.com/rbxRPqLV6g
— NatureReviewsCancer (@NatureRevCancer) May 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-26 datetime="2022-05-26T10:00:00.000Z">2022-05-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-26 datetime="2022-05-26T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35615557"
                        open
                      >
                        <summary class="article-expander__title">Genomic Alteration Spectrum of Non-Small Cell Lung Cancer Patients in East-China Characterized by Tumor Tissue DNA and Cell-Free DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35615557/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220614211251&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The results revealed considerable differences in the alteration characteristics between the two kinds of specimens. To date, this study represents the largest real-world investigation of its kind, derived from the largest number of patients in East-China. It reinforced and expanded the mechanism of molecular analysis of neoplastic genetic profiles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35615557"
                        open
                      >
                        <summary class="article-expander__title">Genomic Alteration Spectrum of Non-Small Cell Lung Cancer Patients in East-China Characterized by Tumor Tissue DNA and Cell-Free DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35615557/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The results revealed considerable differences in the alteration characteristics between the two kinds of specimens. To date, this study represents the largest real-world investigation of its kind, derived from the largest number of patients in East-China. It reinforced and expanded the mechanism of molecular analysis of neoplastic genetic profiles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35615557"
                        open
                      >
                        <summary class="article-expander__title">Genomic Alteration Spectrum of Non-Small Cell Lung Cancer Patients in East-China Characterized by Tumor Tissue DNA and Cell-Free DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35615557/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The results revealed considerable differences in the alteration characteristics between the two kinds of specimens. To date, this study represents the largest real-world investigation of its kind, derived from the largest number of patients in East-China. It reinforced and expanded the mechanism of molecular analysis of neoplastic genetic profiles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35615557"
                        open
                      >
                        <summary class="article-expander__title">Genomic Alteration Spectrum of Non-Small Cell Lung Cancer Patients in East-China Characterized by Tumor Tissue DNA and Cell-Free DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35615557/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The results revealed considerable differences in the alteration characteristics between the two kinds of specimens. To date, this study represents the largest real-world investigation of its kind, derived from the largest number of patients in East-China. It reinforced and expanded the mechanism of molecular analysis of neoplastic genetic profiles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35615557"
                        open
                      >
                        <summary class="article-expander__title">Genomic Alteration Spectrum of Non-Small Cell Lung Cancer Patients in East-China Characterized by Tumor Tissue DNA and Cell-Free DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35615557/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The results revealed considerable differences in the alteration characteristics between the two kinds of specimens. To date, this study represents the largest real-world investigation of its kind, derived from the largest number of patients in East-China. It reinforced and expanded the mechanism of molecular analysis of neoplastic genetic profiles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35615557"
                        open
                      >
                        <summary class="article-expander__title">Genomic Alteration Spectrum of Non-Small Cell Lung Cancer Patients in East-China Characterized by Tumor Tissue DNA and Cell-Free DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35615557/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The results revealed considerable differences in the alteration characteristics between the two kinds of specimens. To date, this study represents the largest real-world investigation of its kind, derived from the largest number of patients in East-China. It reinforced and expanded the mechanism of molecular analysis of neoplastic genetic profiles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35615557"
                        open
                      >
                        <summary class="article-expander__title">Genomic Alteration Spectrum of Non-Small Cell Lung Cancer Patients in East-China Characterized by Tumor Tissue DNA and Cell-Free DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35615557/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The results revealed considerable differences in the alteration characteristics between the two kinds of specimens. To date, this study represents the largest real-world investigation of its kind, derived from the largest number of patients in East-China. It reinforced and expanded the mechanism of molecular analysis of neoplastic genetic profiles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35615557"
                        open
                      >
                        <summary class="article-expander__title">Genomic Alteration Spectrum of Non-Small Cell Lung Cancer Patients in East-China Characterized by Tumor Tissue DNA and Cell-Free DNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35615557/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSION: The results revealed considerable differences in the alteration characteristics between the two kinds of specimens. To date, this study represents the largest real-world investigation of its kind, derived from the largest number of patients in East-China. It reinforced and expanded the mechanism of molecular analysis of neoplastic genetic profiles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fe13755a2693ea65459e5e21df097d2a"
                        open
                      >
                        <summary class="article-expander__title">@UlrikeHarjes: This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer:  do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest☺️</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/UlrikeHarjes/status/1529747629936193537">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>This Review by @MarcusDGon @cantleylab is part of the @NatureRevCancer series on #Diet &amp; #SystemicMetabolism in cancer: https://t.co/p3Y2cdNhco
I do love this topic in general plus I am handling research at @NatureMedicine now.. get in touch if you have something of interest☺️ https://t.co/qWH2dTv8j9
— Ulrike Harjes (@UlrikeHarjes) May 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z">2022-05-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-25 datetime="2022-05-25T17:35:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9ba5279424b31b0122a4ae20c18e6d03"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Developing dietary interventions as therapy for cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1529516411059437568">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Developing dietary interventions as therapy for cancer https://t.co/ijuqctk1wJ pic.twitter.com/KHPpMKnCFa
— NatureReviewsCancer (@NatureRevCancer) May 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-24 datetime="2022-05-24T10:00:00.000Z">2022-05-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-24 datetime="2022-05-24T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35610705"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal profiling of clonal hematopoiesis provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35610705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Targeted smMIPS sequencing can sensitively measure clonal dynamics in CHIP. Mutations that reached the conventional threshold for CHIP (2% frequency) tended to continue growing, indicating that after CHIP is acquired, it is generally not lost. The ability to cost-effectively profile CHIP longitudinally will enable future studies to investigate why some CHIP clones expand, and how their dynamics relate to health outcomes at a biobank scale.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35610705"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal profiling of clonal hematopoiesis provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35610705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Targeted smMIPS sequencing can sensitively measure clonal dynamics in CHIP. Mutations that reached the conventional threshold for CHIP (2% frequency) tended to continue growing, indicating that after CHIP is acquired, it is generally not lost. The ability to cost-effectively profile CHIP longitudinally will enable future studies to investigate why some CHIP clones expand, and how their dynamics relate to health outcomes at a biobank scale.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35610705"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal profiling of clonal hematopoiesis provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35610705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Targeted smMIPS sequencing can sensitively measure clonal dynamics in CHIP. Mutations that reached the conventional threshold for CHIP (2% frequency) tended to continue growing, indicating that after CHIP is acquired, it is generally not lost. The ability to cost-effectively profile CHIP longitudinally will enable future studies to investigate why some CHIP clones expand, and how their dynamics relate to health outcomes at a biobank scale.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35610705"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal profiling of clonal hematopoiesis provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35610705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Targeted smMIPS sequencing can sensitively measure clonal dynamics in CHIP. Mutations that reached the conventional threshold for CHIP (2% frequency) tended to continue growing, indicating that after CHIP is acquired, it is generally not lost. The ability to cost-effectively profile CHIP longitudinally will enable future studies to investigate why some CHIP clones expand, and how their dynamics relate to health outcomes at a biobank scale.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35609925"
                        open
                      >
                        <summary class="article-expander__title">The Role of Telomeres in Human Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35609925/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Telomere biology was first studied in maize, ciliates, yeast, and mice, and in recent decades, it has informed understanding of common disease mechanisms with broad implications for patient care. Short telomere syndromes are the most prevalent premature aging disorders, with prominent phenotypes affecting the lung and hematopoietic system. Less understood are a newly recognized group of cancer-prone syndromes that are associated with mutations that lengthen telomeres. A large body of new data...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35610705"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal profiling of clonal hematopoiesis provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35610705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Targeted smMIPS sequencing can sensitively measure clonal dynamics in CHIP. Mutations that reached the conventional threshold for CHIP (2% frequency) tended to continue growing, indicating that after CHIP is acquired, it is generally not lost. The ability to cost-effectively profile CHIP longitudinally will enable future studies to investigate why some CHIP clones expand, and how their dynamics relate to health outcomes at a biobank scale.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35609925"
                        open
                      >
                        <summary class="article-expander__title">The Role of Telomeres in Human Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35609925/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Telomere biology was first studied in maize, ciliates, yeast, and mice, and in recent decades, it has informed understanding of common disease mechanisms with broad implications for patient care. Short telomere syndromes are the most prevalent premature aging disorders, with prominent phenotypes affecting the lung and hematopoietic system. Less understood are a newly recognized group of cancer-prone syndromes that are associated with mutations that lengthen telomeres. A large body of new data...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35610705"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal profiling of clonal hematopoiesis provides insight into clonal dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35610705/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608003655&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>CONCLUSIONS: Targeted smMIPS sequencing can sensitively measure clonal dynamics in CHIP. Mutations that reached the conventional threshold for CHIP (2% frequency) tended to continue growing, indicating that after CHIP is acquired, it is generally not lost. The ability to cost-effectively profile CHIP longitudinally will enable future studies to investigate why some CHIP clones expand, and how their dynamics relate to health outcomes at a biobank scale.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35609925"
                        open
                      >
                        <summary class="article-expander__title">The Role of Telomeres in Human Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35609925/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608003655&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Telomere biology was first studied in maize, ciliates, yeast, and mice, and in recent decades, it has informed understanding of common disease mechanisms with broad implications for patient care. Short telomere syndromes are the most prevalent premature aging disorders, with prominent phenotypes affecting the lung and hematopoietic system. Less understood are a newly recognized group of cancer-prone syndromes that are associated with mutations that lengthen telomeres. A large body of new data...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35609925"
                        open
                      >
                        <summary class="article-expander__title">The Role of Telomeres in Human Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35609925/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220613211945&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Telomere biology was first studied in maize, ciliates, yeast, and mice, and in recent decades, it has informed understanding of common disease mechanisms with broad implications for patient care. Short telomere syndromes are the most prevalent premature aging disorders, with prominent phenotypes affecting the lung and hematopoietic system. Less understood are a newly recognized group of cancer-prone syndromes that are associated with mutations that lengthen telomeres. A large body of new data...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35609925"
                        open
                      >
                        <summary class="article-expander__title">The Role of Telomeres in Human Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35609925/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220612211220&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Telomere biology was first studied in maize, ciliates, yeast, and mice, and in recent decades, it has informed understanding of common disease mechanisms with broad implications for patient care. Short telomere syndromes are the most prevalent premature aging disorders, with prominent phenotypes affecting the lung and hematopoietic system. Less understood are a newly recognized group of cancer-prone syndromes that are associated with mutations that lengthen telomeres. A large body of new data...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35609925"
                        open
                      >
                        <summary class="article-expander__title">The Role of Telomeres in Human Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35609925/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220611211014&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Telomere biology was first studied in maize, ciliates, yeast, and mice, and in recent decades, it has informed understanding of common disease mechanisms with broad implications for patient care. Short telomere syndromes are the most prevalent premature aging disorders, with prominent phenotypes affecting the lung and hematopoietic system. Less understood are a newly recognized group of cancer-prone syndromes that are associated with mutations that lengthen telomeres. A large body of new data...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35609925"
                        open
                      >
                        <summary class="article-expander__title">The Role of Telomeres in Human Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35609925/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220610210217&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Telomere biology was first studied in maize, ciliates, yeast, and mice, and in recent decades, it has informed understanding of common disease mechanisms with broad implications for patient care. Short telomere syndromes are the most prevalent premature aging disorders, with prominent phenotypes affecting the lung and hematopoietic system. Less understood are a newly recognized group of cancer-prone syndromes that are associated with mutations that lengthen telomeres. A large body of new data...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35609925"
                        open
                      >
                        <summary class="article-expander__title">The Role of Telomeres in Human Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35609925/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Telomere biology was first studied in maize, ciliates, yeast, and mice, and in recent decades, it has informed understanding of common disease mechanisms with broad implications for patient care. Short telomere syndromes are the most prevalent premature aging disorders, with prominent phenotypes affecting the lung and hematopoietic system. Less understood are a newly recognized group of cancer-prone syndromes that are associated with mutations that lengthen telomeres. A large body of new data...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35608750"
                        open
                      >
                        <summary class="article-expander__title">The epigenetic modifier HDAC2 and the checkpoint kinase ATM determine the responses of microsatellite instable colorectal cancer cells to 5-fluorouracil</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35608750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The epigenetic modifier histone deacetylase-2 (HDAC2) is frequently dysregulated in colon cancer cells. Microsatellite instability (MSI), an unfaithful replication of DNA at nucleotide repeats, occurs in about 15% of human colon tumors. MSI promotes a genetic frameshift and consequently a loss of HDAC2 in up to 43% of these tumors. We show that long-term and short-term cultures of colorectal cancers with MSI contain subpopulations of cells lacking HDAC2. These can be isolated as single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35609925"
                        open
                      >
                        <summary class="article-expander__title">The Role of Telomeres in Human Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35609925/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Telomere biology was first studied in maize, ciliates, yeast, and mice, and in recent decades, it has informed understanding of common disease mechanisms with broad implications for patient care. Short telomere syndromes are the most prevalent premature aging disorders, with prominent phenotypes affecting the lung and hematopoietic system. Less understood are a newly recognized group of cancer-prone syndromes that are associated with mutations that lengthen telomeres. A large body of new data...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35608750"
                        open
                      >
                        <summary class="article-expander__title">The epigenetic modifier HDAC2 and the checkpoint kinase ATM determine the responses of microsatellite instable colorectal cancer cells to 5-fluorouracil</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35608750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The epigenetic modifier histone deacetylase-2 (HDAC2) is frequently dysregulated in colon cancer cells. Microsatellite instability (MSI), an unfaithful replication of DNA at nucleotide repeats, occurs in about 15% of human colon tumors. MSI promotes a genetic frameshift and consequently a loss of HDAC2 in up to 43% of these tumors. We show that long-term and short-term cultures of colorectal cancers with MSI contain subpopulations of cells lacking HDAC2. These can be isolated as single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35609925"
                        open
                      >
                        <summary class="article-expander__title">The Role of Telomeres in Human Disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35609925/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Telomere biology was first studied in maize, ciliates, yeast, and mice, and in recent decades, it has informed understanding of common disease mechanisms with broad implications for patient care. Short telomere syndromes are the most prevalent premature aging disorders, with prominent phenotypes affecting the lung and hematopoietic system. Less understood are a newly recognized group of cancer-prone syndromes that are associated with mutations that lengthen telomeres. A large body of new data...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35608750"
                        open
                      >
                        <summary class="article-expander__title">The epigenetic modifier HDAC2 and the checkpoint kinase ATM determine the responses of microsatellite instable colorectal cancer cells to 5-fluorouracil</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35608750/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The epigenetic modifier histone deacetylase-2 (HDAC2) is frequently dysregulated in colon cancer cells. Microsatellite instability (MSI), an unfaithful replication of DNA at nucleotide repeats, occurs in about 15% of human colon tumors. MSI promotes a genetic frameshift and consequently a loss of HDAC2 in up to 43% of these tumors. We show that long-term and short-term cultures of colorectal cancers with MSI contain subpopulations of cells lacking HDAC2. These can be isolated as single...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-23 datetime="2022-05-23T10:00:00.000Z">2022-05-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-23 datetime="2022-05-23T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >#cancer#tumor#evolution#somatic</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;utm_source&#x3D;axios&amp;utm_campaign&#x3D;pubmed-2&amp;v&#x3D;2.17.7&amp;fc&#x3D;20220608010801&amp;utm_medium&#x3D;rss&amp;ff&#x3D;20220714212604&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35605261"
                        open
                      >
                        <summary class="article-expander__title">Comprehensive characterizations of immune receptor repertoire in tumors and cancer immunotherapy studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35605261/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1HuCGFZSJIJipW3YKC8lKs0LOntnvC4VTlDPv37z3kpJCsGYKs&amp;fc&#x3D;20220608010801&amp;ff&#x3D;20220608010931&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We applied our computational algorithm TRUST4 to assemble immune receptor (TCR/BCR) repertoires from approximately twelve thousand RNA-seq samples from The Cancer Genome Atlas (TCGA) and seven immunotherapy studies. From over 35 million assembled complete complementary-determining region 3 (CDR3) sequences, we observed that the expression of CCL5 and MZB1 are the most positively correlated genes with T-cell clonal expansion and B-cell clonal expansion, respectively. We analyzed amino acid...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-19 datetime="2022-05-19T10:00:00.000Z">2022-05-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-19 datetime="2022-05-19T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Clonal Hematopoiesis</button>
                  <a class="source-heading__link" href="https://pubmed.ncbi.nlm.nih.gov/rss-feed/?feed_id&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;ff&#x3D;20220714212604&amp;utm_source&#x3D;axios&amp;fc&#x3D;20220608003622&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;v&#x3D;2.17.7&amp;utm_medium&#x3D;rss">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35587375"
                        open
                      >
                        <summary class="article-expander__title">Erythroid lineage Jak2V617F expression promotes atherosclerosis through erythrophagocytosis and macrophage ferroptosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35587375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220609211054&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Elevated hematocrit is associated with cardiovascular risk; however, the causality and mechanisms are unclear. The JAK2V617F (Jak2VF) mutation increases cardiovascular risk in myeloproliferative disorders and in clonal hematopoiesis (CH). Jak2VF mice with elevated white blood cells, platelets and red blood cells (RBCs) display accelerated atherosclerosis and macrophage erythrophagocytosis. To investigate whether selective erythroid Jak2VF expression promotes atherosclerosis, we developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35587375"
                        open
                      >
                        <summary class="article-expander__title">Erythroid lineage Jak2V617F expression promotes atherosclerosis through erythrophagocytosis and macrophage ferroptosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35587375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608210731&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Elevated hematocrit is associated with cardiovascular risk; however, the causality and mechanisms are unclear. The JAK2V617F (Jak2VF) mutation increases cardiovascular risk in myeloproliferative disorders and in clonal hematopoiesis (CH). Jak2VF mice with elevated white blood cells, platelets and red blood cells (RBCs) display accelerated atherosclerosis and macrophage erythrophagocytosis. To investigate whether selective erythroid Jak2VF expression promotes atherosclerosis, we developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="pubmed:35587375"
                        open
                      >
                        <summary class="article-expander__title">Erythroid lineage Jak2V617F expression promotes atherosclerosis through erythrophagocytosis and macrophage ferroptosis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://pubmed.ncbi.nlm.nih.gov/35587375/?utm_source&#x3D;axios&amp;utm_medium&#x3D;rss&amp;utm_campaign&#x3D;pubmed-2&amp;utm_content&#x3D;1TSNW3JJ2DCkNHmqtItKm189ayyfigsz0lfhc8yQ7h2yfXzYrl&amp;fc&#x3D;20220608003622&amp;ff&#x3D;20220608003655&amp;v&#x3D;2.17.6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Elevated hematocrit is associated with cardiovascular risk; however, the causality and mechanisms are unclear. The JAK2V617F (Jak2VF) mutation increases cardiovascular risk in myeloproliferative disorders and in clonal hematopoiesis (CH). Jak2VF mice with elevated white blood cells, platelets and red blood cells (RBCs) display accelerated atherosclerosis and macrophage erythrophagocytosis. To investigate whether selective erythroid Jak2VF expression promotes atherosclerosis, we developed...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z">2022-05-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-18 datetime="2022-05-18T14:11:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="611d40f2e6ea2796b028bebda41a86dd"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Digging for treasures in the tumour interactome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526928478490095616">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Digging for treasures in the tumour interactome https://t.co/NhHYAwUgLE
— NatureReviewsCancer (@NatureRevCancer) May 18, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Tools of the Trade
  | Nature Reviews Cancer</button>
                  <a class="source-heading__link" href="https://www.nature.com/nrc/articles?type&#x3D;tools-of-the-trade">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="a7dd854ecb00659bf268ce51147a1486"
                        open
                      >
                        <summary class="article-expander__title">Tools of the Trade</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://rss.app/articles/cb4e791f6f6d729c04434f0c71d7a19e191b5e221032fdefd0e1c81b97926ec4ee1ab6032d919e3dfee63522835712cc36c82cb9970f3f56da6f9070d29d1478c6e821">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Browse the archive of articles on Nature Reviews Cancer</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="7a839bbf574698d860220c4191201c86"
                        open
                      >
                        <summary class="article-expander__title">Digging for treasures in the tumour interactome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://rss.app/articles/cb4e791f6f6d729c04434f0c71d7a19e191b5e221032fde0d0f68e1989837488f14bf4096fd0c368f1a0777ddf1007947d92">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Reviews Cancer - In this Tools of the Trade article, Amir Giladi describes the development and use of PIC-seq, which enables the direct enrichment and analysis of conjugates of physically...</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z">2022-05-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-17 datetime="2022-05-17T13:48:21.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9590a971b00448cd312078183965b21d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: As part of our overall commitment to diversity, equity and inclusion at Springer Nature, we’d like to better understand what DEI means within the global research community. Take the survey here</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1526560254091313156">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>As part of our overall commitment to diversity, equity and inclusion at Springer Nature, we’d like to better understand what DEI means within the global research community. Take the survey here https://t.co/03yLiUxFTq
— NatureReviewsCancer (@NatureRevCancer) May 17, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z">2022-05-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-12 datetime="2022-05-12T15:25:46.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="aa38ff4245b93e964773f191844ddc7a"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT 🧰 🛠️ where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial 👇🏻</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524772831904509953">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Newsflash! We have a new article type in the journal - Tools of the Trade #NRCToT 🧰 🛠️ where we hope to enable more #ECRs to interact with the editorial team &amp; have a bigger voice in our content. For more info, check out the Editorial 👇🏻 https://t.co/VbyoUJmTRJ
— NatureReviewsCancer (@NatureRevCancer) May 12, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z">2022-05-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-05-10 datetime="2022-05-10T16:46:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="65aca76b9f794aaa668d84ad32dca4a0"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tools for the next generation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524068259712036864">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tools for the next generation https://t.co/lEzTPFYDf5
— NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="afa9b6784b4ccc7c05ed4efd26ddfb2d"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1524022709386190856">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Hunting down rare drug-tolerant cycling cells with Watermelon https://t.co/XXT0c9SAYX
— NatureReviewsCancer (@NatureRevCancer) May 10, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z">2022-04-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-29 datetime="2022-04-29T14:40:05.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="43e2d5ccebc9030f737d656f47ac9d98"
                        open
                      >
                        <summary class="article-expander__title">@VarunVenkatara2: Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/VarunVenkatara2/status/1520050291986604033">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Very happy that our latest Perspective is now online @NatureRevCancer ! We discuss implications of malignant, multicellular networks and potential therapeutic strategies how to disconnect them. Thanks to everyone involved ! #CancerNeuroscience #TumorMicrotubes #neurotwitter https://t.co/S55cKTGNFd
— Varun Venkataramani (@VarunVenkatara2) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="9f6b8644378881f4c2e6b6833c93fc2f"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! An inhospitable site</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020349068316673">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! An inhospitable site https://t.co/0dvqIKCWxh pic.twitter.com/PtY8J9Fz7r
— NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="f6d91efcd60dae769e2722585bdfa8df"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Disconnecting multicellular networks in brain tumours</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1520020340201598978">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Disconnecting multicellular networks in brain tumours https://t.co/LJrzy8bRma pic.twitter.com/CO7BnooP0r
— NatureReviewsCancer (@NatureRevCancer) April 29, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z">2022-04-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-28 datetime="2022-04-28T14:04:34.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="76e3b436bc8e43ad21366a80c120e395"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.💌</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519678964364623872">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our May issue, out NOW! With Reviews on #CRISPR in cancer biology &amp; #therapy, ROS in tumour development &amp; progression, and cancer #proteogenomics, and Comments on esophageal cancer research in Eastern Africa and on Critical Race Theory.
💌 https://t.co/vQwHFJhpwv@natrescancer pic.twitter.com/6kYgjRYhdT
— NatureReviewsCancer (@NatureRevCancer) April 28, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="fdf224ea293cdf039086d16e5699a59a"
                        open
                      >
                        <summary class="article-expander__title">@tmurraystewart: Our latest publication on #polyamines is online!@HopkinsMedicine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/tmurraystewart/status/1519413460978704384">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Our latest publication on #polyamines is online!@HopkinsMedicine https://t.co/ehRGgN1w9R
— Tracy Murray Stewart (@tmurraystewart) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z">2022-04-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-27 datetime="2022-04-27T18:22:03.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="4f43226ec4dc8e531ff6e64a9c5ab8e6"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519381375874265088">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Polyamines in cancer: integrating organismal metabolism and antitumour immunity https://t.co/YCJX1hRQLO pic.twitter.com/zRUCRNXyxH
— NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="a44ed79b9f76aad377b9272fb392f553"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.📯</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1519317944995569664">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW #ResearchHighlight! Lv, Liu, Mo &amp; co demonstrate that in pancreatic #PDAC cells, gasdermin E transports YBX1 to the nucleus, which promotes mucin expression, thereby providing tumour cells with a barrier against digestive enzymes.
📯 https://t.co/JTUhX0DHQ1@natrescancer pic.twitter.com/2PdCmsDu2p
— NatureReviewsCancer (@NatureRevCancer) April 27, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z">2022-04-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-26 datetime="2022-04-26T14:58:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="06ff7641d8f5fcc0dff91f835ac3bbc1"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino  @natrescancer @Columbia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518967636561956865">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>In the April issue of @NatureRevCancer | Comment, Reimagining cancer research with art, by @CancerSciArtist, Stefani Shoreibah &amp; @tdanino
 https://t.co/GeTVCMJQsp @natrescancer
 @Columbia pic.twitter.com/ZXW6A0Fe0c
— NatureReviewsCancer (@NatureRevCancer) April 26, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z">2022-04-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-04-25 datetime="2022-04-25T12:12:32.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >NatureReviewsCancer (@NatureRevCancer) / Twitter</button>
                  <a class="source-heading__link" href="https://twitter.com/NatureRevCancer">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="96b2ddff4eb269cf343213c6a1b1ce95"
                        open
                      >
                        <summary class="article-expander__title">@NatureRevCancer: NEW content online! Tracking dormant disseminated tumour cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://twitter.com/NatureRevCancer/status/1518563607440621569">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>NEW content online! Tracking dormant disseminated tumour cells https://t.co/kSXyWAvCxF
— NatureReviewsCancer (@NatureRevCancer) April 25, 2022</span>
                                &ensp;<span class="article-reading-time">(&hairsp;1
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>

    <footer>
      <a class="footer-link" href="https://github.com/MacwanEric/cancer.rss.feed/actions/runs/2674104776">
        <time id="build-timestamp" datetime="2022-07-15T01:26:06.699Z">2022-07-15T01:26:06.699Z</time>
      </a>
      <a class="footer-link" href="https://github.com/osmoscraft/osmosfeed">osmosfeed 1.15.1</a>
    </footer>
    <script src="index.js?v1.14.4"></script>
    <!-- %before-body-end.html% -->
  </body>

</html>